{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Modelling of antibodies-like\n",
    "\n",
    "**Author**: Dr. Alexis Salas Burgos\n",
    "**Laboratory**: Nanocell Lab\n",
    "**Date**: May 2019\n",
    "**Version**: 2.0\n",
    "\n",
    "https://en.wikipedia.org/wiki/Single-chain_variable_fragment\n",
    "* A single-chain variable fragment (scFv) is not actually a fragment of an antibody, but instead is a fusion protein of the variable regions of the heavy (VH) and light chains (VL) of immunoglobulins, connected with a short linker peptide of ten to about 25 amino acids. \n",
    "* The linker is usually rich in glycine for flexibility, as well as serine or threonine for solubility, and can either connect the N-terminus of the VH with the C-terminus of the VL, or vice versa."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Requirements\n",
    "\n",
    "**Software**\n",
    "* modeller\n",
    "> export KEY_MODELLER=MODELIRANJE\n",
    "> conda install -c salilab modeller\n",
    "* mafft\n",
    "> conda install -c biocore mafft \n",
    "* dssp\n",
    "> conda install -c salilab dssp\n",
    "\n",
    "**Python**\n",
    "* biopython\n",
    "> conda install biopython\n",
    "* prody \n",
    "> pip install prody"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Query Sequences\n",
    "\n",
    "Separated into light and heavy fragment constant"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "seqMH    = \"MAETQLLESGGGLVQPGGSLRLSCAYSSSSTYYYSMSWVRQAPGKGLEWSSSYSSSYSGYTGYADSVKGRFTISADTSKNTAYLQMNSLRAEDTAVYQETCETYNADYWGQGTLVTVS\"\n",
    "seqML    = \"STDIQMTQSPSSLSASVGDRVTITCRASQDVNTAVAWYQQKPGKAPKLLIYSASFLYSGVPSRFSGSRSGTDFTLTISSLQPEDFATYYCYSYYYYYYTFGQGTKVEIK\"\n",
    "linker   = \"SGGGGSGGGGSGGGG\"\n",
    "his_tail = \"HHHHHH\"\n",
    "\n",
    "pdb_temp = \"2fjh\"\n",
    "chain_H  = \"H\"\n",
    "chain_L  = \"L\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Create the Alignment"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def read_ali(output=\"output\"):\n",
    "    from Bio import SeqIO\n",
    "    handle_ali = open(output, \"r\")\n",
    "    seqs = SeqIO.parse(handle_ali, \"fasta\")\n",
    "    seqs_ali  = []\n",
    "    names_ali = []\n",
    "    for seq in seqs:\n",
    "        seqs_ali.append(seq.seq)\n",
    "        names_ali.append(seq.id)\n",
    "    return seqs_ali, names_ali"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_gaps_index(seq_ali):\n",
    "    import re\n",
    "    rcp = re.compile(\"-+\")\n",
    "    s = str(seq_ali)\n",
    "    find = rcp.findall(s)\n",
    "    gaps_index = []\n",
    "    for gap in find:\n",
    "        gaps_index.append(s.index(gap))\n",
    "    return gaps_index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Connecting wwPDB FTP server RCSB PDB (USA).\n",
      "@> 2fjh downloaded (2fjh.pdb.gz)\n",
      "@> PDB download via FTP completed (1 downloaded, 0 failed).\n",
      "@> 8103 atoms and 1 coordinate set(s) were parsed in 0.06s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--EVQLVESGGGLVQPGGSLRLSCAASGFTINASWIHWVRQAPGKGLEWVGAI-YPYSGYTNYADSVKGRFTISADTSKNTAYLQMNSLRAEDTAVYYCARWGHSTSPWAMDYWGQGTLVTVS\n",
      "--DIQMTQSPSSLSASVGDRVTITCRASQVIRRSLAWYQQKPGKAPKLLIYAASNLASGVPSRFSGSGSGTDFTLTISSLQPEDFATYYCQQSNTSPLTFGQGTKVEIK\n"
     ]
    }
   ],
   "source": [
    "import prody\n",
    "import os\n",
    "\n",
    "# pdb_temp\n",
    "structure = prody.parsePDB(pdb_temp)\n",
    "# chain_H\n",
    "seqH = structure[chain_H].getSequence()\n",
    "# chain_L\n",
    "seqL = structure[chain_L].getSequence()\n",
    "\n",
    "\n",
    "with open(\"seq2aliH.fa\", \"w\") as f:\n",
    "    f.write(\">tempH\\n%s\\n\" % seqH)\n",
    "    f.write(\">model\\n%s\\n\" % seqMH)\n",
    "\n",
    "with open(\"seq2aliL.fa\", \"w\") as f:\n",
    "    f.write(\">tempL\\n%s\\n\" % seqL)\n",
    "    f.write(\">model\\n%s\\n\" % seqML)\n",
    "\n",
    "cmdH = \"mafft --auto seq2aliH.fa > outputH.ali\"\n",
    "os.system(cmdH)\n",
    "seqs_aliH, names_aliH = read_ali(\"outputH.ali\")\n",
    "\n",
    "cmdL = \"mafft --auto seq2aliL.fa > outputL.ali\"\n",
    "os.system(cmdL)\n",
    "seqs_aliL, names_aliL = read_ali(\"outputL.ali\")\n",
    "\n",
    "#print seqs_aliH[1]\n",
    "#print seqs_aliL[1]\n",
    "\n",
    "gaps1 = get_gaps_index(seqs_aliH[1])\n",
    "print (seqs_aliH[0][:gaps1[-1]])\n",
    "limit_H =  len(seqs_aliH[0][:gaps1[-1]]) - seqs_aliH[0][:gaps1[-1]].count(\"-\")\n",
    "\n",
    "gaps2 = get_gaps_index(seqs_aliL[1])\n",
    "print (seqs_aliL[0][:gaps2[-1]])\n",
    "limit_L =  len(seqs_aliL[0][:gaps2[-1]]) - seqs_aliL[0][:gaps2[-1]].count(\"-\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> PDB file is found in working directory (2fjh.pdb.gz).\n",
      "@> 8103 atoms and 1 coordinate set(s) were parsed in 0.07s.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'2fjhHL.pdb'"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import prody\n",
    "# pdb_temp\n",
    "structure = prody.parsePDB(pdb_temp)\n",
    "atoms1 = structure.select(\"chain H and resnum 0 to %d\" % limit_H)\n",
    "atoms1.setChids(\"A\")\n",
    "atoms2 = structure.select(\"chain L and resnum 0 to %d\" % limit_L)\n",
    "atoms2.setChids(\"A\")\n",
    "atomsF = atoms1 + atoms2\n",
    "prody.writePDB(\"%sHL.pdb\"% (pdb_temp), atomsF)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Create the PIR file\n",
    "\n",
    "* Sequence en model in PIR format \"scfv.fa\"\n",
    "* Alignment in PIR format \"scfv.ali\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Modelo\n",
    "SEQH =  seqs_aliH[1][:gaps1[-1]]\n",
    "SEQL = seqs_aliL[1][:gaps2[-1]]\n",
    "MSEQ_FINAL = SEQH + linker + SEQL + his_tail\n",
    "\n",
    "#Templado\n",
    "TSEQ_FINAL = seqs_aliH[0][:gaps1[-1]]+(\"-\"*len(linker))+seqs_aliL[0][:gaps2[-1]]+(\"-\"*len(his_tail))\n",
    "\n",
    "with open(\"scfv.ali\", \"w\") as f:\n",
    "    name =  \"2fjhHL\"\n",
    "    f.write(\">P1;%s\\n\" % name)\n",
    "    f.write(\"structureX:%s.pdb: %d:A:%s: A:::0.00: 0.00\\n\" % (name, 1, \"@\"))\n",
    "    f.write(str(TSEQ_FINAL))\n",
    "    f.write(\"*\\n\")\n",
    "    \n",
    "    name =  \"scfv\"\n",
    "    f.write(\">P1;%s\\n\" % name)\n",
    "    f.write(\"sequenceX:%s: %d:A:%s: A:::0.00: 0.00\\n\" % (name, 1, \"@\"))\n",
    "    f.write(str(MSEQ_FINAL))\n",
    "    f.write(\"*\\n\")\n",
    "\n",
    "with open(\"scfv.fa\", \"w\") as f2:\n",
    "    f2.write(\">P1;%s\\n\" % name)\n",
    "    f2.write(\"sequence:%s: %d:A:%s: A:::0.00: 0.00\\n\" % (name, 1, \"@\"))\n",
    "    seqM_final = str(MSEQ_FINAL).replace(\"-\", \"\") \n",
    "    f2.write(str(MSEQ_FINAL))\n",
    "    f2.write(\"*\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       ">P1;2fjhHL\n",
       "structureX:2fjhHL.pdb: 1:A:@: A:::0.00: 0.00\n",
       "--EVQLVESGGGLVQPGGSLRLSCAASGFTINASWIHWVRQAPGKGLEWVGAI-YPYSGYTNYADSVKGRFTISADTSKNTAYLQMNSLRAEDTAVYYCARWGHSTSPWAMDYWGQGTLVTVS-----------------DIQMTQSPSSLSASVGDRVTITCRASQVIRRSLAWYQQKPGKAPKLLIYAASNLASGVPSRFSGSGSGTDFTLTISSLQPEDFATYYCQQSNTSPLTFGQGTKVEIK------*\n",
       ">P1;scfv\n",
       "sequenceX:scfv: 1:A:@: A:::0.00: 0.00\n",
       "MAETQLLESGGGLVQPGGSLRLSCAYSSSSSYYYSMSWVRQAPGKGLEWSSSYSSSYSGYTGYADSVKGRFTISADTSKNTAYLQMNSLRAEDTAVYQ-----ETCETYNADYWGQGTLVTVSSGGGGSGGGGSGGGGSTDIQMTQSPSSLSASVGDRVTITCRASQDVNTAVAWYQQKPGKAPKLLIYSASFLYSGVPSRFSGSRSGTDFTLTISSLQPEDFATYYCYSYYYYYYTFGQGTKVEIKHHHHHH*\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "%more scfv.ali"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Build the models"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "                         MODELLER 9.21, 2018/12/06, r11301\n",
      "\n",
      "     PROTEIN STRUCTURE MODELLING BY SATISFACTION OF SPATIAL RESTRAINTS\n",
      "\n",
      "\n",
      "                     Copyright(c) 1989-2018 Andrej Sali\n",
      "                            All Rights Reserved\n",
      "\n",
      "                             Written by A. Sali\n",
      "                               with help from\n",
      "              B. Webb, M.S. Madhusudhan, M-Y. Shen, G.Q. Dong,\n",
      "          M.A. Marti-Renom, N. Eswar, F. Alber, M. Topf, B. Oliva,\n",
      "             A. Fiser, R. Sanchez, B. Yerkovich, A. Badretdinov,\n",
      "                     F. Melo, J.P. Overington, E. Feyfant\n",
      "                 University of California, San Francisco, USA\n",
      "                    Rockefeller University, New York, USA\n",
      "                      Harvard University, Cambridge, USA\n",
      "                   Imperial Cancer Research Fund, London, UK\n",
      "              Birkbeck College, University of London, London, UK\n",
      "\n",
      "\n",
      "Kind, OS, HostName, Kernel, Processor: 4, Darwin MacBook-Pro-de-Alexis.local 18.6.0 x86_64\n",
      "Date and time of compilation         : 2018/12/06 10:04:19\n",
      "MODELLER executable type             : mac10v4-intel64\n",
      "Job starting time (YY/MM/DD HH:MM:SS): 2019/06/19 11:51:56\n",
      "\n",
      "readlinef__W> File: 2fjhHL.pdb, Line: 1\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "     98     1   95   101      Q     E   20.163\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        1\n",
      "The following 1 residues contain 6-membered rings with poor geometries\n",
      "after transfer from templates. Rebuilding rings from internal coordinates:\n",
      "   <Residue 104 (type TYR)>\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    23335    21669\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  HIS:OXT  HIS\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21669   21669\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   358154\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -22234.1875\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -22234.187500\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /Users/alsalas/anaconda3/lib/modeller-9.21/modlib/surf5.de\n",
      "Pair library             : /Users/alsalas/anaconda3/lib/modeller-9.21/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    77.093002\n",
      "Sequence length          :          248\n",
      "Compactness              :     0.152676\n",
      "Native energy (pair)     :   -31.203152\n",
      "Native energy (surface)  :   -17.001082\n",
      "Native energy (combined) :    -5.602741\n",
      "Z score (pair)           :    -6.932791\n",
      "Z score (surface)        :    -7.737026\n",
      "Z score (combined)       :    -9.968409\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21669   21669\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     3832\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3035.9229\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1934      10     12   0.018   0.018      192.77       1.000\n",
      " 2 Bond angle potential               :    2624       5     24   2.437   2.437      307.84       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1289       0     49  49.986  49.986      519.45       1.000\n",
      " 4 Stereochemical improper torsion pot:     870       0      0   1.287   1.287      31.377       1.000\n",
      " 5 Soft-sphere overlap restraints     :    3832       0      0   0.003   0.003      4.8484       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    4557      25     47   0.395   0.395      704.07       1.000\n",
      "10 Distance restraints 2 (N-O)        :    4728      20     46   0.408   0.408      683.25       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     247       2      4  16.448  16.448      49.216       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     201       0      4  70.045  70.045      76.648       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     122       0      0  73.268  73.268      38.443       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      48       0      0  73.418  73.418      34.199       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      16       0      0 110.829 110.829      10.739       1.000\n",
      "18 Disulfide distance restraints      :       1       0      0   0.020   0.020     0.69494E-01   1.000\n",
      "19 Disulfide angle restraints         :       2       0      0   2.872   2.872     0.36429       1.000\n",
      "20 Disulfide dihedral angle restraints:       1       0      0   5.692   5.692     0.39674E-01   1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3234       0      0   0.523   0.523      102.38       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     246      32     24  30.528  67.569      178.95       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1549       0      1   0.680   0.680      101.27       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: scfv.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   27636.9023\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    757  98Q  98Q N   CA    735  736    1.58    1.43    0.15    5.06    1.43    0.15    5.06\n",
      "    2    759  98Q  98Q C   CA    742  736    1.79    1.49    0.30    8.67    1.49    0.30    8.67\n",
      "    3    760  98Q  99E C   N     742  744    1.59    1.35    0.25    8.71    1.35    0.25    8.71\n",
      "    4    766  99E  99E N   CA    744  745    1.67    1.43    0.24    7.89    1.43    0.24    7.89\n",
      "    5    768  99E  99E C   CA    751  745    1.75    1.49    0.26    7.52    1.49    0.26    7.52\n",
      "    6    769  99E 100T C   N     751  753    1.56    1.35    0.21    7.55    1.35    0.21    7.55\n",
      "    7    773 100T 100T N   CA    753  754    1.68    1.43    0.25    8.15    1.43    0.25    8.15\n",
      "    8    775 100T 100T C   CA    758  754    1.79    1.49    0.30    8.59    1.49    0.30    8.59\n",
      "    9    776 100T 101C C   N     758  760    1.50    1.35    0.16    5.64    1.35    0.16    5.64\n",
      "   10    779 101C 101C N   CA    760  761    1.59    1.43    0.16    5.41    1.43    0.16    5.41\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2954  98Q  98Q N   CA    735  736  127.38  107.00   20.38    5.86  107.00   20.38    5.86\n",
      "    2   2957  98Q  98Q CA  C     736  742  132.70  116.50   16.20    4.66  116.50   16.20    4.66\n",
      "    3   2963  98Q  99E C   N     742  744  151.49  120.00   31.49    7.16  120.00   31.49    7.16\n",
      "    4   2966  99E  99E N   CA    744  745  136.16  107.00   29.16    8.38  107.00   29.16    8.38\n",
      "    5   2978 100T 100T N   CA    753  754  136.74  107.00   29.74    8.55  107.00   29.74    8.55\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8461  33Y  99E CA  CA    226  745   13.40    8.82    4.57    8.72    8.82    4.57    8.72\n",
      "    2   8484  34Y  99E CA  CA    238  745   10.17    7.65    2.52    5.82    7.65    2.52    5.82\n",
      "    3   9864  99E 185S CA  CA    745 1352   16.38   13.01    3.37    5.83   13.01    3.37    5.83\n",
      "    4   9879 100T 188F CA  CA    754 1369   14.53   11.73    2.79    5.02   11.73    2.79    5.02\n",
      "    5   9880 100T 189L CA  CA    754 1380   14.32   11.85    2.47    4.66   11.85    2.47    4.66\n",
      "    6   9882 100T 191S CA  CA    754 1400   12.62    9.00    3.62    5.88    9.00    3.62    5.88\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14073  99E  30S N   O     744  206   13.36    9.12    4.24    6.78    9.12    4.24    6.78\n",
      "    2  14074  99E  31S N   O     744  212   12.38   10.10    2.28    4.56   10.10    2.28    4.56\n",
      "    3  14075  99E  32Y N   O     744  224   14.79    9.90    4.89    8.59    9.90    4.89    8.59\n",
      "    4  14076  99E  33Y N   O     744  236   12.91    5.55    7.36   13.61    5.55    7.36   13.61\n",
      "    5  14077  99E  34Y N   O     744  248   11.64    8.73    2.91    6.07    8.73    2.91    6.07\n",
      "    6  15654 188F 100T N   O    1368  759   13.64   10.21    3.43    6.15   10.21    3.43    6.15\n",
      "    7  15677 189L 100T N   O    1379  759   12.93    9.47    3.46    6.60    9.47    3.46    6.60\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6722   1M   2A C   N       7    9  -71.41  -68.20    3.66    0.25  -62.50  172.28   28.68\n",
      "    1          2A   2A N   CA      9   10  147.05  145.30                  -40.90\n",
      "    2   6723   2A   3E C   N      12   14 -145.80 -117.80   29.71    0.89  -63.60 -168.50   30.13\n",
      "    2          3E   3E N   CA     14   15  146.74  136.80                  -40.30\n",
      "    3   6748  27S  28S C   N     187  189   80.29   56.90   59.88    2.45  -64.10  145.31   19.09\n",
      "    3         28S  28S N   CA    189  190  -18.72   36.40                  -35.00\n",
      "    4   6753  32Y  33Y C   N     223  225  -73.94  -98.40   62.43    6.29  -63.50  114.83   17.69\n",
      "    4         33Y  33Y N   CA    225  226   70.95  128.40                  -43.40\n",
      "    5   6819  98Q  99E C   N     742  744  -98.10  -69.30   77.98    4.92  -63.60  110.27   16.65\n",
      "    5         99E  99E N   CA    744  745 -145.03  142.50                  -40.30\n",
      "    6   6820  99E 100T C   N     751  753 -153.49 -124.80   43.62    1.37  -63.20  167.90   27.25\n",
      "    6        100T 100T N   CA    753  754  176.35  143.50                  -42.10\n",
      "    7   6824 103T 104Y C   N     780  782   66.92   55.90   28.18    1.36  -63.50  142.32   26.61\n",
      "    7        104Y 104Y N   CA    782  783   13.56   39.50                  -43.40\n",
      "    8   6838 117V 118S C   N     892  894 -142.84 -136.60   14.36    0.57  -64.10  179.10   18.61\n",
      "    8        118S 118S N   CA    894  895  164.14  151.20                  -35.00\n",
      "    9   6839 118S 119S C   N     898  900  -68.96  -72.40   11.73    0.81  -64.10  161.45   11.99\n",
      "    9        119S 119S N   CA    900  901  163.62  152.40                  -35.00\n",
      "   10   6840 119S 120G C   N     904  906   91.76   78.70   25.93    0.39   82.20  163.27    8.41\n",
      "   10        120G 120G N   CA    906  907  171.49 -166.10                    8.50\n",
      "   11   6841 120G 121G C   N     908  910  156.49 -167.20   36.31    0.70   82.20 -178.23   12.25\n",
      "   11        121G 121G N   CA    910  911  174.40  174.60                    8.50\n",
      "   12   6842 121G 122G C   N     912  914   86.34   78.70   14.75    0.23   82.20  172.83    8.60\n",
      "   12        122G 122G N   CA    914  915 -178.72 -166.10                    8.50\n",
      "   13   6843 122G 123G C   N     916  918  145.54 -167.20   48.11    0.74   82.20 -173.79   12.03\n",
      "   13        123G 123G N   CA    918  919 -176.39  174.60                    8.50\n",
      "   14   6844 123G 124S C   N     920  922 -150.28 -136.60   18.34    0.58  -64.10 -176.86   19.46\n",
      "   14        124S 124S N   CA    922  923  163.41  151.20                  -35.00\n",
      "   15   6845 124S 125G C   N     926  928  -75.07  -80.20   14.66    0.36   82.20 -141.37    6.97\n",
      "   15        125G 125G N   CA    928  929  160.37  174.10                    8.50\n",
      "   16   6846 125G 126G C   N     930  932  -62.99  -62.40    1.11    0.20   82.20  153.78   11.71\n",
      "   16        126G 126G N   CA    932  933  -42.14  -41.20                    8.50\n",
      "   17   6847 126G 127G C   N     934  936   76.52   78.70   12.38    0.26   82.20  162.52    8.18\n",
      "   17        127G 127G N   CA    936  937 -153.92 -166.10                    8.50\n",
      "   18   6848 127G 128G C   N     938  940   93.72   78.70   27.23    0.44   82.20  163.09    8.49\n",
      "   18        128G 128G N   CA    940  941  171.18 -166.10                    8.50\n",
      "   19   6849 128G 129S C   N     942  944 -130.30 -136.60   10.22    0.58  -64.10  178.48   17.62\n",
      "   19        129S 129S N   CA    944  945  159.25  151.20                  -35.00\n",
      "   20   6850 129S 130G C   N     948  950  -87.34  -80.20    9.02    0.34   82.20 -119.12    7.50\n",
      "   20        130G 130G N   CA    950  951  179.62  174.10                    8.50\n",
      "   21   6851 130G 131G C   N     952  954 -150.65 -167.20   38.84    1.21   82.20 -177.48   13.96\n",
      "   21        131G 131G N   CA    954  955  139.46  174.60                    8.50\n",
      "   22   6853 132G 133G C   N     960  962   91.46   78.70   40.35    0.68   82.20  147.41    7.62\n",
      "   22        133G 133G N   CA    962  963  155.62 -166.10                    8.50\n",
      "   23   6854 133G 134S C   N     964  966 -141.13 -136.60    9.93    0.39  -64.10 -177.93   18.70\n",
      "   23        134S 134S N   CA    966  967  160.03  151.20                  -35.00\n",
      "   24   6884 163D 164V C   N    1180 1182 -125.78  -62.40   99.67   11.16  -62.40   99.67   11.16\n",
      "   24        164V 164V N   CA   1182 1183   34.52  -42.40                  -42.40\n",
      "   25   6885 164V 165N C   N    1187 1189   49.42  -71.20  139.39   11.25 -119.90 -174.55    9.37\n",
      "   25        165N 165N N   CA   1189 1190 -147.34  142.80                  137.00\n",
      "   26   6907 186A 187S C   N    1360 1362 -147.10  -64.10   95.27    9.56  -64.10   95.27    9.56\n",
      "   26        187S 187S N   CA   1362 1363   11.77  -35.00                  -35.00\n",
      "   27   6962 241I 242K C   N    1813 1815  -86.87  -70.20   24.45    2.04  -62.90  165.06   20.24\n",
      "   27        242K 242K N   CA   1815 1816  122.52  140.40                  -40.80\n",
      "   28   6963 242K 243H C   N    1822 1824 -143.73 -125.60   19.44    0.81  -63.20 -168.17   20.49\n",
      "   28        243H 243H N   CA   1824 1825  131.80  138.80                  -42.30\n",
      "   29   6964 243H 244H C   N    1832 1834 -130.28 -125.60    7.53    0.34  -63.20 -172.39   20.27\n",
      "   29        244H 244H N   CA   1834 1835  132.90  138.80                  -42.30\n",
      "   30   6965 244H 245H C   N    1842 1844 -119.33 -125.60   31.17    1.06  -63.20  160.69   17.40\n",
      "   30        245H 245H N   CA   1844 1845  108.27  138.80                  -42.30\n",
      "   31   6966 245H 246H C   N    1852 1854 -118.38 -125.60   19.14    0.58  -63.20  172.45   18.82\n",
      "   31        246H 246H N   CA   1854 1855  121.08  138.80                  -42.30\n",
      "   32   6967 246H 247H C   N    1862 1864  -77.69  -67.60   10.10    0.75  -63.20  178.00   23.18\n",
      "   32        247H 247H N   CA   1864 1865  140.29  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   14   27   97   99  150  161  181  212  238  315\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  HIS:OXT  HIS\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21669   21669\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   360631\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -22112.3613\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -22112.361328\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /Users/alsalas/anaconda3/lib/modeller-9.21/modlib/surf5.de\n",
      "Pair library             : /Users/alsalas/anaconda3/lib/modeller-9.21/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    77.093002\n",
      "Sequence length          :          248\n",
      "Compactness              :     0.199568\n",
      "Native energy (pair)     :   -56.237529\n",
      "Native energy (surface)  :   -15.077785\n",
      "Native energy (combined) :    -6.453996\n",
      "Z score (pair)           :    -7.396109\n",
      "Z score (surface)        :    -7.723972\n",
      "Z score (combined)       :    -9.832614\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21669   21669\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     3923\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3101.3606\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1934      10     12   0.019   0.019      201.66       1.000\n",
      " 2 Bond angle potential               :    2624       7     21   2.508   2.508      327.81       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1289       0     48  49.830  49.830      517.06       1.000\n",
      " 4 Stereochemical improper torsion pot:     870       0      0   1.174   1.174      28.773       1.000\n",
      " 5 Soft-sphere overlap restraints     :    3923       0      0   0.003   0.003      5.5051       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    4557      24     47   0.394   0.394      701.25       1.000\n",
      "10 Distance restraints 2 (N-O)        :    4728      19     60   0.424   0.424      750.20       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     247       1      4  12.102  12.102      56.982       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     201       0      5  68.176  68.176      54.987       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     122       0      0  68.302  68.302      31.244       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      48       0      0  64.246  64.246      32.667       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      16       0      0 109.489 109.489      12.971       1.000\n",
      "18 Disulfide distance restraints      :       1       0      0   0.017   0.017     0.48361E-01   1.000\n",
      "19 Disulfide angle restraints         :       2       0      0   2.591   2.591     0.29652       1.000\n",
      "20 Disulfide dihedral angle restraints:       1       0      0   5.242   5.242     0.33688E-01   1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3234       0      0   0.440   0.440      83.099       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     246      31     30  32.117  69.478      216.71       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1549       0      0   0.599   0.599      80.065       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: scfv.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   26965.1250\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    757  98Q  98Q N   CA    735  736    1.59    1.43    0.16    5.34    1.43    0.16    5.34\n",
      "    2    759  98Q  98Q C   CA    742  736    1.80    1.49    0.31    8.89    1.49    0.31    8.89\n",
      "    3    760  98Q  99E C   N     742  744    1.60    1.35    0.26    9.13    1.35    0.26    9.13\n",
      "    4    766  99E  99E N   CA    744  745    1.67    1.43    0.24    8.03    1.43    0.24    8.03\n",
      "    5    768  99E  99E C   CA    751  745    1.76    1.49    0.27    7.96    1.49    0.27    7.96\n",
      "    6    769  99E 100T C   N     751  753    1.55    1.35    0.20    7.14    1.35    0.20    7.14\n",
      "    7    773 100T 100T N   CA    753  754    1.69    1.43    0.26    8.49    1.43    0.26    8.49\n",
      "    8    775 100T 100T C   CA    758  754    1.78    1.49    0.29    8.33    1.49    0.29    8.33\n",
      "    9    776 100T 101C C   N     758  760    1.50    1.35    0.16    5.64    1.35    0.16    5.64\n",
      "   10    779 101C 101C N   CA    760  761    1.59    1.43    0.16    5.18    1.43    0.16    5.18\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2954  98Q  98Q N   CA    735  736  127.08  107.00   20.08    5.77  107.00   20.08    5.77\n",
      "    2   2957  98Q  98Q CA  C     736  742  134.17  116.50   17.67    5.08  116.50   17.67    5.08\n",
      "    3   2963  98Q  99E C   N     742  744  145.58  120.00   25.58    5.81  120.00   25.58    5.81\n",
      "    4   2966  99E  99E N   CA    744  745  130.03  107.00   23.03    6.62  107.00   23.03    6.62\n",
      "    5   2975  99E 100T C   N     751  753  151.90  120.00   31.90    7.25  120.00   31.90    7.25\n",
      "    6   2978 100T 100T N   CA    753  754  144.61  107.00   37.61   10.82  107.00   37.61   10.82\n",
      "    7   2985 100T 101C C   N     758  760  141.81  120.00   21.81    4.96  120.00   21.81    4.96\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8461  33Y  99E CA  CA    226  745   13.33    8.82    4.50    8.59    8.82    4.50    8.59\n",
      "    2   8484  34Y  99E CA  CA    238  745   10.07    7.65    2.42    5.60    7.65    2.42    5.60\n",
      "    3   9864  99E 185S CA  CA    745 1352   16.21   13.01    3.20    5.54   13.01    3.20    5.54\n",
      "    4   9879 100T 188F CA  CA    754 1369   14.45   11.73    2.71    4.87   11.73    2.71    4.87\n",
      "    5   9882 100T 191S CA  CA    754 1400   12.47    9.00    3.48    5.64    9.00    3.48    5.64\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14073  99E  30S N   O     744  206   12.88    9.12    3.76    6.02    9.12    3.76    6.02\n",
      "    2  14075  99E  32Y N   O     744  224   14.58    9.90    4.68    8.22    9.90    4.68    8.22\n",
      "    3  14076  99E  33Y N   O     744  236   13.06    5.55    7.52   13.89    5.55    7.52   13.89\n",
      "    4  14077  99E  34Y N   O     744  248   11.80    8.73    3.07    6.40    8.73    3.07    6.40\n",
      "    5  15654 188F 100T N   O    1368  759   13.81   10.21    3.60    6.46   10.21    3.60    6.46\n",
      "    6  15677 189L 100T N   O    1379  759   13.17    9.47    3.69    7.06    9.47    3.69    7.06\n",
      "    7  15699 190Y 100T N   O    1387  759   10.78    8.43    2.35    4.80    8.43    2.35    4.80\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6722   1M   2A C   N       7    9  -78.55  -68.20   22.97    1.51  -62.50  154.13   26.08\n",
      "    1          2A   2A N   CA      9   10  165.81  145.30                  -40.90\n",
      "    2   6723   2A   3E C   N      12   14  -84.42  -69.30   19.36    1.77  -63.60  171.98   21.85\n",
      "    2          3E   3E N   CA     14   15  130.41  142.50                  -40.30\n",
      "    3   6748  27S  28S C   N     187  189   73.06   56.90   40.41    1.65  -64.10  141.40   18.87\n",
      "    3         28S  28S N   CA    189  190   -0.64   36.40                  -35.00\n",
      "    4   6753  32Y  33Y C   N     223  225  -79.71  -98.40   86.42    9.09  -63.50   88.91   13.31\n",
      "    4         33Y  33Y N   CA    225  226   44.02  128.40                  -43.40\n",
      "    5   6819  98Q  99E C   N     742  744 -141.74 -117.80   41.19    1.47  -63.60  168.58   26.83\n",
      "    5         99E  99E N   CA    744  745  170.32  136.80                  -40.30\n",
      "    6   6820  99E 100T C   N     751  753  -77.48  -78.10   27.88    1.21  -63.20  140.96   19.02\n",
      "    6        100T 100T N   CA    753  754  177.67  149.80                  -42.10\n",
      "    7   6824 103T 104Y C   N     780  782   62.86   55.90   22.10    1.12  -63.50  140.72   26.43\n",
      "    7        104Y 104Y N   CA    782  783   18.52   39.50                  -43.40\n",
      "    8   6838 117V 118S C   N     892  894 -134.33 -136.60   13.06    0.61  -64.10 -172.98   11.59\n",
      "    8        118S 118S N   CA    894  895  138.34  151.20                  -35.00\n",
      "    9   6839 118S 119S C   N     898  900 -161.80 -136.60   45.49    1.65  -64.10  167.40   19.18\n",
      "    9        119S 119S N   CA    900  901 -170.93  151.20                  -35.00\n",
      "   10   6840 119S 120G C   N     904  906  -77.23  -80.20    5.53    0.13   82.20 -133.46    7.05\n",
      "   10        120G 120G N   CA    906  907  169.44  174.10                    8.50\n",
      "   11   6842 121G 122G C   N     912  914   95.33   78.70   26.56    1.36   82.20  154.45    6.83\n",
      "   11        122G 122G N   CA    914  915 -145.39 -166.10                    8.50\n",
      "   12   6845 124S 125G C   N     926  928 -107.95  -80.20   40.90    2.85   82.20 -142.70   16.91\n",
      "   12        125G 125G N   CA    928  929  144.05  174.10                    8.50\n",
      "   13   6846 125G 126G C   N     930  932   77.92   78.70   10.79    0.34   82.20  174.69    8.26\n",
      "   13        126G 126G N   CA    932  933 -176.86 -166.10                    8.50\n",
      "   14   6847 126G 127G C   N     934  936   88.84   78.70   37.86    0.69   82.20  149.07    7.57\n",
      "   14        127G 127G N   CA    936  937  157.43 -166.10                    8.50\n",
      "   15   6848 127G 128G C   N     938  940   67.34   78.70   11.45    0.54   82.20  173.79    9.18\n",
      "   15        128G 128G N   CA    940  941 -164.65 -166.10                    8.50\n",
      "   16   6849 128G 129S C   N     942  944 -142.82 -136.60   17.72    0.75  -64.10  175.82   18.38\n",
      "   16        129S 129S N   CA    944  945  167.79  151.20                  -35.00\n",
      "   17   6850 129S 130G C   N     948  950 -100.78  -80.20   31.58    2.17   82.20 -133.29   17.64\n",
      "   17        130G 130G N   CA    950  951  150.15  174.10                    8.50\n",
      "   18   6851 130G 131G C   N     952  954   82.18   78.70    3.79    0.14   82.20  176.09    8.55\n",
      "   18        131G 131G N   CA    954  955 -167.59 -166.10                    8.50\n",
      "   19   6854 133G 134S C   N     964  966  -65.82  -72.40   16.59    1.20  -64.10  157.37   11.50\n",
      "   19        134S 134S N   CA    966  967  167.63  152.40                  -35.00\n",
      "   20   6855 134S 135T C   N     970  972  -58.35  -78.10   29.98    1.05  -63.20  169.42   22.04\n",
      "   20        135T 135T N   CA    972  973  127.25  149.80                  -42.10\n",
      "   21   6870 149S 150V C   N    1076 1078  -66.08  -62.40   16.50    1.90  -73.50  165.68   14.06\n",
      "   21        150V 150V N   CA   1078 1079  -26.31  -42.40                  139.20\n",
      "   22   6871 150V 151G C   N    1083 1085  -91.88  -62.40   34.24    5.20   82.20  177.05   12.90\n",
      "   22        151G 151G N   CA   1085 1086  -23.78  -41.20                    8.50\n",
      "   23   6884 163D 164V C   N    1180 1182 -128.51  -62.40  102.59   11.52  -62.40  102.59   11.52\n",
      "   23        164V 164V N   CA   1182 1183   36.05  -42.40                  -42.40\n",
      "   24   6885 164V 165N C   N    1187 1189   43.36  -71.20  137.39   11.11 -119.90 -177.47    9.35\n",
      "   24        165N 165N N   CA   1189 1190 -141.37  142.80                  137.00\n",
      "   25   6907 186A 187S C   N    1360 1362 -147.19  -64.10   94.30    9.60  -64.10   94.30    9.60\n",
      "   25        187S 187S N   CA   1362 1363    9.60  -35.00                  -35.00\n",
      "   26   6911 190Y 191S C   N    1397 1399  -42.79  -64.10   54.71    3.40  -72.40  125.75    8.42\n",
      "   26        191S 191S N   CA   1399 1400  -85.39  -35.00                  152.40\n",
      "   27   6912 191S 192G C   N    1403 1405  -63.03  -62.40   13.72    2.02   82.20  149.63   11.12\n",
      "   27        192G 192G N   CA   1405 1406  -27.50  -41.20                    8.50\n",
      "   28   6962 241I 242K C   N    1813 1815 -106.78 -118.00   12.42    0.41  -62.90 -179.99   21.38\n",
      "   28        242K 242K N   CA   1815 1816  133.78  139.10                  -40.80\n",
      "   29   6963 242K 243H C   N    1822 1824 -143.46 -125.60   18.96    0.78  -63.20 -167.69   20.54\n",
      "   29        243H 243H N   CA   1824 1825  132.46  138.80                  -42.30\n",
      "   30   6964 243H 244H C   N    1832 1834 -123.89 -125.60   16.00    0.57  -63.20  175.99   19.07\n",
      "   30        244H 244H N   CA   1834 1835  122.89  138.80                  -42.30\n",
      "   31   6967 246H 247H C   N    1862 1864  -68.24  -67.60    1.87    0.12  -63.20  176.01   22.32\n",
      "   31        247H 247H N   CA   1864 1865  141.76  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   18   33   94   96  152  174  184  244  269  307\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.0743\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  HIS:OXT  HIS\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21669   21669\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   357156\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -22360.9551\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -22360.955078\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /Users/alsalas/anaconda3/lib/modeller-9.21/modlib/surf5.de\n",
      "Pair library             : /Users/alsalas/anaconda3/lib/modeller-9.21/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    77.093002\n",
      "Sequence length          :          248\n",
      "Compactness              :     0.184512\n",
      "Native energy (pair)     :   -55.362947\n",
      "Native energy (surface)  :   -18.020615\n",
      "Native energy (combined) :    -6.756710\n",
      "Z score (pair)           :    -6.387277\n",
      "Z score (surface)        :    -7.893026\n",
      "Z score (combined)       :    -9.175090\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21669   21669\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     3836\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3076.2551\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1934      10     12   0.018   0.018      194.78       1.000\n",
      " 2 Bond angle potential               :    2624       7     25   2.446   2.446      310.73       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1289       0     52  50.427  50.427      527.71       1.000\n",
      " 4 Stereochemical improper torsion pot:     870       0      0   1.162   1.162      27.641       1.000\n",
      " 5 Soft-sphere overlap restraints     :    3836       0      0   0.003   0.003      5.0996       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    4557      25     48   0.395   0.395      702.66       1.000\n",
      "10 Distance restraints 2 (N-O)        :    4728      19     53   0.415   0.415      711.23       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     247       2      4  12.040  12.040      52.977       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     201       0      1  72.927  72.927      51.822       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     122       0      0  69.584  69.584      32.917       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      48       0      0  72.605  72.605      32.931       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      16       0      0 112.483 112.483      14.840       1.000\n",
      "18 Disulfide distance restraints      :       1       0      0   0.014   0.014     0.34904E-01   1.000\n",
      "19 Disulfide angle restraints         :       2       0      0   2.196   2.196     0.21291       1.000\n",
      "20 Disulfide dihedral angle restraints:       1       0      0   3.959   3.959     0.19260E-01   1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3234       0      0   0.500   0.500      105.70       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     246      34     30  32.593  71.859      202.72       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1549       0      0   0.677   0.677      102.23       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: scfv.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   28356.9102\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    757  98Q  98Q N   CA    735  736    1.59    1.43    0.16    5.26    1.43    0.16    5.26\n",
      "    2    759  98Q  98Q C   CA    742  736    1.79    1.49    0.30    8.64    1.49    0.30    8.64\n",
      "    3    760  98Q  99E C   N     742  744    1.60    1.35    0.25    8.95    1.35    0.25    8.95\n",
      "    4    766  99E  99E N   CA    744  745    1.67    1.43    0.24    7.86    1.43    0.24    7.86\n",
      "    5    768  99E  99E C   CA    751  745    1.76    1.49    0.27    7.82    1.49    0.27    7.82\n",
      "    6    769  99E 100T C   N     751  753    1.54    1.35    0.20    7.05    1.35    0.20    7.05\n",
      "    7    773 100T 100T N   CA    753  754    1.68    1.43    0.25    8.21    1.43    0.25    8.21\n",
      "    8    775 100T 100T C   CA    758  754    1.79    1.49    0.30    8.86    1.49    0.30    8.86\n",
      "    9    776 100T 101C C   N     758  760    1.51    1.35    0.16    5.74    1.35    0.16    5.74\n",
      "   10    779 101C 101C N   CA    760  761    1.58    1.43    0.15    5.08    1.43    0.15    5.08\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2954  98Q  98Q N   CA    735  736  126.68  107.00   19.68    5.66  107.00   19.68    5.66\n",
      "    2   2957  98Q  98Q CA  C     736  742  133.46  116.50   16.96    4.88  116.50   16.96    4.88\n",
      "    3   2963  98Q  99E C   N     742  744  145.45  120.00   25.45    5.78  120.00   25.45    5.78\n",
      "    4   2966  99E  99E N   CA    744  745  129.75  107.00   22.75    6.54  107.00   22.75    6.54\n",
      "    5   2975  99E 100T C   N     751  753  147.90  120.00   27.90    6.34  120.00   27.90    6.34\n",
      "    6   2978 100T 100T N   CA    753  754  142.83  107.00   35.83   10.30  107.00   35.83   10.30\n",
      "    7   2985 100T 101C C   N     758  760  142.66  120.00   22.66    5.15  120.00   22.66    5.15\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8461  33Y  99E CA  CA    226  745   13.43    8.82    4.61    8.79    8.82    4.61    8.79\n",
      "    2   8484  34Y  99E CA  CA    238  745   10.09    7.65    2.44    5.65    7.65    2.44    5.65\n",
      "    3   9864  99E 185S CA  CA    745 1352   16.24   13.01    3.23    5.60   13.01    3.23    5.60\n",
      "    4   9879 100T 188F CA  CA    754 1369   14.41   11.73    2.68    4.81   11.73    2.68    4.81\n",
      "    5   9882 100T 191S CA  CA    754 1400   12.66    9.00    3.67    5.96    9.00    3.67    5.96\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14073  99E  30S N   O     744  206   12.90    9.12    3.78    6.05    9.12    3.78    6.05\n",
      "    2  14075  99E  32Y N   O     744  224   14.81    9.90    4.90    8.61    9.90    4.90    8.61\n",
      "    3  14076  99E  33Y N   O     744  236   13.07    5.55    7.52   13.90    5.55    7.52   13.90\n",
      "    4  14077  99E  34Y N   O     744  248   11.39    8.73    2.66    5.55    8.73    2.66    5.55\n",
      "    5  15654 188F 100T N   O    1368  759   13.97   10.21    3.76    6.73   10.21    3.76    6.73\n",
      "    6  15677 189L 100T N   O    1379  759   13.30    9.47    3.83    7.32    9.47    3.83    7.32\n",
      "    7  15699 190Y 100T N   O    1387  759   10.96    8.43    2.54    5.18    8.43    2.54    5.18\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7066  99E  99E CA  C     745  751 -157.16 -180.00   22.84    4.57 -180.00   22.84    4.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6722   1M   2A C   N       7    9 -127.58 -134.00   12.33    0.48  -62.50 -171.07   28.58\n",
      "    1          2A   2A N   CA      9   10  136.47  147.00                  -40.90\n",
      "    2   6723   2A   3E C   N      12   14 -169.65 -117.80   54.17    1.66  -63.60 -161.98   32.21\n",
      "    2          3E   3E N   CA     14   15  152.47  136.80                  -40.30\n",
      "    3   6748  27S  28S C   N     187  189   66.15   56.90   24.54    1.01  -64.10  139.05   18.60\n",
      "    3         28S  28S N   CA    189  190   13.67   36.40                  -35.00\n",
      "    4   6753  32Y  33Y C   N     223  225  -85.26  -98.40   62.58    6.59  -63.50  112.74   16.82\n",
      "    4         33Y  33Y N   CA    225  226   67.22  128.40                  -43.40\n",
      "    5   6763  42A  43P C   N     313  315  -41.98  -58.70   40.36    2.37  -64.50  147.30   12.21\n",
      "    5         43P  43P N   CA    315  316  -67.24  -30.50                  147.20\n",
      "    6   6764  43P  44G C   N     320  322  -77.21  -62.40   17.72    2.65   82.20  164.35   12.22\n",
      "    6         44G  44G N   CA    322  323  -31.47  -41.20                    8.50\n",
      "    7   6819  98Q  99E C   N     742  744 -141.07 -117.80   44.10    1.64  -63.60  164.79   26.28\n",
      "    7         99E  99E N   CA    744  745  174.26  136.80                  -40.30\n",
      "    8   6820  99E 100T C   N     751  753  -94.72  -78.10   36.32    1.19  -63.20  139.42   19.98\n",
      "    8        100T 100T N   CA    753  754 -177.91  149.80                  -42.10\n",
      "    9   6824 103T 104Y C   N     780  782   65.77   55.90   26.52    1.29  -63.50  141.81   26.55\n",
      "    9        104Y 104Y N   CA    782  783   14.89   39.50                  -43.40\n",
      "   10   6838 117V 118S C   N     892  894 -129.55 -136.60   16.12    0.64  -64.10 -176.25   11.35\n",
      "   10        118S 118S N   CA    894  895  136.70  151.20                  -35.00\n",
      "   11   6839 118S 119S C   N     898  900  -97.81  -72.40   29.92    1.59  -64.10  160.38   13.90\n",
      "   11        119S 119S N   CA    900  901  168.20  152.40                  -35.00\n",
      "   12   6840 119S 120G C   N     904  906 -114.20  -80.20   43.72    3.16   82.20 -145.90   16.62\n",
      "   12        120G 120G N   CA    906  907  146.61  174.10                    8.50\n",
      "   13   6841 120G 121G C   N     908  910   98.07   78.70   19.46    0.93   82.20  177.27    7.80\n",
      "   13        121G 121G N   CA    910  911 -168.05 -166.10                    8.50\n",
      "   14   6842 121G 122G C   N     912  914 -174.30 -167.20   15.08    0.61   82.20 -175.46   13.45\n",
      "   14        122G 122G N   CA    914  915  161.29  174.60                    8.50\n",
      "   15   6844 123G 124S C   N     920  922 -109.52 -136.60   41.18    1.38  -64.10  161.69    9.99\n",
      "   15        124S 124S N   CA    922  923  120.18  151.20                  -35.00\n",
      "   16   6846 125G 126G C   N     930  932 -139.14 -167.20   28.44    0.45   82.20 -147.16   16.06\n",
      "   16        126G 126G N   CA    932  933  169.98  174.60                    8.50\n",
      "   17   6847 126G 127G C   N     934  936 -114.63  -80.20   47.38    3.37   82.20 -149.47   16.37\n",
      "   17        127G 127G N   CA    936  937  141.55  174.10                    8.50\n",
      "   18   6848 127G 128G C   N     938  940 -156.63 -167.20   10.85    0.16   82.20 -156.37   15.05\n",
      "   18        128G 128G N   CA    940  941  172.16  174.60                    8.50\n",
      "   19   6849 128G 129S C   N     942  944  -86.51  -72.40   30.93    1.43  -64.10  146.80   12.11\n",
      "   19        129S 129S N   CA    944  945  179.92  152.40                  -35.00\n",
      "   20   6850 129S 130G C   N     948  950 -140.57 -167.20   29.69    0.92   82.20 -134.27   16.77\n",
      "   20        130G 130G N   CA    950  951 -172.26  174.60                    8.50\n",
      "   21   6852 131G 132G C   N     956  958 -124.75 -167.20   73.51    1.97   82.20 -173.79   14.55\n",
      "   21        132G 132G N   CA    958  959  114.58  174.60                    8.50\n",
      "   22   6853 132G 133G C   N     960  962  -65.59  -62.40    6.24    1.10   82.20  157.72   12.06\n",
      "   22        133G 133G N   CA    962  963  -46.56  -41.20                    8.50\n",
      "   23   6854 133G 134S C   N     964  966  102.44 -136.60  122.57    4.37  -64.10 -133.17   18.99\n",
      "   23        134S 134S N   CA    966  967  170.99  151.20                  -35.00\n",
      "   24   6855 134S 135T C   N     970  972 -122.63 -124.80   43.09    1.91  -63.20  144.19   22.38\n",
      "   24        135T 135T N   CA    972  973 -173.47  143.50                  -42.10\n",
      "   25   6856 135T 136D C   N     977  979  -55.06  -63.30   19.16    2.04  -70.90  153.23    8.35\n",
      "   25        136D 136D N   CA    979  980  -57.29  -40.00                  150.30\n",
      "   26   6884 163D 164V C   N    1180 1182 -127.68  -62.40  100.60   11.32  -62.40  100.60   11.32\n",
      "   26        164V 164V N   CA   1182 1183   34.15  -42.40                  -42.40\n",
      "   27   6885 164V 165N C   N    1187 1189   51.52  -71.20  140.71   11.34 -119.90 -173.04    9.42\n",
      "   27        165N 165N N   CA   1189 1190 -148.36  142.80                  137.00\n",
      "   28   6907 186A 187S C   N    1360 1362 -148.15  -64.10   95.51    9.71  -64.10   95.51    9.71\n",
      "   28        187S 187S N   CA   1362 1363   10.36  -35.00                  -35.00\n",
      "   29   6962 241I 242K C   N    1813 1815  -97.26 -118.00   54.53    2.29  -62.90  133.96   15.86\n",
      "   29        242K 242K N   CA   1815 1816   88.68  139.10                  -40.80\n",
      "   30   6963 242K 243H C   N    1822 1824 -163.33 -125.60   45.25    2.00  -63.20 -174.51   19.78\n",
      "   30        243H 243H N   CA   1824 1825  113.84  138.80                  -42.30\n",
      "   31   6964 243H 244H C   N    1832 1834 -147.22 -125.60   48.54    1.40  -63.20  159.38   25.32\n",
      "   31        244H 244H N   CA   1834 1835 -177.74  138.80                  -42.30\n",
      "   32   6965 244H 245H C   N    1842 1844  -65.07  -67.60    2.66    0.22  -63.20  176.88   22.23\n",
      "   32        245H 245H N   CA   1844 1845  140.83  140.00                  -42.30\n",
      "   33   6966 245H 246H C   N    1852 1854 -155.18 -125.60   46.56    1.19  -63.20  169.98   27.13\n",
      "   33        246H 246H N   CA   1854 1855  174.75  138.80                  -42.30\n",
      "   34   6967 246H 247H C   N    1862 1864  -76.58  -67.60   10.60    0.93  -63.20  177.19   21.35\n",
      "   34        247H 247H N   CA   1864 1865  134.38  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   12   27   89   92  127  162  160  241  269  301\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "scfv.B99990001.pdb            3035.92285   -22234.18750        1.00000\n",
      "scfv.B99990002.pdb            3101.36060   -22112.36133        1.00000\n",
      "scfv.B99990003.pdb            3076.25513   -22360.95508        1.00000\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "from modeller.automodel import *\n",
    "\n",
    "env = environ()\n",
    "a = automodel(env, alnfile='scfv.ali',\n",
    "              knowns=[\"2fjhHL\"], sequence=\"scfv\",\n",
    "              assess_methods=(assess.DOPE, assess.GA341))\n",
    "a.starting_model = 1\n",
    "a.ending_model = 3\n",
    "a.make()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Seleccionar el mejor modelo"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def do_profile(name):\n",
    "    from modeller import environ, selection\n",
    "    from modeller.scripts import complete_pdb\n",
    "\n",
    "    log.verbose()    # request verbose output\n",
    "    env = environ()\n",
    "    env.libs.topology.read(file='$(LIB)/top_heav.lib') # read topology\n",
    "    env.libs.parameters.read(file='$(LIB)/par.lib') # read parameters\n",
    "    \n",
    "    # read model file\n",
    "    mdl = complete_pdb(env, '%s.pdb'%name)\n",
    "\n",
    "    # Assess with DOPE:\n",
    "    s = selection(mdl)   # all atom selection\n",
    "    s.assess_dope(output='ENERGY_PROFILE NO_REPORT', file='%s.profile' %name,\n",
    "                  normalize_profile=True, smoothing_window=15)\n",
    "\n",
    "    return"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12515140   12221.816    11.935\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12515668   12222.332    11.936\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12564268   12269.793    11.982\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12577568   12282.781    11.995\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12717004   12418.949    12.128\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12718604   12420.512    12.129\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12722584   12424.398    12.133\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12724584   12426.352    12.135\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12727584   12429.281    12.138\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12732084   12433.676    12.142\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12737656   12439.117    12.148\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12740456   12441.852    12.150\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12744656   12445.953    12.154\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12750956   12452.105    12.160\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12753728   12454.812    12.163\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12755128   12456.180    12.164\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           scfv.B99990003.pdb\n",
      "readlinef__W> File: scfv.B99990003.pdb, Line: 6\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12766101   12466.896    12.175\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12766177   12466.970    12.175\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12766348   12467.137    12.175\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12767198   12467.967    12.176\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12768456   12469.195    12.177\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12770360   12471.055    12.179\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12770400   12471.094    12.179\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12770490   12471.182    12.179\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12773346   12473.971    12.182\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12773406   12474.029    12.182\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12773541   12474.161    12.182\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12777825   12478.345    12.186\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12777913   12478.431    12.186\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12778111   12478.624    12.186\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12784537   12484.899    12.192\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12784669   12485.028    12.192\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12784966   12485.318    12.193\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12794588   12494.715    12.202\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12794788   12494.910    12.202\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12795238   12495.350    12.202\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12809688   12509.461    12.216\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12809988   12509.754    12.217\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12810663   12510.413    12.217\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12832321   12531.563    12.238\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12832769   12532.001    12.238\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12833777   12532.985    12.239\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13004465   12699.673    12.402\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     13012209   12707.235    12.409\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     13153901   12845.606    12.545\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     13156309   12847.958    12.547\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     13298001   12986.329    12.682\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13327481   13015.118    12.710\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933      0         0       876\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13408741   13094.474    12.788\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933   2622      3114       876\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13485517   13169.450    12.861\n",
      "patch_______> segment topology patched using RTF:   248  ;  HIS  ;  CTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1884        0\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  HIS:OXT  HIS\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     13734697   13412.790    13.098\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14072517   13742.692    13.421\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14117733   13786.849    13.464\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14208165   13875.161    13.550\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14389029   14051.786    13.722\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14750757   14405.036    14.067\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     15474213   15111.536    14.757\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     16921125   16524.535    16.137\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   357151\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -22361.1484\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "31 Non-bonded restraints              :  357108       0    534   6.789   6.789     -22361.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           scfv.B99990003.profile\n",
      "# Energy of each residue is written to: scfv.B99990003.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:      -6.5476\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -22361.148438\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12515140   12221.816    11.935\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12515668   12222.332    11.936\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12564268   12269.793    11.982\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12577568   12282.781    11.995\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12717004   12418.949    12.128\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12718604   12420.512    12.129\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12722584   12424.398    12.133\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12724584   12426.352    12.135\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12727584   12429.281    12.138\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12732084   12433.676    12.142\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12737656   12439.117    12.148\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12740456   12441.852    12.150\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12744656   12445.953    12.154\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12750956   12452.105    12.160\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12753728   12454.812    12.163\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     12755128   12456.180    12.164\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           2fjhHL.pdb\n",
      "readlinef__W> File: 2fjhHL.pdb, Line: 1\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12766101   12466.896    12.175\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12766177   12466.970    12.175\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12766348   12467.137    12.175\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12767198   12467.967    12.176\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12768456   12469.195    12.177\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12770360   12471.055    12.179\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12770400   12471.094    12.179\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12770490   12471.182    12.179\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12773346   12473.971    12.182\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12773406   12474.029    12.182\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12773541   12474.161    12.182\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12777825   12478.345    12.186\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12777913   12478.431    12.186\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12778111   12478.624    12.186\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12784537   12484.899    12.192\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12784669   12485.028    12.192\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12784966   12485.318    12.193\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12794588   12494.715    12.202\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12794788   12494.910    12.202\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12795238   12495.350    12.202\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12809688   12509.461    12.216\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12809988   12509.754    12.217\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12810663   12510.413    12.217\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12832321   12531.563    12.238\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12832769   12532.001    12.238\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     12833777   12532.985    12.239\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13004465   12699.673    12.402\n",
      "read_mo_297_> Segments, residues, atoms:        1      227     1727\n",
      "read_mo_298_> Segment:        1     1 A   107 A     1727\n",
      "relabel_469W> At least one residue number in the model is duplicated.\n",
      "              (First duplication is residue \"    1\", chain \" A\".)\n",
      "              Suggest you renumber the residues and/or chains to\n",
      "              avoid selecting the wrong residues (e.g. with model_segment).\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     13011600   12706.641    12.409\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     13141381   12833.380    12.533\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     13143600   12835.547    12.535\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     13273381   12962.286    12.658\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13299653   12987.942    12.684\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      227  1727  1770      0         0       757\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13374561   13061.095    12.755\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  GLU  ;  NTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      227  1727  1770   2410      2876       757\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13451337   13136.071    12.828\n",
      "patch_______> segment topology patched using RTF:   227  ;  LYS  ;  CTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      227  1728  1771   2412      2878       758\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      227  1728  1771   2412      2878       758\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1727        1\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LYS:OXT  LYS\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     13680209   13359.579    13.046\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14014285   13685.825    13.365\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14055757   13726.325    13.405\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14138701   13807.325    13.484\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14304589   13969.325    13.642\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14636365   14293.325    13.958\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     15299917   14941.325    14.591\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     16627021   16237.325    15.857\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     19281229   18829.324    18.388\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      227\n",
      "Number of all, selected real atoms                :     1728    1728\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   338408\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -25187.8945\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "31 Non-bonded restraints              :  338260       0    143   6.817   6.817     -25188.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           2fjhHL.profile\n",
      "# Energy of each residue is written to: 2fjhHL.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:      -8.2093\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -25187.894531\n"
     ]
    }
   ],
   "source": [
    "modelsel = 3\n",
    "from modeller import *\n",
    "from modeller.automodel import *\n",
    "\n",
    "pdb_model_name = '%s.B9999000%d' % (name, modelsel)\n",
    "pdb_template_name = '%s' % (\"2fjhHL\")\n",
    "\n",
    "do_profile(pdb_model_name)\n",
    "do_profile(pdb_template_name)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Obtain DSSP information\n",
    "\n",
    "Code from dssp\n",
    "\n",
    "    H = alpha helix\n",
    "    B = residue in isolated beta-bridge\n",
    "    E = extended strand, participates in beta ladder\n",
    "    G = 3-helix (3/10 helix)\n",
    "    I = 5 helix (pi helix)\n",
    "    T = hydrogen bonded turn\n",
    "    S = bend\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def do_dssp(pdbIN):\n",
    "    import os\n",
    "    dssp = \"/Users/alsalas/anaconda3/bin/mkdssp\"\n",
    "    comando = \"%s -i %s.pdb -o %s.dssp\" % (dssp, pdbIN, pdbIN)\n",
    "    os.system(comando)\n",
    "    \n",
    "    with open(\"%s.dssp\"%pdbIN, \"r\") as f:\n",
    "        #head = \"#  RESIDUE AA STRUCTURE BP1 BP2  ACC     N-H-->O    O-->H-N    N-H-->O    O-->H-N    TCO  KAPPA ALPHA  PHI   PSI    X-CA   Y-CA   Z-CA\"\n",
    "        #print [x for x in head.split(\" \") if len(x) > 0]\n",
    "        residue_num  = [0]\n",
    "        residue_name = \"\"\n",
    "        structure2D  = \"\"\n",
    "        resnumber = 0\n",
    "        for l in f.readlines()[25:]:\n",
    "            residue_name += l[13]\n",
    "            structure2D  += l[16]\n",
    "            try:\n",
    "                resnumber0 = residue_num[-1]\n",
    "                resnumber = int(l[6:10])\n",
    "                residue_num.append(resnumber)\n",
    "            except:\n",
    "                residue_name = residue_name[:-1]\n",
    "                structure2D  = structure2D[:-1]\n",
    "                resnumber0 = residue_num[-1]\n",
    "            \n",
    "            if resnumber - resnumber0 > 1:\n",
    "                gap = \"-\"*(resnumber-resnumber0-1)\n",
    "                residue_name = residue_name[:-1]+gap+residue_name[-1]\n",
    "                structure2D = structure2D[:-1]+gap+structure2D[-1]\n",
    "     \n",
    "    return residue_name,structure2D"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "231\n"
     ]
    }
   ],
   "source": [
    "seq, structure2D = do_dssp(\"2fjhHL\")\n",
    "print (len(structure2D))\n",
    "structure2D_gaps = \"\"\n",
    "counter = 0\n",
    "for aa in str(TSEQ_FINAL):\n",
    "    if aa == \"-\":\n",
    "        structure2D_gaps += \"-\"\n",
    "    else:\n",
    "        structure2D_gaps += structure2D[counter]\n",
    "        counter +=1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_2dprofile(structure2D, value=-0.04):\n",
    "    valsH = [None]\n",
    "    for e in structure2D:\n",
    "        if e == \"E\":\n",
    "            valsH.append(value)\n",
    "        else:\n",
    "            valsH.append(None)\n",
    "    \n",
    "    valsS = [None]\n",
    "    for e in structure2D:\n",
    "        if e == \"S\":\n",
    "            valsS.append(value)\n",
    "        else:\n",
    "            valsS.append(None)\n",
    "    \n",
    "    valsT = [None]\n",
    "    for e in structure2D:\n",
    "        if e == \"T\":\n",
    "            valsT.append(value)\n",
    "        else:\n",
    "            valsT.append(None)\n",
    "    #print valsH\n",
    "    return valsH, valsS, valsT\n",
    "    \n",
    "def get_profileline(structure2D):\n",
    "    vals = [None]\n",
    "    for e in structure2D:\n",
    "        vals.append(-0.03)\n",
    "    return vals\n",
    "\n",
    "def get_rangeprofile(structure2D, lista, value=-0.05):\n",
    "    valsR = [None]\n",
    "    largo = len(structure2D)\n",
    "    j = 0\n",
    "    counter = 0\n",
    "    end = 100\n",
    "    for i in range(largo):\n",
    "        if counter > end and (j< len(lista)-1):\n",
    "            j += 1\n",
    "        start = lista[j][0]\n",
    "        end = lista[j][1]\n",
    "        \n",
    "        if (counter >= start) and (counter <= end):\n",
    "            valsR.append(value)\n",
    "        else:\n",
    "            valsR.append(None)\n",
    "        counter += 1\n",
    "    #print valsR\n",
    "    return valsR"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Plot DOPE with 2D"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%matplotlib inline\n",
    "import pylab\n",
    "import modeller"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12515140   12221.816    11.935\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12515668   12222.332    11.936\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12564268   12269.793    11.982\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12577568   12282.781    11.995\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           scfv.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     12579754   12284.916    11.997\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     12581204   12286.332    11.998\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     12584104   12289.164    12.001\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     12589904   12294.828    12.007\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     12706960   12409.141    12.118\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12707864   12410.023    12.119\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     12708852   12410.988    12.120\n",
      "\n",
      "Read the alignment from file       : scfv.ali\n",
      "Total number of alignment positions:   253\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1     2fjhHL     227      1  2fjhHL.pdb\n",
      "  2       scfv     248      1        scfv\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABKEAAAF3CAYAAAB9mFiyAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4yLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvOIA7rQAAIABJREFUeJzs3XuclGX5+PHPDYugiJiAR1ROHgDBBRExz0cUUTyVmqfKUjLLNP1FffOYlVmZWamp9cXUNLOvSoWpEVaeASFFQQRFDgoCJqJy3vv3x70LKyzLLDuzz8zs5/167es5zDPPc82KM7PXc93XHWKMSJIkSZIkSYXUIusAJEmSJEmSVP5MQkmSJEmSJKngTEJJkiRJkiSp4ExCSZIkSZIkqeBMQkmSJEmSJKngTEJJkiRJkiSp4ExCSZIkSZIkqeBMQkmSJEmSJKngTEJJkiRJkiSp4ExCSZIkSZIkqeAqsg6gKXXs2DF26dIl6zAkSZIkSZLKxoQJExbGGDtt7LhmlYTq0qUL48ePzzoMSZIkSZKkshFCeCuX4xyOJ0mSJEmSpIIzCSVJkiRJkqSCMwklSZIkSZKkgmtWPaEkSZIkSVLzs3LlSubMmcOyZcuyDqWktWnThs6dO9OqVatNer5JKEmSJEmSVNbmzJlDu3bt6NKlCyGErMMpSTFGFi1axJw5c+jatesmncPheJIkSZIkqawtW7aMDh06mIBqhBACHTp0aFQ1mUkoSZIkSZJU9kxANV5jf4cmoSRJkiRJkgoshMDZZ5+9ZnvVqlV06tSJoUOHNug8Xbp0YeHChY0+JgsmoSRJkiRJkgqsbdu2TJ48maVLlwLwxBNPsNNOO2UcVdMyCSVJkiRJktQEjj32WP76178CcN9993HGGWeseey9997jxBNPpG/fvgwaNIiXXnoJgEWLFnH00UfTr18/LrjgAmKMa55zzz33MHDgQCorK7ngggtYvXp1076gBjIJJUmSJEmSmo8QCvOTg9NPP53777+fZcuW8dJLL7Hffvuteeyqq66iX79+vPTSS/zgBz/gnHPOAeCaa67hwAMPZOLEiZxwwgnMmjULgClTpvCHP/yBp59+mkmTJtGyZUvuvffe/P++8qgi6wAkSZKURzHCffdBt24waFDW0UiSpFr69u3LzJkzue+++xgyZMgnHnvqqaf405/+BMDhhx/OokWLWLx4Mf/617/4v//7PwCOO+44PvWpTwEwZswYJkyYwL777gvA0qVL2XbbbZvw1TScSShJkqRyMno0nHkmtGgBN9wAl16a891ZSZKahVrD2bJwwgkncNlll/Hkk0+yaNGiNftjHXHVzEZX16x0MUbOPfdcfvjDHxYu2DxzOJ4kSVI5ueWWtKyqgssug3POgeoGqJIkKXtf/OIXufLKK+nTp88n9h988MFrhtM9+eSTdOzYka222uoT+x999FH++9//AnDEEUfw4IMP8u677wKpp9Rbb73VhK+k4UxCSZIklYs334RHH4XNNoM77oC2beGee+Cgg2D27KyjkyRJQOfOnbn44ovX23/11Vczfvx4+vbty4gRI7jrrruA1CvqX//6F/379+fxxx9nl112AaBXr15cd911HH300fTt25ejjjqKd955p0lfS0OFusq9ytWAAQPi+PHjsw5DkiSpMEaMgB/9CM46C+6+G15+GU48Ed54Aw4/HMaMyTpCSZIyMWXKFHr27Jl1GGWhrt9lCGFCjHHAxp5rJZQkSVI5WL4cfvObtH7hhWnZpw889VRaf+65NERPkiQpIyahJEmSysGDD8LChbD33p+cFW+HHWDbbeHjj2Hu3OzikyRJzZ5JKEmSpHJw661peeGF68+Gt+eeaTl1atPGJEmSVItJKEmSpFL30kvw9NPQrh187nPrP77HHmn52mtNG5ckSVItJqEkSZJKXU0V1LnnwpZbrv+4lVCSJKkImISSJEkqZR98APfck9aHD6/7mJoklJVQkiQpQyahJEmSStk998CHH8Ihh0Dv3nUfUzMcz0ooSZIysWjRIiorK6msrGT77bdnp512WrO9YsWKvF9v1apVbL311vUe895773Hbbbfl/dr1MQklSZJUqmJcOxTvK1/Z8HFdusBmm8GcOSlhJUmSmlSHDh2YNGkSkyZNYvjw4VxyySVrtjfbbLNMYjIJJUmSpNw99RRMngzbbQcnnbTh41q2hN12S+vTpjVNbJIkKSd33XUXAwcOpLKykgsvvJCqqqo1lUyXX345/fv3Z/DgwTz//PMccsghdOvWjdGjRwNw5513ctJJJzF48GD22GMPrrvuuvXO/8EHH3D44YfTv39/+vbty1/+8hcARowYwWuvvUZlZSUjRowA4Prrr2fgwIH07duXa6+9Nu+vtSLvZ2yAEMIxwM+BlsCdMcbr13m8NfA7YB9gEXBajHFmrcd3AV4Fro4x/qSp4pYkSSoKNVVQX/pSqnSqz557wiuvpCF5/fsXPjZJkopUuCYU5Lzxqtjg50yePJmHHnqIZ555hoqKCs4//3zuv/9+PvvZz7J48WKOPvpofvzjH3P88cdz9dVXM2bMGP7zn/9wwQUXMGTIEABeeOEFJk+ezGabbca+++7L0KFD2WuvvdZcY/PNN+eRRx6hXbt2vPvuuxxwwAEMHTqU66+/nunTpzNp0iQARo8ezaxZs3j++eeJMTJkyBCeeeYZPv3pT+fnF0SGSagQQkvgV8BRwBxgXAhhVIzx1VqHnQf8N8bYI4RwOvAj4LRaj/8MeLSpYpYkSSoa774LDz4ILVrA+edv/PiavlA2J5ckqWj8/e9/Z9y4cQwYMACApUuXsvPOOwMpeXTUUUcB0KdPH9q3b09FRQV9+vRh5syZa84xePBgPvWpTwFw4okn8tRTT30iCRVj5Fvf+hZPPfUULVq0YPbs2SxcuHC9WB5//HEeffRR+vXrB8CHH37ItGnTyiMJBQwEpscY3wAIIdwPDCNVNtUYBlxdvf4g8MsQQogxxhDCicAbwEdNF7IkSVKR+M1vYOVKOOEE2GWXjR9fM0OezcklSc3cplQsFUqMkS9+8Yt873vf+8T+VatWfaJXVIsWLWjduvWa9VWrVq15LIRPVnatu/273/2OxYsX8+KLL1JRUUHnzp1ZtmxZnbF897vf5bzzzmv069qQLHtC7QTMrrU9p3pfncfEGFcBi4EOIYS2wLeAa5ogTkmSpOKyejX8+tdpvb6G5LVZCSVJUtE58sgjeeCBB9ZUJi1atIhZs2Y16ByPP/4477//Ph9//DGPPPIIBxxwwCceX7x4Mdtuuy0VFRU88cQTzJ07F4B27dqxZMmSNccNHjyY3/zmN3z0Uar1mTNnTp0VU42RZSVUXYMw101HbuiYa4CfxRg/XDfDt95FQjgfOB9gl1zuEkqSJBW7Rx+Ft96Cbt3g6KNze05NEmraNKiqSsP4JElSpvr06cNVV13FkUceSVVVFa1ateK2225jxx13zPkcBx54IJ/73OeYMWMGZ599NpWVlZ+olDr77LM5/vjjGTBgAP3792e36slKtttuOwYMGECfPn047rjjuP7665k6dSqDBg0CUpLq97//PR07dszb6w0xZlOGFkLYn9RQfHD19rcBYow/rHXMY9XHPBtCqADmAZ2AfwE7Vx+2NVAFXBlj/GV91xwwYEAcP3583l+LJElSkzruOBg9Gm64AS6/PPfn7bADzJsHM2fCrrsWLDxJkorNlClT6NmzZ9Zh5N2dd97J5MmTuemmm5rsmnX9LkMIE2KMAzb23CxvgY0DdgshdA0hbAacDoxa55hRwLnV66cC/4jJQTHGLjHGLsBNwA82loCSJEkqC489liqhWreGL3yhYc+1L5QkScpQZsPxYoyrQggXAY8BLYHfxhhfCSFcC4yPMY4CfgPcHUKYDrxHSlRJkiQ1PzHCT38K3/pWWh8+HBpaHr/HHvDkkykJNXhwQcKUJElN50tf+lLWITRIlj2hiDGOBkavs+/KWuvLgM9s5BxXFyQ4SZKkYrF0KXz5y3DvvWn7yivhqqsafp6aSiibk0uSpAxkmoSSJEnSRixZAocdBhMmQNu28Lvfwcknb9q5HI4nSZIyZBJKkiSpmP361ykB1aULjBoFffps+rlqZsizEkqSJGXAuXklSZKKVVUV3HZbWr/55sYloAB22QXatIG334YPPmh8fJIkSQ1gEkqSJKlYPfEEzJiRkkdDhjT+fC1bwm67pfVp0xp/PkmSlJPZs2dz2GGH0bNnT3r37s3Pf/7zNY99/vOfp2vXruy9997svvvunHPOOcydO3e9c5x00klUVlbSo0cP2rdvT2VlJZWVlTzzzDNN+VIaxSSUJElSsbr11rQ8//yUQMoH+0JJktTkKioq+OlPf8qUKVN47rnn+NWvfsWrr7665vEf//jH/Oc//+G1116jX79+HHbYYaxYseIT53jooYeYNGkSd955JwcddBCTJk1i0qRJfPrTn27ql7PJ7AklSZJUjGbNgj//GVq1gvPOy9957QslSWrOfh8Ke/7PxTp377DDDuywww4AtGvXjp49ezJ37lx69er1ieNCCFxyySU89NBDPProowwbNiyny44bN47LLruMDz/8kG233ZaRI0ey3XbbceCBBzJw4EDGjRvHokWL+N3vfsf3v/99Jk+ezJlnnsnVV1/N9OnTGTZsGP3792fSpEn07NmTu+66i80337xxv4s6WAklSZJUjO64I/WEOvlk2H77/J3XSihJkjI1c+ZMJk6cyH777bfBY/r378/UHD+rly9fzsUXX8yf/vQnJkyYwFlnncUVV1yx5vHNN9+cf//735x33nmceOKJ3Hbbbbz88svcfvvtvP/++wC8+uqrfPWrX+Xll1+mTZs2/PrXv27ci9wAK6EkSZKKzYoVKQkFcOGF+T13TSXUlCn5Pa8kSdqoDz/8kFNOOYWbbrqJrbbaaoPHxVh3RVVdpkyZwiuvvMKRRx4JwOrVq+ncufOax0844QQA+vTpQ58+fdhuu+0A6NKlC3PmzKFNmzZ07dqVQYMGAXDWWWdx++23841vfKPBr29jTEJJkiQVm4cfhvnzoXdvOOig/J67Z08IITUmX7kyDfeTJEkFt3LlSk455RTOPPNMTj755HqPnThxIkcccURO540x0rdvX/7973/X+Xjr1q0BaNGixZr1mu1Vq1YBaRhgbetu54tJKEmSpGJzyy1p+ZWvpIRRPrVtC926pVn3XnsN9torv+eXJKmYbaBnU6HFGDnvvPPo2bMnl156ab3H/eIXv+Cdd97hmGOOyencvXr1Yu7cubzwwgsMHDiQFStW8Prrr9O7d++c43vzzTcZN24c++67L/fddx8HHnhgzs9tCHtCSZIkFZPXXoN//jMli84+uzDX6NMnLV96qTDnlyRJn/D0009z9913849//IPKykoqKysZPXr0mscvv/xy9t57b3bffXfGjRvH2LFj2WyzzXI6d+vWrXnwwQe59NJL2XvvvenXrx/PP/98g+Lr3bs3d9xxB3379uWjjz7i/PPPb9Dzc2UllCRJUjGp+dJ47LFQT6+IRunTJw35e/nlwpxfkiR9woEHHrjBPk8jR45s0LkOPfRQDj300E/s69+/P0899dR6x9bed+SRR67pG1X7senTp9OyZUtuv/32BsWxKayEkiRJKiZvvZWWPXoU7hp9+6alSShJktSETEJJkiQVk5kz07JLl8Jdo2Y4nkkoSZKavR49ejBp0qQmuZZJKEmSpGJSk4TaddfCXaNHD2jTBmbNgsWLC3cdSZKkWkxCSZIkFZOa4XiFrIRq2RJ69UrrVkNJkqQmYhJKkiSpWKxenaqToLCVUOCQPEmS1ORMQkmSJBWLd96BlSth221h880Ley2TUJIkqYmZhJIkSSoWTTEUr4Yz5EmS1KTmzZvH6aefTvfu3enVqxdDhgxh2rRpbL755vTr14+ePXsycOBA7rrrrjXPGTlyJJ06daKyspI999yTn/3sZxm+gsaryDoASZIkVWuKmfFq1K6EihFCKPw1JUnKUFN91MVY177ISSedxLnnnsv9998PwKRJk5g/fz7du3dn4sSJALzxxhucfPLJVFVV8YUvfAGA0047jV/+8pcsWrSIPfbYg1NPPZWdd965aV5MnlkJJUmSVCxqKqEK3Q8KYLvtoGPHNDvenDmFv54kSc3Y2LFjadWqFcOHD1+zr7Kycr1kUrdu3bjxxhu5+eab1ztHhw4d6NGjB++88w4Af/zjH9lrr73Ye++9Ofjggwv7AvLESihJkqRi0ZSVUCGkaqixY+Gll6BE76hKklQKJk+ezD777JPTsf3792fq1Knr7Z81axbLli2jb/WQ+muvvZbHHnuMnXbaiffffz+v8RaKlVCSJEnFoiYJ1RSVUGBzckmSilBcZzzfH/7wB3r37k23bt24+OKLadOmDQAHHHAAn//857njjjtYvXp1FqE2mEkoSZKkYtGUjcnBJJQkSU2kd+/eTJgwIadjJ06cSM+ePddsn3baabzyyiv8+9//5pvf/Cbz5s0D4LbbbuO6665j9uzZVFZWsmjRooLEnk8moSRJkopBVVXT9oQCZ8iTJKmJHH744Sxfvpw77rhjzb5x48bxVs1nf7WZM2dy2WWX8bWvfW29c+y///6cffbZ/PznPwdgxowZ7Lffflx77bV07NiR2bNnF/ZF5IE9oSRJkorB/PmwfDl06ABbbtk01+zdO/WGmjoVVq6EVq2a5rqSJGWgrlnrmkoIgYceeohvfOMbXH/99bRp04YuXbpw0003MWPGDPr168eyZcto164dX/va19bMjLeub33rW/Tv35/vfOc7XH755bz++uvEGDniiCPYe++9m/hVNVxYd6xhORswYEAcP3581mFIkiSt77nnYP/9YZ99oCm/r/ToATNmpGqovfZquutKktSEpkyZ8okhbtp0df0uQwgTYowDNvZch+NJkiQVg6acGa+2mr5QL73UtNeVJEnNjkkoSZKkYtDUM+PVsDm5JElqIiahJEmSikFTz4xXw+bkkiSpiZiEkiRJKgZZVULVJKGefx5WrGjaa0uSpGbFJJQkSVIxyKoSarfd0ix5CxfCww837bUlSVKzYhJKkiQpazFmVwkVAgwfntZvvbVpry1JkpoVk1CSJElZW7AAli6FrbeG9u2b/vpnnw1t28KTT8KUKU1/fUmSmoF58+Zx+umn0717d3r16sWQIUOYNm3aBo+/6aab+Pjjjxt8nTPOOIO+ffvys5/9jKlTp1JZWUm/fv2YMWNGY8LPC5NQkiRJWctqKF6N9u3hzDPTutVQkiTlXYyRk046iUMPPZQZM2bw6quv8oMf/ID58+dv8DmbkoSaN28ezzzzDC+99BKXXHIJDz/8MMOGDWPixIl07969sS+j0UxCSZIkZa1mKF5WSSiAr3wlLe+6Cz76KLs4JEkqQ2PHjqVVq1YMrxkCD1RWVrJ69WqGDh26Zt9FF13EyJEjufnmm3n77bc57LDDOOyww9Y737Jly/jCF75Anz596NevH2PHjgXg6KOP5t1336WyspJrrrmGm266iTvvvLPOc2ShIusAJEmSmr2s+kHVVlkJ++8Pzz4Lv/89fPnL2cUiSVKBXXPNNRt8bOjQoeyzzz4ATJgwgb/85S8bPPaqq67K6XqTJ09ec85cfP3rX+fGG29k7NixdOzYcb3Hf/WrXwHw8ssvM3XqVI4++mimTZvGqFGjGDp0KJMmTQJSBdaWW27JZZddlvO1C8lKKEmSpKxlPRyvxoUXpuWtt6Zm6ZIkqSg99dRTnH322QDsueee7LrrrvX2lyoWVkJJkiRlrRiG4wGceip84xswcSK88ALst1+28UiSVCC5VjDts88+Dapg2pDevXvz4IMPrre/oqKCqqqqNdvLli2r8/kPPfTQmuqtO++8k1iiN4ushJIkScpaTSVUlsPxANq0gfPOS+u33JJtLJIklZHDDz+c5cuXc8cdd6zZN27cOFavXs2rr77K8uXLWbx4MWPGjFnzeLt27ViyZAkAJ510EpMmTWLSpEkMGDCAgw8+mHvvvReAadOmMWvWLPbYY4+mfVGbwCSUJElSlmIsnkoogAsuSMs//hGWLs02FkmSykQIgYceeognnniC7t2707t3b66++mp23HFHPvvZz9K3b1/OPPNM+vXrt+Y5559/Pscee2ydTcUvvPBCVq9eTZ8+fTjttNMYOXIkrVu3bsqXtElCqZZwbYoBAwbE8ePHZx2GJEnSWosWQceO0K4dLF4MIWQdEQwYABMmwKhRcPzxWUcjSVKjTZkyhZ49e2YdRlmo63cZQpgQYxywsedaCSVJkpSl2k3JiyEBBXDiiWn58MPZxiFJksqKSShJkqQsFUs/qNqGDUvLP/8ZVq/ONhZJklQ2TEJJkiRl6Z130nKnnbKNo7a99oKuXWHBAnjuuayjkSRJZcIklCRJUpZqklDbb59tHLWFsLYa6pFHso1FkqQ8aU49sQulsb9Dk1CSJElZmjcvLYspCQUmoSRJZaVNmzYsWrTIRFQjxBhZtGgRbdq02eRzVOQxHkmSJDVUTRJqhx2yjWNdBx4I22wD06bB1Kmw555ZRyRJ0ibr3Lkzc+bMYcGCBVmHUtLatGlD586dN/n5JqEkSZKyVIzD8QAqKuC44+Duu1M1lEkoSVIJa9WqFV27ds06jGbP4XiSJElZKtbheOCQPEmSlFcmoSRJkrJSVQXz56f1YkxCDR4MrVunGfJq4pQkSdpEJqEkSZKysmgRrFoFn/pUSvYUmy23hCOOgBjhz3/OOhpJklTiTEJJkiRlpZiH4tVwSJ4kScoTk1CSJElZKdaZ8Wo7/vi0HDs2DR+UJEnaRCahJEmSslKsM+PVtsMOsO228NFHMHdu1tFIkqQSZhJKkiQpK6UwHA9gjz3ScurUbOOQJEklLdMkVAjhmBDCayGE6SGEEXU83jqE8Ifqx58PIXSp3n9UCGFCCOHl6uXhTR27JElSo5XCcDyAPfdMy9deyzYOSZJU0jJLQoUQWgK/Ao4FegFnhBB6rXPYecB/Y4w9gJ8BP6revxA4PsbYBzgXuLtpopYkScqjUhiOB1ZCSZKkvMiyEmogMD3G+EaMcQVwPzBsnWOGAXdVrz8IHBFCCDHGiTHGt6v3vwK0CSEU4bzGkiRJ9SiV4XhWQkmSpDzIMgm1EzC71vac6n11HhNjXAUsBjqsc8wpwMQY4/ICxSlJklQYpTIcz0ooSZKUB1kmoUId+2JDjgkh9CYN0btggxcJ4fwQwvgQwvgFCxZsUqCSJEkFUSrD8bp0gc02gzlz4MMPs45GkiSVqCyTUHOAnWttdwbe3tAxIYQKoD3wXvV2Z+Ah4JwY44wNXSTGeHuMcUCMcUCnTp3yGL4kSVIjLF0KixdDq1awzTZZR1O/igro0SOtT5uWbSySJKlkZZmEGgfsFkLoGkLYDDgdGLXOMaNIjccBTgX+EWOMIYStgb8C344xPt1kEUuSJOXL/Plpuf32EOoq/i4y9oWSJEmNlFkSqrrH00XAY8AU4IEY4yshhGtDCCdUH/YboEMIYTpwKTCiev9FQA/gihDCpOqfbZv4JUiSJG26UhmKV8O+UJIkqZEqsrx4jHE0MHqdfVfWWl8GfKaO510HXFfwACVJkgqlVGbGq2EllCRJaqQsh+NJkiQ1X6UyM16NmiSUlVCSJGkTmYSSJEnKQqkOx5s2Daqqso1FkiSVJJNQkiRJWSi14Xjt26dYly6F2bOzjkaSJJUgk1CSJElZKLXheLC2Gsq+UJIkaRNsNAkVQtguhPCbEMKj1du9QgjnFT40SZKkMlZqw/HAvlCSJKlRcqmEGgk8BuxYvT0N+EahApIkSWoWSm04HlgJJUmSGiWXJFTHGOMDQBVAjHEVsLqgUUmSJJWzqiqYPz+tl1ISykooSZLUCLkkoT4KIXQAIkAIYRCwuKBRSZIklbP33oOVK2HrraFNm6yjyZ2VUJIkqREqcjjmUmAU0D2E8DTQCTi1oFFJkiSVs1Icigew667QujXMnQtLlkC7dllHJEmSSki9lVAhhBZAG+AQ4NPABUDvGONLTRCbJElSeSrVJFTLlrDbbml92rRsY5EkSSWn3iRUjLEK+GmMcVWM8ZUY4+QY48omik2SJKk81cyMt8MO2caxKewLJUmSNlEuPaEeDyGcEkIIBY9GkiSpOSjVSiiwL5QkSdpkufaEagusDiEsBQIQY4xbFTQySZKkclXKSSgroSRJ0ibaaBIqxmjHSUmSpHxyOJ4kSWqGcqmEIoRwAnBw9eaTMca/FC4kSZKkMlfKlVC7756Wr78OMYIdGyRJUo422hMqhHA9cDHwavXPxdX7JEmStClKOQm11Vaw9dawbBksXJh1NJIkqYTkUgk1BKisnimPEMJdwERgRCEDkyRJKlulPBwPYJdd4P33YfZs6NQp62gkSVKJyGV2PICta623L0QgkiRJzcKyZSmBU1EB22yTdTSbZued03L27GzjkCRJJSWXSqgfAhNDCGNJM+MdDHy7oFFJkiSVq/nz03K77aBFrvcDi0xNEmrWrGzjkCRJJSWX2fHuCyE8CexLSkJ9K8Y4r9CBSZIklaVSH4oHVkJJkqRNkktj8pOAj2OMo2KMjwDLQggnFj40SZKkMlTKTclr7LJLWpqEkiRJDZBLDfhVMcbFNRsxxveBqwoXkiRJUhl7++20LIdKKIfjSZKkBsglCVXXMbn0kpIkSdK65s5Ny512yjaOxnA4niRJ2gS5JKHGhxBuDCF0DyF0CyH8DJhQ6MAkSZLKUjkkoXbaCUJIVV2rV2cdjSRJKhG5JKG+BqwA/gD8EVgGfLWQQUmSJJWtmiTUjjtmG0djtG6dZvdbvXpto3VJkqSNyGV2vI+AEQAhhJZA2+p9kiRJaqianlClXAkFaUjevHmpL1TnzllHI0mSSkAus+P9PoSwVQihLfAK8FoI4fLChyZJklSGymE4HtgXSpIkNVguw/F6xRg/AE4ERgO7AGcXNCpJkqRy9NFHsHhxGs7WoUPW0TTOLrukpUkoSZKUo1ySUK1CCK1ISahHYowrgVjYsCRJkspQ7X5QIWQbS2PVVELNmpWCw6RBAAAgAElEQVRtHJIkqWTkkoT6NTATaAv8K4SwK/BBIYOSJEkqSzX9oEq5KXkNh+NJkqQG2mgSKsZ4c4xxpxjjkBhjBGYBhxU+NEmSpDJTLv2gwOF4kiSpwTY6O966qhNRqwoQiyRJUnkrpySUlVCSJKmBchmOJ0mSpHwopyTUdttBRQW8+y4sW5Z1NJIkqQSYhJIkSWoqtRuTl7qWLdcm0+bMyTYWSZJUEjaahAohbBFCuCKEcEf19m4hhKGFD02SJKnM1DQmL4dKKLAvlCRJapBcKqH+F1gO7F+9PQe4rmARSZIklatyGo4Ha/tCzZqVbRySJKkk5JKE6h5jvAFYCRBjXAqEgkYlSZJUbqqq1lZClcNwPLA5uSRJapBcklArQgibAxEghNCdVBklSZKkXC1YAKtWwTbbwOabZx1NfpiEkiRJDVCRwzFXAX8Ddg4h3AscAHy+kEFJkiSVnXKrggJ7QkmSpAbZaBIqxvhECOFFYBBpGN7FMcaFBY9MkiSpnJRbPyiwJ5QkSWqQjSahQggHV68uqV72CiEQY/xX4cKSJEkqM+WchLISSpIk5SCX4XiX11pvAwwEJgCHFyQiSZKkclSOSahttoEttoAPPoDFi6F9+6wjkiRJRSyX4XjH194OIewM3FCwiCRJkspROSahQkjVUK+9lqqhTEJJkqR65DI73rrmAHvlOxBJkqSyVo6NycEheZIkKWe59IT6BRCrN1sAlcB/ChmUJElS2SnHSigwCSVJknKWS0+o8bXWVwH3xRifLlA8kiRJ5alck1C77JKWJqEkSdJG5NIT6q6mCESSJKlsLV0K770HFRXQqVPW0eRXTSXUrFnZxiFJkoreBpNQIYSXWTsMbz0xxr4FiUiSJKncvPNOWu6wA7TYlJacRczheJIkKUf1VUINrV5+tXp5d/XyTODjgkUkSZJUbsp1KB44HE+SJOVsg0moGONbACGEA2KMB9R6aEQI4Wng2kIHJ0mSVBbKOQlVuxKqqqr8Kr0kSVLe5PItoW0I4cCajRDCp4G2hQtJkiSpzJRzEqpt29TnavlymDcv62gkSVIRy2V2vPOA34YQ2ldvvw98sXAhSZIklZlyTkIBdOsGCxbAG2/AjjtmHY0kSSpSG62EijFOiDHuDfQF9o4xVsYYXyx8aJIkSWXi7bfTslwTNN26peUbb2QbhyRJKmr1zY53VozxnhDCpevsByDGeGOBY5MkSSoPzaESCkxCSZKketU3HK+m71O7pghEkiSpbJmEkiRJqnd2vF9XL69punAkSZLKTIwOx5MkSSKHnlAhhBtCCFuFEFqFEMaEEBaGEM7Kx8VDCMeEEF4LIUwPIYyo4/HWIYQ/VD/+fAihS63Hvl29/7UQwuB8xCNJkpR3772XZo7baivYcsusoykMk1CSJCkHG01CAUfHGD8AhgJzgN2Byxt74RBCS+BXwLFAL+CMEEKvdQ47D/hvjLEH8DPgR9XP7QWcDvQGjgFuqT6fJElScSn3oXiQXlurVvDOO/Dxx1lHI0mSilQuSahW1cshwH0xxvfydO2BwPQY4xsxxhXA/cCwdY4ZBtxVvf4gcERIndGHAffHGJfHGN8EplefT5Ikqbg0hyRUy5bQpUtanzkzy0gkSVIRyyUJ9ecQwlRgADAmhNAJWJaHa+8EzK61Pad6X53HxBhXAYuBDjk+V5IkKXtz5qRlOSehYO2QvBkzso1DkiQVrY0moWKMI4D9gQExxpXAx6xfsbQpQl2Xy/GYXJ6bThDC+SGE8SGE8QsWLGhgiJIkSY305ptpWVMpVK7sCyVJkjYil8bkWwBfBW6t3rUjqSqqseYAO9fa7gy8vaFjQggVQHvgvRyfC0CM8fYY44AY44BOnTrlIWxJkqQGqElCde2abRyFZhJKkiRtRC7D8f4XWAF8unp7DnBdHq49DtgthNA1hLAZqdH4qHWOGQWcW71+KvCPGGOs3n969ex5XYHdgBfyEJMkSVJ+mYSSJEkCoCKHY7rHGE8LIZwBEGNcWt0cvFFijKtCCBcBjwEtgd/GGF8JIVwLjI8xjgJ+A9wdQphOqoA6vfq5r4QQHgBeBVYBX40xrm5sTJIkSXlnEkqSJAnILQm1IoSwOdU9l0II3YHl+bh4jHE0MHqdfVfWWl8GfGYDz/0+8P18xCFJklQQH30E774LrVrBjjtmHU1h1STZ3ngDYoTG37OUJEllJpfheFcBfwN2DiHcC4wB/l9Bo5IkSSoHM2em5a67QsuWmYZScO3bQ4cOsGwZzJuXdTSSJKkI1VsJVT3sbipwMjCINCvdxTHGhU0QmyRJUmlrLkPxanTrBosWpWqoHXbIOhpJklRk6q2Eqm4C/nCMcVGM8a8xxr+YgJIkScpRc0xCgX2hJElSnXIZjvdcCGHfgkciSZJUbkxCSZIkrZFLY/LDgAtCCG8BH5GG5MUYY9+CRiZJklTqTEJJkiStkUsS6tiCRyFJklSOTEJJkiStsdHheDHGt2p+gMG11iVJkrQhMZqEkiRJqiWXnlC1DS9IFJIkSeXmv/+FDz6Atm2hY8eso2kanTtDRQW8/TYsXZp1NJIkqcg0NAkVChKFJElSuZk5My27doXQTL5CVVTArrum9ZrXL0mSVK3eJFQIoUUI4bO1dh1f4HgkSZLKQ3MbilfDIXmSJGkD6k1CxRirgItqbc8peESSJEnlwCRUtnFIkqSik8twvCdCCJeFEHYOIWxT81PwyCRJkkqZSahs45AkqZg98ACMG5d1FE2uIodjvli9/GqtfRHolv9wJEmSyoRJqGzjkCSpWE2aBOeeC1VV8PLLsPvuWUfUZDaahIoxNrNvTpIkSXlgEirbOCRJKkbvvw+nnALLlsGXvtSsElCQw3C8EMIWIYTvhhBur97eLYQwtPChSZIklagYPzk7XnNSOwlVVZVtLJIkFZOqqlQB9cYb0L8//OIXWUfU5HLpCfW/wArg09Xbc4DrChaRJElSqZs3L93h7NAB2rXLOpqmtfXWsP328PHHMGNG1tFIklQ8fvxjGDUqfVY++CC0aZN1RE0ulyRU9xjjDcBKgBjjUiAUNCpJkqRS1lyH4tXYd9+0fOGFbOOQJKlYjB0L3/lOWr/77mb7HSGXJNSKEMLmpGbkhBC6A8sLGpUkSVIpa+5JqIED09IklCRJsGQJnHFGGo73P/8DQ5tvh6NcZse7CvgbsHMI4V7gAODzhQxKkiSppJmESstmOPW0JEnr+dWvYP582G8/uOaarKPJVC6z4z0RQngRGEQahndxjHFhwSOTJEkqVc09CTVgQFpOnAgrV0KrVtnGI0lSVj78EH7607T+ve9By5bZxpOxXIbjARwCHAEcBhxUuHAkSZLKQHNPQm2zDfTokZqzT56cdTSSJGXn1lth4UIYNAiOPDLraDK30SRUCOEWYDjwMjAZuCCE8KtCByZJklSymnsSCuwLJUnSRx+lGfEArroKgnO85VIJdQgwOMb4vzHG/wWGAIcWNCpJkqRStWoVzJ6dvmjuumvW0WSnZoY8+0JJkpqrX/8aFixIn4mDB2cdTVHIJQn1GrBLre2dgZcKE44kSVKJmz0bVq+GHXeE1q2zjiY7VkJJkpqzjz+GG25I61ZBrZHL7HgdgCkhhJpvEPsCz4YQRgHEGE8oVHCSJEklx6F4Sb9+qfnqK6+k4Qht22YdkSRJTeeOO9KMePvsA0OGZB1N0cglCXVlwaOQJEkqFyahks03hz59YNIkePFFOMi5bSRJzcSKFfCjH6X1K6+0CqqWjSahYoz/bIpAJEmSysLrr6dljx7ZxlEMBg5MSagXXjAJJUlqPv72N3jnHejVC44/PutoikouPaEkSZKUq+nT03K33bKNoxjYnFyS1Bz9/vdpec45VkGtwySUJElSPlkJtZbNySVJzc2SJTBqVFo//fRsYylCG0xChRC2quexXTb0mCRJUrMVo5VQtfXqBVtskfpkLVyYdTSSJBXeww/D0qVpGPquu2YdTdGprxLqyZqVEMKYdR57uCDRSJIklbK3305TMnfsCFtvnXU02auogP7907pD8iRJzUHNULzPfS7bOIpUfUmo2gMXt6nnMUmSJIFVUHWpGZJnEkqSVO7efReeeCLdhDn11KyjKUr1JaHiBtbr2pYkSZL9oNZnXyhJUnPxwAOwejUcc0yqitZ6Kup5bNsQwqWkqqeadaq3OxU8MkmSpFJTk4SyEmqtmhnyXngh9cxyliBJUrlyKN5G1VcJdQfQDtiy1nrN9p2FD02SJKnEmIRaX9eu0KkTLFgAM2ZkHY0kSYXxxhvw7LPQti2ccELW0RStDVZCxRivAQghdIwxOp2JJEnSxtgTan0hwMEHw5/+BE8+6VBFSVJ5uu++tDzxxJSIUp02WAkVQhgaQlgAvBRCmBNC+HQTxiVJklRaqqrWJqFMtHzSIYek5T//mW0ckiQVQoxw771p3aF49apvON4PgINijDsCpwA/bJqQJEmSStDbb8PSpWnoWfv2WUdTXGonoaLz20iSysy4cTBlSmpGftRRWUdT1OpLQq2KMU4FiDE+T+oHJUmSpLrYD2rD9toLttkGZs+GN9/MOhpJkvLr1lvT8vOfh1atMg2l2OUyO16d2zHGGwsXliRJUokxCbVhLVqkvlAPP5yqobp1yzoiSZLy47334P770/rw4dnGUgJymR2v5mfdbUmSJNWwKXn97AslSSpHI0fCsmUweDB07551NEVvo7PjSZIkKQdWQtXv0EPT0iSUJKlcVFXBbbel9a98JdtYSkR9lVCEEI4NIfwrhLAwhLAghPDPEMKQpgpOajZs0ipJpa8mCeXMeHXr0we23hpmzoS33so6GkmSGm/MmPT5v/POcNxxWUdTEjaYhAohfBn4HnA10A3oDlwDXB1COL9JopPK3bvvwtFHw447whNPZB2NJGlTVVXBjBlp3UqourVsCQcdlNathpIklYOahuTnnw8V9bXcVo36KqEuAY6OMf4jxvhB9c8/gGOrH5PUGC++CAMGpOTTvHlwzDFw441WRUlSKZo7N/WD2G47aGfrzA1ySJ4kqVzMmQOPPJKST1/6UtbRlIz6klAhxvjeujtjjIsKGI/UPNx3HxxwQJqqetAguPzydBf9m9+Ec86BpUuzjlCS1BD2g8pNTXPyJ5/MNAxJkhrtjjvS33Annwzbb591NCWjviTUByGEvdfdWb1vSeFCksrc978Pn/tcumN+3nnpi/gNN8Af/whbbAH33JOmsV7i/2aSVDLsB5WbykrYait44410B1mSpFK0cmVKQoENyRuoviTUN4FRIYSrQwjHhxCGhhCuAR4BLm2a8KQyM348XHEFtGgBv/hFeuNq3To9duqp8Oyz0KVLOu7qq7OMVJLUEFZC5aZlSzjwwLTukDxJUqn65z/hnXdg993XVvkqJxtMQsUYnwL2qz7m88AXq9cHVT8mqSFWrYILLkg9n775TbjoIgjhk8f07QsPPpiSVD//OUyalE2skqSGMQmVO/tCSZJK3SOPpOUpp6z/N53qVV8lFDHGeTHGK4HhwAUxxitijPOaJjSpzNxyS2pGvssucNVVGz5un31Sgmr1ahg+PC0lScVt+vS0NAm1cfaFkiSVshjXJqGGDcs2lhK0wSRUSK4OISwApgKvhRAWhBCubLrwpDIxdy5897tp/Re/gLZt6z/+e9+DHXeE559fO9ZYklScqqpgxoy0bk+ojevfH7bcMlWPzZ+fdTSSJDXMpElpgqkddoB99806mpJTXyXUN4ADgIExxg4xxm1Iw/MOCCFc0iTRSeXikktSo/Fhw+CEEzZ+/FZbpeF4ACNG+CVdkorZ7NmwfHmaGWfLLbOOpvhVVKSqX0g9ECVJKiU1VVDHH5/aqKhB6vuNnQOcEWN8s2ZHjPEN4KzqxyTl4tFH08x3bdvCzTfn/rxTToFjj4XFi1MPKRXO9OnORihp09kPquFq7hyPG5dtHJIkNVRNEurEE7ONo0TVl4RqFWNcuO7OGOMCoFXhQpLKzOWXp+U116R+ULkKAX75S2jTBu69F8aMKUx8zVlVFVx7bfrDsWdP/xiStGlMQjWcSShJUimaOTMNx9tySzj88KyjKUn1JaFWbOJjkmpMnQqvvALbbANf/3rDn9+tG1xxRVr/yldg2bL8xtecLVkCp566tkn83Llw0EHwu99lG5ek0vPaa2m5xx7ZxlFKBgxIy/HjU4NXSZJKwahRaXnMMdC6dbaxlKj6klB7hxA+qONnCdCnqQKUSlpNqeZxx0GrTSwgvOyyVKXz+utwww35i605mzED9t8fHnoI2rdP/52GD089Xc49Fy69FFatyjpKSaXCJFTDde0KHTrAu++mnlqSJJUCZ8VrtA0moWKMLWOMW9Xx0y7G2KjheCGEbUIIT4QQXq9efmoDx51bfczrIYRzq/dtEUL4awhhagjhlRDC9Y2JRSqofLxJbbYZ3HprWv/BD9YO+1DdpkyBbbdNwxk39NOjR6pQ69kTXnghNYu/9Va47bbUMPdnP0t3NxYtyvrVSCoFU6em5Z57ZhtHKQlhbTWUQ/IkSaXgv/+Ff/4TWraEIUOyjqZkZdXKfQQwJsa4GzCmevsTQgjbAFeRZuQbCFxVK1n1kxjjnkA/0mx9xzZN2FIDzJ8Pzz2XyjQHD27cuQ45JFXoLF8OF17o0IUNqaqC88+HBQvqPy4E+Mxn0n+f3Xdfu/+CC2Ds2JTEGjMm9Sx5+eXCxiyptC1dCm+9lRLY3bplHU1pMQklSSolo0fD6tVw8MGp3Yo2SVZJqGHAXdXrdwF1tZUfDDwRY3wvxvhf4AngmBjjxzHGsQAxxhXAi0DnJohZapg//zkli448Mj9Tdv/4x+nN7u9/h/vvb/z5ytFdd8FTT6Uk0nvvpd9/XT9VVfDAA7DVVuuf48ADU4+SffaBN99Mw/b+9Kemfy2SSsPrr6f3le7dN33YdXNV05x8/Phs45AkKRcOxcuLrJJQ28UY3wGoXm5bxzE7AbWbBMyp3rdGCGFr4HhSNZVUXPL9JtWp09qeUJdcAosX5+e85WLhwrUzEd54I3yqzlG+udl5Z/j3v+Gss+Cjj1ID85/+ND9xSiov9oPadLWTUFVV2cYiSVJ9li+HRx9N6yahGqVgSagQwt9DCJPr+Mn1v1ioY9+aMUghhArgPuDmGOMb9cRxfghhfAhh/IKNDdGR8uWjj1LFUghw/PH5O+8XvgD77ZeG+lmd80nf+lbq4XT44fC5zzX+fJtvnmbK+8lP0vb//I89oiStz35Qm27HHWGHHdJNlenTs45GkqQNGzsWPvwQ+vaFLl2yjqakFSwJFWM8Msa4Vx0/jwDzQwg7AFQv363jFHOAnWttdwberrV9O/B6jPGmjcRxe4xxQIxxQKdOnRr3oqRcPf44LFuWEkbbb5+/87ZokXpDwdpKK6Wqpd/+NjVxv+WWlPzLhxDgm99MPb2WL4eRI/NzXknlw0qoxnFIniSpFDgUL28qMrruKOBc4PrqZV1/TT8G/KBWM/KjgW8DhBCuA9oDXyp8qM3ckiWpz8777+f+nJYt4bTTmneD1kK+SZ1wQmpO/sQT8PHHsMUW+b9GKVmxAr7ylbQ+YkRh/hC88EJ47LE0e94ll6RkoKS1xo1Lyff6Jk3YZRc488z0GVFOrIRqnH33hVGj0r+hfFSxSpKUb1VV6bMKTELlQVZJqOuBB0II5wGzgM8AhBAGAMNjjF+KMb4XQvgeUDNlyrXV+zoD/wNMBV4MqeLhlzHGO5v8VZS76dPhxBPTVPYNdc89MHFiqkxpblatgr/8Ja0X4k1qp53SjELjx6dEVHN/I7zxxvRvtEcP+Pa3C3ON445LfaKmT0/DLI8+ujDXkUpNjPDzn8Nll6XZYjbmvvvg979vXM+2YhKjlVCN5Qx5kqRiN2ECvP02dO4M/ftnHU3JyyQJFWNcBBxRx/7x1KpuijH+FvjtOsfMoe5+Ucqnxx9P1Uzvv5/u7p5ySu7P/f3v4dVXU3JgxIjCxVisnnkm9Q7abbfC3RkfNiwloR55pHknod58E669Nq3fcgu0aVOY67RsCRdcAN/9Ltx6q0koCdKQ4+HDU7UswOc/n5LkdVm9Gu68E/72Nxg4ML139erVZKEWzNtvp/4QHTtChw5ZR1OaapJQL76YbuJUZHV/VJKkDag9yiVfbT+aMT/p9UkxpuTR//t/qezw+ONTVVNdU9lvyCGHpD/Sr702JbK6di1cvMWoKd6kTjwRrrgiVVytXl1+w1tyESNcdBEsXQpnnAFHHVXY6513Hlx9dSrFnT07VUZJzcW4cbBy5drtFSvS58S4cWlI8MiR8JnP1H+OCy5I713/+Q8MGgS//GWqYKyx/falN4zbKqjG69gxfU94802YMgX69Mk6IklSPsQIM2ZA9+6ln7h5+OG0bM43//PIxib6pFtuScMqqqpSkuPhhxuWgIKUDDjjjJQcuOii+nuElKM//zktC/km1bt3+mNtwQJ49tnCXaeYPfQQjB4N7dunxGmhbb99qgisqoI77ij89aRiMmwYHHDA2p/DDksJqF13TdWfG0tAQZpJ5umn4bOfTf0Gzz33k+fs3h2uvDL9P1Yq7AeVHw7Jk6Tysnhx6mO7225puXhx1hFtuhkzUuuPrbZKxRZqNJNQWuvtt9f21Bk5MlUybWoD5htvTMmB0aPh//4vbyEWvTlz4PXX05vUoEGFu04Ia5NczXGWvCVL4OtfT+s/+EF+ZyCsT00D9Dvu+GRViFTuBgyA/ff/5M8556RhwXvvnft52raF+++Hm25Kiaeacw0alD5vvvc9OOkk+OCDwr2WfKpJQlkJ1Tg1M+SZhJKk0jdtWvpcr+mR+5e/pBnDa6qHS03N31pDhjTPfscFYBJKa11ySfrj/oQT0h3qxth+e/jhD9P617+eztsc/POfaXnggYXva1E7CdXcqs2uugrmzk29ZS64oOmue/DBqY/NvHlry3Kl5mDUqFTxVPvnrrvSUKqGCgEuvhieemrtuZ59NvWL+tSn0rUGDUpfYotdzRdqK6EaxySUJJWH0aPT9/OpU9PIjSeegL32Sp+XAwemx0tNIWc9b6ZMQin529/ggQdSb4+bb87POc8/P73ZvP12GmLRHNQkoQ49tPDXOuAA2GabVHlVcze+OXj44TQbV4sWcNttTdsPK4S11VC33tp015Wag6OOSkmI3r1Tb6CePdNkAzU/O++8drhzsbASKj/690/vry+9lIbyS5KK3333pdEftT+rjzsuDb076aR0g+nII9Py5JNTlfPQoalQoVRuoC9cmG6atWoFxx6bdTRlwySU0he+Cy9M69dck3p85EPLlilJ0KJFSmxNnJif8xazmiRUU4wXrqhIb+TQPIbkVVWlf58nnZTWL78c+vVr+jjOPjsla8eOTX8oS8qf7t3Tl9XTTkv/ny9fvvZnzpx0F/K664rjy+vHH8OsWemLaXObgCPfttoK9tknDXN+8MGso5Ekbcy778JXv5pGu9T+rG7dOrV0efBBaNcuHbvllvDHP6b9McJ3vpP6B3/0UbavIRd//Wv6PnLooanVjPLCJJTg+99Ps9L06ZOGSORTv35pOF5VVRo2tXp1fs9fTN55Jw0f2XLLdFe3KTSXvlBLlqSm4Fdfne6W33DD2uGeTa19ezjrrLR+223ZxCCtq5zeW9u1S32jli1LN0mWLk0Jn5r/56+4IjVC//DDbOOsGS7YvXtKRKlxaoZWW2UqScXvssvgv/9NM6J//PHaz+slS9Ln9Lp9hVu0SPsfeSR9zv/hD2lUx8yZmYSfM4fiFYRJqOZuypT0Bz2kP6gL8UX62mthp53SMItf/zr/5y8WNVVQBxxQ+H5QNY4+Ot1xeP751KeoHE2fnhoXP/wwbL11Gkt++eXZTvVaMyRv5MjSuIuj8hZjKhG/7LLsEzP51Lr12vL+zTeHESPScLyttoI//Qk+/ek0Q2hW7AeVX2eckZL8zz4LkyZlHY0kaUPGjoW7706f07fckj6jaz6vN/a35AknpL9bdtsN/vOf1BNw/Pimibuhli6Fxx5L6yeckG0sZcYkVHMWY/pjeuVK+PKX0xf6QmjXLvXwgTT7XrkmS5qyH1SNLbdMiagY4d57m+66TeXxx9OH0yuvpP4wL7wAxxyTdVRQWZkSYx98kMbDS1m6997U+HPkyFQKX86OOy69D+yxB7z8cppQIyv2g8qvtm3XTopiNZQkFafly9fejP3ud1M1cEPV/k6/cGFqdbFiRX7jzIcxY1KVV//+qS+l8sYkVHP2u9+lxEnHjnD99YW91sknpz8ePvgALr20sNfKypNPpmVT9IOq7ctfTstbb03DHstBjPDTn6bqjvffT3cfnnsu3TUpFjUfwLfcUhz9adQ8vffe2vfUn/wEOnTINp6msMceqSKyTZuUgBszJps4rITKv+HD0/Kee1JjW0lScfnxj9Pn3x57pJEJm2rrrdMoh913Tzd1fvKT/MWYLw7FKxiTUM3VokVp6AakP/a32aaw1wsBfvGLVK55333prn05mT8/vYFusQUMGNC01x4yBHbZBWbMgL//vWmvnQ8ffQS/+U36d1jz89nPpn+fVVVpZsWHHkpDcIrJZz6T/uCfODHdzZGy8O1vpyFpBx+8toqkOejWLfWWgJQQXras6WOwEir/evaEww5Ld57vvjvraCRJtbw7+XnueegaPtyMdPO7devGnbB167WVr9/7HrzxRqNjzJuqqrUz8p54YraxlCGTUM3ViP/P3lmHRZW9cfw7tBhgICqs3d3t2rEWmFirqy52i2utP8fubuwkRAHBbsUCbFBMQiQkpGvi/f1xFkaUmIGZuTDcz/PcR5l74r137px7znveWMTMH7t0YSaQ6qBaNaZQAFg2Pi4WDari3j32b4cO6g9Qq60tC+i6d696+84vAQHsnv39N1M6pR9OTsw149w5ltybnOIAACAASURBVBHv5+CGBQEDA2DCBPZ/3nWEhwsePgRsbdmYs28ft3HSuMDGhiktPnyQxTZUF1KpzBKKV0Ipl3Qr0337eCtTHh4engKCZ/ATNDvTGX8OFKP9fGN8bqqkbOrdurGEPykpLNteQRn3nzxhRgZVq7LkXTxKpQCu7HhUzoMHwKFD3Cxc5s0D6tdnwaZV7QKoTtLjQanbFS+diRPZ9+nmBnz5wo0MinLnDrMae/kSqFmTPRvpx6JFLJD94MFcS5kzkyez34+9PbMu5OFRFyKRzHVpwQI2rhY19PRkCuC1a5kySl18/cqsdcqXV70lcVHD0hKoUAF480a2wcPDw8PDwxnHXxzH70c6IUQ/Ffpi4LV+DFodbIUbn5XkgbF5M3PPu3KFbUQXBH50xStqm3xqgFdCFTV+XLgsXJinWBYBMQF4Hvo82+NV+CuIJKKsK+vpydLar1sn20ku7HAVDyodU1NgyBC2O29ry40M8kLEXDN79GCKmz59mMLpR3e8deuYhUNBp0YNoHdvFqTx2DGupeEpSuzcyQJzV6sGLF3KtTTc0bkz8Ndf7Dc4bZr6dlB5KyjVoaubOdYhDw8PDw8niKVizLkyB3+5/oVUEmGaJxBstgX9a/dHdHI0ep/qjU0PNuW4Lnwb8RaU27vZ1FRmnDB7NoshzDUuLuxfPh6UShDk+lBoEC1btiTvgpoCUl1s2gT88w+Lp+Hjw2I0yYlYKobNNRvseLIj17JWDaxgP9Q++wITJwJHjjATzBs3CreGOTISMDFh7lkxMfn3j84r9+6xBVmFCkBQkPrdAuUhfaF45Aj7e+FCYM0a5lJYWHFzY4HTq1QB3r5V6DfFw5MngoKY5VNiInDxIosLV5SJiGAbKtHRwKxZTJGto6PaPnfvBmbOZK7EBw+qtq+iSHAwc4EQCFjsrbxkX+Lh4eHhyTNRSVEY7jQct/xvQZe0sMdNCmv9doCHB6QC4H+3/4c199fI1dbabmuxuNPinAtJpSxEx+PHbKPawYE7S+N379i8onRp4Ns31c8pNAiBQPCUiHINkMxbQhUlAgMBoZD9f88ehRbLkUmR6H2qN3Y82QFdLV00rdA028NAxwAOvg64+P5i9g1u2MCCOt+6BZw5k7/r4pp0d4H27blTQAFAp05AgwZAWJhMe1+QCAlhSrIjR9izd+YM2/UozAoogCkAGjZkv69167iWhqcoMGsWU0ANHcoroAC2CZAeG2vnTmZdqWr32Nev2b8NGqi2n6KKuTmLESIWAzNmFJwYITw8PDxFgFfhr9DyYEvc8r8FU/0yuH1UCuuX2sybRUsLWgItrO62Gk7DnNDOvF22a8Impk0AACvursDH6I85d6qlxcLFmJgwA4VWrZjBBBeku+L168croFQFERWZo0WLFlRkkUqJ+vcnAoiGD1eo6suwl1R1e1WCEGS6yZQ8Aj1yLL/l4RaCEFR1e1VKTEvMvuCRI0ye8uWJoqMVkqlAMXMmu44VK7iWhGj3biZLly5cS5KZx4+JKlZkslWuTPTsGdcSKRcPD3ZturpEb99yLQ2PJuPiwp61EiWIgoO5lqZgcf8+kakpuz/VqhG9eqW6vtq1Y/3cuKG6Poo64eFExsbsPjs4cC0NDw8PT5HA0ceRDNcYEoSglgda0pf2Ddk4bGOTp/bGOY8jCEE9T/QkqVSae4XAQKJmzVifxYsTnT+fp37zRfv2rP+zZ9XfdyEHgDfJoZfhLaGKCq6ugLs7ULIksG2b3NXO+p5Fu8PtEBATgFaVWsF7kjc6VO6QY51ZbWahiWkTBMQEYPW91dkXHDeOWe98+8ZcGgprtjyu40H9yJ9/sqxyd+7Iduq55uhRlj4+NJT96+0NNGvGtVTKJT3Dn0jEMjvxu/Y8qiAhgbmAAcDq1YCZGbfyFDQ6dmTjS8uWgL8/0K4dizenbIhku7N8xhzVUb48s5oGgDlzgNhYbuXh4eFRLp8/s7h6pUvLjrJlWdiG1FSupSuS7Pfej+FOw5EkSsKfjf/EvZQRMH/oA/z2G7B8eZ7a3NRzE0oblMb1z9fh4OuQe4XKlQEPD2DkSGb1PXgw8+SRSvPUv8KEhwOPHrE4xr17q6fPIgivhCoK/LhwWbMGqFQp1ypSkmLpzaUZA9HYJmNxb/w9mJcyz7WujpYO9vffDwEE2PRwE3y/+WZdUEuLmXUWLw6cP88UFF+/KnJl3PPiBVP2GBkBbdpwLQ1QqhQwYQL7/8yZ3CpDRCLmNjRhApCWxtKu3rjBzGw1kfXrgXLlmALw1CmupeHRRFauZNkvmzVjvyeeXzE3Zy7SVlZs8pquHFYmgYFAfDxTkpQvr9y2eTLz999A27ZsE2PZMq6l4eHhURZEbNPu/XsWTzX9iI5mCQk6d2ZhHHjURlBsEOZfmw+AKY6Ot16HYv+uYCd37QJKlMhTuybFTbCx50YAwNyrcxGTEpN7JUND4PRpYONGtl5csYIpo+Lj8ySDQri7s+eze3dmvMGjEnglVFFAKGRBPlu0YLsLuRCbEgsLewus9VgLLYEWtvXehmMWx2CgYyB3l23N22JSi0kQS8WYcnEKpJSN9rp+feDBAxbU2cuLyfjwodz9cE565p5x41hg8oKAUMgUPXfvAidOcCNDZCTbPdi1i8VpOXiQBfItiMHSlUXZsizFLADMn88mUjw8yuL1a2DrVhao+cABPkZBThQrxmLPVasGvHrF4kQpk3Qr08aNldsuz6+kb1Zpa7NYlkU9uQwPj6bg6AhcuwYYGwOfPrE5U3Q0C0pduTLw5Amzan38mGtJiwwzL89EkigJw+oPg017GwjmzmVKn4ED850hbkKzCWj/W3uEJYTh31v/yldJIAAWLAAuXWLPiasr25T4mEtsqfySHg+Kz4qnUngllKbz+DGwfTubyB04kGsQ6OC4YLQ51Abu791R2qA0ro65ijlt50CQh+x167qvQ/ni5eER5IHjL45nX7BJEzax7NqVmUB26VI4gpXHxjItPcB2cwoKZcqw7FAAYGOj+gC9P/PyJQsmePs2y9R39y7bzS4KjB3Ldu8iIpj7iLpMh3k0m6QkYNIkQCJhGwmtWnEtUcHH0JApvgHmQhAUpLy2X71i//KueOqhSRPZeDp5MrPu5uHhKbzExrLfNMBcbqtXl7njtWnD1gTpYRw6d2ZhHXhUiqufKy68u4CSeiWxrfc24PJl4OxZ9i7dtSvf7WsJtLC/335oC7Sx12sv7gbclb9y796ApydQrx7w5g2bA3l45FumLImKAq5fZ/8fMEA1ffAA4JVQmo2dHdCtG1u4TJ/OrIxygIgwyW0S3kW9Q8PyDeFl7YUe1XvkufvSxUpja6+tAIAF1xcgMiky+8LlygFXrzL3LZEIsLZmLg8FmZMnmbtH164sjWdBYswYJldkJLBokfr6dXRkWQIDAthLwtubxWUpKggEzDpOV5c9H4MGAXFxXEvFU5gJCmKxjh4/Zkrd1TnE2ePJTN++LINgYiIwe7by2k23hOKVUOpDKGQxSZ49Y++Yz5+5loiHhyev/Psvy+Tcrl3Wm5Tp2dGmT2fhHCZMkK0PeJROQloCZl5mYVtWd1sNM90yMpf/FSuYZZoSaGTaCPPazQOB0PtU75wNFH6mVi02D7KwYG6bo0ezd7sy8fVlllYpKSxmsRzha3jygTzRyzXlKDLZ8cRiogULWFR/gGjCBKKUlFyrOfk6EYQgo3VGFBofqhRRpFIpdTvejSAETXCZIF+l4cOZ3AMHKkUGlSCVEtWvX7AzJ/j5EenpMRk9cs5omG/EYqLFi2XP3LhxRMnJqu2zIHP1qiyrU716RO/fcy0RT2Hk7l0iExP2HFWvTvT6NdcSFT6Cg1kmQYDI1VU5baaP/V5eymmPRz78/Ijq1GH3vkwZPjMhD09hxMuLSCAg0tYmevky9/IHD7LMw+mZnyMiVC9jEcPmqg1BCGp+oDmJJCKipUvZ/W7UiCgtTal9pYnTaKr7VIIQBCFo9uXZrE95EYlkmfP++Ud5grm4yOYKTZsSBQQor+0iBuTMjiegIpTFqWXLluSt6fEEvn8HRo0CrlxhrnfbtzNtdi7udPGp8ai3px6+xn/Fnr57MK1V7rGj5OVd5Ds03t8YaZI03PvrHjpV6ZRzhZAQZlkUHw+4uBRMn9y7d5nbYMWKzGKroMY6+t//gFWrgIYN2Q6yKuSMjWU7Ehcvsmduyxa2Y5UHF06N4uNH9uy+ecN82R0cgF69uJaKpzBAxOLgzJoFiMVAz56AvT1zteVRnB07mOtH5crs91i8eN7bSk1l9aVS5hZmaKg8OXly5+f3jY0Ns5CSl0qV2LisxTsCFElevGBxSHOiVi3+Xa0qxGJmafL0KfvtbtokX71Hj1hQ6rAwFkP2wgU+Jp+SeBX+Cs0PNIeUpHjy9xO0iivBXKBFIvZbad9eJf0efHoQ0y9Nh0gqQrdq3eA41BFlDcvKV9nTkz1H2tpsbZMfq2Qitk5Kz/xnZcViSvLv9jwjEAieElHLXAvKo6nSlEPjLaF8fYlq1mRa3HLliG7flrvq3CtzCUJQ64OtSSwRK120/936H0EIarCnAaWKU3OvsHMnu47ffiOKj1e6PPnGyorJt2wZ15LkTFISUY0aTNYNG5Tf/tu3RLVr8zvT2REXR2Rpye5PsWJE/v5cS8RT0ElJIbK2llkVzp/Pdv548s6PO6f5HQdfvGDt1K6tHNl4FEcsJlqyRPYbUfSwtGRjM0/RQSol2r6dWd/I84xMmkSUKsdclUd+oqKIevbM+9w+OJioVauCvTYoZEikEmp3qB1BCJpxcQb7cMgQdo+trVXev0egB5luMiUIQVW3V6WXYXJYxqUzbRqTs0MHIokk70KcOMHaEQiI1q9nYwVPvgBvCfUrGm0JdeECiwMUHw80bQo4OwNVq8pV9Xnoc7Q8yBSW3tbeaFaxmdLFSxGnoOHehvj0/RPWd1+PhR0X5lxBImHBCRXdLVEHYWFs55WIxT4yN+daopy5ehXo04dljHrzRu7nIlcuXmRWd3FxbBfCxYUFl+TJjFQKjBjBAjz2789+q0XdSowna8LCgCFDWIZQAwOWVXLMGK6l0gwuXmS/v2rVmJViXi1hTp5kCQiGDAGcnJQrI49iXL7Mvld557FSKbMojIlhmXldXYGaNVUrIw/3pKQAU6YAx/+LP2NlxbLZZkVaGvuNp6YCHToA584Bpqbqk1VT8fVlFoifPrF4TxcuMEsWRUlJYTESC+LaoBBy8OlBTHKfhAolKsBvuh+MohKYpZlAwLw81BATKTguGIMcBsE7xBuGuoY4ZnEMwxoMy71iTAzzmgkPBw4dAiZOVLzz6GjWRkQEYGvL4hHz5BveEqqoWEJJJEQrV8p2b6ysiBIT5a4uloip9cHWBCFozuU5KhSU6OrHqwQhqNjqYuT/3T/3Cor6jauL1atlu6mFhXTLrf7986/ll0qJ1qxh3w1ANHQovyOVG6GhREZG7H6dP8+1NDwFEU9PIjMz9oyYmxN5e3MtkWYhFhNVqcLu76VLeW8nPd6iUKg00XiUg0giou/J33Mu9P49i9MHsLh9V66oRzgeTkjwf09vujaiN+VAb8wN6M2JLfTm25scjyiP67Kx2MyMj/2WX5ydZbF2mjcnCgzMX3ve3kRaWgVvbVDICE8Ip9LrSxOEIPvX9uzD5ctl83o1kpSWRH+e/zMjTtTSm0sz/SY/Rn0kaVZrlzNnZJ4Y374p3vGkSax+5868BZQSAW8J9SuaYgm1YsWKbM/179cPLVoy5ePTp0/h7u6ebdnF/y7GzEszYfvMFtO1psNEapJluebNm2PAf2kqQ0JCcPDgwWzbtLa2RqX/NOdubm549uxZluWSDZKxfuF6+a7JzQ0t4uMBNzc8jY/P8ZqWp/v0ArC1tUVoaKjarqlixYqYNGmSfNfUvz9a/JetMLfvib8mDbsmPz8MsLUFSpbUnGvSxO+JvybNuyZdXVRauhQYMABu1taacU2a+D3l4ZreR72H3W47+a7JwwPuN28W+GsCNO97AtR/TU/xFG5wY7KgIiZjcrZtHsABJOon4vnga/DddAjPsrF05/qaCtX3JBSy/4wcCdvff0doeHj+rykmBpW2bwfatoXb4sV49vy5eq9JE7+nJUuYp0RICGw3b0ZoQoJar8na2hrbH2+HzXUb/I/+J981eXvD/eLF7K9J3u+penUM+PNPpV9TXr4nTUBeSyg+MqOmoYCbT/cT3WH7zBb62vqoYlxFhUL9SkxKDILjguUrbGYGvH8PtG4N+PioVrDciIjgtn9lkZyseB1/f+XLURRJTGTpxnl4eNSLhQVLzuDuzoKK82gEVz5eQeuDreWvUKyY6oThKXDoSQSoa1wLdcvVRVXjqjmWLVusLOJS4zDdSwj066cW+TQeLS3mNnf6tPISAkydyhIDPX4M+Pkpp82izoULssRQJUuqvXuBQIC57ebi6pirOZYLjf9BkaSs0BbGxspph0cx5DGX0pRDY9zxrl9npocAUd26RO/eKVTd66sXmW81JwhBZlvMyDPYU0WCZs0wx2EEIeh/t/4nX4X4eKLBgykjcNy6ddyZTf77L5Nj6lRu+s8PYjFRy5ZM/nnzFKt78yZR2bKsbq1aRG/eqEZGTefpU5kZ+fPnXEvDwyWRkUTdu7PflI4O0f79XEtUNBg1it3zJUsUrxsVxeoaGrLxlIdTpFIpbfDYQAKhgCAEWdhZ0F7PvaS3So8gBP1+9HcKTwjPvoHkZPY+A9i8gkdjeHXfiXSWgQTLQU+uHpG7Xlh8GBmtMyIIQWd9zxJFRMjmPqdPq1BiDeLDB6L69VXv8urgIOsjPIffOU8mUkQpVGdXHYIQtPLOStmJ9PnIjh3cCZcN/t/9qen+pgQhqMTaEuTy1iVzga9fidq0oYwkQHZ2OTe4aRMrW62aQiFseOQDcrrjca4YUuehEUqobdvYIhYgGjCAKCZGoeqnXp4ig9UGBCGo/eH2FBofqiJBs+e2/22CEFRxc0VKE6fJV+nn2FdTpqhWyOxo2JD1f/UqN/3nlx996eWJiyKVshdSekaZP/4g+p5LzA2enJk9m93Lpk35iVNRJTJSlrWyfHmi+/e5lqjocP++7L4rmv3qzh1Wt1Ur1cjGIzeJaYk00mlkRgwR4W0hSaQsQ9KjL4+o4uaKBCGo+JriVGFzhYyj+YHmmTferl+XLVw+f+boaniUiUQsog6zSxKEoGn/NFS4/j6vfQQhqNKWShSbEkt05IhszIiOVoHEGsSVK0wpBDBF1Pv3qutLKiXq3Zv19ddfqutHgxBLxDT94nSCEFRnVx1KEaWwE35+sg2WAjrHT0xLJKuzVhlj/qGnhzIXSE4mGj9etk40NSWqUCHrQ0eH8h0fkidbeCWUpiqh0oOi/vuvQikpRRIRzb86P+PHa33BWjb4qBmpVEr1dteT7TQpgosLkYEBuwfXrqlGwOz49In1W6pU4U7dO38+uw4tLaKNG7O3KktOZi/29AF90SJ+918ZxMURVa3K7mnlysw6iqdoMXEi+/4bNyYKCuJamqKFVErUqBG7/7ntlv7Mrl2s3oQJqpGNRy4CYwKp2f5mGbvizm+dfykTEhdCHQ53yJjz/Hjor9Kn4y+OywqnW8f17csHp9UADm4fSxCCTP/Rou9hAQrXl0gl1OZgG4IQNPvybDbX7tSJ2w3Qgo5UyqxL0jfJLSzYXEfVfPzINkl1dIhCQlTfXyEmJjmG+p7uSxCCtFdo023/27KTc+aw7+3vvzmTTx6kUimtubcmY+z/Evvl5wLsPa2nJ1u7ZHeMGcPNRRQBeCWUpiqhxGKi27cVqhKVFEU9T/QkCEE6K3Vor+ferLMMqJGdj3cShKBux7spXnndOvbo1qzJFCXqYutWyshAWJiRSFhmp/SBeOTIX81RfzZttbfnRlZN5etXorZtZff3zBmuJeJRF+mWOHp6bPeRR/3s3cu+g99/V6xeeiadbdtUIxdPrtwNuEsmG00IQlCNHTXIJ9wn27JSqZTC4sMoJC6EQuJCKDg2mKa4TclQRs29MpdEElHm7KVOTmq8Gh5l8y3wDZVexNwzz+yfked2noc+J+0V2qS1Qouehjwl8vFhig6BgOjRIyVKrAEkJRGNHi2bU/7vfwptkueb9HAdK1fmXraI8jbiLdXeVZsgBJXZUIZufr4pO5mYKLNeKySbooMdBhOEoCEOQ7IuEB/PlJLZHWFh/IaDCpFXCcVnx9MwHH0dceHdhUyfPfzyEP4x/jAxNIHTcCf8XuV3jqSTEZsSi0pbKyFJlIS309+ibrm68ldOSwOaNQPevAGWL1dfkOcuXYC7d4EzZ4CRI9XTpypxcQH+/JMF6G3QAGjaVHbu5k0gLAyoUoWV+/Ecj3JITQWmTQOOHGF///MPsHYtoK2duRwRYGsL3L+v3P6rVAEWLeIkAGWRRSRiY5evL7BsGbByJdcSKQ2713a49PESfpxTVDWuikUdF6GEXol8t3/z802cfHUSYqk447NyhuWwsMNCVCxZUbHG4uOBSpXY2Pf6NdCwoXz12rcHHj0CbtwAundXrE+efHP0+VFMcp8EsVSMXjV6wW6IHcoUK6NwOwe8D2DG5RkQS8XoUb0HXEe4wvDQcTYeV6rEAh3z46LCJKQlYL3HegTEBGR8JhAI0LdmX4xspPo5k1Qixph/asKuVCB6fi+Dq1sjIMhHIOz5V+dj6+OtaFmpJe7+dReG/1sFrF8PNGkCeHgAJfI/rhV6goKAQYOAZ8+A4sWBEyeAwYPVK8ONG0DPnoC5OUugo6Oj3v4LOO7v3TH6/GjEpcahsWljuFi5oFrparICR48CEyYAbdqwQO+FgOC4YNTbUw8JaQlwH+mOfrX5JAIFCXmz4/FKKA1j6c2lWOux9pfPm1dsDmcrZ1Q2qsyBVFljfcEah54fwuw2s7G9z3bFKt+/D/z+O6CnxxYRtWurRsh0oqKA8uVZZo+ICM3JpODjwzJGff7867nOnYGzZwETE/XLVVQgAvbsAebMASQSoE8fpuQsXZqdT0wExo9n34MqaNAAcHUFatRQTfs8mdm4EVi4kN3v1681IktXmiQNsy/Pxv6n+7M8n+WkVwGICBsebMCSm0tA+HW+UqlkJThbOaO1mQLZ0QBg+nRg716WAcvNLfcsO1IpG/fj44HwcPY+4FEbfpF+aLyvMURSEWza2WBdj3XQ0cr7YtMjyANDHIfgW+I3LOywEOu7rmFKRk9PNh5v26ZE6TWfz98/w9LeEq+/vc7y/JQWU7Djjx3Q09ZTSf9xEcH4c20rXDAOg74YeG15DbVa9MxXm/Gp8ai/tz6C44LRrEIzuFjYoXK7PkBAANCoEXt3VsvbuKYR3L8PDB0KfPsGVK/O7oe8Cn1lIpUC9eqxLNrOzoClpfplKIAQEdZ5rMO/t/4FgTCs/jActTiK4nrFZYUSE9k8MDAQOH4cGDuWO4EVZNujbZh3bR6qGFXBm+lvYKhryLVIPP/BK6GyoCgooV6EvYDPN59MnxXXLY4+NfugmG7BWvA8C32GFrYtYKRvhK/zvmYeGOVh4kRmRdK9O3D9uvJSdWbFiRPAuHFAjx6sL00iNha4epVZmKVjZMQUIrq63MlVlLhzBxg2DIiMBGrWZJO5YsXYZOrVK7Yrv3o1UEbxXf8skUjYjq6fH1N4OTqyZ5tHdQQEAPXrA8nJ7PfWqxfXEuWb8IRwDD07FB5BHtDX1seKLitgVsoMACCWirHOYx3eR71H2WJl4TjMEd2qdVOo/SRREia4ToCDrwMAYEH7BWhs2hgAm2DbPrPN6Nt2gC3GNlFgAh0WxlJRx8YC58+z3fyc8PdnCy1TU1aXR20QEbqf6I7bAbcxoekEHLY4rJR2Pb96ou2httDW0sbzyc/RMEQMtGjBTnp7M6tFnly5+fkmhjsNR3RyNOqUrYNFHRdlKAiD44IhvCNEqiQVnSp3gtNwJ5QvrlwF7oen12Fxuj/eGqXBOEUAx+Zr0XPYIqW0/SbiDQbaDcSn75+YN0Hbrfj971VM4VGmDHt3FkWryP37gZkzAbGYWSHZ2ytvfpIXtm0D5s1j79WrV7mTo4CQmJaI8a7jcfbNWQggwOpuq7G442IIfl4nLVoEbNjAxjpPz0JlRSaWitHStiVehr/Eog6LsK7HOq5F4vkPXgmVBUVBCVXYaHuoLZ58fYJDAw5hYvOJilWOjGSLiKgo4NQpYPRo1QgJAEOGsIXKrl3AjBmq60cDSRGnwO2dG+LT4rMto6+tj361+8HYQEMszPJCQABTOr18ycz89fXZs12rFlNK1aun3P5iY5k7ppsbs/DbtAmYO1e1ytyiChGzOHRzA0aMAOzsuJYoV95EvMHj4OxN89MkaVhzfw2C44JhVtIMzlbOaGXWKlOZmJQYjD4/Gpc+XIK2QBuLOy6W2yKKiLDbazdehL1ACb0SODXoFCzqWvwiw49WWHPazMGmXpvkt5DZu5dZRJmZAW/f5uyCdeEC+w41cSOigHPy5UmMdRmLssXK4t2MdyhrWFZpbU+/OB17vfeiw28dcG/8PWjNt2EL2latmOvlz+7RRRki5qpfty5gbg4iwo4nO2BzzQYSkqBfrX44Pfg0jAyMMlXz/OqJQQ6DEBIfgt9K/QZnK2e0qNQiTyI8DXmKl+EvM/6Oe+WJFX62iDEgNIjVh8vYi6jZVLlKoejkaIw8NxLXPl2DjpYOdnZejykbbkJw6TJ7PrZuBWbNUmqfBZa0NKZ8srVlf8+bx5QYciovpCTFpQ+X8C3xm9xdagm00L1ad/xm9Fv2haKj2TieksIUhLVqyd0+53z8CNy7l/mzatWArl3z1Jz/d39YOljiVfgrlNIvhdODT6N/7f6/FvTxYconiYS54bVW0Jq4APAk+AnaHW4n20goz4ElHs8v8EqoLOCVUAWPEy9PYJzLOBgbGMNu0XjMrwAAIABJREFUiB361OyjWAPpvswGBsDhw8CoUcoXMiUFKFeOma0GBgKVC45LY0HnS+wXDHIYhKehT3MtW824GlxGuGRYOhRJEhOZhZ8Ds/zAH38w9zxVuX9KpSyu2urV7O8xY9jkUgPcxAoMKSnA5MnMmrJUKWZ9VlHBGEZqJiwhDHV310VsamyuZdv/1h7nhp9DhRIVsjwvkUqw7PYyrPPI2y5lzTI14TrCFfVN6mdbxvapLWZcmgGRVITu1brDYaiDfIoKiUTmgjV3LltMZsekScDBg2zRtWVLHq6EJy9EJ0ej7u66iEiKwFGLo/ir6V9KbT8mJQb19tRDWEIY2wyrNZwp/L9+Za7S06Yptb9CS1IS8PffTIFubIyUMycwOdUJJ16eAAAs6bgEK7uuhLZW1kq70PhQDHEcgkfBj2CgY4DDAw9jVCP552tEhLX312LZ7WVZuuVaxlTEiX+9UbJspbxdXy6IpWIsvrEYmx9tBgBYN/sbu56Ugf7ajaxAUXADCw9nG7IPHrBNsoMH2UaWnMSlxmHM+TFwe++mcNdG+kawG2KHP2r9kX2hCRPYmmD+fGDzZoX74AQ7OzbnS07+9dzkycDOnSzsiJzc8r+F4WeHIyo5CrXL1obrCNesY+5KpSykyYMHbIzbsycfF8EtU92nYv/T/ahUshLODz+PNuZtuBapyMMrobKAV0IVPEQSEaycrODs5wwtgRbWd18Pm/Y2v5qMZgcRMGWKbFfGxoa5GSlz9/LiRaB/f7Zj8OyZ8trVcH6MuVHVuCq6Vs1+V8c7xBuvv72Goa4hjlsex9D6Q9UoaQGDiClUk5KYlYY6duKdnJi7aVIS0LIlm1Cbm6u+X00nOJgFafXyAgwN2YRz4ECupcqVUedGwc7HDo1NG6NFxewtFmqVqYV57eZBX0c/1zbd37vD+a1zlgvI7DAtbop/OvyD0sVK51r2x/FGIYX28+fsmQeyd8F6/Jgpq7S1gRcvWAwNHrUw2W0ybJ/ZonOVzrg97rb8cwMFsPexx8hzI1GmWBn4TfeDydX7bLFdqhTw7h1QIWsFa5EhKIgpWJ4/BwQCBJckDLYCvMwAQ11DHLM4hmENhuXaTKo4FdMvTcfh58yd0qadDdb3WJ+t4iqdhLQEjHcdD6c3ThBAgGENhqG4riFw9RoQEoJWlVpi8q5H0NJWvSvR6Ven8bfb30gRpzAFfFQPVPhnJfDbbyxZjqYGK/f2Zi7LwcFsbuDsLBs35eB91HtY2FvAL9IPpQ1Kw6KuBQSQ77f8+ftn3A28CwEEWNt9LRZ2WJj1OODlxax5ypRhchbkzTSJBFi8mFmgA0Dv3iwpAsBcHB0dWfKajh3Z/MzUNMfmiAi7PHdh3tV5kJAEfWv1xenBp7P3Ljh8mCmVTU3ZxlghjnMblxqH/mf6437Qfehp6+FA/wNK36zgUQx5lVC5ps/TpKNFixZypxfkUR8SqYSW316ekTZ5pNNISkxLlL8BqZRozx6WPhcg6tWLKDpaeQJaW7N2hULltanh7PfaT7ordQlCUPfj3SkyMTLH8klpSTTm/JiMZ+Dfm/+SRKrGFL+FDJFERP7f/TMdCv1msuLlS6Jq1dizbmpK5OGhHGGLCtHRRP7+suPaNXYfAaKqVYlevOBYQPm49vEaQQgqtroYfY7+zLU4ChEUE0QtDrQgCEGGawzprO9Z+SrOmcO+p9aticTizOdEIqImTdj5RYuULzRPtjwIekAQgnRX6tKbb29U1o9UKqWeJ3oShKC/XP5ic4p+/dh3PnKkyvotFNy9S2Riwu5FjRr04PYJMl1enCAEVZ0NevFXH6KPH2XjXkAAkST7d7dUKqU9nntIZ6UOQQjqdbIXRSdlP1/7HP2ZGu1tRBCCSq0rRe7v3NkJOzsmU+nSRN++Kfeac8H7qzeZbzUnCEFmW8zIs1tdJsv8+WqVQ22cPElkYMCusUMHotBQhapfen+JjNYZEYSgBnsa0MeojwrVl0gltPLOyoz5odVZq+znOy1aMDmPHVOoD7USHU3Upw+TU1ubxDt3kH/050zzuYSHd4nMzFgZc3MiL69sm0sWJdNfLn9l3J/FNxaTWCLOtjx9+0ZUpgxr+8wZFVyg+kkVp9JU96kZ92DWpVmUJk7jWqwiCwBvkkMvw7liSJ0Hr4Qq2Jx/c55KrC1BEIKa7W9GgTGBijVw545sstSsGVGaEgag+HhZm8+f5789DSdVnEpT3KZkvAjmXplLIolIrrpSqZS2PNxCWiu0CELQgDMDKDYlVsUSFz5ehb2iGjtqZNzj9MNonRFd8LuQv8YjI4m6dWPPu64u0YEDyhFa03n8mEhPj923n4+uXYkiIriWUC6SRclUc2dNghC07v46rsXJE3lSaMfFySb8U6Zkfnds3SpTJCbmU9HLIzdp4rQM5cOSG0tU3t/7yPekv0qfIATd8b9D9PkzUbFi7Ltfv54ppooSWWzu2d7dlrG51HVTQ4ooZ5j1mNe0Kbt/OXDH/w6V21iOIATV2FGDfMJ9filz49MNKrOhDEEIqr2rNr2NeMtOfP9OVKEC6+vgQVVcfa6ExYdRxyMdCUKQwUp9em0qINLWLjSbDXIhEhHNmyf7XidNIkpNlbu6VCqldffXkUAoIAhBg+wHUVxKXJ7FcfVzpZJrSxKEoKb7m1LA94BfCx0+LHsGf95QKAj4+hLVrMlkLFeO/C4ep/p76v8ynyu+pjjZ399P1L49K2tgQHTq1C/NfY37Sq0Pts7YeHHwcci5/8hIoi5dWJs9emjcuGbrbSsbo451pYjEwjH30jR4JRSvhCqU+IT7ZCywTTaa0N2Au4o1EBjIFgsA0caN+RdowQLWVosWGjdYK5sfJ2X6q/Tp+IvjeWrn2sdrVHp9aYIQVHd3XXoX+U7JkhZenHydqPgatgtddkNZqrKtClXZVoUqbq5IEIIEQgGturuKpPl5VkUimWVI+qJcgYlnkUMkImrcmN2r8uWJqlRhR7VqRAsXKkcZribSLVLr76lPqeLC+53nSaF96ZJMkdi5M9stDgoiKl6cfeburhbZeRibH2wmCEHVtlfLv5WnnKy4s4IgBNXbXY89//v2ycbBkSOLjhIyJYXo778zrj3VZi5N+2FzKcPK4PVroo4dZWNelSpExsasXtmyRDdv5thNwPcAara/GUEIKrG2BLm8dSEi9vvd/mg7aa/QJghBfU/3pe/J32UVp09nfbRvn6PVlapJFafSCKcRBCGow9KKJBGAqG1bTmVSGlFRRD17svuso8N+CwqQmJZIVmetMp6ZFXdWKMW63febb8ZGSbmN5ei2/+2fOk5klkMA0e7d+e5Pqbi4EJUokaEku3j/CJVaV4ogBJVeXzpjPme2xUxm1XR1AYmtJ8rGofnz2ZyDiB4GPaQKmysQhKAq26rQ89BcNspfvZJZu5cvzywYNRCPQA8y3WTKrDW3V6UXoRqkGC4k8EooXglVaIlKisowjddZqUN7Pfcqtqi+fJk92oaGzDQ8r7x6xXa2BIIcTWF5iLy+emUyT38S/CRf7X2M+kgN9jTIsPC59P6SkiQtnEikElp2a1nGxGTM+TGUlJaUcV4qldLae2szdhyHOAyh+NT4/HV67BiRvj77LXXsSBQWls+r0FC2bCFNsJTxi/AjvVV6BCEUV/4XUBRWaD96RFSxIvs+K1eW7RgPHqwegXmIiCgwJjBD2X7x/UW19ZsiSqHau2oThKA199awD52dZQvHZs3YRpcmExJC1K4dpVtfhB/bQ78f/Z0gBOmt0qMjz47kXP/7d6K+fSnd1Yi2b89xAy8xLTFDkQMhaPnt5TTOeVzG30tuLMnsWuTpyeZk2tpsjsYx35O/Zyx4D3UxYtdd2C2IX78mql6dXYuJCdG9ewpVD/geQE33N/1FuagsopOiqc+pPgQhSHuFNu16sivzGuH8eSZ7qVLseeYaiYRoxYoMRZLUajitu7UiY7422GFwpvnaL0rYU30pZvcWmVViz550+P7OjPd1l2Nd6FtCLi6pTk6yDZWWLYm+fFHxRXPLl9gv1Mq2VYaFmKOPI9ciFSnkVULxgcl5CiRiqRiLbizClkcsC5F1c2vs+mOXXAFwAQBWViyw38CBLL29okilQKdOwMOHwIwZwK5dirdRRLj5+Sb62/WXBerMIVOWIsSnxmOcyzg4+znnHpCyEBOdHI1pF6fB86tntmVSxCkITQiFlkALm3puwty2c7O8DxffX8So86MQlxqHMsXKwEhfliq7Wulq2N9vP2qVVSB1sacnC0YaEsKCkbq4AC3yllpbI/nyhWXSSkwE3N2Bfv24lihPxKbEou+Zvnj45SHGNx2PIxZHuBZJaXyK/gQLewv4RvjCSN8It8bdQvOKzbOvEBLCgsk/ecL+LlECePuWD9SvRiztLeH6zhVD6w/F2WFn1dr3zc830eNkDxjoGMB3mi+ql64O+PoCFhbAp0+AiQlw/z5Qp45a5VILoaFAq1YsM6C5OULsD6Kd12QExQahYomKcLZyli/zlEQCLFsGrPsvI+a4ccD+/SyLcRYQETY93IRFNxZlJC4w1DXEUYujGN5guKygWMwCTz9/DixYAGzcmN8rVgp2r+0w6vwolNEugXfrElBOz5gFtC9fXqn9EBF2e+7GHq89SJOkyVcpIQElIuMgfG6EwUHF5asTGsqyujZvzt75v/2WbdEbn29g4Y2F+J78PeOzb4nfkChKlCuzaV6RSCVYcnMJNj5kz4BNOxts6vVfkG8iNvd3dwdGjGAJQbgiPh746y/g/HlAIEDiGiEm1noDB1+WAXlll5VY+vtSaAm0fql68/NNDHcajujkaJQ2KA1jGADh4ZCSFIH/xRGf+bYUtniVgS7lMC8mAgIC2P+LUAbkFHEKJrtPzsjgub33dsxuO5tjqYoGfHa8LOCVUIWPXzKRyKvgCAkB6tZlLwAXFzaBVIRDhwBra5YVx88PMDLKvU4RJDEtEQ32NkBgbCDGNx2Pff32ya8olAMpSbH63mosv7McAGDVwApHLI7AUNdQaX1wyevw17B0sMTn759zLVumWBnYD7FHzxo9cyz3LvIdLB0s4Rfp98s5YwNj2A2xQ5+afeQXMjSUZYp69IgtIg4dAkaPlr++JjN4MMsSNGQIy2BTCPGL9IOFvQXeR72HaXFT+EzzQTnDclyLpVTiU+MxxnkMLry7gGYVmsHT2hM6Wjlk0kpJYZkpjx0D9u5lqbJ51IKrnyssHSxRUq8k3k5/C7NSZmqXYcz5MTj9+jT61OyDS6MuMYV/dDQwfDhw8ybQuTNw+zagYRsiWLoUWLuWKXouXMCwezPg9MYJrSq1gssIF1QqWUmx9hwdgfHjWdbV1q3ZQtws++/zyscrGHVuFIwNjOFs5YwmFZpkLrBzJzB7NlC5MstEV1xOpYqKISL0OtULNz7fwPhvZjiy9yvw55/AiRNK6yNZlIwpF6dkLKjzwrK7gPAOoCXPsi8XZQURYfvj7bC5bgMpSX85/0fNP3B68Gm5MpvmB7vXdhjnMg4iqQj3/rqHTlU6sRMBAUD9+kByMnD1KtCrl0rlyJJPn9jaw9cXMDJCwNFtsAzfgZfhL1FSryRODT6FgXVyzpbr/90fgxwG4WX4y0yf64uBvReBCc/llEVHB9iwAZg7V/PGrRxIf07nXZsHPW09+Ez1UWwjlidP8EqoLOCVUIWTpyFPYelgieC4YJiVNIOzlTNambXKveKuXcCsWYqnzo2IYLuc378D9vbMqoonSxbdWIQNDzagaYWm8LL2ynlhlw9c/VwxxnkMEtIS0LRCU7hYuaCKcRWV9KUuzr89j7HOY5EoSkTzis1xeOBhlNIvlW35iiUqopiufLtXEqkEgbGBGX+LpWIsvLEQLn4u0BJoYX339bBpbyO/VVlqKrMIPHSI/W1jA6xfz1LWF1Xc3NhuayG2lHF/747R50cjLjUOjco3gssIF2b5oYEkpiWi/t76CIoNkn9HNCkJMNQMhXdhICEtAfX31MeXuC/Y0WcHZrWZxYkc4QnhqLunLmJSYnB22FkMrT+UnYiOZnODyEjg+HFg7FhO5FMJaWlsrvTtG+DhgUvlY9HvTD8U1y2ON9PfoLJR5by1++IFYGkJBAayTb3z54F27bItnixKhq627q9zia9fmdVpfDxw4QIwYEDe5FERH6I+oNG+RkiVpOLuKV38/lEE3LoFdO2a77aD44IxyGEQvEO8YahrCNv+tmj3W/b3ENHRwMyZwOPHgI4Ozi8ciIWJLpCSFAPMu+NUhy0opVcy+/rFigEVK2Z7OlmUjMnuk3Hy1UkAwJKOSzCx+cSM87paujAvZa42q/Xlt5dj5b2VaGDSAM8mP4Oeth47sWEDsGgRUKMG8Pq1eq1/btxgSuvv34G6dXHHdgmGPZ6HyKRIhS3EpCRFYExghpUgpFKUi0pGKR0FlLClS7OjiDLedTyOvTiGHtV74NqYaxrnUVHQ4JVQWcAroQov4QnhGHp2KDyCPKCvrQ/bAbYY2ySXCaBEArRpAzx9ynYua9aUnevYkZmI/zwQSaVsB+vMGbZzcuVKkdo1UASfbz5odqAZJFIJHk18JJ+Zfj7w/eYLSwdLfIz+iHKG5eA20g1tzduqtE9VICUphHeEWHVvFQBgdKPRODjgoNwKpvz0u/LuSqy4uwIA0KN6D1QxkinyzEuZw6a9DUroZaOsJQL27WM70WIxW0jU/2ESVa4cU06V0ywrGgBAVBSweTNTUKdz+TKzuNy2DZgzhzvZ8siWh1uw4PoCEAhD6g3BMctj2X/3GoLbOzcMtB+IEnol4DfdjxMrG57s+ef6P9j0cBOaV2yOJ38/Udmmhjwc8D6AKRenoFLJSng7/a1sg+DECTZ3KFeOuVyVKcOZjErF3h4YORJo3BhJXg/RcF8j+Mf4Y3PPzZjffn7+2o6IYAvyO3cAPT1mXThxYq7VMjFsGLM2tbRk1qcFkBV3VkB4V4j6MMGTtREoUa0O8PIloC+/dfijL49w4uUJiKSijM/c37sjPDEcVY2rwsXK5VcLsR95+ZLdo4AAwNSUKf3at8f1T9dh5WSF7ynfUbtsbXSq3CmjiqGuIaa1moa65er+0lxQbBC2P96OuNS4jM+8Q7zxMvwlDHUNccziGIY1GCb39amCFHEKGu1rhI/RH7G++3os7LiQnRCJgGbNmCXS//4HrFihemGI2JxgwQK2nujfHwcX9sTUW/MgIQl61+gNuyF2KrcQ48lMRGIE6u6pi+jkaJwZfAYjG43kWiSNRl4lFOfBwtV58IHJCzep4lSa8kOGlrlX5pJIIsq5kpcXkZZWRkDATMeYMURJsuDOFBdHNGgQO6evT/Thg2ovqBAjkUoyMuFNdZ+qtn6jk6Kp98neGdlAElIT1Na3MohNiaWBdgMJQpDWCi3a9GBT/jLZ5YHzb85nBP39+Wi8rzF9js45tTbduUNUrlzWv6mqVYlevlTPhaiLHzPK/Hw0bZqRqaYw8SDoQUY2xdV3V6v9GeQSS3tLghA01HEo16Lw/MCrsFekvUKbBEIBeQZ7ci0OSaQSanuoLUEImn15tuyEVCoLWG9tzZ2AyqZTJ3ZN+/bRkhtLMt4HaWIlZfdMSyOaOVM2dk6fLn/m0IsXWZ3ixQt0YPhkUXJGYPtGc/ToU2kQrVolV12pVEp7PPeQzkqdLN/NcqWbd3BgCXkAolatiIKDM53+MeHLz0fJtSXpgt+FTOXv+N+hchvLZVm+oGUdu/bxGkEIKra6WOY5zP377H7o6RH5+alWiKQktq5If8aXLqVXIS8yAoz/c+2fzEH2edTKoaeHCEKQ6SbTzNk2eZQO+MDkv8JbQmkGB7wPYMblGRBLxeherTschjqgrGHZ7Cs8eMDcZdKJjgZWrmTBhFu0YLtqaWmZfLfh4AD07q36iymkHHl+BBMvTET54uXxbsY7GBsYq61vsVSM1gdb43nYcyzssBDre6xXW9/54UPUB1jYW+Bt5FsYGxjDfog9etfk5hkLiAnAzc83M8y7pSTFlkdb8D7qPcoWKwvHYY7oVq1b9g2EhTErQbGY/U3E4kd4ezP3pWPH2M51YefcOWb1kD5WTJ4ss4zU0gL++CNHt4WCiEgiQgvbFnj97TWWdlqK1d1Wcy2SWgmKDUL9PfWRKErExVEX0bdWX65FKvJISYpORzvh4ZeHmN5qOnb33c21SACAl2Ev0cK2BQgEL2svWUB7Pz+gcWNmafHgAdC+PbeC5hcfH6BRI6BECbx5fQtNT3aASCrCwwkPc3b7ygtHjwJTprA5V+fOwNmzLNh7diQlAQ0bAv7+zBp1fj6tslTMh6gPGGg/EH6RfiidDDi66KKH+5vMlvg/kSpOxYxLM3DoOXN3n9pyaqbkCWWKlcGA2gOgq62bdQMKBIJPSEuA2zs3JIoSMz678vEKzr09BwEEWNl1JZZ2Woq9Xnsx5+ociKVi9KzeM1OAeH1tffSv3b/AWfOMOjcKdj526FurL9xHustcrv7+Gzh8mLlG3rypGu+G4GCWxCV9DnT8OKRDBmeMa9NaTsOefnuU3y+P3EhJis7HOsMjyIP/PlQM746XBbwSSnPwCPLAEMch+Jb4DdWMq+HCyAtoWL6h/A28fs1Mlj9/ZhlMRCLmu12vHsumV4sPXJcdkUmRqLO7DqKTo3F68GmMajRK7TJ4fvVE20Ntoa2ljeeTnyv23XPA1Y9XMeLcCMSkxKC+SX24jnBFzTLZT0q5ICYlBqPPj8alD5egLdDGll5bMKvNLPl955OTmZLmJIsTgblzgZa5W+PKTYkSTDGsgGtDnpFKgeXLgdX/KWg0KKPM5oebseD6AlQvXR0+U31U7gZaENnycAtsrtugqnFV+E7z1ZhEB4WVQ88OwdrNGhVKVIDfdD8YGRScRCDzr87H1sdb0bJSSzye+BjaWv/FwVu2jI0PDRsCz54ButkoCAoD06cDe/eCpk1Fl1a+uBd4D5OaT8KBAQdU09/jxyypQ2goCzIuFGY/rl+/zjY1GjdmC/xCcJ9jU2IxxnkM3N+7Q0sK/Pu1OurMWpml8oOIsNd7Lx5+eQgDHQMcGnAIoxsrkPgjNpYlCrl4kcVo3LKFxUJVQNFCRFjvsR5Lby0FgVC3XN2MxCY27Wywrsc6Tl1j5SUsIQx1d9dFbGoszg0/h8H1BrMTUVEslltUFJufjBmj3I4fPGDJScLDgapV2RqiceMCPa4VVdQdRqSowrvj8e54Gk9QTBC1ONCCIARV2lKJYlNiFWsgKoqoRw+Z6ezAgUSxCrZRxPgc/Zka72tMEIJ6nOjBqRvPNPdpBCGow+EOJJFKOJMjN/wi/EhvlR5BCLK0t6S4lDiuRcoWsUSc4YoBIWic8zhKFiXL34BUSrRtW/YusPk9Wrf+xcVA6cTGEg0YwPrT0iLasoVdlwYQGBNIhmsMCULQ5Q+XuRaHM0QSETXZ14QgBC2+sZhrcYo04QnhVHp9aYIQZPfajmtxfiEuJY7Mt5oThKDdT3bLTiQlEVWvzsaJPn2IoqO5EzI/xMURlSxJBNCV6/sJQpDJRhOKTlLx9Xz9StSmjXzjvkBA9OiRauVRMhKphJZemJOlK1tWh/lWc/L+6q1YJ35+RHXqsHtUpgzRjRv5ktn9nTuVWleKIAQZrDagUy9P5as9LtjjuYcgBJltMcs81zp6lN0nExPl/lYPHCDS1WVtd+tGFBlJRETfEr4V6HGtKLPw+kKCEFR+U3m6H3ifa3E0EvDueL/CW0JpHsmiZHQ93hVPvj7BrNazsOOPHYo1IBazIIIGBmw3UEtLNYJqALf8b2H42eGISo5C7bK1cW3MNU4z1MWkxKDu7roITwzHoQGHMmVnKSgQEbqf6I7bAbcxpvEYHLc8Di1BwX/GHH0dMd51PJJESWht1hrnh59XLIjz3bvAkSPMwlBZeHgAX77IlWEpz7x/z9xy/fxYJhkHB6BnT+X3wxEW9ha48O4ChjcYDoehDlyLwymPgx+j/eH20NbSxovJL9CgfAOuRSqSjHMZhxMvT6BXjV64MvpKgcxa5PzWGYMdB6OUfin4TfdDxZL/ueB6egL9+rFseTVrMguI+vJlvCow7N8PTJ0KdOyIyf/Uh+0zWwg7C7G8y3LV952SAmzcyMbbnOjWjblUFULcTy+Hg+saSKQSFsS+U6dfMm6aGJpgSaclMC1hKn/DFy8Co0YBcXHMldLFBaie/8ym7yLfYb/3foxtMhbNKjbLd3vqRiKVoN3hdvAK8cLctnOxtfdWdoII6NIFuHePWWzv35+/jtLSWJKW9HbmzAE2bQJ0mMVY+rjWs3pPXB1ztUCOa0WVJFESLOwtcOPzDehq6WLXH7swueVkrsXSKHh3vCzglVCayY9xGzz/9kSLSi24FkmjICLsfLIT86/Nh4Qk6FurL04PPq3WOFDZYffaDqPOj0KZYmXwaOIjWQYjAHraeihTjNvMRadencKfzn+ibLGyeDfjXc6xywoYL8JewNLeEoGxgahQogLODT+H9r9xGPvk5wxLu3fnnKZbXz/nlMTx8SzWUzqenizlemws0KABW1DWqKE08ZNFyYhNjc34WwAByhcvr7bJqaufKywdLFFKvxTeTn+LSiUrqaXfgswU9yk48PQAOlXuhLt/3eUXCmrmtv9tdDvRDfra+vCZ5lPgXJTTISIMtB8I9/fuGNFwBOyG2MlOBgYy1/4XL4CSJZm7j4UFd8IqAhHQtCnw6hWkp0/BLNQGYQlheD75OZpWaMq1dJrDq1fsGfH3ZxnrTp9m75i8cvQosHQp+/6GDmV/l9DszKaK8Cz0GVodbAUA8Lb2linT3rwBmjRhm88PH+Z9Iys8nMW8vH+fzTMOHGBxuP6jsIxrRRmxVIx/rv+DbY+3AQAmNZ+EXX13QU9bj2PJNAPeHY93xytSzL86nyAEtbRtyWefUDLr76/PMBlffGOblSVdAAAgAElEQVRxgbq/UqmUepzokaV5e88TPTmVLSopikw2mhCEoKPPj3IqS175lvCNuhzrQhCC9FbpkddXL24F+jnDUm7HhAlEyT+5E0okLGORjk7WdQYNYi4qSsTutR2VWFvil2e0zcE2FBQTpNS+ssLrqxeZbjIlCEG7nuxSeX+FheikaCq/qXyh/o0WVlJEKVRnVx2CELTyzkquxckV/+/+VGx1MYIQdPXj1cwnExOJrKxkY8iKFWycKejcuJHhovTo092MrLNFKVum2oiMZO5aynJNFwiIVq/WGFdxZTPnMnOFbH2wdeY56+LF7P41bpy3zLYJCUS1arE2KlUievIk02mfcB+qsq1KoRnXijrHXxwn/VX6fMZcJQM53fEKvl8ID48cCLsIYV7KHN4h3tjnvY9rcTSGj9EfsfwOM8s/Oegk1nZfKwvMWgAQCATY328/Gpg0gGlx00wH15lbFt9YjIikCPxe5XeMazIu9woFEJPiJrg25hrGNRmHNEkaJrtPZm4FXKGrC+zcyVz9qldnu8rZHbq6rFyXLkBICKufkMCsqZYtYxmFfixvbs4CDTs5MYsGJSCRSrDoxiKMPDcSCWkJKGdYLuP5LK5bHE++PkHLgy3hEeShlP6y4uTLk+h4pCPCE8PRrVo3TG05VWV9FTZKFyuNrb2Yu4bNNRtEJUVxLFHRYfPDzXgX9Q61y9bGPx3+4VqcXKlqXBXCLkIAwLSL05AsSpadNDQE7OyA9etZQOjly5mlREICN8LKQ1oaMHMm+/+sWXD9fBkAYFHHgrcIVAVlywJXrwILFwKVKuX87srtqFuXWeouXaqaTG8awMquK2FW0gyeXz1h+9RWduLff4Fq1Zh12g4Fw3cAwKpVwIcPzJLN2xto3TrjlIufC9oebovA2EC0NmtdKMa1os7YJmNxf/x9lNQrCac3TnD1c+VapKKFPJoqTTl4SyjNxvmtM0EIKrm2JH2N+8q1OIUeqVRKvU72IghBf57/k2txChUPgh4QhCDdlbr05tsbrsXJNwmpCVR5W2WCELTz8U6uxZGPZ8+IKldmO5YVKxI5OhI1asT+LlWKyN1dpd1/T/5Of5z6gyAEaa/Qpp2Pd2ayMIhMjKTux7tnPCcHvA8otX+RRETzrszLsLqa7DaZUsWpSu1DE5BKpdTteDeCEDTRdSLX4hQJPkZ9JIPVBgQh6Obnm1yLIzdp4jRqsKcBQQhadmtZ1oUuXiQyMmLjTMOGRJ8+qVdIeVm7lslYqxZRcjLV212v0H0fPDw54eTrRBCCjNYZUWh8qOzExYvs2S9enCgwUP4GX79mFtQCAdHjxxkfS6QSWn57eca7dtS5UZSYlqjEK+FRNTse7yAIQb9t/Y3iU+O5FqfQg4IcmFwgEJQB4ACgKoAAAMOJ6HsW5cYB+Pe/P1cT0fGfzl8AUJ2I5MrPzseE0myICBb2FnB774a+tfrCaZhTgUs/niZJw7+3/oVfpB92992NykaVuRYpWxx8HDDi3AiUNigNvxl+KF+8PNciFUjCEsIw49IMvI18m/FZaHwovqd8x5KOS7Cm+xoOpVMeF95dgIW9BUrqlcTb6W8zApUTsZhhbu/dsL7HerSslLsbuNqIiGAxM+7dk31Wpw7bRa5TR2nd+HzzwZwrcxCaECrrOjECEUkRKFusLByHOaJbtW6/1Ps5LsGUFlOw448dCscliEyKxMzLM/Eq/FXGZ/Gp8fgS9wU6WjrY9ccuTGk5JY9Xp/m8i3yHxvsbI02ShgcTHnAb+0zDISL0PdMXVz5ewZjGY3By0EmuRVIIjyAPdDraCbpaung19RXqlqv7a6GfExw4OgI9eqhf2Oz4/JlZcqSkANev40OzKqi9uzaMDYzxzeYbdLV1uZaQhyffEBH62/XHpQ+X0L92f5wddhYGOgbs5LBhzOrZ0hJwds69MakU6NyZJUiZOhXYuxcAe8+OdRkLFz8XaAm0sKHHBsxvN5+3JixkSKQStD7UGs9Cn2FB+wXY2HMj1yIVagp0YHKBQLARQDQRrRcIBIsAlCaihT+VKQPAG0BLAATgKYAW6coqgUAwGMBQAI15JRRPOoExgWi0rxHi0+LRslJLOFs5w7yUOddiAQDCE8Ix9OzQDNcbE0MTOA13wu9VfudYsl+JTYlF3T11EZYQBtv+trBuYc21SAUSr69eGOQwCF/jv/5yrnbZ2ng++TkMdQ2zqFk4sbS3hOs7VwyrPwyOwxyRJEqCtZs1zrw+AwAw0DHAwQEHMabxGI4l/QGRCJg7F9izB+jfHzh1CjAyUlrzzm+d8afzn0gUJf5yrrFpY7hYuaBa6Wo5tnHi5QlMcpuEVEkqOlXuhLPDzsqdKenHAPI/U754eTgNc0KnKp3ku5gizNKbS7HWYy0s61rC2UqORQlPnnD0dYSVkxWMDYzhN91PsYxgBYS/L/yNw88Po2vVrrg59mbWC864OGDMGMDNjWXd3byZZdDienFKxDL6Xb4MjB4NnDqFzQ83Y8H1BRjdaDRODT7FrXw8PErE/7s/Gu9vjIS0BLQ2aw1nK2eWlOPrV+bWmJDANqUGDsy5oSNHgIkTmTuknx9gbIyP0R9haW8J3whfGBsYw36IPXrX7K2eC+NROt4h3mh9sDW0BFp4Pvk5Gpk24lqkQktBV0K9A9CFiEIFAkFFAHeIqM5PZUb+V2byf38f+K+cnUAgKAHgCoBJABx5JZQaOaPkCdQoBZ+/3PofRXgZ9hKWDpYIiAnItTlankv/yr5eRRmdi3zK/vlycb0/PQOCFTnLoPTv7OdnUMn3QPAhf/WpVj4FUPD+5pd8y/szWYwRil4DLSdmFVWuHFsEqvg59+3hAy2BFmqXrS13DDXPr54Y5DAIIfEhMC9lDhcrl1wzfTr4OGC863gki5PR2qw1dv+xG4a6hmhwQ65XoupQ9DctzPk3LRDmc0z4mVy+/y/9gvCb0W9KbVPhd50GEpsSi3p76iE0IVTxcSK/9y+/v/kf+o9KikKd3XUQlRyFk4NOMqW7EttPR+3vwvySz/mKwter6utT9tzkZziea9ByUnjcyu89yLW+CuYPP27U5PauwWkVP1PqntP/jLrHhKzG7fyKoMJ7NPPSTOyK2Z1zIf5dnivyKqF01CFMFpgSUSgA/KeIysrPxwzAlx/+Dv7vMwBYBWALgCSVSslTKGlSoQm8rL1g5WSFW/63uBaHh4cnH8y8xILnltArgamtpubdhdXERIlS5UyD8oqn325t1hre1t4Y4jgEj4IfoePRjhjbeGy2rnmRyZGw97EHAIxrMg77+++XuRrw5IuDzw5iZdeVXIuhcSy7vQyhCaFoa94WwP/bu/c4m6v9j+OvzwwGkXI5iNBN6PwYougidEK6GI9cu+FEOehGzlGdikfH45SOOkSK3KpTKOOaikrSiWaQ3OWkSBTpMuM+l/X74/vFGHv2zDB79p497+fj4bH3Xuv7Xd/Pvqz5bp+91vquCHc4p61S2Ur8q+2/6D23N4M+GETbi9qiyeoikSm+WjzJfZPp8naXcIciEe4fbf4BibkkoaTAhCwJZWYfAtUCVD2e1yYClDkziwcuds49bGZ18hDHvXgjpqhVK3LX35GCVblsZT648wNKPh18bYMj6UeIKxFXSFFJJHLOaf5+BBubfOILwatfvso7Xd4JYzShVb18dZb0XMKAhQOY9OUkJqyeEHT7WItlVNtRPHDlA/oMF6CJqyfyRMsntDZOAVq5ayXjkscRa7G8fNPL8HF8uEM6Iz0b9WTKmil8uv1Trph4Bd8pCyUSsaqcVYXFdy2G3uGORCJZhdIFt1SD5C5kSSjnXI6rMJrZT2ZWPct0vD0BNtsJtMryuCbwCdACuNzMvsOL/w9m9olzrhUBOOcmABPAm46X/2ciRVWJmNw/3m1ea8OsrrOoVi5QvrT4yHSZDPtkGMXxt/8es3owuePkqFq7KZqMaT8GgPlfz2fxtsX86fUIWuA3BOJKxDHxlol0btCZrfuCz6+4ptY1NK7euJAiKz5+3P8jczbPoctl+uW8IGRkZtBvQT8yXSaDWwymUbVG4Q7pjJkZ02+bTsKMBJJ+SEJDoUQim35UEIks4VoT6jlgX5aFySs65/6abZuKeIuRN/GLVuMtTP5Llm3qAAu0JpTk16pdq+g0oxPfp3xPjfI1mN1tNs1qNDuttrKuQVWtXDVmdZ1V4FdX+uXQL3R/pzuLty2mREwJnm79NJdWOrGMWtVyVWlRs0WeRkPsP7qfT777hLSMtONlU7+ayrwt84ixGEb+aSSDWgyK+pEV87bM487EO0k9mkp8tXgev/ZxYs1bv+f7lO8Z+uFQDqUf4soaV5LYLdFbzFLCJiMzg0c/epTnPn8OgN7xvbml7i3H68uULEPrOq0LZGSjc45lO5ax7+C+42Vf/fQVT3/6NJkuk46XduT1Tq9TPq78GR9LIsvYpLHc/979tK7Tmo97ajp3QTj2mtY8uyabBmyiXKly4Q6pwBxOP8xf3v0LU9dMBWDo1UMZcf0IYiwm8A7p6TBkCPz7397je++F9u1P1Jcu7V1Jr2QB/oc5IwNatIDkZBg0CEaNAuBg2kEqj6zMofRD/DDoB53jpFjYd3Af9cbV4+eDP/Nawmvc1eiuPO036vNRPLL4ES445wLW91+vHy5FchDpC5NXAmYCtYAdQBfn3C9m1hTo55zr42/3Z+Axf7cRzrkp2dqpg5JQcpr2HNhD55mdWbZjGXGxcUy4ZQJ3N7o7X23M3DCT3nN7czDtIM3Oa8bsbrOPX7q+oKVnpjP0w6GMWj4qYH2v+F6Mv2l80HVhNu3dRMKMBL7e9/UpdeeWPpfpnafT9qK2BRZzpNu4dyMJ0xPY+kvgESe943vz0k0vaa2dCPKftf+hz/w+HE4/fEpdQfTBA0cP0Htub97e+HbA+qeue4onr3sy5/9kSpH2++HfqfF8DQ6kHWBj/43Ur1I/3CEVabtSd1FvbD1Sj6Yyu9tsEuolhDukAuec48WkFxn0wSAyXAbPXP8Mf7vmb8F3mjYN7rsPjhw5te7BB08kqQrCuHEwcCDUrAkbN0J5L3n+6upX6Tu/L83Oa0ZS36SCO55IhJu2Zhq95vaiStkqbB64mYplKgbdfsfvO2gwrgEH0g7w7u3v0uGSDoUUqUjRE9FJqHBREkqyO5pxlIfef4jxK8cD8NCVD/Fc2+dyncqXkZnBE0ue4J+f/RMo3IWB39n4DtPXTyfDZQDeF+BF3yzKddTO/C3zuSPxDlKPplK3Ul0aVGlwvO7c0ufy2LWPcXHFi0Mef6T57fBvDPtk2EmXuDeMW+reQq/4XlE/IqwoWr17Nc8vf54DaQeOl63atYrvU76n6llVSeyWeFqjEb/99VsSZiSw9qe1lC9VnjYXtDn+/peIKUHPRj25ue7NBfY8JDLdN/8+JqyewANXPMDoG0eHO5wirds73Zi5YSa3Xnorc7vPDXc4ITV381wSZiRQpkQZNg7YSJ1z6gTfITnZSzYd9K+x4xwsWODdJiXB5cGvjpknu3d7l6JPSYHEROjUCYCfD/5MvbH12HdoH290eoM7Gt5x5scSKSKcc7Se1pql25fSt0lfJtwSfP3FTjM6MWfzHDo36MzbXQL/QCUiHiWhAlASSnIyYdUEBi4cSFpmGtdfcD1TE6ZSvlTgqTYH0w7SZ34fFm5dGDELA2e9BG21ctWY2XkmDas2BMDhGJs0lieXPInD0aVBF6Z0nMJZpc4KW7wiBe3ngz/T9e2uLPluCSVjSjK2w1i6XdYtz/sn/ZBEj1k92HdoH3Ur1WVu97nUq1wvhBFLpFrz4xoav9KYCnEV2DJwy0k/LsSViNPIyACcc6QcSTmpbOn2pXSc3pGyJcuysf9Gap9TO0zRFZ4es3owff10brrkJub3mJ//7wWPPOJNl2vaFFasgNjYMwyoB0yfDjffDPPmgR/PPXPvYfKaybS5oA0f3vWhfmyRYmfT3k00erkRaZlpfHT3R1xePXDS98NtH9L57c6UL1WeTQM2hWy2g0i0UBIqACWhJJj/7vgvt828jZ8O/JSn7SuWqcjMzjO5/sLrQxxZ3uw9sJcub3dh6falAesNY0SbEQy9Zqi+cEpUSstIY8jiIYz+4vRHr9x48Y28edubnFP6nAKMTIqaqyZdxfKdy08pf+yaxxhx/YgwRBTZ0jPTc7wa7cg/jWTI1UMKOaLw2J26m3rj6pFyJIXErol0qt8pfw3s3w/168POnfDii940utPhHIweDQ8/DGXLetPwantJwGXbl9FyaktKxZZibb+1XFr50lwaE4lOf//474xYlre/56Pbj+aBKx8IcUQiRZ+SUAEoCSW52Zmyk15zepG8Kznodg2rNmRawjQuPPfCQoosb9Iy0hj64VCmfjWV9Mz04+WVy1ZmTPsx3FT3pjBGJ1I4pq6ZyhNLnjhlZEYwJWNK0q9pP4a3Gk5szBmOPpAib9E3i7h79t0cSj90UvngFoN58ronwxRV5ErPTKfSyEqnlDev2ZwFPRYUqytTjUsax8D3BlLz7Jps7L8x/xcwmDPHmzZXvjxs3gzn5XPB8MOHoV8/b90pgLFjYcAAwFuCoMkrTdiwdwNPtnyS4a2H569tkShyKO0Q7d5ox1c/fRV0u2trXcuc7nPydNVtkeJOSagAlIQSERERkVDJyMyg+aTmrNy1kkHNBzGqXeCLieTIOejYEebPh27dvOl0ebVrl5fASkqCMmVgyhSvDd+znz3L0I+GcnHFi1n3l3WaWioiIgUqr0koXd5HRERERKQAxMbE8srNrxBjMYz+YjQff/tx/how86bilS0LM2bAW2/lbb/ly73FzJOSvKl3n39+UgLq21+/ZfhSb+TTSx101VcREQkfJaFERERERApIk+pNePDKB8lwGbR7ox3jk8eTr5kHtWvDcH+q3O23e/czM3PeftIkaNUKfvzRu01Ohvj449XOOQa+N5BD6Yfo8cce3HDRDaf1vERERAqCklAiIiIiIgVo5A0jGdxiMOmZ6fRf2J/7FtzHkfQjeW9g8GB49llvZNSwYXDbbZCaevI2aWlw//3Qpw8cPerdX7QIqlQ5abPZm2ezcOtCKsRV4Pl2z5/5kxMRETkDWhNKRERERCQE3lj7Bn3n9+Vw+mGuqHEFLWu1PF5XrlQ5+l7el/PKB1l8/P33oXt3+P13aNAAOnQ4Uff5596/UqVg/Hj4859P2T31SCr1x9Xnh9QfGNdhHP2b9S/IpyciInKcFiYPQEkoERERESlMq3atImFGAjtTdp5SV61cNRK7JtLi/BY5N7B1q7dY+aZNp9ZVrw6JidC8ecBdB30wiBdWvECz85qx/J7luvqniIiEjJJQASgJJYXJLNwRBBbNXT5SX/Nwi+b3XIIrqn1Cn1mJNnsP7GXGhhkcSjt0vGzB1gV8uv1TSsWW4qUOL3FPk3tybiAlxVukPCXlRFlcHHTtCtWqnbSp+n1kiZT3I1pfX8ldpHwGA9HnMrooCRWAklBSmCL1D340d/lIfc3DLZrfcwmuqPYJfWalOEjLSGPwosG8mPQiAAOaDeCFdi9QMrZkvttyzrFi5wp+O/wbHereWNChFopo7feR8nc4Wl9fyV2kfAYD0ecyuuQ1CaWFyUVEREREClnJ2JKMuXEMk2+dTKnYUoxLHscNr9/A3gN789XO/qP76fx2Z66afBUd3uyQ+w4iIiJhpCSUiIiIiEiY9G7cm6W9llK9XHWWbl9K04lN+XL3l3nad9uv22gxqQWJmxI5O+5s2l/cPsTRioiInBkloUREREREwqh5zeasvHclV9a4kh2/7+DqyVfz1rq3OJJ+JMd/i79ZTLOJzVi/Zz2XVrqUpD5JvHfHe+F+KiIiIkFpTSiREInU+dfR3OUj9TUPt2h+zyW4oton9JmV4upw+mH6v9ufKWum5Hmfm+vezBud3qBC6QqA+n2kiZT3I1pfX8ldpHwGA9HnMrrkdU2oEoURjEhxpD+qhU+vucjJ1CdEipbSJUoz6dZJNK7WmOFLh5N6NDXHbeNi43io+UMMazWMGDsxuUH9PrLo/ZBw02dQIo1GQomIiIiIiIiIyGnT1fFERERERERERCRiKAklIiIiIiIiIiIhpySUiIiIiIiIiIiEnJJQIiIiIiIiIiISckpCiYiIiIiIiIhIyCkJJSIiIiIiIiIiIacklIiIiIiIiIiIhJySUCIiIiIiIiIiEnJKQomIiIiIiIiISMgpCSUiIiIiIiIiIiGnJJSIiIiIiIiIiIScklAiIiIiIiIiIhJySkKJiIiIiIiIiEjImXMu3DEUGjPbC2wPdxynoTLwc7iDEIlw6iciwamPiASnPiKSO/UTkeCKcx+p7ZyrkttGxSoJVVSZ2UrnXNNwxyESydRPRIJTHxEJTn1EJHfqJyLBqY/kTtPxREREREREREQk5JSEEhERERERERGRkFMSqmiYEO4ARIoA9ROR4NRHRIJTHxHJnfqJSHDqI7nQmlAiIiIiIiIiIhJyGgklIiIiIiIiIiIhpyRUhDOz9ma2xcz+Z2ZDwx2PSCQws+/MbJ2ZrTGzlX5ZRTNbbGZb/dtzwx2nSGEys8lmtsfM1mcpC9gvzDPGP7esNbMm4YtcpHDk0EeGmdkP/vlkjZl1yFL3qN9HtphZu/BELVJ4zOx8M1tiZpvMbIOZPeiX61wiQtA+onNJPigJFcHMLBYYB9wINAB6mFmD8EYlEjFaO+fis1wCdSjwkXPuEuAj/7FIcTIVaJ+tLKd+cSNwif/vXmB8IcUoEk5TObWPALzgn0/inXMLAfzvW92By/x9XvK/l4lEs3RgsHOuPtAcGOD3BZ1LRDw59RHQuSTPlISKbFcA/3PObXPOHQWmAx3DHJNIpOoITPPvTwMSwhiLSKFzzn0K/JKtOKd+0RF4zXlWAOeYWfXCiVQkPHLoIznpCEx3zh1xzn0L/A/ve5lI1HLO7XbOrfbvpwKbgBroXCICBO0jOdG5JAAloSJbDeD7LI93EvxDLlJcOGCRma0ys3v9sqrOud3gnSCAP4QtOpHIkVO/0PlF5ISB/lSiyVmmcquPSLFmZnWAxsAX6FwicopsfQR0LskzJaEimwUo0+UMReBq51wTvGHgA8ysZbgDEilidH4R8YwHLgLigd3AKL9cfUSKLTMrB8wCHnLOpQTbNECZ+olEvQB9ROeSfFASKrLtBM7P8rgmsCtMsYhEDOfcLv92DzAbb1jrT8eGgPu3e8IXoUjEyKlf6PwiAjjnfnLOZTjnMoGJnJgmoT4ixZKZlcT7z/V/nHOJfrHOJSK+QH1E55L8URIqsiUDl5jZBWZWCm9Rs3lhjkkkrMzsLDMrf+w+0BZYj9c3evqb9QTmhidCkYiSU7+YB9ztX9moOfD7sakWIsVJtvVrOuGdT8DrI93NLM7MLsBbeDmpsOMTKUxmZsAkYJNz7vksVTqXiJBzH9G5JH9KhDsAyZlzLt3MBgIfALHAZOfchjCHJRJuVYHZ3jmAEsCbzrn3zSwZmGlm9wA7gC5hjFGk0JnZW0AroLKZ7QSeAp4hcL9YCHTAWyDzINC70AMWKWQ59JFWZhaPNz3iO+A+AOfcBjObCWzEuxrSAOdcRjjiFilEVwN3AevMbI1f9hg6l4gck1Mf6aFzSd6Zc8V+SqKIiIiIiIiIiISYpuOJiIiIiIiIiEjIKQklIiIiIiIiIiIhpySUiIiIiIiIiIiEnJJQIiIiIiIiIiISckpCiYiIiIiIiIhIyCkJJSIiIlHPzDqZmTOzelnK6pjZev9+UzMbE74IAzOzBDNrEOJjfO7f1jGz27OUR+RrIiIiIkWXklAiIiJSHPQAPgO6B6p0zq10zj1QuCHlSQIQ0iSUc+4q/24d4PYs5ZH6moiIiEgRpSSUiIiIRDUzKwdcDdxDDkkoM2tlZgv8+1XMbLGZrTazV8xsu5lV9kcKbTKziWa2wcwWmVkZf59PzOwFM/vU36aZmSWa2VYz+0eW49xpZklmtsZvO9Yv329mI8zsKzNbYWZVzewq4FbgOX/7i7LFPNXMXjazZWb2tZnd7JeXNrMpZrbOzL40s9Z++WVZjr3WzC45dmy/yWeAa/36h7O9JhXNbI6/3woza+iXDzOzyf7z32ZmSlqJiIhIjpSEEhERkWiXALzvnPsa+MXMmuSy/VPAx865JsBsoFaWukuAcc65y4DfgNuy1B11zrUEXgbmAgOAPwK9zKySmdUHugFXO+figQzgDn/fs4AVzrlGwKdAX+fc58A8YIhzLt45902AWOsA1wE3AS+bWWn/uDjn/g9vBNg0v7wfMNo/dlNgZ7a2hgLL/GO9kK1uOPClc64h8BjwWpa6ekA74ArgKTMrGSBOEREREUqEOwARERGREOsB/Nu/P91/vDrI9tcAnQCcc++b2a9Z6r51zq3x76/CSwIdM8+/XQdscM7tBjCzbcD5fruXA8lmBlAG2OPvcxRYkKXdG/L43GY65zKBrf5x6vnHedGPf7OZbQfqAsuBx82sJpDonNuax2Pgt3mb3+bHflKtgl/3rnPuCHDEzPYAVTk1wSUiIiKiJJSIiIhELzOrBLQB/mhmDogFnJn9NdhuQeqOZLmfgZdIyl6XmW27TLzvXAZMc849GqDdNOecy9JuXr+juQCPA8bvnHvTzL7AGzX1gZn1cc59nMfjBGrz2LGzvyb6fikiIiIBaTqeiIiIRLPOwGvOudrOuTrOufOBb/FG9uTkM6ArgJm1Bc4toFg+Ajqb2R/8tiuaWe1c9kkFygep72JmMf56URcCW/Cm893hH6Mu3nTCLWZ2IbDNOTcGb9RWw3wcK2ubrYCfnXMpucQuIiIichIloURERCSa9cBb1ymrWWS5ClwAw4G2ZrYauBHYjZegOSPOuY3A34FFZrYWWAxUz2W36cAQf4HxiwLUbwGWAu8B/Zxzh4GXgFgzW3uO53IAAADMSURBVAfMAHr50+W6AevNbA3etL3XsrW1Fkj3F0d/OFvdMKCpH/czQM88PWkRERGRLOzEyG8RERERMbM4IMM5l25mLYDx/mLeEcXMpgILnHPvhDsWERERkbzQnH0RERGRk9UCZppZDN6C4X3DHI+IiIhIVNBIKBERERERERERCTmtCSUiIiIiIiIiIiGnJJSIiIiIiIiIiIScklAiIiIiIiIiIhJySkKJiIiIiIiIiEjIKQklIiIiIiIiIiIhpySUiIiIiIiIiIiE3P8Di7D2oRqMlQsAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 1440x432 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "def get_profile(profile_file, seq):\n",
    "    \"\"\"Read `profile_file` into a Python array, and add gaps corresponding to\n",
    "       the alignment sequence `seq`.\"\"\"\n",
    "    # Read all non-comment and non-blank lines from the file:\n",
    "    f = open(profile_file)\n",
    "    vals = []\n",
    "    for line in f:\n",
    "        if not line.startswith('#') and len(line) > 10:\n",
    "            spl = line.split()\n",
    "            vals.append(float(spl[-1]))\n",
    "    # Insert gaps into the profile corresponding to those in seq:\n",
    "    for n, res in enumerate(seq.residues):\n",
    "        for gap in range(res.get_leading_gaps()):\n",
    "            vals.insert(n, None)\n",
    "    # Add a gap at position '0', so that we effectively count from 1:\n",
    "    vals.insert(0, None)\n",
    "    return vals\n",
    "\n",
    "e = modeller.environ()\n",
    "a = modeller.alignment(e, file='%s.ali' % name)\n",
    "template = get_profile('%s.profile' % pdb_template_name, a[pdb_template_name])\n",
    "model = get_profile('%s.profile' % pdb_model_name, a[name])\n",
    "\n",
    "#Plot CDR ranges\n",
    "import numpy as np\n",
    "cdrH = np.array([[27,35],[50,66],[99,109]])\n",
    "cdrL = np.array([[24,34],[49,56],[89,97]])+len(SEQH + linker)\n",
    "lista = np.append(cdrH, cdrL, 0)\n",
    "cdr   = get_rangeprofile(structure2D_gaps, lista, value=-0.045)\n",
    "\n",
    "#secondary elements\n",
    "sheet, beta, turn = get_2dprofile(structure2D_gaps)\n",
    "line_cutoff = get_profileline(structure2D_gaps)\n",
    "\n",
    "# Plot the template and model profiles in the same plot for comparison:\n",
    "pylab.figure(1, figsize=(20,6))\n",
    "pylab.xlabel('Alignment position')\n",
    "pylab.ylabel('DOPE per-residue score')\n",
    "pylab.plot(model, color='red', linewidth=2, label='Model')\n",
    "pylab.plot(template, color='green', linewidth=2, label='Template')\n",
    "\n",
    "\n",
    "pylab.plot(sheet, color='orange', linewidth=4, label='2D Temp')\n",
    "pylab.plot(beta, color='magenta', linewidth=4)\n",
    "pylab.plot(turn, color='green', linewidth=4)\n",
    "pylab.plot(cdr, color='blue', linewidth=6, label='CDRs')\n",
    "pylab.plot(line_cutoff, color='gray', linestyle='--', linewidth=2, label='Cut-off')\n",
    "\n",
    "pylab.legend()\n",
    "pylab.savefig('dope_profile.png', dpi=65)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Loop refinenment"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12900418   12598.064    12.303\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12900946   12598.580    12.303\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12949546   12646.041    12.350\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     12962846   12659.029    12.362\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "io_data____W> Setting io.atom_files_directory to a colon-delimited string is\n",
      "              deprecated, as it is not robust on Windows systems. Set it to\n",
      "              a list of directories instead. For example:\n",
      "              env.io.atom_files_directory = ['./', '../atom_files']\n",
      "openf___224_> Open           scfv.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     12974339   12670.253    12.373\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     12975789   12671.669    12.375\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     12978689   12674.501    12.377\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     12984489   12680.165    12.383\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     13101545   12794.478    12.495\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13102449   12795.360    12.495\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13103437   12796.325    12.496\n",
      "\n",
      "Read the alignment from file       : scfv.ali\n",
      "Total number of alignment positions:   253\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1     2fjhHL     227      1  2fjhHL.pdb\n",
      "  2       scfv     248      1        scfv\n",
      "check_a_343_> >> BEGINNING OF COMMAND\n",
      "openf___224_> Open           2fjhHL.pdb\n",
      "readlinef__W> File: 2fjhHL.pdb, Line: 1\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13173262   12864.514    12.563\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13173262   12864.514    12.563\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13173433   12864.681    12.563\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13174283   12865.511    12.564\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13175541   12866.739    12.565\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13177445   12868.599    12.567\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13177445   12868.599    12.567\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13177535   12868.687    12.567\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13180391   12871.476    12.570\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13180391   12871.476    12.570\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13180526   12871.607    12.570\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13184810   12875.791    12.574\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13184810   12875.791    12.574\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13185008   12875.984    12.574\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13191434   12882.260    12.580\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13191434   12882.260    12.580\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13191731   12882.550    12.581\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13201353   12891.946    12.590\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13201353   12891.946    12.590\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13201803   12892.386    12.590\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13216253   12906.497    12.604\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13216253   12906.497    12.604\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13216928   12907.156    12.605\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13238586   12928.307    12.625\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     13238586   12928.307    12.625\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     13239594   12929.291    12.626\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     13238604   12928.324    12.625\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "     98     1   95   101      Q     E   20.163\n",
      "END OF TABLE\n",
      "check_a_344_> << END OF COMMAND\n",
      "openf___224_> Open           ${LIB}/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           ${LIB}/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13377856   13064.312    12.758\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13379456   13065.875    12.760\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13383436   13069.762    12.763\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13385436   13071.715    12.765\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13388436   13074.645    12.768\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13392936   13079.039    12.772\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13398508   13084.480    12.778\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13401308   13087.215    12.780\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13405508   13091.316    12.784\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13411808   13097.469    12.790\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13414580   13100.176    12.793\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     13415980   13101.543    12.794\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13454759   13139.413    12.831\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13641155   13321.440    13.009\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13670635   13350.229    13.037\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933      0         0       876\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13751895   13429.585    13.115\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933   2622      3114       876\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13828671   13504.562    13.188\n",
      "patch_______> segment topology patched using RTF:   248  ;  HIS  ;  CTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     13848015   13523.452    13.206\n",
      "patch_______> segment topology patched using RTF:   158   223  ;  CYS  CYS  ;  DISU\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1935   2626      3119       877\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        1\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1578      306\n",
      "The following 1 residues contain 6-membered rings with poor geometries\n",
      "after transfer from templates. Rebuilding rings from internal coordinates:\n",
      "   <Residue 104 (type TYR)>\n",
      "openf___224_> Open           scfv.ini\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     13955064   13627.992    13.309\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     13963256   13635.992    13.316\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     13979640   13651.992    13.332\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14012408   13683.992    13.363\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14045176   13715.992    13.395\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14110712   13779.992    13.457\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14143480   13811.992    13.488\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14176248   13843.992    13.520\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14241784   13907.992    13.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14372856   14035.992    13.707\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14438392   14099.992    13.770\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14503928   14163.992    13.832\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14635000   14291.992    13.957\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :     1935    2626    2956     870\n",
      "              Total number of restraints before, now         :        0     8387\n",
      "make_re_422_> Number of previous, current restraints         :        0     8387\n",
      "make_re_423_> Number of previous, current selected restraints:        0     8387\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     14775123   14428.831    14.091\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  2fjhHL\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :     8387     8633\n",
      "make_re_423_> Number of previous, current selected restraints:     8387     8633\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     14775139   14428.847    14.091\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  2fjhHL\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:      254\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :     8633     8880\n",
      "make_re_423_> Number of previous, current selected restraints:     8633     8626\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     14775139   14428.847    14.091\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  2fjhHL\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :     8880     9081\n",
      "make_re_423_> Number of previous, current selected restraints:     8626     8827\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     14775139   14428.847    14.091\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  2fjhHL\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14906211   14556.847    14.216\n",
      "make_re_422_> Number of previous, current restraints         :     9081     9203\n",
      "make_re_423_> Number of previous, current selected restraints:     8827     8949\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     14906211   14556.847    14.216\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  2fjhHL\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :     9203     9251\n",
      "make_re_423_> Number of previous, current selected restraints:     8949     8997\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     14906211   14556.847    14.216\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  2fjhHL\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     15168355   14812.847    14.466\n",
      "make_re_422_> Number of previous, current restraints         :     9251     9267\n",
      "make_re_423_> Number of previous, current selected restraints:     8997     9013\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     15710911   15342.687    14.983\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     15841983   15470.687    15.108\n",
      "make_re_422_> Number of previous, current restraints         :     9267    13824\n",
      "make_re_423_> Number of previous, current selected restraints:     9013    13570\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     16104127   15726.687    15.358\n",
      "make_re_422_> Number of previous, current restraints         :    13824    18552\n",
      "make_re_423_> Number of previous, current selected restraints:    13570    18298\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     16628415   16238.687    15.858\n",
      "make_re_422_> Number of previous, current restraints         :    18552    21786\n",
      "make_re_423_> Number of previous, current selected restraints:    18298    21532\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     16890559   16494.688    16.108\n",
      "make_re_422_> Number of previous, current restraints         :    21786    23335\n",
      "make_re_423_> Number of previous, current selected restraints:    21532    23081\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23335    23335\n",
      "make_re_423_> Number of previous, current selected restraints:    23081    23081\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23335    23335\n",
      "make_re_423_> Number of previous, current selected restraints:    23081    23081\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23335    23335\n",
      "make_re_423_> Number of previous, current selected restraints:    23081    23081\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23335    23335\n",
      "make_re_423_> Number of previous, current selected restraints:    23081    23081\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "make_re_417_> Restraint type to be calculated:  DISTANCE\n",
      "make_re_422_> Number of previous, current restraints         :    23335    23335\n",
      "make_re_423_> Number of previous, current selected restraints:    23081    23081\n",
      "rmdupl__427_>     1412 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    23335    21669\n",
      "openf___224_> Open           scfv.rsr\n",
      "openf___224_> Open           scfv.rsr\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     17007860   16609.238    16.220\n",
      "rdcsr2__307_> Number of restraints read    :    21669\n",
      "              Number of excluded pairs read:        0\n",
      "              Number of pseudo atoms read  :        0\n",
      "rdcsrs__304_> Restraints in memory, selected restraints:    21669    21669\n",
      "              Explicitly excluded atom pairs in memory :        0\n",
      "              Pseudo atoms in memory                   :        0\n",
      "openf___224_> Open           scfv.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     17007884   16609.262    16.220\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884     1884        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    11636\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    12758\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    13207\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    13818\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    15457\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    16513\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    17658\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    19939\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     17345680   16939.141    16.542\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    21089\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    21669\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    21669\n",
      "iupac_m_486_> OE1/2 will be swapped:     -134.9896        8     8\n",
      "iupac_m_487_> NH1/2 swapped:     -179.3412       21    21\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -101.7679       32    32\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      177.1912       70    70\n",
      "iupac_m_487_> NH1/2 swapped:     -179.8658       90    90\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       99.0370      108   108\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      121.2221      152   152\n",
      "iupac_m_487_> NH1/2 swapped:      179.6608      153   153\n",
      "iupac_m_487_> NH1/2 swapped:     -176.1318      201   201\n",
      "iupac_m_486_> OE1/2 will be swapped:     -101.1555      216   216\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       95.2833      224   224\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       97.7073      230   230\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21669   21669\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     3832\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3035.9229\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1934      10     12   0.018   0.018      192.77       1.000\n",
      " 2 Bond angle potential               :    2624       5     24   2.437   2.437      307.84       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1289       0     49  49.986  49.986      519.45       1.000\n",
      " 4 Stereochemical improper torsion pot:     870       0      0   1.287   1.287      31.377       1.000\n",
      " 5 Soft-sphere overlap restraints     :    3832       0      0   0.003   0.003      4.8484       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    4557      25     47   0.395   0.395      704.07       1.000\n",
      "10 Distance restraints 2 (N-O)        :    4728      20     46   0.408   0.408      683.25       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     247       2      4  16.448  16.448      49.216       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     201       0      4  70.045  70.045      76.648       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     122       0      0  73.268  73.268      38.443       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      48       0      0  73.418  73.418      34.199       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      16       0      0 110.829 110.829      10.739       1.000\n",
      "18 Disulfide distance restraints      :       1       0      0   0.020   0.020     0.69494E-01   1.000\n",
      "19 Disulfide angle restraints         :       2       0      0   2.872   2.872     0.36429       1.000\n",
      "20 Disulfide dihedral angle restraints:       1       0      0   5.692   5.692     0.39674E-01   1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3234       0      0   0.523   0.523      102.38       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     246      32     24  30.528  67.569      178.95       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1549       0      1   0.680   0.680      101.27       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           scfv.V99990001\n",
      "# Heavy relative violation of each residue is written to: scfv.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   27636.9023\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    757  98Q  98Q N   CA    735  736    1.58    1.43    0.15    5.06    1.43    0.15    5.06\n",
      "    2    759  98Q  98Q C   CA    742  736    1.79    1.49    0.30    8.67    1.49    0.30    8.67\n",
      "    3    760  98Q  99E C   N     742  744    1.59    1.35    0.25    8.71    1.35    0.25    8.71\n",
      "    4    766  99E  99E N   CA    744  745    1.67    1.43    0.24    7.89    1.43    0.24    7.89\n",
      "    5    768  99E  99E C   CA    751  745    1.75    1.49    0.26    7.52    1.49    0.26    7.52\n",
      "    6    769  99E 100T C   N     751  753    1.56    1.35    0.21    7.55    1.35    0.21    7.55\n",
      "    7    773 100T 100T N   CA    753  754    1.68    1.43    0.25    8.15    1.43    0.25    8.15\n",
      "    8    775 100T 100T C   CA    758  754    1.79    1.49    0.30    8.59    1.49    0.30    8.59\n",
      "    9    776 100T 101C C   N     758  760    1.50    1.35    0.16    5.64    1.35    0.16    5.64\n",
      "   10    779 101C 101C N   CA    760  761    1.59    1.43    0.16    5.41    1.43    0.16    5.41\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2954  98Q  98Q N   CA    735  736  127.38  107.00   20.38    5.86  107.00   20.38    5.86\n",
      "    2   2957  98Q  98Q CA  C     736  742  132.70  116.50   16.20    4.66  116.50   16.20    4.66\n",
      "    3   2963  98Q  99E C   N     742  744  151.49  120.00   31.49    7.16  120.00   31.49    7.16\n",
      "    4   2966  99E  99E N   CA    744  745  136.16  107.00   29.16    8.38  107.00   29.16    8.38\n",
      "    5   2978 100T 100T N   CA    753  754  136.74  107.00   29.74    8.55  107.00   29.74    8.55\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8461  33Y  99E CA  CA    226  745   13.40    8.82    4.57    8.72    8.82    4.57    8.72\n",
      "    2   8484  34Y  99E CA  CA    238  745   10.17    7.65    2.52    5.82    7.65    2.52    5.82\n",
      "    3   9864  99E 185S CA  CA    745 1352   16.38   13.01    3.37    5.83   13.01    3.37    5.83\n",
      "    4   9879 100T 188F CA  CA    754 1369   14.53   11.73    2.79    5.02   11.73    2.79    5.02\n",
      "    5   9880 100T 189L CA  CA    754 1380   14.32   11.85    2.47    4.66   11.85    2.47    4.66\n",
      "    6   9882 100T 191S CA  CA    754 1400   12.62    9.00    3.62    5.88    9.00    3.62    5.88\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14073  99E  30S N   O     744  206   13.36    9.12    4.24    6.78    9.12    4.24    6.78\n",
      "    2  14074  99E  31S N   O     744  212   12.38   10.10    2.28    4.56   10.10    2.28    4.56\n",
      "    3  14075  99E  32Y N   O     744  224   14.79    9.90    4.89    8.59    9.90    4.89    8.59\n",
      "    4  14076  99E  33Y N   O     744  236   12.91    5.55    7.36   13.61    5.55    7.36   13.61\n",
      "    5  14077  99E  34Y N   O     744  248   11.64    8.73    2.91    6.07    8.73    2.91    6.07\n",
      "    6  15654 188F 100T N   O    1368  759   13.64   10.21    3.43    6.15   10.21    3.43    6.15\n",
      "    7  15677 189L 100T N   O    1379  759   12.93    9.47    3.46    6.60    9.47    3.46    6.60\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6722   1M   2A C   N       7    9  -71.41  -68.20    3.66    0.25  -62.50  172.28   28.68\n",
      "    1          2A   2A N   CA      9   10  147.05  145.30                  -40.90\n",
      "    2   6723   2A   3E C   N      12   14 -145.80 -117.80   29.71    0.89  -63.60 -168.50   30.13\n",
      "    2          3E   3E N   CA     14   15  146.74  136.80                  -40.30\n",
      "    3   6748  27S  28S C   N     187  189   80.29   56.90   59.88    2.45  -64.10  145.31   19.09\n",
      "    3         28S  28S N   CA    189  190  -18.72   36.40                  -35.00\n",
      "    4   6753  32Y  33Y C   N     223  225  -73.94  -98.40   62.43    6.29  -63.50  114.83   17.69\n",
      "    4         33Y  33Y N   CA    225  226   70.95  128.40                  -43.40\n",
      "    5   6819  98Q  99E C   N     742  744  -98.10  -69.30   77.98    4.92  -63.60  110.27   16.65\n",
      "    5         99E  99E N   CA    744  745 -145.03  142.50                  -40.30\n",
      "    6   6820  99E 100T C   N     751  753 -153.49 -124.80   43.62    1.37  -63.20  167.90   27.25\n",
      "    6        100T 100T N   CA    753  754  176.35  143.50                  -42.10\n",
      "    7   6824 103T 104Y C   N     780  782   66.92   55.90   28.18    1.36  -63.50  142.32   26.61\n",
      "    7        104Y 104Y N   CA    782  783   13.56   39.50                  -43.40\n",
      "    8   6838 117V 118S C   N     892  894 -142.84 -136.60   14.36    0.57  -64.10  179.10   18.61\n",
      "    8        118S 118S N   CA    894  895  164.14  151.20                  -35.00\n",
      "    9   6839 118S 119S C   N     898  900  -68.96  -72.40   11.73    0.81  -64.10  161.45   11.99\n",
      "    9        119S 119S N   CA    900  901  163.62  152.40                  -35.00\n",
      "   10   6840 119S 120G C   N     904  906   91.76   78.70   25.93    0.39   82.20  163.27    8.41\n",
      "   10        120G 120G N   CA    906  907  171.49 -166.10                    8.50\n",
      "   11   6841 120G 121G C   N     908  910  156.49 -167.20   36.31    0.70   82.20 -178.23   12.25\n",
      "   11        121G 121G N   CA    910  911  174.40  174.60                    8.50\n",
      "   12   6842 121G 122G C   N     912  914   86.34   78.70   14.75    0.23   82.20  172.83    8.60\n",
      "   12        122G 122G N   CA    914  915 -178.72 -166.10                    8.50\n",
      "   13   6843 122G 123G C   N     916  918  145.54 -167.20   48.11    0.74   82.20 -173.79   12.03\n",
      "   13        123G 123G N   CA    918  919 -176.39  174.60                    8.50\n",
      "   14   6844 123G 124S C   N     920  922 -150.28 -136.60   18.34    0.58  -64.10 -176.86   19.46\n",
      "   14        124S 124S N   CA    922  923  163.41  151.20                  -35.00\n",
      "   15   6845 124S 125G C   N     926  928  -75.07  -80.20   14.66    0.36   82.20 -141.37    6.97\n",
      "   15        125G 125G N   CA    928  929  160.37  174.10                    8.50\n",
      "   16   6846 125G 126G C   N     930  932  -62.99  -62.40    1.11    0.20   82.20  153.78   11.71\n",
      "   16        126G 126G N   CA    932  933  -42.14  -41.20                    8.50\n",
      "   17   6847 126G 127G C   N     934  936   76.52   78.70   12.38    0.26   82.20  162.52    8.18\n",
      "   17        127G 127G N   CA    936  937 -153.92 -166.10                    8.50\n",
      "   18   6848 127G 128G C   N     938  940   93.72   78.70   27.23    0.44   82.20  163.09    8.49\n",
      "   18        128G 128G N   CA    940  941  171.18 -166.10                    8.50\n",
      "   19   6849 128G 129S C   N     942  944 -130.30 -136.60   10.22    0.58  -64.10  178.48   17.62\n",
      "   19        129S 129S N   CA    944  945  159.25  151.20                  -35.00\n",
      "   20   6850 129S 130G C   N     948  950  -87.34  -80.20    9.02    0.34   82.20 -119.12    7.50\n",
      "   20        130G 130G N   CA    950  951  179.62  174.10                    8.50\n",
      "   21   6851 130G 131G C   N     952  954 -150.65 -167.20   38.84    1.21   82.20 -177.48   13.96\n",
      "   21        131G 131G N   CA    954  955  139.46  174.60                    8.50\n",
      "   22   6853 132G 133G C   N     960  962   91.46   78.70   40.35    0.68   82.20  147.41    7.62\n",
      "   22        133G 133G N   CA    962  963  155.62 -166.10                    8.50\n",
      "   23   6854 133G 134S C   N     964  966 -141.13 -136.60    9.93    0.39  -64.10 -177.93   18.70\n",
      "   23        134S 134S N   CA    966  967  160.03  151.20                  -35.00\n",
      "   24   6884 163D 164V C   N    1180 1182 -125.78  -62.40   99.67   11.16  -62.40   99.67   11.16\n",
      "   24        164V 164V N   CA   1182 1183   34.52  -42.40                  -42.40\n",
      "   25   6885 164V 165N C   N    1187 1189   49.42  -71.20  139.39   11.25 -119.90 -174.55    9.37\n",
      "   25        165N 165N N   CA   1189 1190 -147.34  142.80                  137.00\n",
      "   26   6907 186A 187S C   N    1360 1362 -147.10  -64.10   95.27    9.56  -64.10   95.27    9.56\n",
      "   26        187S 187S N   CA   1362 1363   11.77  -35.00                  -35.00\n",
      "   27   6962 241I 242K C   N    1813 1815  -86.87  -70.20   24.45    2.04  -62.90  165.06   20.24\n",
      "   27        242K 242K N   CA   1815 1816  122.52  140.40                  -40.80\n",
      "   28   6963 242K 243H C   N    1822 1824 -143.73 -125.60   19.44    0.81  -63.20 -168.17   20.49\n",
      "   28        243H 243H N   CA   1824 1825  131.80  138.80                  -42.30\n",
      "   29   6964 243H 244H C   N    1832 1834 -130.28 -125.60    7.53    0.34  -63.20 -172.39   20.27\n",
      "   29        244H 244H N   CA   1834 1835  132.90  138.80                  -42.30\n",
      "   30   6965 244H 245H C   N    1842 1844 -119.33 -125.60   31.17    1.06  -63.20  160.69   17.40\n",
      "   30        245H 245H N   CA   1844 1845  108.27  138.80                  -42.30\n",
      "   31   6966 245H 246H C   N    1852 1854 -118.38 -125.60   19.14    0.58  -63.20  172.45   18.82\n",
      "   31        246H 246H N   CA   1854 1855  121.08  138.80                  -42.30\n",
      "   32   6967 246H 247H C   N    1862 1864  -77.69  -67.60   10.10    0.75  -63.20  178.00   23.18\n",
      "   32        247H 247H N   CA   1864 1865  140.29  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   14   27   97   99  150  161  181  212  238  315\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.B99990001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     17345704   16939.164    16.542\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884     1884        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    11636\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    12758\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    13207\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    13818\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    15457\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    16513\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    17658\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    19939\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    21089\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    21669\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    21669\n",
      "iupac_m_486_> OE1/2 will be swapped:      107.6858        3     3\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -93.4747       32    32\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -96.3547       33    33\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.6286       34    34\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:      171.7923       40    40\n",
      "iupac_m_486_> OE1/2 will be swapped:      154.7669       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       94.2932      108   108\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_485_> OD1/2 will be swapped:      170.5291      152   152\n",
      "iupac_m_485_> OD1/2 will be swapped:      166.5229      163   163\n",
      "iupac_m_487_> NH1/2 swapped:     -177.9960      201   201\n",
      "iupac_m_486_> OE1/2 will be swapped:      160.4200      216   216\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.6361      228   228\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      108.4663      230   230\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      123.3825      240   240\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21669   21669\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     3923\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3101.3606\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1934      10     12   0.019   0.019      201.66       1.000\n",
      " 2 Bond angle potential               :    2624       7     21   2.508   2.508      327.81       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1289       0     48  49.830  49.830      517.06       1.000\n",
      " 4 Stereochemical improper torsion pot:     870       0      0   1.174   1.174      28.773       1.000\n",
      " 5 Soft-sphere overlap restraints     :    3923       0      0   0.003   0.003      5.5051       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    4557      24     47   0.394   0.394      701.25       1.000\n",
      "10 Distance restraints 2 (N-O)        :    4728      19     60   0.424   0.424      750.20       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     247       1      4  12.102  12.102      56.982       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     201       0      5  68.176  68.176      54.987       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     122       0      0  68.302  68.302      31.244       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      48       0      0  64.246  64.246      32.667       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      16       0      0 109.489 109.489      12.971       1.000\n",
      "18 Disulfide distance restraints      :       1       0      0   0.017   0.017     0.48361E-01   1.000\n",
      "19 Disulfide angle restraints         :       2       0      0   2.591   2.591     0.29652       1.000\n",
      "20 Disulfide dihedral angle restraints:       1       0      0   5.242   5.242     0.33688E-01   1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3234       0      0   0.440   0.440      83.099       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     246      31     30  32.117  69.478      216.71       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1549       0      0   0.599   0.599      80.065       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           scfv.V99990002\n",
      "# Heavy relative violation of each residue is written to: scfv.V99990002\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   26965.1250\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    757  98Q  98Q N   CA    735  736    1.59    1.43    0.16    5.34    1.43    0.16    5.34\n",
      "    2    759  98Q  98Q C   CA    742  736    1.80    1.49    0.31    8.89    1.49    0.31    8.89\n",
      "    3    760  98Q  99E C   N     742  744    1.60    1.35    0.26    9.13    1.35    0.26    9.13\n",
      "    4    766  99E  99E N   CA    744  745    1.67    1.43    0.24    8.03    1.43    0.24    8.03\n",
      "    5    768  99E  99E C   CA    751  745    1.76    1.49    0.27    7.96    1.49    0.27    7.96\n",
      "    6    769  99E 100T C   N     751  753    1.55    1.35    0.20    7.14    1.35    0.20    7.14\n",
      "    7    773 100T 100T N   CA    753  754    1.69    1.43    0.26    8.49    1.43    0.26    8.49\n",
      "    8    775 100T 100T C   CA    758  754    1.78    1.49    0.29    8.33    1.49    0.29    8.33\n",
      "    9    776 100T 101C C   N     758  760    1.50    1.35    0.16    5.64    1.35    0.16    5.64\n",
      "   10    779 101C 101C N   CA    760  761    1.59    1.43    0.16    5.18    1.43    0.16    5.18\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2954  98Q  98Q N   CA    735  736  127.08  107.00   20.08    5.77  107.00   20.08    5.77\n",
      "    2   2957  98Q  98Q CA  C     736  742  134.17  116.50   17.67    5.08  116.50   17.67    5.08\n",
      "    3   2963  98Q  99E C   N     742  744  145.58  120.00   25.58    5.81  120.00   25.58    5.81\n",
      "    4   2966  99E  99E N   CA    744  745  130.03  107.00   23.03    6.62  107.00   23.03    6.62\n",
      "    5   2975  99E 100T C   N     751  753  151.90  120.00   31.90    7.25  120.00   31.90    7.25\n",
      "    6   2978 100T 100T N   CA    753  754  144.61  107.00   37.61   10.82  107.00   37.61   10.82\n",
      "    7   2985 100T 101C C   N     758  760  141.81  120.00   21.81    4.96  120.00   21.81    4.96\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8461  33Y  99E CA  CA    226  745   13.33    8.82    4.50    8.59    8.82    4.50    8.59\n",
      "    2   8484  34Y  99E CA  CA    238  745   10.07    7.65    2.42    5.60    7.65    2.42    5.60\n",
      "    3   9864  99E 185S CA  CA    745 1352   16.21   13.01    3.20    5.54   13.01    3.20    5.54\n",
      "    4   9879 100T 188F CA  CA    754 1369   14.45   11.73    2.71    4.87   11.73    2.71    4.87\n",
      "    5   9882 100T 191S CA  CA    754 1400   12.47    9.00    3.48    5.64    9.00    3.48    5.64\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14073  99E  30S N   O     744  206   12.88    9.12    3.76    6.02    9.12    3.76    6.02\n",
      "    2  14075  99E  32Y N   O     744  224   14.58    9.90    4.68    8.22    9.90    4.68    8.22\n",
      "    3  14076  99E  33Y N   O     744  236   13.06    5.55    7.52   13.89    5.55    7.52   13.89\n",
      "    4  14077  99E  34Y N   O     744  248   11.80    8.73    3.07    6.40    8.73    3.07    6.40\n",
      "    5  15654 188F 100T N   O    1368  759   13.81   10.21    3.60    6.46   10.21    3.60    6.46\n",
      "    6  15677 189L 100T N   O    1379  759   13.17    9.47    3.69    7.06    9.47    3.69    7.06\n",
      "    7  15699 190Y 100T N   O    1387  759   10.78    8.43    2.35    4.80    8.43    2.35    4.80\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6722   1M   2A C   N       7    9  -78.55  -68.20   22.97    1.51  -62.50  154.13   26.08\n",
      "    1          2A   2A N   CA      9   10  165.81  145.30                  -40.90\n",
      "    2   6723   2A   3E C   N      12   14  -84.42  -69.30   19.36    1.77  -63.60  171.98   21.85\n",
      "    2          3E   3E N   CA     14   15  130.41  142.50                  -40.30\n",
      "    3   6748  27S  28S C   N     187  189   73.06   56.90   40.41    1.65  -64.10  141.40   18.87\n",
      "    3         28S  28S N   CA    189  190   -0.64   36.40                  -35.00\n",
      "    4   6753  32Y  33Y C   N     223  225  -79.71  -98.40   86.42    9.09  -63.50   88.91   13.31\n",
      "    4         33Y  33Y N   CA    225  226   44.02  128.40                  -43.40\n",
      "    5   6819  98Q  99E C   N     742  744 -141.74 -117.80   41.19    1.47  -63.60  168.58   26.83\n",
      "    5         99E  99E N   CA    744  745  170.32  136.80                  -40.30\n",
      "    6   6820  99E 100T C   N     751  753  -77.48  -78.10   27.88    1.21  -63.20  140.96   19.02\n",
      "    6        100T 100T N   CA    753  754  177.67  149.80                  -42.10\n",
      "    7   6824 103T 104Y C   N     780  782   62.86   55.90   22.10    1.12  -63.50  140.72   26.43\n",
      "    7        104Y 104Y N   CA    782  783   18.52   39.50                  -43.40\n",
      "    8   6838 117V 118S C   N     892  894 -134.33 -136.60   13.06    0.61  -64.10 -172.98   11.59\n",
      "    8        118S 118S N   CA    894  895  138.34  151.20                  -35.00\n",
      "    9   6839 118S 119S C   N     898  900 -161.80 -136.60   45.49    1.65  -64.10  167.40   19.18\n",
      "    9        119S 119S N   CA    900  901 -170.93  151.20                  -35.00\n",
      "   10   6840 119S 120G C   N     904  906  -77.23  -80.20    5.53    0.13   82.20 -133.46    7.05\n",
      "   10        120G 120G N   CA    906  907  169.44  174.10                    8.50\n",
      "   11   6842 121G 122G C   N     912  914   95.33   78.70   26.56    1.36   82.20  154.45    6.83\n",
      "   11        122G 122G N   CA    914  915 -145.39 -166.10                    8.50\n",
      "   12   6845 124S 125G C   N     926  928 -107.95  -80.20   40.90    2.85   82.20 -142.70   16.91\n",
      "   12        125G 125G N   CA    928  929  144.05  174.10                    8.50\n",
      "   13   6846 125G 126G C   N     930  932   77.92   78.70   10.79    0.34   82.20  174.69    8.26\n",
      "   13        126G 126G N   CA    932  933 -176.86 -166.10                    8.50\n",
      "   14   6847 126G 127G C   N     934  936   88.84   78.70   37.86    0.69   82.20  149.07    7.57\n",
      "   14        127G 127G N   CA    936  937  157.43 -166.10                    8.50\n",
      "   15   6848 127G 128G C   N     938  940   67.34   78.70   11.45    0.54   82.20  173.79    9.18\n",
      "   15        128G 128G N   CA    940  941 -164.65 -166.10                    8.50\n",
      "   16   6849 128G 129S C   N     942  944 -142.82 -136.60   17.72    0.75  -64.10  175.82   18.38\n",
      "   16        129S 129S N   CA    944  945  167.79  151.20                  -35.00\n",
      "   17   6850 129S 130G C   N     948  950 -100.78  -80.20   31.58    2.17   82.20 -133.29   17.64\n",
      "   17        130G 130G N   CA    950  951  150.15  174.10                    8.50\n",
      "   18   6851 130G 131G C   N     952  954   82.18   78.70    3.79    0.14   82.20  176.09    8.55\n",
      "   18        131G 131G N   CA    954  955 -167.59 -166.10                    8.50\n",
      "   19   6854 133G 134S C   N     964  966  -65.82  -72.40   16.59    1.20  -64.10  157.37   11.50\n",
      "   19        134S 134S N   CA    966  967  167.63  152.40                  -35.00\n",
      "   20   6855 134S 135T C   N     970  972  -58.35  -78.10   29.98    1.05  -63.20  169.42   22.04\n",
      "   20        135T 135T N   CA    972  973  127.25  149.80                  -42.10\n",
      "   21   6870 149S 150V C   N    1076 1078  -66.08  -62.40   16.50    1.90  -73.50  165.68   14.06\n",
      "   21        150V 150V N   CA   1078 1079  -26.31  -42.40                  139.20\n",
      "   22   6871 150V 151G C   N    1083 1085  -91.88  -62.40   34.24    5.20   82.20  177.05   12.90\n",
      "   22        151G 151G N   CA   1085 1086  -23.78  -41.20                    8.50\n",
      "   23   6884 163D 164V C   N    1180 1182 -128.51  -62.40  102.59   11.52  -62.40  102.59   11.52\n",
      "   23        164V 164V N   CA   1182 1183   36.05  -42.40                  -42.40\n",
      "   24   6885 164V 165N C   N    1187 1189   43.36  -71.20  137.39   11.11 -119.90 -177.47    9.35\n",
      "   24        165N 165N N   CA   1189 1190 -141.37  142.80                  137.00\n",
      "   25   6907 186A 187S C   N    1360 1362 -147.19  -64.10   94.30    9.60  -64.10   94.30    9.60\n",
      "   25        187S 187S N   CA   1362 1363    9.60  -35.00                  -35.00\n",
      "   26   6911 190Y 191S C   N    1397 1399  -42.79  -64.10   54.71    3.40  -72.40  125.75    8.42\n",
      "   26        191S 191S N   CA   1399 1400  -85.39  -35.00                  152.40\n",
      "   27   6912 191S 192G C   N    1403 1405  -63.03  -62.40   13.72    2.02   82.20  149.63   11.12\n",
      "   27        192G 192G N   CA   1405 1406  -27.50  -41.20                    8.50\n",
      "   28   6962 241I 242K C   N    1813 1815 -106.78 -118.00   12.42    0.41  -62.90 -179.99   21.38\n",
      "   28        242K 242K N   CA   1815 1816  133.78  139.10                  -40.80\n",
      "   29   6963 242K 243H C   N    1822 1824 -143.46 -125.60   18.96    0.78  -63.20 -167.69   20.54\n",
      "   29        243H 243H N   CA   1824 1825  132.46  138.80                  -42.30\n",
      "   30   6964 243H 244H C   N    1832 1834 -123.89 -125.60   16.00    0.57  -63.20  175.99   19.07\n",
      "   30        244H 244H N   CA   1834 1835  122.89  138.80                  -42.30\n",
      "   31   6967 246H 247H C   N    1862 1864  -68.24  -67.60    1.87    0.12  -63.20  176.01   22.32\n",
      "   31        247H 247H N   CA   1864 1865  141.76  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   18   33   94   96  152  174  184  244  269  307\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.B99990002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.ini\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     17345704   16939.164    16.542\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884     1884        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    11636\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    12758\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    13207\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    13818\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    15457\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    16513\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    17658\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    19939\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    21089\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    21669\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;     1884    21669\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -95.1079       32    32\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      119.9334       48    48\n",
      "iupac_m_486_> OE1/2 will be swapped:     -123.0852       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       96.0858      108   108\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_487_> NH1/2 swapped:     -175.3731      153   153\n",
      "iupac_m_485_> OD1/2 will be swapped:      161.2492      163   163\n",
      "iupac_m_487_> NH1/2 swapped:      179.1091      201   201\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -90.2828      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.0743\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      106.0435      229   229\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      104.1229      230   230\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       91.1840      233   233\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    21669   21669\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     3836\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3076.2551\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1934      10     12   0.018   0.018      194.78       1.000\n",
      " 2 Bond angle potential               :    2624       7     25   2.446   2.446      310.73       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1289       0     52  50.427  50.427      527.71       1.000\n",
      " 4 Stereochemical improper torsion pot:     870       0      0   1.162   1.162      27.641       1.000\n",
      " 5 Soft-sphere overlap restraints     :    3836       0      0   0.003   0.003      5.0996       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    4557      25     48   0.395   0.395      702.66       1.000\n",
      "10 Distance restraints 2 (N-O)        :    4728      19     53   0.415   0.415      711.23       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     247       2      4  12.040  12.040      52.977       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     201       0      1  72.927  72.927      51.822       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     122       0      0  69.584  69.584      32.917       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      48       0      0  72.605  72.605      32.931       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      16       0      0 112.483 112.483      14.840       1.000\n",
      "18 Disulfide distance restraints      :       1       0      0   0.014   0.014     0.34904E-01   1.000\n",
      "19 Disulfide angle restraints         :       2       0      0   2.196   2.196     0.21291       1.000\n",
      "20 Disulfide dihedral angle restraints:       1       0      0   3.959   3.959     0.19260E-01   1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    3234       0      0   0.500   0.500      105.70       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     246      34     30  32.593  71.859      202.72       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1549       0      0   0.677   0.677      102.23       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           scfv.V99990003\n",
      "# Heavy relative violation of each residue is written to: scfv.V99990003\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   28356.9102\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    757  98Q  98Q N   CA    735  736    1.59    1.43    0.16    5.26    1.43    0.16    5.26\n",
      "    2    759  98Q  98Q C   CA    742  736    1.79    1.49    0.30    8.64    1.49    0.30    8.64\n",
      "    3    760  98Q  99E C   N     742  744    1.60    1.35    0.25    8.95    1.35    0.25    8.95\n",
      "    4    766  99E  99E N   CA    744  745    1.67    1.43    0.24    7.86    1.43    0.24    7.86\n",
      "    5    768  99E  99E C   CA    751  745    1.76    1.49    0.27    7.82    1.49    0.27    7.82\n",
      "    6    769  99E 100T C   N     751  753    1.54    1.35    0.20    7.05    1.35    0.20    7.05\n",
      "    7    773 100T 100T N   CA    753  754    1.68    1.43    0.25    8.21    1.43    0.25    8.21\n",
      "    8    775 100T 100T C   CA    758  754    1.79    1.49    0.30    8.86    1.49    0.30    8.86\n",
      "    9    776 100T 101C C   N     758  760    1.51    1.35    0.16    5.74    1.35    0.16    5.74\n",
      "   10    779 101C 101C N   CA    760  761    1.58    1.43    0.15    5.08    1.43    0.15    5.08\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2954  98Q  98Q N   CA    735  736  126.68  107.00   19.68    5.66  107.00   19.68    5.66\n",
      "    2   2957  98Q  98Q CA  C     736  742  133.46  116.50   16.96    4.88  116.50   16.96    4.88\n",
      "    3   2963  98Q  99E C   N     742  744  145.45  120.00   25.45    5.78  120.00   25.45    5.78\n",
      "    4   2966  99E  99E N   CA    744  745  129.75  107.00   22.75    6.54  107.00   22.75    6.54\n",
      "    5   2975  99E 100T C   N     751  753  147.90  120.00   27.90    6.34  120.00   27.90    6.34\n",
      "    6   2978 100T 100T N   CA    753  754  142.83  107.00   35.83   10.30  107.00   35.83   10.30\n",
      "    7   2985 100T 101C C   N     758  760  142.66  120.00   22.66    5.15  120.00   22.66    5.15\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   8461  33Y  99E CA  CA    226  745   13.43    8.82    4.61    8.79    8.82    4.61    8.79\n",
      "    2   8484  34Y  99E CA  CA    238  745   10.09    7.65    2.44    5.65    7.65    2.44    5.65\n",
      "    3   9864  99E 185S CA  CA    745 1352   16.24   13.01    3.23    5.60   13.01    3.23    5.60\n",
      "    4   9879 100T 188F CA  CA    754 1369   14.41   11.73    2.68    4.81   11.73    2.68    4.81\n",
      "    5   9882 100T 191S CA  CA    754 1400   12.66    9.00    3.67    5.96    9.00    3.67    5.96\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14073  99E  30S N   O     744  206   12.90    9.12    3.78    6.05    9.12    3.78    6.05\n",
      "    2  14075  99E  32Y N   O     744  224   14.81    9.90    4.90    8.61    9.90    4.90    8.61\n",
      "    3  14076  99E  33Y N   O     744  236   13.07    5.55    7.52   13.90    5.55    7.52   13.90\n",
      "    4  14077  99E  34Y N   O     744  248   11.39    8.73    2.66    5.55    8.73    2.66    5.55\n",
      "    5  15654 188F 100T N   O    1368  759   13.97   10.21    3.76    6.73   10.21    3.76    6.73\n",
      "    6  15677 189L 100T N   O    1379  759   13.30    9.47    3.83    7.32    9.47    3.83    7.32\n",
      "    7  15699 190Y 100T N   O    1387  759   10.96    8.43    2.54    5.18    8.43    2.54    5.18\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7066  99E  99E CA  C     745  751 -157.16 -180.00   22.84    4.57 -180.00   22.84    4.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6722   1M   2A C   N       7    9 -127.58 -134.00   12.33    0.48  -62.50 -171.07   28.58\n",
      "    1          2A   2A N   CA      9   10  136.47  147.00                  -40.90\n",
      "    2   6723   2A   3E C   N      12   14 -169.65 -117.80   54.17    1.66  -63.60 -161.98   32.21\n",
      "    2          3E   3E N   CA     14   15  152.47  136.80                  -40.30\n",
      "    3   6748  27S  28S C   N     187  189   66.15   56.90   24.54    1.01  -64.10  139.05   18.60\n",
      "    3         28S  28S N   CA    189  190   13.67   36.40                  -35.00\n",
      "    4   6753  32Y  33Y C   N     223  225  -85.26  -98.40   62.58    6.59  -63.50  112.74   16.82\n",
      "    4         33Y  33Y N   CA    225  226   67.22  128.40                  -43.40\n",
      "    5   6763  42A  43P C   N     313  315  -41.98  -58.70   40.36    2.37  -64.50  147.30   12.21\n",
      "    5         43P  43P N   CA    315  316  -67.24  -30.50                  147.20\n",
      "    6   6764  43P  44G C   N     320  322  -77.21  -62.40   17.72    2.65   82.20  164.35   12.22\n",
      "    6         44G  44G N   CA    322  323  -31.47  -41.20                    8.50\n",
      "    7   6819  98Q  99E C   N     742  744 -141.07 -117.80   44.10    1.64  -63.60  164.79   26.28\n",
      "    7         99E  99E N   CA    744  745  174.26  136.80                  -40.30\n",
      "    8   6820  99E 100T C   N     751  753  -94.72  -78.10   36.32    1.19  -63.20  139.42   19.98\n",
      "    8        100T 100T N   CA    753  754 -177.91  149.80                  -42.10\n",
      "    9   6824 103T 104Y C   N     780  782   65.77   55.90   26.52    1.29  -63.50  141.81   26.55\n",
      "    9        104Y 104Y N   CA    782  783   14.89   39.50                  -43.40\n",
      "   10   6838 117V 118S C   N     892  894 -129.55 -136.60   16.12    0.64  -64.10 -176.25   11.35\n",
      "   10        118S 118S N   CA    894  895  136.70  151.20                  -35.00\n",
      "   11   6839 118S 119S C   N     898  900  -97.81  -72.40   29.92    1.59  -64.10  160.38   13.90\n",
      "   11        119S 119S N   CA    900  901  168.20  152.40                  -35.00\n",
      "   12   6840 119S 120G C   N     904  906 -114.20  -80.20   43.72    3.16   82.20 -145.90   16.62\n",
      "   12        120G 120G N   CA    906  907  146.61  174.10                    8.50\n",
      "   13   6841 120G 121G C   N     908  910   98.07   78.70   19.46    0.93   82.20  177.27    7.80\n",
      "   13        121G 121G N   CA    910  911 -168.05 -166.10                    8.50\n",
      "   14   6842 121G 122G C   N     912  914 -174.30 -167.20   15.08    0.61   82.20 -175.46   13.45\n",
      "   14        122G 122G N   CA    914  915  161.29  174.60                    8.50\n",
      "   15   6844 123G 124S C   N     920  922 -109.52 -136.60   41.18    1.38  -64.10  161.69    9.99\n",
      "   15        124S 124S N   CA    922  923  120.18  151.20                  -35.00\n",
      "   16   6846 125G 126G C   N     930  932 -139.14 -167.20   28.44    0.45   82.20 -147.16   16.06\n",
      "   16        126G 126G N   CA    932  933  169.98  174.60                    8.50\n",
      "   17   6847 126G 127G C   N     934  936 -114.63  -80.20   47.38    3.37   82.20 -149.47   16.37\n",
      "   17        127G 127G N   CA    936  937  141.55  174.10                    8.50\n",
      "   18   6848 127G 128G C   N     938  940 -156.63 -167.20   10.85    0.16   82.20 -156.37   15.05\n",
      "   18        128G 128G N   CA    940  941  172.16  174.60                    8.50\n",
      "   19   6849 128G 129S C   N     942  944  -86.51  -72.40   30.93    1.43  -64.10  146.80   12.11\n",
      "   19        129S 129S N   CA    944  945  179.92  152.40                  -35.00\n",
      "   20   6850 129S 130G C   N     948  950 -140.57 -167.20   29.69    0.92   82.20 -134.27   16.77\n",
      "   20        130G 130G N   CA    950  951 -172.26  174.60                    8.50\n",
      "   21   6852 131G 132G C   N     956  958 -124.75 -167.20   73.51    1.97   82.20 -173.79   14.55\n",
      "   21        132G 132G N   CA    958  959  114.58  174.60                    8.50\n",
      "   22   6853 132G 133G C   N     960  962  -65.59  -62.40    6.24    1.10   82.20  157.72   12.06\n",
      "   22        133G 133G N   CA    962  963  -46.56  -41.20                    8.50\n",
      "   23   6854 133G 134S C   N     964  966  102.44 -136.60  122.57    4.37  -64.10 -133.17   18.99\n",
      "   23        134S 134S N   CA    966  967  170.99  151.20                  -35.00\n",
      "   24   6855 134S 135T C   N     970  972 -122.63 -124.80   43.09    1.91  -63.20  144.19   22.38\n",
      "   24        135T 135T N   CA    972  973 -173.47  143.50                  -42.10\n",
      "   25   6856 135T 136D C   N     977  979  -55.06  -63.30   19.16    2.04  -70.90  153.23    8.35\n",
      "   25        136D 136D N   CA    979  980  -57.29  -40.00                  150.30\n",
      "   26   6884 163D 164V C   N    1180 1182 -127.68  -62.40  100.60   11.32  -62.40  100.60   11.32\n",
      "   26        164V 164V N   CA   1182 1183   34.15  -42.40                  -42.40\n",
      "   27   6885 164V 165N C   N    1187 1189   51.52  -71.20  140.71   11.34 -119.90 -173.04    9.42\n",
      "   27        165N 165N N   CA   1189 1190 -148.36  142.80                  137.00\n",
      "   28   6907 186A 187S C   N    1360 1362 -148.15  -64.10   95.51    9.71  -64.10   95.51    9.71\n",
      "   28        187S 187S N   CA   1362 1363   10.36  -35.00                  -35.00\n",
      "   29   6962 241I 242K C   N    1813 1815  -97.26 -118.00   54.53    2.29  -62.90  133.96   15.86\n",
      "   29        242K 242K N   CA   1815 1816   88.68  139.10                  -40.80\n",
      "   30   6963 242K 243H C   N    1822 1824 -163.33 -125.60   45.25    2.00  -63.20 -174.51   19.78\n",
      "   30        243H 243H N   CA   1824 1825  113.84  138.80                  -42.30\n",
      "   31   6964 243H 244H C   N    1832 1834 -147.22 -125.60   48.54    1.40  -63.20  159.38   25.32\n",
      "   31        244H 244H N   CA   1834 1835 -177.74  138.80                  -42.30\n",
      "   32   6965 244H 245H C   N    1842 1844  -65.07  -67.60    2.66    0.22  -63.20  176.88   22.23\n",
      "   32        245H 245H N   CA   1844 1845  140.83  140.00                  -42.30\n",
      "   33   6966 245H 246H C   N    1852 1854 -155.18 -125.60   46.56    1.19  -63.20  169.98   27.13\n",
      "   33        246H 246H N   CA   1854 1855  174.75  138.80                  -42.30\n",
      "   34   6967 246H 247H C   N    1862 1864  -76.58  -67.60   10.60    0.93  -63.20  177.19   21.35\n",
      "   34        247H 247H N   CA   1864 1865  134.38  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   12   27   89   92  127  162  160  241  269  301\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.B99990003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           $(LIB)/atmcls-melo.lib\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17346114   16939.564    16.543\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17346178   16939.627    16.543\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17346242   16939.689    16.543\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17346370   16939.814    16.543\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17346562   16940.002    16.543\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17346818   16940.252    16.543\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17347202   16940.627    16.544\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17347778   16941.189    16.544\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17347828   16941.238    16.544\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17347886   16941.295    16.544\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17348782   16942.170    16.545\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17350126   16943.482    16.546\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17350192   16943.547    16.546\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17352176   16945.484    16.548\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17352332   16945.637    16.548\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17355340   16948.574    16.551\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17355634   16948.861    16.552\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17356138   16949.354    16.552\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17357134   16950.326    16.553\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17361614   16954.701    16.557\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17363648   16956.688    16.559\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17368100   16961.035    16.564\n",
      "openf___224_> Open           $(LIB)/melo1-dist.lib\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17368496   16961.422    16.564\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17369292   16962.199    16.565\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17370088   16962.977    16.565\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17371276   16964.137    16.567\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17371676   16964.527    16.567\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17372276   16965.113    16.567\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17373176   16965.992    16.568\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17374524   16967.309    16.570\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17376548   16969.285    16.572\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17376748   16969.480    16.572\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17379784   16972.445    16.575\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17380084   16972.738    16.575\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17384636   16977.184    16.579\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17385084   16977.621    16.580\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17391912   16984.289    16.586\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17392584   16984.945    16.587\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17402828   16994.949    16.597\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17403836   16995.934    16.598\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17404436   16996.520    16.598\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17404836   16996.910    16.599\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17420200   17011.914    16.613\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17421712   17013.391    16.615\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17422612   17014.270    16.615\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17423212   17014.855    16.616\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17446260   17037.363    16.638\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17448528   17039.578    16.640\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17449872   17040.891    16.641\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17450772   17041.770    16.642\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17485344   17075.531    16.675\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17488748   17078.855    16.679\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17490764   17080.824    16.680\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17492112   17082.141    16.682\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17543968   17132.781    16.731\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17549072   17137.766    16.736\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17552096   17140.719    16.739\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17554120   17142.695    16.741\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17631904   17218.656    16.815\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17639560   17226.133    16.822\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17644096   17230.562    16.827\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17647132   17233.527    16.830\n",
      "\n",
      "Dynamically allocated memory at amaxgroup_restr [B,KiB,MiB]:     17763808   17347.469    16.941\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                                0       0       0       0     820\n",
      "openf___224_> Open           scfv.B99990001.pdb\n",
      "readlinef__W> File: scfv.B99990001.pdb, Line: 6\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     17763808   17347.469    16.941\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17771552   17355.031    16.948\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     17913244   17493.402    17.083\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     17915652   17495.754    17.086\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18057344   17634.125    17.221\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933      0         0       876\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933   2622      3114       876\n",
      "\n",
      "patch_______> segment topology patched using RTF:   248  ;  HIS  ;  CTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "openf___224_> Open           scfv.B99990001.pdb\n",
      "readlinef__W> File: scfv.B99990001.pdb, Line: 6\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18068317   17644.840    17.231\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18068393   17644.914    17.231\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18068564   17645.082    17.232\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18069414   17645.912    17.232\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18070672   17647.141    17.234\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18072576   17649.000    17.235\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18072616   17649.039    17.235\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18072706   17649.127    17.235\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18075562   17651.916    17.238\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18075622   17651.975    17.238\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18075757   17652.105    17.238\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18080041   17656.289    17.242\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18080129   17656.375    17.243\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18080327   17656.570    17.243\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18086753   17662.844    17.249\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18086885   17662.973    17.249\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18087182   17663.264    17.249\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18096804   17672.660    17.258\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18097004   17672.855    17.259\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18097454   17673.295    17.259\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18111904   17687.406    17.273\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18112204   17687.699    17.273\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18112879   17688.359    17.274\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18134537   17709.508    17.294\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18134985   17709.945    17.295\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18135993   17710.930    17.296\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18306681   17877.617    17.459\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  scfv\n",
      "  2  scfv\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1884        0\n",
      "openf___224_> Open           scfv.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18059346   17636.080    17.223\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18060796   17637.496    17.224\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18063696   17640.328    17.227\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18069496   17645.992    17.232\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18186552   17760.305    17.344\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18187456   17761.188    17.345\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18188444   17762.152    17.346\n",
      "\n",
      "Read the alignment from file       : scfv.ali\n",
      "Total number of alignment positions:   253\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1     2fjhHL     227      1  2fjhHL.pdb\n",
      "  2       scfv     248      1        scfv\n",
      "132 atoms selected for loop refinement\n",
      "make_re_417_> Restraint type to be calculated:  bond\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      141       0       0       0\n",
      "              Total number of restraints before, now         :        0      141\n",
      "make_re_422_> Number of previous, current restraints         :        0      141\n",
      "make_re_423_> Number of previous, current selected restraints:        0      141\n",
      "make_re_417_> Restraint type to be calculated:  angle\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :        0     194       0       0\n",
      "              Total number of restraints before, now         :      141      335\n",
      "make_re_422_> Number of previous, current restraints         :      141      335\n",
      "make_re_423_> Number of previous, current selected restraints:      141      335\n",
      "make_re_417_> Restraint type to be calculated:  improper\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :        0       0       0      81\n",
      "              Total number of restraints before, now         :      335      416\n",
      "make_re_422_> Number of previous, current restraints         :      335      416\n",
      "make_re_423_> Number of previous, current selected restraints:      335      416\n",
      "make_re_417_> Restraint type to be calculated:  dihedral\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :        0       0     235       0\n",
      "              Total number of restraints before, now         :      416      651\n",
      "make_re_422_> Number of previous, current restraints         :      416      651\n",
      "make_re_423_> Number of previous, current selected restraints:      416      651\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18057344   17634.125    17.221\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      651      670\n",
      "make_re_423_> Number of previous, current selected restraints:      651      670\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18057360   17634.141    17.221\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      670      687\n",
      "make_re_423_> Number of previous, current selected restraints:      670      670\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18057360   17634.141    17.221\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      687      702\n",
      "make_re_423_> Number of previous, current selected restraints:      670      685\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18057360   17634.141    17.221\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      702      713\n",
      "make_re_423_> Number of previous, current selected restraints:      685      696\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18057360   17634.141    17.221\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      713      716\n",
      "make_re_423_> Number of previous, current selected restraints:      696      699\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18057360   17634.141    17.221\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      716      717\n",
      "make_re_423_> Number of previous, current selected restraints:      699      700\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;      132      717\n",
      "rmdupl__427_>      103 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:      717      614\n",
      "openf___224_> Open           scfv.lrsr\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4401\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         351.9498\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       6      7   0.051   0.051      110.33       1.000\n",
      " 2 Bond angle potential               :     194       4     10   4.644   4.644      85.946       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      5  44.976  44.976      38.151       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.145   1.145      2.2546       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4401       0      0   0.002   0.002      1.1885       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      0   5.699   5.699      6.5166       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0  80.423  80.423      3.9215       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  96.766  96.766      5.2741       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 122.148 122.148      3.8646       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 107.859 107.859      1.1037       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      2  32.863 152.604      22.625       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4401       0     19   1.924   1.924      70.775       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     45  98Q  98Q N   CA    735  736    1.58    1.43    0.15    5.06    1.43    0.15    5.06\n",
      "    2     47  98Q  98Q C   CA    742  736    1.79    1.49    0.30    8.68    1.49    0.30    8.68\n",
      "    3     48  98Q  99E C   N     742  744    1.59    1.35    0.25    8.70    1.35    0.25    8.70\n",
      "    4     54  99E  99E N   CA    744  745    1.67    1.43    0.24    7.88    1.43    0.24    7.88\n",
      "    5     56  99E  99E C   CA    751  745    1.75    1.49    0.26    7.54    1.49    0.26    7.54\n",
      "    6     57  99E 100T C   N     751  753    1.56    1.35    0.21    7.53    1.35    0.21    7.53\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  127.41  107.00   20.41    5.86  107.00   20.41    5.86\n",
      "    2    202  98Q  98Q CA  C     736  742  132.73  116.50   16.23    4.66  116.50   16.23    4.66\n",
      "    3    208  98Q  99E C   N     742  744  151.51  120.00   31.51    7.16  120.00   31.51    7.16\n",
      "    4    211  99E  99E N   CA    744  745  136.20  107.00   29.20    8.39  107.00   29.20    8.39\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   58.96    1.83  -64.10 -164.71   23.22\n",
      "    1         51S  51S N   CA    376  377  175.85  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -137.67 -136.60    8.63    0.41  -64.10 -179.13   18.35\n",
      "    2         52S  52S N   CA    382  383  159.77  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -106.78  -98.40   39.64    4.09  -63.50  139.92   20.20\n",
      "    3         53Y  53Y N   CA    388  389   89.66  128.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -156.46 -136.60   32.66    1.85  -64.10 -175.03   12.17\n",
      "    4         55S  55S N   CA    406  407  125.26  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -122.86 -124.30   12.73    0.74  -63.50  178.70   31.35\n",
      "    5         97Y  97Y N   CA    723  724  148.05  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -152.90 -121.10   58.70    2.12  -63.80  158.15   27.73\n",
      "    6         98Q  98Q N   CA    735  736 -170.96  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744  -98.05  -69.30   77.99    4.92  -63.60  110.23   16.65\n",
      "    7         99E  99E N   CA    744  745 -145.01  142.50                  -40.30\n",
      "    8    559  99E 100T C   N     751  753 -153.53 -124.80   43.67    1.38  -63.20  167.89   27.25\n",
      "    8        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -119.65 -117.80    9.80    0.45  -63.60 -177.88   27.45\n",
      "    9        240E 240E N   CA   1798 1799  146.43  136.80                  -40.30\n",
      "   10    562 241I 242K C   N    1813 1815  -86.88  -70.20   24.43    2.04  -62.90  165.10   20.25\n",
      "   10        242K 242K N   CA   1815 1816  122.55  140.40                  -40.80\n",
      "   11    563 242K 243H C   N    1822 1824 -143.75 -125.60   19.46    0.81  -63.20 -168.18   20.48\n",
      "   11        243H 243H N   CA   1824 1825  131.79  138.80                  -42.30\n",
      "   12    564 243H 244H C   N    1832 1834 -130.30 -125.60    7.56    0.34  -63.20 -172.41   20.26\n",
      "   12        244H 244H N   CA   1834 1835  132.88  138.80                  -42.30\n",
      "   13    565 244H 245H C   N    1842 1844 -119.33 -125.60   31.18    1.06  -63.20  160.68   17.39\n",
      "   13        245H 245H N   CA   1844 1845  108.26  138.80                  -42.30\n",
      "   14    566 245H 246H C   N    1852 1854 -118.36 -125.60   19.14    0.58  -63.20  172.44   18.81\n",
      "   14        246H 246H N   CA   1854 1855  121.08  138.80                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864  -77.62  -67.60   10.02    0.75  -63.20  178.01   23.17\n",
      "   15        247H 247H N   CA   1864 1865  140.28  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   14   27   95  101  150  163  181  213  236  316\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18349996   17919.918    17.500\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4476\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         325.1158\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.049   0.049      104.60       1.000\n",
      " 2 Bond angle potential               :     194       4     11   5.169   5.169      105.46       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  43.363  43.363      40.165       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      2   2.628   2.628      12.569       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4476       0      0   0.001   0.001     0.87514       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      4   9.621   9.621      18.572       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      3  96.849  96.849      16.804       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      1  90.746  90.746      8.8188       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 114.945 114.945      3.3865       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 130.696 130.696      1.1221       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      17      6  65.386 160.342      65.932       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4476       0      7   1.926   1.926     -53.187       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.17    5.87    1.35    0.17    5.87\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.11    1.43    0.19    6.11\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.56    1.49    0.26    7.56\n",
      "    4     48  98Q  99E C   N     742  744    1.52    1.35    0.18    6.31    1.35    0.18    6.31\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.24    1.43    0.22    7.24\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.73    1.49    0.27    7.73\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.22    7.72    1.35    0.22    7.72\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  129.05  107.00   22.05    6.34  107.00   22.05    6.34\n",
      "    2    208  98Q  99E C   N     742  744  143.41  120.00   23.41    5.32  120.00   23.41    5.32\n",
      "    3    211  99E  99E N   CA    744  745  138.57  107.00   31.57    9.07  107.00   31.57    9.07\n",
      "    4    214  99E  99E CA  C     745  751  134.27  116.50   17.77    5.11  116.50   17.77    5.11\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   54.36    1.93  -64.10 -152.75   24.11\n",
      "    1         51S  51S N   CA    376  377  160.49  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -166.88 -136.60   39.78    1.24  -64.10 -179.82   20.47\n",
      "    2         52S  52S N   CA    382  383  177.01  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -145.68 -124.30   48.06    1.90  -63.50  160.76   29.43\n",
      "    3         53Y  53Y N   CA    388  389  178.44  135.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400 -120.60 -136.60   23.71    0.78  -64.10  177.92   10.96\n",
      "    4         54S  54S N   CA    400  401  133.71  151.20                  -35.00\n",
      "    5    553  54S  55S C   N     404  406  101.87 -136.60  168.51    5.39  -64.10  175.51   19.71\n",
      "    5         55S  55S N   CA    406  407  -92.07  151.20                  -35.00\n",
      "    6    554  55S  56S C   N     410  412   98.65  -64.10  162.88   21.09  -64.10  162.88   21.09\n",
      "    6         56S  56S N   CA    412  413  -28.43  -35.00                  -35.00\n",
      "    7    556  96V  97Y C   N     721  723 -122.86 -124.30   12.43    0.72  -63.50  178.98   31.40\n",
      "    7         97Y  97Y N   CA    723  724  147.75  135.40                  -43.40\n",
      "    8    557  97Y  98Q C   N     733  735 -165.37 -121.10   60.08    1.92  -63.80  172.47   30.40\n",
      "    8         98Q  98Q N   CA    735  736 -179.68  139.70                  -40.30\n",
      "    9    558  98Q  99E C   N     742  744  -93.72  -69.30   90.04    5.99  -63.60   95.42   14.43\n",
      "    9         99E  99E N   CA    744  745 -130.84  142.50                  -40.30\n",
      "   10    559  99E 100T C   N     751  753 -154.71 -124.80   44.46    1.41  -63.20  168.53   27.40\n",
      "   10        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "   11    560 239V 240E C   N    1796 1798 -119.65 -117.80    4.20    0.17  -63.60 -172.30   28.20\n",
      "   11        240E 240E N   CA   1798 1799  140.57  136.80                  -40.30\n",
      "   12    561 240E 241I C   N    1805 1807  -86.44  -63.40   25.83    4.87 -120.60  177.74   11.47\n",
      "   12        241I 241I N   CA   1807 1808  -55.27  -43.60                  130.30\n",
      "   13    562 241I 242K C   N    1813 1815 -129.05 -118.00   34.63    1.50  -62.90  161.45   24.93\n",
      "   13        242K 242K N   CA   1815 1816  171.92  139.10                  -40.80\n",
      "   14    564 243H 244H C   N    1832 1834  -73.22  -67.60    7.77    0.70  -63.20  177.21   21.54\n",
      "   14        244H 244H N   CA   1834 1835  134.63  140.00                  -42.30\n",
      "   15    565 244H 245H C   N    1842 1844 -108.75 -125.60   41.05    1.81  -63.20  148.62   21.56\n",
      "   15        245H 245H N   CA   1844 1845  176.23  138.80                  -42.30\n",
      "   16    566 245H 246H C   N    1852 1854  -86.48  -67.60   23.56    1.41  -63.20  165.25   22.16\n",
      "   16        246H 246H N   CA   1854 1855  154.10  140.00                  -42.30\n",
      "   17    567 246H 247H C   N    1862 1864  -68.68  -67.60   24.33    1.90  -63.20  158.09   19.43\n",
      "   17        247H 247H N   CA   1864 1865  115.70  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   17   22   97   97  144  164  186  213  247  321\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.BL00010001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18349996   17919.918    17.500\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4869\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         213.9255\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.049   0.049      104.95       1.000\n",
      " 2 Bond angle potential               :     194       4     11   5.016   5.016      99.432       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      7  45.106  45.106      42.240       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   2.000   2.000      7.2109       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4869       0      0   0.002   0.002      1.6056       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      2   8.074   8.074      13.081       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0  86.628  86.628      7.4002       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  90.819  90.819      10.882       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  78.771  78.771      2.7302       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.514   0.514     0.22388       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      6  44.558 144.034      33.588       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4869       0      8   1.886   1.886     -109.41       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.50    1.35    0.16    5.63    1.35    0.16    5.63\n",
      "    2     45  98Q  98Q N   CA    735  736    1.61    1.43    0.18    5.78    1.43    0.18    5.78\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.68    1.49    0.26    7.68\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.71    1.35    0.19    6.71\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.31    1.43    0.22    7.31\n",
      "    6     56  99E  99E C   CA    751  745    1.77    1.49    0.28    8.02    1.49    0.28    8.02\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.21    7.46    1.35    0.21    7.46\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  129.79  107.00   22.79    6.55  107.00   22.79    6.55\n",
      "    2    208  98Q  99E C   N     742  744  139.81  120.00   19.81    4.50  120.00   19.81    4.50\n",
      "    3    211  99E  99E N   CA    744  745  135.42  107.00   28.42    8.17  107.00   28.42    8.17\n",
      "    4    214  99E  99E CA  C     745  751  134.77  116.50   18.27    5.25  116.50   18.27    5.25\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   56.99    1.83  -64.10 -160.73   23.52\n",
      "    1         51S  51S N   CA    376  377  170.67  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -157.59 -136.60   37.46    1.35  -64.10  170.66   19.11\n",
      "    2         52S  52S N   CA    382  383 -177.77  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -117.54 -124.30   27.09    1.73  -63.50  164.12   28.77\n",
      "    3         53Y  53Y N   CA    388  389  161.63  135.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -154.62 -136.60   19.98    1.01  -64.10 -160.67   12.74\n",
      "    4         55S  55S N   CA    406  407  142.59  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -122.86 -124.30   16.55    0.94  -63.50  175.08   30.77\n",
      "    5         97Y  97Y N   CA    723  724  151.89  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -152.68 -121.10   62.11    2.29  -63.80  154.62   27.17\n",
      "    6         98Q  98Q N   CA    735  736 -166.82  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -126.10 -117.80   93.98    4.56  -63.60  108.99   17.92\n",
      "    7         99E  99E N   CA    744  745 -129.58  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753 -156.83 -124.80   45.90    1.48  -63.20  169.68   27.66\n",
      "    8        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -119.65 -117.80   19.76    0.96  -63.60  172.58   26.18\n",
      "    9        240E 240E N   CA   1798 1799  156.47  136.80                  -40.30\n",
      "   10    562 241I 242K C   N    1813 1815  -97.65 -118.00   26.14    0.89  -62.90  167.15   20.07\n",
      "   10        242K 242K N   CA   1815 1816  122.70  139.10                  -40.80\n",
      "   11    563 242K 243H C   N    1822 1824 -122.99 -125.60   58.05    2.26  -63.20  134.88   20.75\n",
      "   11        243H 243H N   CA   1824 1825 -163.21  138.80                  -42.30\n",
      "   12    564 243H 244H C   N    1832 1834 -129.77 -125.60   13.63    0.59  -63.20 -179.18   19.49\n",
      "   12        244H 244H N   CA   1834 1835  125.82  138.80                  -42.30\n",
      "   13    565 244H 245H C   N    1842 1844   51.21   56.30   24.26    1.82  -63.20  156.53   26.41\n",
      "   13        245H 245H N   CA   1844 1845   64.52   40.80                  -42.30\n",
      "   14    566 245H 246H C   N    1852 1854 -133.85 -125.60   70.93    2.53  -63.20  129.43   20.70\n",
      "   14        246H 246H N   CA   1854 1855 -150.75  138.80                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864  -71.66  -67.60   73.68    5.79  -63.20  109.06   13.12\n",
      "   15        247H 247H N   CA   1864 1865   66.43  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   17   23   98   97  149  164  188  218  245  305\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:      166.9966       99    99\n",
      "openf___224_> Open           scfv.BL00020001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18349996   17919.918    17.500\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4946\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         245.9939\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      105.96       1.000\n",
      " 2 Bond angle potential               :     194       4     14   5.380   5.380      113.76       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      5  37.175  37.175      34.468       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   2.310   2.310      9.1802       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4946       0      0   0.001   0.001     0.30031       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      2   8.644   8.644      14.992       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2  91.713  91.713      17.550       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0 121.064 121.064      10.163       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  94.032  94.032      1.9630       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  93.361  93.361      1.1710       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      9  54.688 142.899      52.868       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4946       0      7   1.914   1.914     -116.37       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.17    5.94    1.35    0.17    5.94\n",
      "    2     45  98Q  98Q N   CA    735  736    1.61    1.43    0.18    6.03    1.43    0.18    6.03\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.63    1.49    0.26    7.63\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.47    1.35    0.18    6.47\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.22    1.43    0.22    7.22\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.89    1.49    0.27    7.89\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.21    7.47    1.35    0.21    7.47\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  129.83  107.00   22.83    6.56  107.00   22.83    6.56\n",
      "    2    208  98Q  99E C   N     742  744  140.17  120.00   20.17    4.58  120.00   20.17    4.58\n",
      "    3    211  99E  99E N   CA    744  745  136.65  107.00   29.65    8.52  107.00   29.65    8.52\n",
      "    4    214  99E  99E CA  C     745  751  134.09  116.50   17.59    5.06  116.50   17.59    5.06\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   55.46    1.86  -64.10 -156.78   23.81\n",
      "    1         51S  51S N   CA    376  377  165.60  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  145.23 -136.60   78.34    2.98  -64.10 -133.87   27.21\n",
      "    2         52S  52S N   CA    382  383  156.39  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -163.53  -63.50  108.60   20.28  -63.50  108.60   20.28\n",
      "    3         53Y  53Y N   CA    388  389  -85.69  -43.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -84.94  -72.40   45.08    2.25  -64.10  130.97   10.87\n",
      "    4         54S  54S N   CA    400  401 -164.30  152.40                  -35.00\n",
      "    5    553  54S  55S C   N     404  406 -110.76 -136.60   28.38    1.43  -64.10  168.66   15.45\n",
      "    5         55S  55S N   CA    406  407  162.92  151.20                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -122.86 -124.30   14.95    0.85  -63.50  176.59   31.02\n",
      "    6         97Y  97Y N   CA    723  724  150.28  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -157.68 -121.10   67.07    2.42  -63.80  155.36   27.47\n",
      "    7         98Q  98Q N   CA    735  736 -164.09  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -116.23 -117.80   92.40    4.67  -63.60  104.70   16.87\n",
      "    8         99E  99E N   CA    744  745 -130.82  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753 -155.52 -124.80   45.00    1.43  -63.20  168.97   27.50\n",
      "    9        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.65 -117.80    5.81    0.25  -63.60 -173.96   27.98\n",
      "   10        240E 240E N   CA   1798 1799  142.31  136.80                  -40.30\n",
      "   11    561 240E 241I C   N    1805 1807  -85.11  -63.40   21.99    3.80 -120.60  174.02    8.60\n",
      "   11        241I 241I N   CA   1807 1808  -40.06  -43.60                  130.30\n",
      "   12    563 242K 243H C   N    1822 1824 -121.65 -125.60   73.85    2.88  -63.20  120.31   18.82\n",
      "   12        243H 243H N   CA   1824 1825 -147.46  138.80                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834  -18.09  -67.60   66.46    3.94  -63.20  145.15   21.10\n",
      "   13        244H 244H N   CA   1834 1835   95.67  140.00                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844  -92.41  -67.60   26.59    1.71  -63.20  170.64   23.22\n",
      "   14        245H 245H N   CA   1844 1845  149.58  140.00                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864 -118.17 -125.60   78.08    3.10  -63.20  115.15   17.95\n",
      "   15        247H 247H N   CA   1864 1865 -143.48  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   19   22   98  100  149  170  186  222  246  311\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -104.5336       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.2109       99    99\n",
      "openf___224_> Open           scfv.BL00030001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5278\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1172.7306\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       9     10   0.059   0.059      142.64       1.000\n",
      " 2 Bond angle potential               :     194      10     24   8.317   8.317      219.90       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0     10  52.714  52.714      53.904       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       1      6   4.099   4.099      34.796       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5278      20     28   0.029   0.029      529.89       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      4  10.608  10.608      22.579       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      3  95.901  95.901      14.757       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0 110.446 110.446      7.3457       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 136.951 136.951      3.4611       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.034   0.034     0.21532       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      8  54.484 145.026      48.321       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5278       0     49   1.922   1.922      94.915       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1      9  53Y  53Y CG  CB    391  390    1.79    1.49    0.30    8.28    1.49    0.30    8.28\n",
      "    2     39  97Y  98Q C   N     733  735    1.50    1.35    0.15    5.39    1.35    0.15    5.39\n",
      "    3     45  98Q  98Q N   CA    735  736    1.59    1.43    0.16    5.22    1.43    0.16    5.22\n",
      "    4     47  98Q  98Q C   CA    742  736    1.74    1.49    0.25    7.38    1.49    0.25    7.38\n",
      "    5     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.42    1.35    0.18    6.42\n",
      "    6     54  99E  99E N   CA    744  745    1.64    1.43    0.21    6.80    1.43    0.21    6.80\n",
      "    7     56  99E  99E C   CA    751  745    1.79    1.49    0.30    8.67    1.49    0.30    8.67\n",
      "    8     57  99E 100T C   N     751  753    1.57    1.35    0.22    7.81    1.35    0.22    7.81\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    143  51S  52S C   N     380  382  141.70  120.00   21.70    4.93  120.00   21.70    4.93\n",
      "    2    159  53Y  53Y CB  CG    390  391  147.71  122.30   25.41    5.53  122.30   25.41    5.53\n",
      "    3    160  53Y  53Y CG  CD1   391  392  152.68  120.00   32.68    6.64  120.00   32.68    6.64\n",
      "    4    164  53Y  53Y CD2 CE2   393  395  154.82  120.00   34.82    7.08  120.00   34.82    7.08\n",
      "    5    167  53Y  53Y CE2 CZ    395  396  143.44  120.00   23.44    5.06  120.00   23.44    5.06\n",
      "    6    199  98Q  98Q N   CA    735  736  131.55  107.00   24.55    7.06  107.00   24.55    7.06\n",
      "    7    211  99E  99E N   CA    744  745  136.22  107.00   29.22    8.40  107.00   29.22    8.40\n",
      "    8    214  99E  99E CA  C     745  751  138.00  116.50   21.50    6.18  116.50   21.50    6.18\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    336  51S  51S C   CA    380  377   15.20    0.00   15.20    5.35    0.00   15.20    5.35\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    569  52S  52S CA  C     383  386 -152.12  180.00   27.88    5.58  180.00   27.88    5.58\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   53.79    2.26  -64.10 -141.26   24.95\n",
      "    1         51S  51S N   CA    376  377  146.24  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  -66.22  -72.40   73.30    4.10  -64.10  114.38    8.17\n",
      "    2         52S  52S N   CA    382  383   79.37  152.40                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388  -79.92  -63.50   86.51   12.93  -63.50   86.51   12.93\n",
      "    3         53Y  53Y N   CA    388  389   41.54  -43.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  105.43 -136.60  119.38    5.22  -64.10 -121.37   29.41\n",
      "    4         54S  54S N   CA    400  401  132.95  151.20                  -35.00\n",
      "    5    553  54S  55S C   N     404  406 -108.04 -136.60   67.79    2.72  -64.10  132.23    8.15\n",
      "    5         55S  55S N   CA    406  407   89.72  151.20                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -122.86 -124.30   16.09    0.91  -63.50  175.51   30.84\n",
      "    6         97Y  97Y N   CA    723  724  151.43  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -151.40 -121.10   63.30    2.38  -63.80  152.17   26.75\n",
      "    7         98Q  98Q N   CA    735  736 -164.73  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -137.74 -117.80  102.27    4.73  -63.60  110.99   18.67\n",
      "    8         99E  99E N   CA    744  745 -122.89  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753 -157.33 -124.80   46.26    1.49  -63.20  169.96   27.72\n",
      "    9        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.65 -117.80   10.02    0.47  -63.60 -178.09   27.43\n",
      "   10        240E 240E N   CA   1798 1799  146.64  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815  -42.49  -70.20   33.24    2.03  -62.90  164.13   22.49\n",
      "   11        242K 242K N   CA   1815 1816  122.05  140.40                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824   49.02   56.30   12.78    1.03  -63.20  146.13   24.84\n",
      "   12        243H 243H N   CA   1824 1825   51.31   40.80                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834 -113.41 -125.60   27.01    0.77  -63.20  164.83   18.07\n",
      "   13        244H 244H N   CA   1834 1835  114.70  138.80                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844 -112.53 -125.60   46.79    1.99  -63.20  142.77   21.08\n",
      "   14        245H 245H N   CA   1844 1845 -176.28  138.80                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864 -114.51 -125.60   13.10    0.58  -63.20  179.41   25.79\n",
      "   15        247H 247H N   CA   1864 1865  145.78  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   373  395   2.208\n",
      "  serious non-bonded atom clash:   379  395   2.214\n",
      "  serious non-bonded atom clash:   394  628   2.219\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    3    4    7   23   28  111  107  152  165  194  229  253  320\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.BL00040001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4578\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1338.1558\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.052   0.052      117.25       1.000\n",
      " 2 Bond angle potential               :     194      11     19  12.810  12.810      1139.8       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      7  44.503  44.503      39.704       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      2   2.212   2.212      8.3880       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4578       0      1   0.003   0.003      5.2709       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      2   7.911   7.911      12.556       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0  80.808  80.808      7.9183       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0 106.505 106.505      9.0939       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  69.940  69.940      2.7463       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   9.267   9.267     0.27938       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      17      9  57.218 149.441      55.230       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4578       0     14   1.898   1.898     -60.084       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.53    1.35    0.18    6.46    1.35    0.18    6.46\n",
      "    2     45  98Q  98Q N   CA    735  736    1.64    1.43    0.21    6.90    1.43    0.21    6.90\n",
      "    3     47  98Q  98Q C   CA    742  736    1.74    1.49    0.25    7.35    1.49    0.25    7.35\n",
      "    4     48  98Q  99E C   N     742  744    1.55    1.35    0.20    7.10    1.35    0.20    7.10\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.15    1.43    0.22    7.15\n",
      "    6     56  99E  99E C   CA    751  745    1.77    1.49    0.28    8.23    1.49    0.28    8.23\n",
      "    7     57  99E 100T C   N     751  753    1.57    1.35    0.23    8.00    1.35    0.23    8.00\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    193  56S  57Y C   N     416  418  144.95  120.00   24.95    5.67  120.00   24.95    5.67\n",
      "    2    199  98Q  98Q N   CA    735  736  133.85  107.00   26.85    7.72  107.00   26.85    7.72\n",
      "    3    208  98Q  99E C   N     742  744  141.90  120.00   21.90    4.98  120.00   21.90    4.98\n",
      "    4    211  99E  99E N   CA    744  745  131.77  107.00   24.77    7.12  107.00   24.77    7.12\n",
      "    5    214  99E  99E CA  C     745  751  134.81  116.50   18.31    5.26  116.50   18.31    5.26\n",
      "    6    220  99E 100T C   N     751  753  140.86  120.00   20.86    4.74  120.00   20.86    4.74\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   57.99    1.83  -64.10 -162.85   23.36\n",
      "    1         51S  51S N   CA    376  377  173.41  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -161.17 -136.60   44.64    1.62  -64.10  167.52   19.14\n",
      "    2         52S  52S N   CA    382  383 -171.53  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -119.24 -124.30   39.44    2.31  -63.50  152.62   27.01\n",
      "    3         53Y  53Y N   CA    388  389  174.52  135.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400 -129.57 -136.60   59.95    2.89  -64.10  142.58    9.14\n",
      "    4         54S  54S N   CA    400  401   91.66  151.20                  -35.00\n",
      "    5    553  54S  55S C   N     404  406   99.23 -136.60  129.37    4.29  -64.10 -146.50   18.57\n",
      "    5         55S  55S N   CA    406  407 -172.49  151.20                  -35.00\n",
      "    6    554  55S  56S C   N     410  412 -146.94  -64.10   86.23    9.94  -64.10   86.23    9.94\n",
      "    6         56S  56S N   CA    412  413  -11.03  -35.00                  -35.00\n",
      "    7    556  96V  97Y C   N     721  723 -122.86 -124.30   23.24    1.29  -63.50  168.79   29.77\n",
      "    7         97Y  97Y N   CA    723  724  158.59  135.40                  -43.40\n",
      "    8    557  97Y  98Q C   N     733  735 -145.05 -121.10   71.78    2.94  -63.80  138.64   24.43\n",
      "    8         98Q  98Q N   CA    735  736 -152.63  139.70                  -40.30\n",
      "    9    558  98Q  99E C   N     742  744 -133.75 -117.80   85.48    3.97  -63.60  121.27   19.97\n",
      "    9         99E  99E N   CA    744  745 -139.23  136.80                  -40.30\n",
      "   10    559  99E 100T C   N     751  753 -158.03 -124.80   46.75    1.52  -63.20  170.35   27.81\n",
      "   10        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "   11    560 239V 240E C   N    1796 1798 -119.65 -117.80   15.27    0.73  -63.60  176.86   26.75\n",
      "   11        240E 240E N   CA   1798 1799  151.96  136.80                  -40.30\n",
      "   12    562 241I 242K C   N    1813 1815  -58.91  -70.20   31.76    2.04  -62.90  151.57   19.91\n",
      "   12        242K 242K N   CA   1815 1816  110.72  140.40                  -40.80\n",
      "   13    563 242K 243H C   N    1822 1824 -121.64 -125.60   68.93    2.69  -63.20  124.63   19.37\n",
      "   13        243H 243H N   CA   1824 1825 -152.38  138.80                  -42.30\n",
      "   14    564 243H 244H C   N    1832 1834  -82.96  -67.60   22.35    1.33  -63.20  162.66   21.60\n",
      "   14        244H 244H N   CA   1834 1835  156.24  140.00                  -42.30\n",
      "   15    565 244H 245H C   N    1842 1844  -62.59  -67.60   21.33    1.49  -63.20  161.57   20.23\n",
      "   15        245H 245H N   CA   1844 1845  119.27  140.00                  -42.30\n",
      "   16    566 245H 246H C   N    1852 1854 -108.30 -125.60   20.35    0.89  -63.20  174.12   24.71\n",
      "   16        246H 246H N   CA   1854 1855  149.52  138.80                  -42.30\n",
      "   17    567 246H 247H C   N    1862 1864  -79.59  -67.60   60.71    5.00  -63.20  123.87   14.53\n",
      "   17        247H 247H N   CA   1864 1865   80.48  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   18   23   99  101  150  161  187  218  252  318\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -108.4668       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      160.5058       99    99\n",
      "openf___224_> Open           scfv.BL00050001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5147\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         353.3295\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      107.41       1.000\n",
      " 2 Bond angle potential               :     194       5     14   5.584   5.584      122.27       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      5  41.060  41.060      35.826       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      2   2.901   2.901      15.957       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5147       0      0   0.003   0.003      6.3709       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      2   8.806   8.806      15.557       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      3 105.519 105.519      19.766       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      1 102.409 102.409      14.062       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  90.385  90.385      3.3107       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 118.903 118.903     0.95300       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      14      7  54.321 144.973      51.388       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5147       0      6   1.888   1.888     -39.538       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.16    5.80    1.35    0.16    5.80\n",
      "    2     45  98Q  98Q N   CA    735  736    1.61    1.43    0.18    6.08    1.43    0.18    6.08\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.46    1.49    0.26    7.46\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.46    1.35    0.18    6.46\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.31    1.43    0.22    7.31\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.88    1.49    0.27    7.88\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.21    7.61    1.35    0.21    7.61\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    155  53Y  53Y CA  CB    389  390  127.26  107.50   19.76    4.57  107.50   19.76    4.57\n",
      "    2    199  98Q  98Q N   CA    735  736  129.03  107.00   22.03    6.33  107.00   22.03    6.33\n",
      "    3    208  98Q  99E C   N     742  744  142.08  120.00   22.08    5.02  120.00   22.08    5.02\n",
      "    4    211  99E  99E N   CA    744  745  136.99  107.00   29.99    8.62  107.00   29.99    8.62\n",
      "    5    214  99E  99E CA  C     745  751  134.43  116.50   17.93    5.15  116.50   17.93    5.15\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   53.61    2.07  -64.10 -147.08   24.53\n",
      "    1         51S  51S N   CA    376  377  153.40  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  -95.77  -64.10   88.25    8.63  -64.10   88.25    8.63\n",
      "    2         52S  52S N   CA    382  383 -117.37  -35.00                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388  128.87 -124.30  108.67    8.68  -63.50 -129.03   32.23\n",
      "    3         53Y  53Y N   CA    388  389  115.49  135.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406  -59.06  -72.40   15.21    0.83  -64.10  179.98   12.75\n",
      "    4         55S  55S N   CA    406  407  145.10  152.40                  -35.00\n",
      "    5    554  55S  56S C   N     410  412  175.50  -64.10  123.29   16.41  -64.10  123.29   16.41\n",
      "    5         56S  56S N   CA    412  413  -61.53  -35.00                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -122.86 -124.30   13.12    0.76  -63.50  178.33   31.29\n",
      "    6         97Y  97Y N   CA    723  724  148.44  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -161.98 -121.10   60.35    2.00  -63.80  167.42   29.50\n",
      "    7         98Q  98Q N   CA    735  736 -175.91  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -102.71  -69.30   93.04    5.90  -63.60   98.46   15.33\n",
      "    8         99E  99E N   CA    744  745 -130.66  142.50                  -40.30\n",
      "    9    559  99E 100T C   N     751  753 -155.05 -124.80   44.68    1.42  -63.20  168.71   27.44\n",
      "    9        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.65 -117.80   11.02    0.52  -63.60 -179.06   27.30\n",
      "   10        240E 240E N   CA   1798 1799  147.66  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815 -103.41 -118.00   14.97    0.52  -62.90 -178.88   21.64\n",
      "   11        242K 242K N   CA   1815 1816  135.73  139.10                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824  -85.57  -67.60   22.17    1.33  -63.20  166.22   22.22\n",
      "   12        243H 243H N   CA   1824 1825  152.99  140.00                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834 -108.41 -125.60   34.35    1.54  -63.20  155.86   22.44\n",
      "   13        244H 244H N   CA   1834 1835  168.54  138.80                  -42.30\n",
      "   14    567 246H 247H C   N    1862 1864 -124.84 -125.60   22.53    0.87  -63.20  168.09   25.05\n",
      "   14        247H 247H N   CA   1864 1865  161.32  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   16   27  100   99  143  167  190  227  243  330\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      156.9268       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "openf___224_> Open           scfv.BL00060001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4811\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1256.3607\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      107.68       1.000\n",
      " 2 Bond angle potential               :     194       9     17  12.627  12.627      1127.5       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  44.585  44.585      41.205       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   2.032   2.032      7.4344       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4811       0      0   0.001   0.001     0.28528       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      2   7.645   7.645      11.726       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2  75.804  75.804      14.082       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2  81.763  81.763      12.364       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 111.077 111.077      2.7958       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   1.561   1.561     0.22857       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      14      3  37.845 141.738      24.451       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4811       0     12   1.862   1.862     -93.353       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.17    5.97    1.35    0.17    5.97\n",
      "    2     45  98Q  98Q N   CA    735  736    1.61    1.43    0.18    6.07    1.43    0.18    6.07\n",
      "    3     47  98Q  98Q C   CA    742  736    1.76    1.49    0.27    7.72    1.49    0.27    7.72\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.56    1.35    0.19    6.56\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.32    1.43    0.22    7.32\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.89    1.49    0.27    7.89\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.21    7.49    1.35    0.21    7.49\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  129.84  107.00   22.84    6.56  107.00   22.84    6.56\n",
      "    2    208  98Q  99E C   N     742  744  140.50  120.00   20.50    4.66  120.00   20.50    4.66\n",
      "    3    211  99E  99E N   CA    744  745  135.62  107.00   28.62    8.22  107.00   28.62    8.22\n",
      "    4    214  99E  99E CA  C     745  751  134.07  116.50   17.57    5.05  116.50   17.57    5.05\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   56.94    1.83  -64.10 -160.61   23.53\n",
      "    1         51S  51S N   CA    376  377  170.52  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -141.53 -136.60    7.89    0.27  -64.10 -175.34   18.91\n",
      "    2         52S  52S N   CA    382  383  157.36  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -105.73  -98.40   25.69    2.60  -63.50  153.12   22.31\n",
      "    3         53Y  53Y N   CA    388  389  103.78  128.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -143.02 -136.60   21.79    1.21  -64.10 -176.76   11.57\n",
      "    4         55S  55S N   CA    406  407  130.37  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -122.86 -124.30   14.96    0.85  -63.50  176.58   31.02\n",
      "    5         97Y  97Y N   CA    723  724  150.29  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -157.72 -121.10   66.14    2.37  -63.80  156.30   27.62\n",
      "    6         98Q  98Q N   CA    735  736 -165.23  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -115.33 -117.80   92.68    4.70  -63.60  104.02   16.73\n",
      "    7         99E  99E N   CA    744  745 -130.55  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753 -156.42 -124.80   45.62    1.46  -63.20  169.46   27.61\n",
      "    8        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -119.65 -117.80   11.42    0.54  -63.60 -179.44   27.25\n",
      "    9        240E 240E N   CA   1798 1799  148.07  136.80                  -40.30\n",
      "   10    563 242K 243H C   N    1822 1824  -86.55  -67.60   73.70    6.18  -63.20  113.51   12.90\n",
      "   10        243H 243H N   CA   1824 1825   68.78  140.00                  -42.30\n",
      "   11    564 243H 244H C   N    1832 1834  -85.54  -67.60   19.03    1.24  -63.20  172.81   23.04\n",
      "   11        244H 244H N   CA   1834 1835  146.34  140.00                  -42.30\n",
      "   12    565 244H 245H C   N    1842 1844   47.29   56.30    9.25    1.32  -63.20  139.52   23.81\n",
      "   12        245H 245H N   CA   1844 1845   42.89   40.80                  -42.30\n",
      "   13    566 245H 246H C   N    1852 1854 -137.70 -125.60   18.77    0.48  -63.20 -179.37   27.44\n",
      "   13        246H 246H N   CA   1854 1855  153.15  138.80                  -42.30\n",
      "   14    567 246H 247H C   N    1862 1864  -57.92  -67.60   19.26    1.20  -63.20  165.74   21.05\n",
      "   14        247H 247H N   CA   1864 1865  123.35  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   17   22   95   99  148  162  195  227  246  314\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      128.5285       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      164.4407       99    99\n",
      "openf___224_> Open           scfv.BL00070001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4493\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         329.9474\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      9   0.051   0.051      112.13       1.000\n",
      " 2 Bond angle potential               :     194       4     13   4.978   4.978      99.240       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      8  46.043  46.043      42.946       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.920   1.920      6.0590       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4493       1      1   0.005   0.005      14.158       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      2   9.276   9.276      17.263       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2  93.904  93.904      15.795       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  96.931  96.931      9.6109       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 115.033 115.033      3.4916       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  25.053  25.053     0.70233       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      16      8  65.562 144.269      87.261       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4493       0      9   1.876   1.876     -78.708       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.17    6.01    1.35    0.17    6.01\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.23    1.43    0.19    6.23\n",
      "    3     47  98Q  98Q C   CA    742  736    1.76    1.49    0.27    7.89    1.49    0.27    7.89\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.66    1.35    0.19    6.66\n",
      "    5     54  99E  99E N   CA    744  745    1.66    1.43    0.23    7.58    1.43    0.23    7.58\n",
      "    6     56  99E  99E C   CA    751  745    1.74    1.49    0.25    7.26    1.49    0.25    7.26\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.21    7.26    1.35    0.21    7.26\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  129.27  107.00   22.27    6.40  107.00   22.27    6.40\n",
      "    2    208  98Q  99E C   N     742  744  142.63  120.00   22.63    5.14  120.00   22.63    5.14\n",
      "    3    211  99E  99E N   CA    744  745  137.27  107.00   30.27    8.70  107.00   30.27    8.70\n",
      "    4    214  99E  99E CA  C     745  751  132.48  116.50   15.98    4.59  116.50   15.98    4.59\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    573  56S  56S CA  C     413  416 -155.62  180.00   24.38    4.88  180.00   24.38    4.88\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   57.32    1.83  -64.10 -161.46   23.47\n",
      "    1         51S  51S N   CA    376  377  171.61  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -154.35 -136.60   34.70    1.30  -64.10  169.93   18.82\n",
      "    2         52S  52S N   CA    382  383 -178.98  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -111.24 -124.30   29.16    2.10  -63.50  162.31   28.18\n",
      "    3         53Y  53Y N   CA    388  389  161.47  135.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -140.66  -64.10   80.40    9.13  -64.10   80.40    9.13\n",
      "    4         55S  55S N   CA    406  407  -10.45  -35.00                  -35.00\n",
      "    5    554  55S  56S C   N     410  412   56.52  -72.40  137.93    8.21  -64.10 -176.45   21.97\n",
      "    5         56S  56S N   CA    412  413  103.36  152.40                  -35.00\n",
      "    6    555  56S  57Y C   N     416  418  148.00  -63.50  148.55   25.80  -63.50  148.55   25.80\n",
      "    6         57Y  57Y N   CA    418  419  -47.44  -43.40                  -43.40\n",
      "    7    556  96V  97Y C   N     721  723 -122.86 -124.30   14.53    0.83  -63.50  176.99   31.08\n",
      "    7         97Y  97Y N   CA    723  724  149.86  135.40                  -43.40\n",
      "    8    557  97Y  98Q C   N     733  735 -155.93 -121.10   65.43    2.38  -63.80  154.96   27.35\n",
      "    8         98Q  98Q N   CA    735  736 -164.90  139.70                  -40.30\n",
      "    9    558  98Q  99E C   N     742  744 -111.98 -117.80   89.87    4.61  -63.60  105.03   16.70\n",
      "    9         99E  99E N   CA    744  745 -133.52  136.80                  -40.30\n",
      "   10    559  99E 100T C   N     751  753 -155.38 -124.80   44.90    1.43  -63.20  168.89   27.48\n",
      "   10        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "   11    560 239V 240E C   N    1796 1798 -119.65 -117.80   12.81    0.61  -63.60  179.21   27.07\n",
      "   11        240E 240E N   CA   1798 1799  149.48  136.80                  -40.30\n",
      "   12    562 241I 242K C   N    1813 1815 -124.33 -118.00   25.97    1.33  -62.90  166.46   19.25\n",
      "   12        242K 242K N   CA   1815 1816  113.91  139.10                  -40.80\n",
      "   13    563 242K 243H C   N    1822 1824  -75.86  -67.60   67.42    5.42  -63.20  116.08   13.76\n",
      "   13        243H 243H N   CA   1824 1825   73.09  140.00                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844 -139.52 -125.60   47.36    1.51  -63.20  153.88   24.16\n",
      "   14        245H 245H N   CA   1844 1845 -175.93  138.80                  -42.30\n",
      "   15    566 245H 246H C   N    1852 1854 -124.88 -125.60   46.18    1.77  -63.20  146.36   22.32\n",
      "   15        246H 246H N   CA   1854 1855 -175.03  138.80                  -42.30\n",
      "   16    567 246H 247H C   N    1862 1864  -68.30  -67.60   33.98    2.63  -63.20  148.41   18.24\n",
      "   16        247H 247H N   CA   1864 1865  106.02  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   224  417   2.203\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    1    0    0   18   22   98   97  147  156  188  219  253  305\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -142.8391       99    99\n",
      "openf___224_> Open           scfv.BL00080001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4473\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         258.3735\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.049   0.049      103.84       1.000\n",
      " 2 Bond angle potential               :     194       3      9   4.870   4.870      95.908       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      5  40.541  40.541      36.572       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      1   2.253   2.253      10.550       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4473       0      0   0.002   0.002      2.3458       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      2   8.460   8.460      14.361       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      3  93.720  93.720      20.651       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  98.941  98.941      7.7026       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 125.336 125.336      3.5835       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  37.192  37.192      1.3287       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      14      4  41.494 144.563      30.746       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4473       0      9   1.919   1.919     -69.217       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.50    1.35    0.16    5.57    1.35    0.16    5.57\n",
      "    2     45  98Q  98Q N   CA    735  736    1.60    1.43    0.17    5.63    1.43    0.17    5.63\n",
      "    3     47  98Q  98Q C   CA    742  736    1.76    1.49    0.27    7.75    1.49    0.27    7.75\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.69    1.35    0.19    6.69\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.32    1.43    0.22    7.32\n",
      "    6     56  99E  99E C   CA    751  745    1.77    1.49    0.28    8.08    1.49    0.28    8.08\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.21    7.46    1.35    0.21    7.46\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  129.93  107.00   22.93    6.59  107.00   22.93    6.59\n",
      "    2    211  99E  99E N   CA    744  745  135.81  107.00   28.81    8.28  107.00   28.81    8.28\n",
      "    3    214  99E  99E CA  C     745  751  134.59  116.50   18.09    5.20  116.50   18.09    5.20\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   53.79    2.01  -64.10 -149.31   24.36\n",
      "    1         51S  51S N   CA    376  377  156.19  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -162.18 -136.60   37.47    1.23  -64.10  176.24   19.85\n",
      "    2         52S  52S N   CA    382  383  178.58  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -159.69 -124.30   56.01    2.32  -63.50  168.05   31.12\n",
      "    3         53Y  53Y N   CA    388  389  178.81  135.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -172.59 -136.60   36.54    1.29  -64.10 -160.41   22.29\n",
      "    4         55S  55S N   CA    406  407  157.48  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -122.86 -124.30   16.14    0.92  -63.50  175.47   30.84\n",
      "    5         97Y  97Y N   CA    723  724  151.47  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -152.13 -121.10   61.70    2.28  -63.80  154.43   27.12\n",
      "    6         98Q  98Q N   CA    735  736 -166.97  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -125.64 -117.80   93.51    4.55  -63.60  109.08   17.91\n",
      "    7         99E  99E N   CA    744  745 -130.02  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753 -156.32 -124.80   45.55    1.46  -63.20  169.40   27.60\n",
      "    8        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -119.65 -117.80   10.56    0.49  -63.60 -178.62   27.36\n",
      "    9        240E 240E N   CA   1798 1799  147.20  136.80                  -40.30\n",
      "   10    563 242K 243H C   N    1822 1824 -101.34 -125.60   54.96    1.58  -63.20  137.19   15.17\n",
      "   10        243H 243H N   CA   1824 1825   89.48  138.80                  -42.30\n",
      "   11    564 243H 244H C   N    1832 1834 -125.49 -125.60   46.40    1.77  -63.20  146.41   22.36\n",
      "   11        244H 244H N   CA   1834 1835 -174.80  138.80                  -42.30\n",
      "   12    565 244H 245H C   N    1842 1844  -87.97  -67.60   20.41    1.49  -63.20  178.09   23.86\n",
      "   12        245H 245H N   CA   1844 1845  141.34  140.00                  -42.30\n",
      "   13    566 245H 246H C   N    1852 1854  -59.32  -67.60   21.37    1.39  -63.20  162.64   20.57\n",
      "   13        246H 246H N   CA   1854 1855  120.30  140.00                  -42.30\n",
      "   14    567 246H 247H C   N    1862 1864 -104.72 -125.60   33.80    1.53  -63.20  157.87   22.45\n",
      "   14        247H 247H N   CA   1864 1865  165.39  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   18   23  100  101  147  157  190  217  254  307\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -172.2015       99    99\n",
      "openf___224_> Open           scfv.BL00090001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000001.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4263\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         241.1616\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.049   0.049      102.96       1.000\n",
      " 2 Bond angle potential               :     194       4     10   4.995   4.995      97.973       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0     11  48.470  48.470      44.439       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.967   1.967      6.4658       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4263       0      0   0.001   0.001     0.19557       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      2   6.606   6.606      8.7555       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2  89.662  89.662      9.8319       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  98.061  98.061      6.4807       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  75.478  75.478      2.5603       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  24.896  24.896     0.60908       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      16      5  58.955 157.531      43.072       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4263       0      5   1.880   1.880     -82.176       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.52    1.35    0.17    6.03    1.35    0.17    6.03\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.26    1.43    0.19    6.26\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.56    1.49    0.26    7.56\n",
      "    4     48  98Q  99E C   N     742  744    1.52    1.35    0.18    6.29    1.35    0.18    6.29\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.30    1.43    0.22    7.30\n",
      "    6     56  99E  99E C   CA    751  745    1.75    1.49    0.26    7.55    1.49    0.26    7.55\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.22    7.64    1.35    0.22    7.64\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  129.67  107.00   22.67    6.51  107.00   22.67    6.51\n",
      "    2    208  98Q  99E C   N     742  744  145.80  120.00   25.80    5.86  120.00   25.80    5.86\n",
      "    3    211  99E  99E N   CA    744  745  138.27  107.00   31.27    8.99  107.00   31.27    8.99\n",
      "    4    214  99E  99E CA  C     745  751  133.32  116.50   16.82    4.83  116.50   16.82    4.83\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  169.84 -136.60   58.14    1.83  -64.10 -163.16   23.34\n",
      "    1         51S  51S N   CA    376  377  173.82  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  151.24 -136.60  100.60    3.23  -64.10  177.99   23.00\n",
      "    2         52S  52S N   CA    382  383 -138.70  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388   49.90  -98.40  149.54    6.31  -63.50 -169.89   35.35\n",
      "    3         53Y  53Y N   CA    388  389  109.18  128.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  155.55 -136.60  111.17    3.86  -64.10  164.47   21.58\n",
      "    4         54S  54S N   CA    400  401 -120.74  151.20                  -35.00\n",
      "    5    553  54S  55S C   N     404  406 -161.23 -136.60   25.16    0.87  -64.10 -165.40   21.09\n",
      "    5         55S  55S N   CA    406  407  156.37  151.20                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -122.86 -124.30   14.28    0.82  -63.50  177.23   31.12\n",
      "    6         97Y  97Y N   CA    723  724  149.61  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -168.79 -121.10   61.91    1.95  -63.80  175.41   30.98\n",
      "    7         98Q  98Q N   CA    735  736  179.18  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744  -85.39  -69.30   83.48    5.79  -63.60   97.74   14.26\n",
      "    8         99E  99E N   CA    744  745 -135.58  142.50                  -40.30\n",
      "    9    559  99E 100T C   N     751  753 -154.85 -124.80   44.54    1.41  -63.20  168.60   27.41\n",
      "    9        100T 100T N   CA    753  754  176.38  143.50                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.65 -117.80    6.52    0.29  -63.60 -174.67   27.88\n",
      "   10        240E 240E N   CA   1798 1799  143.05  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815  -65.13  -70.20   21.84    1.46  -62.90  159.97   20.65\n",
      "   11        242K 242K N   CA   1815 1816  119.16  140.40                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824  -67.85  -67.60   36.15    2.79  -63.20  146.22   17.99\n",
      "   12        243H 243H N   CA   1824 1825  103.85  140.00                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834  -99.33 -125.60   30.46    1.33  -63.20  167.42   23.28\n",
      "   13        244H 244H N   CA   1834 1835  154.23  138.80                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844 -123.16 -125.60    2.60    0.07  -63.20 -170.47   27.62\n",
      "   14        245H 245H N   CA   1844 1845  137.91  138.80                  -42.30\n",
      "   15    566 245H 246H C   N    1852 1854   49.85   56.30   10.60    0.89  -63.20  145.44   24.76\n",
      "   15        246H 246H N   CA   1854 1855   49.21   40.80                  -42.30\n",
      "   16    567 246H 247H C   N    1862 1864  -94.69  -67.60   27.35    2.17  -63.20 -178.70   20.88\n",
      "   16        247H 247H N   CA   1864 1865  136.25  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.050\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   15   23   97   95  147  162  181  219  240  309\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.BL00100001.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.B99990002.pdb\n",
      "readlinef__W> File: scfv.B99990002.pdb, Line: 6\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18101636   17677.379    17.263\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18109380   17684.941    17.270\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18251072   17823.312    17.406\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18253480   17825.664    17.408\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395172   17964.035    17.543\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933      0         0       876\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933   2622      3114       876\n",
      "\n",
      "patch_______> segment topology patched using RTF:   248  ;  HIS  ;  CTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "openf___224_> Open           scfv.B99990002.pdb\n",
      "readlinef__W> File: scfv.B99990002.pdb, Line: 6\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18406145   17974.750    17.553\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18406221   17974.824    17.554\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18406392   17974.992    17.554\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18407242   17975.822    17.555\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18408500   17977.051    17.556\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18410404   17978.910    17.558\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18410444   17978.949    17.558\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18410534   17979.037    17.558\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18413390   17981.826    17.560\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18413450   17981.885    17.560\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18413585   17982.016    17.561\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18417869   17986.199    17.565\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18417957   17986.285    17.565\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18418155   17986.480    17.565\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18424581   17992.754    17.571\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18424713   17992.883    17.571\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18425010   17993.174    17.571\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18434632   18002.570    17.581\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18434832   18002.766    17.581\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18435282   18003.205    17.581\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18449732   18017.316    17.595\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18450032   18017.609    17.595\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18450707   18018.270    17.596\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18472365   18039.418    17.617\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18472813   18039.855    17.617\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18473821   18040.840    17.618\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18644509   18207.527    17.781\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  scfv\n",
      "  2  scfv\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1884        0\n",
      "openf___224_> Open           scfv.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18397174   17965.990    17.545\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18398624   17967.406    17.546\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18401524   17970.238    17.549\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18407324   17975.902    17.555\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18524380   18090.215    17.666\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18525284   18091.098    17.667\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18526272   18092.062    17.668\n",
      "\n",
      "Read the alignment from file       : scfv.ali\n",
      "Total number of alignment positions:   253\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1     2fjhHL     227      1  2fjhHL.pdb\n",
      "  2       scfv     248      1        scfv\n",
      "132 atoms selected for loop refinement\n",
      "make_re_417_> Restraint type to be calculated:  bond\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      141       0       0       0\n",
      "              Total number of restraints before, now         :        0      141\n",
      "make_re_422_> Number of previous, current restraints         :        0      141\n",
      "make_re_423_> Number of previous, current selected restraints:        0      141\n",
      "make_re_417_> Restraint type to be calculated:  angle\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :        0     194       0       0\n",
      "              Total number of restraints before, now         :      141      335\n",
      "make_re_422_> Number of previous, current restraints         :      141      335\n",
      "make_re_423_> Number of previous, current selected restraints:      141      335\n",
      "make_re_417_> Restraint type to be calculated:  improper\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :        0       0       0      81\n",
      "              Total number of restraints before, now         :      335      416\n",
      "make_re_422_> Number of previous, current restraints         :      335      416\n",
      "make_re_423_> Number of previous, current selected restraints:      335      416\n",
      "make_re_417_> Restraint type to be calculated:  dihedral\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :        0       0     235       0\n",
      "              Total number of restraints before, now         :      416      651\n",
      "make_re_422_> Number of previous, current restraints         :      416      651\n",
      "make_re_423_> Number of previous, current selected restraints:      416      651\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395172   17964.035    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      651      670\n",
      "make_re_423_> Number of previous, current selected restraints:      651      670\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      670      687\n",
      "make_re_423_> Number of previous, current selected restraints:      670      670\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      687      702\n",
      "make_re_423_> Number of previous, current selected restraints:      670      685\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      702      713\n",
      "make_re_423_> Number of previous, current selected restraints:      685      696\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      713      716\n",
      "make_re_423_> Number of previous, current selected restraints:      696      699\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      716      717\n",
      "make_re_423_> Number of previous, current selected restraints:      699      700\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;      132      717\n",
      "rmdupl__427_>      103 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:      717      614\n",
      "openf___224_> Open           scfv.lrsr\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4524\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         361.0470\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       6      7   0.052   0.052      116.34       1.000\n",
      " 2 Bond angle potential               :     194       5     10   4.638   4.638      82.094       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      3  41.444  41.444      37.009       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   0.731   0.731      1.1164       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4524       0      0   0.001   0.001     0.91108       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      1   5.982   5.982      7.1799       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      1  92.597  92.597      7.8626       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  80.227  80.227      7.4185       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 120.405 120.405      3.3189       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 128.854 128.854      1.2700       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      13      2  27.123 144.828      14.769       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4524       0     21   1.912   1.912      81.754       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     45  98Q  98Q N   CA    735  736    1.59    1.43    0.16    5.34    1.43    0.16    5.34\n",
      "    2     47  98Q  98Q C   CA    742  736    1.80    1.49    0.31    8.91    1.49    0.31    8.91\n",
      "    3     48  98Q  99E C   N     742  744    1.60    1.35    0.26    9.11    1.35    0.26    9.11\n",
      "    4     54  99E  99E N   CA    744  745    1.67    1.43    0.24    8.06    1.43    0.24    8.06\n",
      "    5     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.96    1.49    0.27    7.96\n",
      "    6     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.13    1.35    0.20    7.13\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  127.09  107.00   20.09    5.77  107.00   20.09    5.77\n",
      "    2    202  98Q  98Q CA  C     736  742  134.18  116.50   17.68    5.08  116.50   17.68    5.08\n",
      "    3    208  98Q  99E C   N     742  744  145.60  120.00   25.60    5.82  120.00   25.60    5.82\n",
      "    4    211  99E  99E N   CA    744  745  130.04  107.00   23.04    6.62  107.00   23.04    6.62\n",
      "    5    220  99E 100T C   N     751  753  151.92  120.00   31.92    7.25  120.00   31.92    7.25\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   58.92    1.83  -64.10 -165.20   23.17\n",
      "    1         51S  51S N   CA    376  377  176.28  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -137.49 -136.60    8.98    0.44  -64.10 -179.54   18.31\n",
      "    2         52S  52S N   CA    382  383  160.13  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -107.69  -98.40   36.89    3.77  -63.50  143.10   20.67\n",
      "    3         53Y  53Y N   CA    388  389   92.70  128.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -167.42 -136.60   40.18    2.20  -64.10 -169.17   12.97\n",
      "    4         55S  55S N   CA    406  407  125.43  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -124.06 -124.30   14.16    0.76  -63.50  177.68   31.25\n",
      "    5         97Y  97Y N   CA    723  724  149.56  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -155.35 -121.10   58.79    2.07  -63.80  160.82   28.23\n",
      "    6         98Q  98Q N   CA    735  736 -172.52  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -141.78 -117.80   41.21    1.47  -63.60  168.61   26.84\n",
      "    7         99E  99E N   CA    744  745  170.31  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753  -77.52  -78.10   27.85    1.21  -63.20  140.98   19.03\n",
      "    8        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -119.93 -117.80   10.82    0.50  -63.60 -178.73   27.36\n",
      "    9        240E 240E N   CA   1798 1799  147.41  136.80                  -40.30\n",
      "   10    562 241I 242K C   N    1813 1815 -106.79 -118.00   12.42    0.41  -62.90  179.99   21.38\n",
      "   10        242K 242K N   CA   1815 1816  133.76  139.10                  -40.80\n",
      "   11    563 242K 243H C   N    1822 1824 -143.47 -125.60   18.96    0.78  -63.20 -167.67   20.54\n",
      "   11        243H 243H N   CA   1824 1825  132.47  138.80                  -42.30\n",
      "   12    564 243H 244H C   N    1832 1834 -123.93 -125.60   15.97    0.57  -63.20  176.03   19.07\n",
      "   12        244H 244H N   CA   1834 1835  122.92  138.80                  -42.30\n",
      "   13    567 246H 247H C   N    1862 1864  -68.23  -67.60    1.85    0.12  -63.20  176.03   22.32\n",
      "   13        247H 247H N   CA   1864 1865  141.74  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   18   33   94   96  152  177  184  244  268  306\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5356\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         355.8945\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.052   0.052      115.48       1.000\n",
      " 2 Bond angle potential               :     194       4     13   5.536   5.536      113.83       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  41.292  41.292      36.189       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.652   1.652      5.8443       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5356       0      0   0.002   0.002      1.8431       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      2   9.973   9.973      19.955       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2  90.193  90.193      21.214       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2  99.646  99.646      14.285       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  48.422  48.422      3.0954       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  13.792  13.792     0.33504       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      14      5  39.926 139.106      31.844       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5356       0     16   1.901   1.901     -8.0235       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.53    1.35    0.18    6.39    1.35    0.18    6.39\n",
      "    2     45  98Q  98Q N   CA    735  736    1.63    1.43    0.20    6.71    1.43    0.20    6.71\n",
      "    3     47  98Q  98Q C   CA    742  736    1.76    1.49    0.27    7.84    1.49    0.27    7.84\n",
      "    4     48  98Q  99E C   N     742  744    1.54    1.35    0.19    6.82    1.35    0.19    6.82\n",
      "    5     54  99E  99E N   CA    744  745    1.66    1.43    0.23    7.51    1.43    0.23    7.51\n",
      "    6     56  99E  99E C   CA    751  745    1.75    1.49    0.26    7.68    1.49    0.26    7.68\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.14    1.35    0.20    7.14\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  127.87  107.00   20.87    6.00  107.00   20.87    6.00\n",
      "    2    208  98Q  99E C   N     742  744  142.26  120.00   22.26    5.06  120.00   22.26    5.06\n",
      "    3    211  99E  99E N   CA    744  745  131.12  107.00   24.12    6.93  107.00   24.12    6.93\n",
      "    4    220  99E 100T C   N     751  753  152.05  120.00   32.05    7.28  120.00   32.05    7.28\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    572  55S  55S CA  C     407  410 -152.29  180.00   27.71    5.54  180.00   27.71    5.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   54.65    1.88  -64.10 -155.05   23.92\n",
      "    1         51S  51S N   CA    376  377  163.22  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  -98.40  -72.40   32.75    2.63  -64.10  170.96   10.83\n",
      "    2         52S  52S N   CA    382  383  132.48  152.40                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388  -79.59  -98.40   44.09    4.22  -63.50  149.19   24.51\n",
      "    3         53Y  53Y N   CA    388  389  168.28  128.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -63.07  -72.40   71.06    3.87  -64.10  116.96    8.54\n",
      "    4         54S  54S N   CA    400  401   81.95  152.40                  -35.00\n",
      "    5    553  54S  55S C   N     404  406  -40.39  -72.40   56.01    2.56  -64.10  143.41   11.96\n",
      "    5         55S  55S N   CA    406  407  106.44  152.40                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -124.06 -124.30   14.71    0.79  -63.50  177.16   31.16\n",
      "    6         97Y  97Y N   CA    723  724  150.11  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -166.55 -121.10   61.15    1.95  -63.80  172.92   30.52\n",
      "    7         98Q  98Q N   CA    735  736 -179.39  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -130.12 -117.80   35.43    1.50  -63.60  163.80   25.58\n",
      "    8         99E  99E N   CA    744  745  170.02  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -76.65  -78.10   27.89    1.24  -63.20  140.90   18.96\n",
      "    9        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.93 -117.80    9.99    0.46  -63.60 -177.93   27.46\n",
      "   10        240E 240E N   CA   1798 1799  146.56  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815  -75.92  -70.20   26.00    2.01  -62.90  156.38   19.63\n",
      "   11        242K 242K N   CA   1815 1816  115.04  140.40                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824 -159.82 -125.60   81.85    2.40  -63.20  142.35   23.72\n",
      "   12        243H 243H N   CA   1824 1825 -146.84  138.80                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834  -65.53  -67.60    2.07    0.15  -63.20  177.75   22.36\n",
      "   13        244H 244H N   CA   1834 1835  139.97  140.00                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844 -119.02 -125.60   44.68    1.57  -63.20  147.85   15.90\n",
      "   14        245H 245H N   CA   1844 1845   94.61  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   25   32   99   95  154  186  192  245  265  317\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:      146.6991       99    99\n",
      "openf___224_> Open           scfv.BL00010002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4912\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1337.0236\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141      10     14   0.071   0.071      198.91       1.000\n",
      " 2 Bond angle potential               :     194      10     21   8.636   8.636      224.76       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      7  47.833  47.833      48.089       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       4      8   6.150   6.150      70.724       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4912      16     24   0.031   0.031      553.70       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      4  11.009  11.009      24.316       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      1  80.797  80.797      12.892       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2  76.248  76.248      10.206       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  97.365  97.365      3.6682       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  13.460  13.460     0.33003       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      16      6  46.239 154.792      39.105       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4912       0     50   1.891   1.891      150.33       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1      8  53Y  53Y CB  CA    390  389    1.90    1.54    0.36   10.02    1.54    0.36   10.02\n",
      "    2      9  53Y  53Y CG  CB    391  390    1.85    1.49    0.36   10.04    1.49    0.36   10.04\n",
      "    3     39  97Y  98Q C   N     733  735    1.52    1.35    0.18    6.33    1.35    0.18    6.33\n",
      "    4     45  98Q  98Q N   CA    735  736    1.63    1.43    0.20    6.56    1.43    0.20    6.56\n",
      "    5     47  98Q  98Q C   CA    742  736    1.76    1.49    0.27    7.95    1.49    0.27    7.95\n",
      "    6     48  98Q  99E C   N     742  744    1.54    1.35    0.19    6.80    1.35    0.19    6.80\n",
      "    7     54  99E  99E N   CA    744  745    1.66    1.43    0.23    7.46    1.43    0.23    7.46\n",
      "    8     56  99E  99E C   CA    751  745    1.75    1.49    0.26    7.56    1.49    0.26    7.56\n",
      "    9     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.15    1.35    0.20    7.15\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    155  53Y  53Y CA  CB    389  390  143.10  107.50   35.60    8.23  107.50   35.60    8.23\n",
      "    2    160  53Y  53Y CG  CD1   391  392  163.82  120.00   43.82    8.90  120.00   43.82    8.90\n",
      "    3    164  53Y  53Y CD2 CE2   393  395  163.83  120.00   43.83    8.91  120.00   43.83    8.91\n",
      "    4    165  53Y  53Y CE1 CZ    394  396  144.09  120.00   24.09    5.20  120.00   24.09    5.20\n",
      "    5    168  53Y  53Y C   CA    398  389  133.45  108.00   25.45    5.90  108.00   25.45    5.90\n",
      "    6    199  98Q  98Q N   CA    735  736  129.48  107.00   22.48    6.46  107.00   22.48    6.46\n",
      "    7    211  99E  99E N   CA    744  745  129.98  107.00   22.98    6.60  107.00   22.98    6.60\n",
      "    8    220  99E 100T C   N     751  753  151.58  120.00   31.58    7.18  120.00   31.58    7.18\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    336  51S  51S C   CA    380  377   14.52    0.00   14.52    5.11    0.00   14.52    5.11\n",
      "    2    346  53Y  53Y CE2 CD2   395  393 -160.11  180.00   19.89    6.06  180.00   19.89    6.06\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    569  52S  52S CA  C     383  386 -148.09  180.00   31.91    6.38  180.00   31.91    6.38\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   53.38    2.19  -64.10 -143.20   24.79\n",
      "    1         51S  51S N   CA    376  377  148.44  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  -57.11  -72.40   62.52    3.18  -64.10  126.97    9.63\n",
      "    2         52S  52S N   CA    382  383   91.78  152.40                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388  -87.98  -63.50   87.66   12.76  -63.50   87.66   12.76\n",
      "    3         53Y  53Y N   CA    388  389   40.77  -43.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  124.40 -136.60   99.81    4.29  -64.10 -116.02   19.65\n",
      "    4         54S  54S N   CA    400  401  138.54  151.20                  -35.00\n",
      "    5    553  54S  55S C   N     404  406 -129.01 -136.60   60.32    2.89  -64.10  142.06    9.09\n",
      "    5         55S  55S N   CA    406  407   91.36  151.20                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -124.06 -124.30   16.54    0.89  -63.50  175.44   30.89\n",
      "    6         97Y  97Y N   CA    723  724  151.94  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -161.19 -121.10   65.84    2.28  -63.80  160.66   28.42\n",
      "    7         98Q  98Q N   CA    735  736 -168.07  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -145.48 -117.80   43.04    1.46  -63.60  170.84   27.33\n",
      "    8         99E  99E N   CA    744  745  169.76  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -76.99  -78.10   27.87    1.23  -63.20  140.93   18.98\n",
      "    9        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.93 -117.80    8.20    0.37  -63.60 -176.17   27.70\n",
      "   10        240E 240E N   CA   1798 1799  144.72  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815 -122.62 -118.00    5.14    0.24  -62.90 -172.58   21.86\n",
      "   11        242K 242K N   CA   1815 1816  136.85  139.10                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824 -113.27 -125.60   30.17    1.33  -63.20  159.43   23.21\n",
      "   12        243H 243H N   CA   1824 1825  166.34  138.80                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834  -85.39  -67.60   18.64    1.23  -63.20  173.56   23.12\n",
      "   13        244H 244H N   CA   1834 1835  145.57  140.00                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844  -71.95  -67.60   21.22    1.75  -63.20  161.77   19.69\n",
      "   14        245H 245H N   CA   1844 1845  119.23  140.00                  -42.30\n",
      "   15    566 245H 246H C   N    1852 1854  -93.59  -67.60   36.14    2.14  -63.20  155.58   21.42\n",
      "   15        246H 246H N   CA   1854 1855  165.12  140.00                  -42.30\n",
      "   16    567 246H 247H C   N    1862 1864 -131.92 -125.60   43.51    1.77  -63.20  154.20   16.42\n",
      "   16        247H 247H N   CA   1864 1865   95.75  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   274  390   2.095\n",
      "  serious non-bonded atom clash:   373  395   2.079\n",
      "  serious non-bonded atom clash:   379  391   2.234\n",
      "  serious non-bonded atom clash:   379  392   2.034\n",
      "  serious non-bonded atom clash:   381  390   2.243\n",
      "  serious non-bonded atom clash:   389  548   2.279\n",
      "  serious non-bonded atom clash:   390  548   2.266\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    0    4    0    3   25   37  105  101  154  184  184  239  268  312\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.BL00020002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4590\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         241.0219\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      109.24       1.000\n",
      " 2 Bond angle potential               :     194       4     10   5.230   5.230      101.70       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      2  36.952  36.952      31.294       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.499   1.499      4.4873       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4590       0      0   0.001   0.001     0.92859       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      2   8.384   8.384      14.103       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      3  75.688  75.688      14.101       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      3 105.513 105.513      10.908       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  35.781  35.781      2.3421       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   2.504   2.504     0.23303       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      14      6  38.734 144.888      30.334       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4590       0      8   1.883   1.883     -78.656       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.53    1.35    0.18    6.40    1.35    0.18    6.40\n",
      "    2     45  98Q  98Q N   CA    735  736    1.63    1.43    0.20    6.73    1.43    0.20    6.73\n",
      "    3     47  98Q  98Q C   CA    742  736    1.77    1.49    0.28    8.06    1.49    0.28    8.06\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.62    1.35    0.19    6.62\n",
      "    5     54  99E  99E N   CA    744  745    1.66    1.43    0.23    7.44    1.43    0.23    7.44\n",
      "    6     56  99E  99E C   CA    751  745    1.74    1.49    0.25    7.36    1.49    0.25    7.36\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.25    1.35    0.20    7.25\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  129.44  107.00   22.44    6.45  107.00   22.44    6.45\n",
      "    2    208  98Q  99E C   N     742  744  143.05  120.00   23.05    5.24  120.00   23.05    5.24\n",
      "    3    211  99E  99E N   CA    744  745  132.65  107.00   25.65    7.37  107.00   25.65    7.37\n",
      "    4    220  99E 100T C   N     751  753  151.41  120.00   31.41    7.14  120.00   31.41    7.14\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   58.49    1.82  -64.10 -164.42   23.23\n",
      "    1         51S  51S N   CA    376  377  175.26  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -162.23 -136.60   42.68    1.49  -64.10  170.71   19.45\n",
      "    2         52S  52S N   CA    382  383 -174.68  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -115.07 -124.30   35.83    2.30  -63.50  155.39   27.26\n",
      "    3         53Y  53Y N   CA    388  389  170.02  135.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -148.82 -136.60   14.82    0.79  -64.10 -163.03   12.44\n",
      "    4         55S  55S N   CA    406  407  142.82  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -124.06 -124.30   15.13    0.81  -63.50  176.77   31.10\n",
      "    5         97Y  97Y N   CA    723  724  150.53  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -175.01 -121.10   62.40    1.92  -63.80 -174.41   32.79\n",
      "    6         98Q  98Q N   CA    735  736  171.12  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -115.03 -117.80   35.98    1.85  -63.60  155.76   23.68\n",
      "    7         99E  99E N   CA    744  745  172.67  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753  -77.12  -78.10   27.87    1.22  -63.20  140.94   18.99\n",
      "    8        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -119.93 -117.80    7.58    0.33  -63.60 -175.57   27.78\n",
      "    9        240E 240E N   CA   1798 1799  144.08  136.80                  -40.30\n",
      "   10    561 240E 241I C   N    1805 1807  -71.23  -63.40   13.03    2.41 -120.60 -177.51   12.29\n",
      "   10        241I 241I N   CA   1807 1808  -54.02  -43.60                  130.30\n",
      "   11    562 241I 242K C   N    1813 1815 -148.18 -118.00   32.90    1.49  -62.90 -172.65   21.52\n",
      "   11        242K 242K N   CA   1815 1816  126.01  139.10                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824 -103.77 -125.60   47.72    2.13  -63.20  142.37   20.45\n",
      "   12        243H 243H N   CA   1824 1825 -178.77  138.80                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834 -116.74  -63.20   96.13   15.43  -63.20   96.13   15.43\n",
      "   13        244H 244H N   CA   1834 1835 -122.14  -42.30                  -42.30\n",
      "   14    567 246H 247H C   N    1862 1864 -108.68 -125.60   32.07    1.44  -63.20  158.33   22.77\n",
      "   14        247H 247H N   CA   1864 1865  166.04  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   23   30   90   96  148  177  179  238  275  317\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:      139.7218       99    99\n",
      "openf___224_> Open           scfv.BL00030002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4675\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         316.3915\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      105.86       1.000\n",
      " 2 Bond angle potential               :     194       5     12   5.502   5.502      110.37       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0     10  48.291  48.291      48.076       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.764   1.764      5.6939       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4675       0      1   0.003   0.003      5.9186       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      2   9.957   9.957      19.892       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0 104.678 104.678      9.3522       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2 109.923 109.923      11.579       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 116.668 116.668      3.3459       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  16.435  16.435     0.38117       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      9  56.093 149.449      53.785       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4675       0      6   1.890   1.890     -57.862       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.16    5.76    1.35    0.16    5.76\n",
      "    2     45  98Q  98Q N   CA    735  736    1.61    1.43    0.18    5.92    1.43    0.18    5.92\n",
      "    3     47  98Q  98Q C   CA    742  736    1.74    1.49    0.25    7.42    1.49    0.25    7.42\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.48    1.35    0.18    6.48\n",
      "    5     54  99E  99E N   CA    744  745    1.63    1.43    0.20    6.55    1.43    0.20    6.55\n",
      "    6     56  99E  99E C   CA    751  745    1.79    1.49    0.30    8.66    1.49    0.30    8.66\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.22    7.75    1.35    0.22    7.75\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    143  51S  52S C   N     380  382  145.03  120.00   25.03    5.69  120.00   25.03    5.69\n",
      "    2    199  98Q  98Q N   CA    735  736  127.32  107.00   20.32    5.84  107.00   20.32    5.84\n",
      "    3    211  99E  99E N   CA    744  745  131.70  107.00   24.70    7.10  107.00   24.70    7.10\n",
      "    4    215  99E  99E CB  CG    746  747  128.92  108.00   20.92    4.85  108.00   20.92    4.85\n",
      "    5    220  99E 100T C   N     751  753  148.89  120.00   28.89    6.56  120.00   28.89    6.56\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -146.54  180.00   33.46    6.69  180.00   33.46    6.69\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   55.38    1.84  -64.10 -157.40   23.75\n",
      "    1         51S  51S N   CA    376  377  166.20  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382   96.68 -136.60  131.37    6.12  -64.10 -139.06   27.47\n",
      "    2         52S  52S N   CA    382  383  116.53  151.20                  -35.00\n",
      "    3    552  53Y  54S C   N     398  400 -153.09 -136.60   57.24    3.17  -64.10  158.69   10.98\n",
      "    3         54S  54S N   CA    400  401   96.39  151.20                  -35.00\n",
      "    4    553  54S  55S C   N     404  406 -125.64 -136.60   30.64    1.28  -64.10  169.18   10.46\n",
      "    4         55S  55S N   CA    406  407  122.59  151.20                  -35.00\n",
      "    5    554  55S  56S C   N     410  412  174.07 -136.60  114.51    4.56  -64.10  140.74   18.54\n",
      "    5         56S  56S N   CA    412  413 -105.46  151.20                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -124.06 -124.30   10.55    0.57  -63.50 -178.92   31.79\n",
      "    6         97Y  97Y N   CA    723  724  145.94  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -164.06 -121.10   63.21    2.09  -63.80  167.07   29.52\n",
      "    7         98Q  98Q N   CA    735  736 -173.93  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -157.24 -117.80   55.40    1.77  -63.60  171.76   28.02\n",
      "    8         99E  99E N   CA    744  745  175.71  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -81.74  -78.10   28.08    1.09  -63.20  141.47   19.37\n",
      "    9        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.93 -117.80   13.59    0.64  -63.60  178.59   27.00\n",
      "   10        240E 240E N   CA   1798 1799  150.22  136.80                  -40.30\n",
      "   11    563 242K 243H C   N    1822 1824 -123.74 -125.60   36.97    1.37  -63.20  156.37   16.79\n",
      "   11        243H 243H N   CA   1824 1825  101.88  138.80                  -42.30\n",
      "   12    564 243H 244H C   N    1832 1834 -133.05 -125.60   45.35    1.86  -63.20  153.21   16.30\n",
      "   12        244H 244H N   CA   1834 1835   94.06  138.80                  -42.30\n",
      "   13    565 244H 245H C   N    1842 1844 -127.65  -63.20   92.85   10.38  -63.20   92.85   10.38\n",
      "   13        245H 245H N   CA   1844 1845   24.54  -42.30                  -42.30\n",
      "   14    566 245H 246H C   N    1852 1854 -111.92 -125.60   21.46    0.97  -63.20  169.52   24.38\n",
      "   14        246H 246H N   CA   1854 1855  155.33  138.80                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864  -62.33  -67.60    5.31    0.42  -63.20  177.04   22.08\n",
      "   15        247H 247H N   CA   1864 1865  140.66  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   21   32   96   95  142  179  186  231  263  315\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.BL00040002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5132\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         268.3043\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.051   0.051      109.53       1.000\n",
      " 2 Bond angle potential               :     194       4     11   5.238   5.238      103.81       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  45.022  45.022      42.008       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   2.042   2.042      7.4872       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5132       0      0   0.001   0.001     0.54209       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      1   7.611   7.611      11.623       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      1  92.217  92.217      12.622       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      3 103.044 103.044      14.310       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  86.339  86.339      3.6045       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 111.407 111.407      1.1092       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      13      6  53.906 141.304      36.155       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5132       0     12   1.887   1.887     -74.493       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.17    5.99    1.35    0.17    5.99\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.38    1.43    0.19    6.38\n",
      "    3     47  98Q  98Q C   CA    742  736    1.77    1.49    0.28    8.04    1.49    0.28    8.04\n",
      "    4     48  98Q  99E C   N     742  744    1.54    1.35    0.19    6.75    1.35    0.19    6.75\n",
      "    5     54  99E  99E N   CA    744  745    1.66    1.43    0.23    7.45    1.43    0.23    7.45\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.71    1.49    0.27    7.71\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.22    1.35    0.20    7.22\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  128.48  107.00   21.48    6.17  107.00   21.48    6.17\n",
      "    2    208  98Q  99E C   N     742  744  139.96  120.00   19.96    4.53  120.00   19.96    4.53\n",
      "    3    211  99E  99E N   CA    744  745  131.30  107.00   24.30    6.98  107.00   24.30    6.98\n",
      "    4    220  99E 100T C   N     751  753  151.12  120.00   31.12    7.07  120.00   31.12    7.07\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   56.90    1.82  -64.10 -161.21   23.47\n",
      "    1         51S  51S N   CA    376  377  171.08  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  154.33 -136.60  103.03    3.42  -64.10  171.81   22.30\n",
      "    2         52S  52S N   CA    382  383 -132.35  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388   36.86  -98.40  135.69    5.48  -63.50 -170.28   34.86\n",
      "    3         53Y  53Y N   CA    388  389  117.60  128.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  155.42 -136.60   93.12    2.96  -64.10  178.53   22.82\n",
      "    4         54S  54S N   CA    400  401 -145.16  151.20                  -35.00\n",
      "    5    553  54S  55S C   N     404  406 -141.44 -136.60   14.51    0.81  -64.10 -170.94   11.84\n",
      "    5         55S  55S N   CA    406  407  137.51  151.20                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -124.06 -124.30   13.74    0.74  -63.50  178.08   31.31\n",
      "    6         97Y  97Y N   CA    723  724  149.14  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -168.26 -121.10   61.92    1.96  -63.80  174.57   30.84\n",
      "    7         98Q  98Q N   CA    735  736  179.83  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -132.61 -117.80   40.38    1.69  -63.60  160.89   25.32\n",
      "    8         99E  99E N   CA    744  745  174.36  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -77.10  -78.10   27.87    1.22  -63.20  140.94   18.99\n",
      "    9        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.93 -117.80    9.52    0.43  -63.60 -177.47   27.53\n",
      "   10        240E 240E N   CA   1798 1799  146.08  136.80                  -40.30\n",
      "   11    564 243H 244H C   N    1832 1834 -108.20 -125.60   78.34    3.27  -63.20  111.95   16.93\n",
      "   11        244H 244H N   CA   1834 1835 -144.81  138.80                  -42.30\n",
      "   12    565 244H 245H C   N    1842 1844  -69.69  -67.60    2.14    0.17  -63.20  178.24   22.69\n",
      "   12        245H 245H N   CA   1844 1845  139.57  140.00                  -42.30\n",
      "   13    567 246H 247H C   N    1862 1864   48.74   56.30    7.91    1.09  -63.20  140.80   24.04\n",
      "   13        247H 247H N   CA   1864 1865   43.11   40.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   21   32   95  100  155  187  195  251  270  322\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      -97.7742       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      120.3612       99    99\n",
      "openf___224_> Open           scfv.BL00050002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5162\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         409.2365\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      106.44       1.000\n",
      " 2 Bond angle potential               :     194       5     17   6.048   6.048      134.41       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  42.139  42.139      38.333       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      1   2.174   2.174      9.0225       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5162       0      1   0.003   0.003      6.4998       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      4  12.141  12.141      29.576       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      3  99.925  99.925      16.616       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2  99.718  99.718      12.849       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 141.794 141.794      3.9679       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   6.523   6.523     0.25661       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      7  39.059 151.694      43.027       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5162       0     16   1.886   1.886      8.2408       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.16    5.68    1.35    0.16    5.68\n",
      "    2     45  98Q  98Q N   CA    735  736    1.60    1.43    0.17    5.73    1.43    0.17    5.73\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.49    1.49    0.26    7.49\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.49    1.35    0.18    6.49\n",
      "    5     54  99E  99E N   CA    744  745    1.63    1.43    0.20    6.59    1.43    0.20    6.59\n",
      "    6     56  99E  99E C   CA    751  745    1.79    1.49    0.30    8.66    1.49    0.30    8.66\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.22    7.73    1.35    0.22    7.73\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  127.66  107.00   20.66    5.94  107.00   20.66    5.94\n",
      "    2    211  99E  99E N   CA    744  745  131.65  107.00   24.65    7.08  107.00   24.65    7.08\n",
      "    3    215  99E  99E CB  CG    746  747  128.58  108.00   20.58    4.77  108.00   20.58    4.77\n",
      "    4    220  99E 100T C   N     751  753  148.77  120.00   28.77    6.54  120.00   28.77    6.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -146.25  180.00   33.75    6.75  180.00   33.75    6.75\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   57.16    1.82  -64.10 -161.77   23.43\n",
      "    1         51S  51S N   CA    376  377  171.81  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -115.61 -136.60   21.65    0.73  -64.10 -173.64   17.07\n",
      "    2         52S  52S N   CA    382  383  145.90  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388  -66.93  -98.40   31.99    1.29  -63.50  177.57   28.01\n",
      "    3         53Y  53Y N   CA    388  389  134.14  128.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -74.05  -72.40   74.48    4.32  -64.10   98.65    7.78\n",
      "    4         54S  54S N   CA    400  401 -133.14  152.40                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -124.06 -124.30   11.92    0.64  -63.50  179.79   31.58\n",
      "    5         97Y  97Y N   CA    723  724  147.32  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -165.47 -121.10   62.42    2.03  -63.80  169.88   30.01\n",
      "    6         98Q  98Q N   CA    735  736 -176.39  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -157.92 -117.80   55.65    1.77  -63.60  172.42   28.14\n",
      "    7         99E  99E N   CA    744  745  175.37  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753  -81.90  -78.10   28.11    1.08  -63.20  141.49   19.39\n",
      "    8        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -119.93 -117.80    7.88    0.35  -63.60 -175.86   27.74\n",
      "    9        240E 240E N   CA   1798 1799  144.38  136.80                  -40.30\n",
      "   10    562 241I 242K C   N    1813 1815  -63.44  -70.20   37.86    2.58  -62.90  143.95   18.67\n",
      "   10        242K 242K N   CA   1815 1816  103.15  140.40                  -40.80\n",
      "   11    563 242K 243H C   N    1822 1824  -82.51  -67.60   25.60    2.29  -63.20  162.64   19.20\n",
      "   11        243H 243H N   CA   1824 1825  119.18  140.00                  -42.30\n",
      "   12    564 243H 244H C   N    1832 1834 -143.95 -125.60   21.55    0.55  -63.20 -173.95   28.52\n",
      "   12        244H 244H N   CA   1834 1835  150.09  138.80                  -42.30\n",
      "   13    565 244H 245H C   N    1842 1844 -121.93 -125.60    8.47    0.25  -63.20 -176.86   19.98\n",
      "   13        245H 245H N   CA   1844 1845  131.16  138.80                  -42.30\n",
      "   14    566 245H 246H C   N    1852 1854  -93.58  -67.60   29.08    1.81  -63.20  167.42   22.90\n",
      "   14        246H 246H N   CA   1854 1855  153.06  140.00                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864  -62.96  -67.60   18.97    1.32  -63.20  163.91   20.50\n",
      "   15        247H 247H N   CA   1864 1865  121.61  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    1   19   36  103   96  148  183  201  236  268  323\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.BL00060002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4896\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         279.0291\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      107.06       1.000\n",
      " 2 Bond angle potential               :     194       5     13   5.428   5.428      109.05       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      5  43.006  43.006      35.785       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.624   1.624      4.5450       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4896       0      0   0.002   0.002      3.4382       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      2   9.009   9.009      16.282       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      3  71.776  71.776      16.691       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2  94.620  94.620      12.849       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 120.562 120.562      2.7474       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  93.167  93.167      1.1729       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      17      7  45.265 153.027      51.394       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4896       0      9   1.896   1.896     -81.977       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.17    5.90    1.35    0.17    5.90\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.15    1.43    0.19    6.15\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.60    1.49    0.26    7.60\n",
      "    4     48  98Q  99E C   N     742  744    1.54    1.35    0.19    6.82    1.35    0.19    6.82\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.29    1.43    0.22    7.29\n",
      "    6     56  99E  99E C   CA    751  745    1.75    1.49    0.26    7.58    1.49    0.26    7.58\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.21    7.56    1.35    0.21    7.56\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    193  56S  57Y C   N     416  418  145.06  120.00   25.06    5.69  120.00   25.06    5.69\n",
      "    2    199  98Q  98Q N   CA    735  736  128.12  107.00   21.12    6.07  107.00   21.12    6.07\n",
      "    3    208  98Q  99E C   N     742  744  140.32  120.00   20.32    4.62  120.00   20.32    4.62\n",
      "    4    211  99E  99E N   CA    744  745  131.84  107.00   24.84    7.14  107.00   24.84    7.14\n",
      "    5    220  99E 100T C   N     751  753  148.67  120.00   28.67    6.51  120.00   28.67    6.51\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -156.05  180.00   23.95    4.79  180.00   23.95    4.79\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   57.70    1.82  -64.10 -162.90   23.34\n",
      "    1         51S  51S N   CA    376  377  173.28  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -140.06 -136.60    3.51    0.13  -64.10 -170.85   19.11\n",
      "    2         52S  52S N   CA    382  383  151.77  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -103.83  -98.40   43.75    4.60  -63.50  134.57   19.48\n",
      "    3         53Y  53Y N   CA    388  389   84.99  128.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -143.82 -136.60   47.67    2.55  -64.10  160.31   10.53\n",
      "    4         55S  55S N   CA    406  407  104.08  151.20                  -35.00\n",
      "    5    555  56S  57Y C   N     416  418 -135.04  -63.50   71.90   12.72  -63.50   71.90   12.72\n",
      "    5         57Y  57Y N   CA    418  419  -50.57  -43.40                  -43.40\n",
      "    6    556  96V  97Y C   N     721  723 -124.06 -124.30   10.04    0.54  -63.50 -178.44   31.87\n",
      "    6         97Y  97Y N   CA    723  724  145.43  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -161.24 -121.10   61.63    2.07  -63.80  165.06   29.11\n",
      "    7         98Q  98Q N   CA    735  736 -173.54  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -147.92 -117.80   43.72    1.42  -63.60  173.14   27.76\n",
      "    8         99E  99E N   CA    744  745  168.49  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -79.74  -78.10   27.90    1.14  -63.20  141.22   19.21\n",
      "    9        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.93 -117.80   12.62    0.59  -63.60  179.53   27.12\n",
      "   10        240E 240E N   CA   1798 1799  149.24  136.80                  -40.30\n",
      "   11    561 240E 241I C   N    1805 1807  -69.71  -63.40    6.47    1.19 -120.60 -177.44    8.56\n",
      "   11        241I 241I N   CA   1807 1808  -45.02  -43.60                  130.30\n",
      "   12    562 241I 242K C   N    1813 1815  -95.80  -62.90   80.13   12.38  -62.90   80.13   12.38\n",
      "   12        242K 242K N   CA   1815 1816 -113.87  -40.80                  -40.80\n",
      "   13    563 242K 243H C   N    1822 1824 -120.32 -125.60    8.18    0.21  -63.20 -176.05   20.13\n",
      "   13        243H 243H N   CA   1824 1825  132.55  138.80                  -42.30\n",
      "   14    564 243H 244H C   N    1832 1834  -82.03  -67.60   27.90    1.73  -63.20  154.97   20.58\n",
      "   14        244H 244H N   CA   1834 1835  163.88  140.00                  -42.30\n",
      "   15    565 244H 245H C   N    1842 1844  -88.05  -67.60   29.27    1.73  -63.20  158.73   21.45\n",
      "   15        245H 245H N   CA   1844 1845  160.93  140.00                  -42.30\n",
      "   16    566 245H 246H C   N    1852 1854  -59.54  -67.60   21.64    1.42  -63.20  162.26   20.51\n",
      "   16        246H 246H N   CA   1854 1855  119.92  140.00                  -42.30\n",
      "   17    567 246H 247H C   N    1862 1864 -106.25 -125.60   52.79    2.31  -63.20  136.73   19.91\n",
      "   17        247H 247H N   CA   1864 1865 -172.08  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   25   31   96   95  146  184  188  248  271  314\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      140.9568       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "openf___224_> Open           scfv.BL00070002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4684\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         255.4299\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.051   0.051      111.14       1.000\n",
      " 2 Bond angle potential               :     194       4      9   5.145   5.145      98.936       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      5  45.432  45.432      39.721       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.598   1.598      4.6262       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4684       0      0   0.001   0.001     0.25702       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      3   8.403   8.403      14.167       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      1  87.245  87.245      9.8456       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  87.452  87.452      7.7963       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  84.987  84.987      2.3768       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  15.547  15.547     0.36438       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      5  57.141 147.003      54.688       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4684       0     13   1.874   1.874     -88.489       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.53    1.35    0.18    6.43    1.35    0.18    6.43\n",
      "    2     45  98Q  98Q N   CA    735  736    1.63    1.43    0.20    6.63    1.43    0.20    6.63\n",
      "    3     47  98Q  98Q C   CA    742  736    1.76    1.49    0.27    7.95    1.49    0.27    7.95\n",
      "    4     48  98Q  99E C   N     742  744    1.54    1.35    0.19    6.90    1.35    0.19    6.90\n",
      "    5     54  99E  99E N   CA    744  745    1.66    1.43    0.23    7.63    1.43    0.23    7.63\n",
      "    6     56  99E  99E C   CA    751  745    1.75    1.49    0.26    7.53    1.49    0.26    7.53\n",
      "    7     57  99E 100T C   N     751  753    1.54    1.35    0.20    7.07    1.35    0.20    7.07\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  128.48  107.00   21.48    6.17  107.00   21.48    6.17\n",
      "    2    208  98Q  99E C   N     742  744  143.65  120.00   23.65    5.37  120.00   23.65    5.37\n",
      "    3    211  99E  99E N   CA    744  745  131.15  107.00   24.15    6.94  107.00   24.15    6.94\n",
      "    4    220  99E 100T C   N     751  753  152.12  120.00   32.12    7.30  120.00   32.12    7.30\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   56.29    1.82  -64.10 -159.80   23.57\n",
      "    1         51S  51S N   CA    376  377  169.27  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  178.05 -136.60   77.09    2.72  -64.10  162.20   20.16\n",
      "    2         52S  52S N   CA    382  383 -146.46  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388   35.31  -98.40  146.84    8.74  -63.50  148.68   27.92\n",
      "    3         53Y  53Y N   CA    388  389   67.69  128.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400 -158.44 -136.60   48.62    1.91  -64.10  160.92   18.47\n",
      "    4         54S  54S N   CA    400  401 -165.36  151.20                  -35.00\n",
      "    5    553  54S  55S C   N     404  406   28.87  -72.40  130.96    6.48  -64.10  139.76   16.81\n",
      "    5         55S  55S N   CA    406  407   69.36  152.40                  -35.00\n",
      "    6    554  55S  56S C   N     410  412 -143.98 -136.60   58.96    2.83  -64.10  140.26   15.93\n",
      "    6         56S  56S N   CA    412  413 -150.30  151.20                  -35.00\n",
      "    7    556  96V  97Y C   N     721  723 -124.06 -124.30   16.23    0.87  -63.50  175.74   30.94\n",
      "    7         97Y  97Y N   CA    723  724  151.62  135.40                  -43.40\n",
      "    8    557  97Y  98Q C   N     733  735 -168.43 -121.10   60.57    1.90  -63.80  176.56   31.15\n",
      "    8         98Q  98Q N   CA    735  736  177.49  139.70                  -40.30\n",
      "    9    558  98Q  99E C   N     742  744 -127.18 -117.80   34.19    1.51  -63.60  162.94   25.30\n",
      "    9         99E  99E N   CA    744  745  169.67  136.80                  -40.30\n",
      "   10    559  99E 100T C   N     751  753  -76.62  -78.10   27.89    1.24  -63.20  140.89   18.95\n",
      "   10        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "   11    560 239V 240E C   N    1796 1798 -119.93 -117.80    8.22    0.37  -63.60 -176.20   27.69\n",
      "   11        240E 240E N   CA   1798 1799  144.74  136.80                  -40.30\n",
      "   12    563 242K 243H C   N    1822 1824 -102.52 -125.60   29.74    1.34  -63.20  164.90   23.18\n",
      "   12        243H 243H N   CA   1824 1825  157.56  138.80                  -42.30\n",
      "   13    565 244H 245H C   N    1842 1844 -114.70 -125.60   17.73    0.81  -63.20  172.77   24.97\n",
      "   13        245H 245H N   CA   1844 1845  152.79  138.80                  -42.30\n",
      "   14    566 245H 246H C   N    1852 1854  -72.88  -67.60   25.03    2.07  -63.20  158.13   19.18\n",
      "   14        246H 246H N   CA   1854 1855  115.54  140.00                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864  -82.69  -67.60   16.07    1.36  -63.20  177.86   21.08\n",
      "   15        247H 247H N   CA   1864 1865  134.49  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   24   31   93   96  146  184  186  242  268  318\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.BL00080002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5766\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1004.4657\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141      10     12   0.067   0.067      184.24       1.000\n",
      " 2 Bond angle potential               :     194      11     23   7.146   7.146      193.43       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  44.670  44.670      44.079       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       1      2   3.476   3.476      19.867       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5766       7     12   0.016   0.016      179.56       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       4      6  17.476  17.476      61.279       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2  95.165  95.165      21.123       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  95.255  95.255      8.5065       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 108.624 108.624      3.0100       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.208   0.208     0.21710       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      9  58.303 145.983      75.372       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5766       0     72   1.911   1.911      213.78       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.50    1.35    0.16    5.61    1.35    0.16    5.61\n",
      "    2     45  98Q  98Q N   CA    735  736    1.60    1.43    0.17    5.61    1.43    0.17    5.61\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.46    1.49    0.26    7.46\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.61    1.35    0.19    6.61\n",
      "    5     54  99E  99E N   CA    744  745    1.64    1.43    0.21    6.80    1.43    0.21    6.80\n",
      "    6     56  99E  99E C   CA    751  745    1.79    1.49    0.30    8.61    1.49    0.30    8.61\n",
      "    7     57  99E 100T C   N     751  753    1.56    1.35    0.22    7.76    1.35    0.22    7.76\n",
      "    8     75 242K 243H C   N    1822 1824    1.48    1.35    0.13    4.60    1.35    0.13    4.60\n",
      "    9     82 243H 243H N   CA   1824 1825    1.59    1.43    0.16    5.38    1.43    0.16    5.38\n",
      "   10     84 243H 243H C   CA   1832 1825    1.94    1.49    0.45   13.21    1.49    0.45   13.21\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  127.58  107.00   20.58    5.92  107.00   20.58    5.92\n",
      "    2    211  99E  99E N   CA    744  745  132.08  107.00   25.08    7.21  107.00   25.08    7.21\n",
      "    3    220  99E 100T C   N     751  753  148.17  120.00   28.17    6.40  120.00   28.17    6.40\n",
      "    4    223 240E 241I C   N    1805 1807  147.49  120.00   27.49    6.25  120.00   27.49    6.25\n",
      "    5    245 242K 243H C   N    1822 1824  143.93  120.00   23.93    5.44  120.00   23.93    5.44\n",
      "    6    248 243H 243H N   CA   1824 1825  132.02  107.00   25.02    7.19  107.00   25.02    7.19\n",
      "    7    250 243H 243H CA  C    1825 1832  132.58  116.50   16.08    4.62  116.50   16.08    4.62\n",
      "    8    263 244H 244H N   CA   1834 1835  125.51  107.00   18.51    5.32  107.00   18.51    5.32\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -145.76  180.00   34.24    6.85  180.00   34.24    6.85\n",
      "    2    580 243H 243H CA  C    1825 1832 -138.03  180.00   41.97    8.39  180.00   41.97    8.39\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   54.73    1.87  -64.10 -155.34   23.90\n",
      "    1         51S  51S N   CA    376  377  163.57  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -144.47 -136.60    8.82    0.27  -64.10 -172.11   19.36\n",
      "    2         52S  52S N   CA    382  383  155.17  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -154.46 -124.30   32.74    1.96  -63.50 -168.57   34.78\n",
      "    3         53Y  53Y N   CA    388  389  148.16  135.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -63.24  -72.40   43.32    2.25  -64.10  145.06   10.56\n",
      "    4         54S  54S N   CA    400  401  110.06  152.40                  -35.00\n",
      "    5    553  54S  55S C   N     404  406  -67.82  -72.40   40.19    2.21  -64.10  147.52   10.47\n",
      "    5         55S  55S N   CA    406  407  112.47  152.40                  -35.00\n",
      "    6    554  55S  56S C   N     410  412   14.73  -64.10  110.92   13.67  -64.10  110.92   13.67\n",
      "    6         56S  56S N   CA    412  413   43.04  -35.00                  -35.00\n",
      "    7    556  96V  97Y C   N     721  723 -124.06 -124.30   11.36    0.61  -63.50 -179.69   31.67\n",
      "    7         97Y  97Y N   CA    723  724  146.76  135.40                  -43.40\n",
      "    8    557  97Y  98Q C   N     733  735 -164.68 -121.10   61.73    2.01  -63.80  169.55   29.93\n",
      "    8         98Q  98Q N   CA    735  736 -176.57  139.70                  -40.30\n",
      "    9    558  98Q  99E C   N     742  744 -159.29 -117.80   57.46    1.82  -63.60  172.18   28.17\n",
      "    9         99E  99E N   CA    744  745  176.56  136.80                  -40.30\n",
      "   10    559  99E 100T C   N     751  753  -81.76  -78.10   28.09    1.09  -63.20  141.48   19.37\n",
      "   10        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "   11    560 239V 240E C   N    1796 1798 -119.93 -117.80   18.75    0.90  -63.60  173.66   26.34\n",
      "   11        240E 240E N   CA   1798 1799  155.43  136.80                  -40.30\n",
      "   12    562 241I 242K C   N    1813 1815 -118.02 -118.00    1.58    0.08  -62.90 -173.35   21.88\n",
      "   12        242K 242K N   CA   1815 1816  137.52  139.10                  -40.80\n",
      "   13    563 242K 243H C   N    1822 1824 -133.30 -125.60   49.57    1.73  -63.20  147.63   23.00\n",
      "   13        243H 243H N   CA   1824 1825 -172.23  138.80                  -42.30\n",
      "   14    564 243H 244H C   N    1832 1834  -25.37  -67.60   86.36    7.63  -63.20  109.14   11.82\n",
      "   14        244H 244H N   CA   1834 1835 -144.67  140.00                  -42.30\n",
      "   15    565 244H 245H C   N    1842 1844 -144.29  -63.20   81.98   13.53  -63.20   81.98   13.53\n",
      "   15        245H 245H N   CA   1844 1845  -54.41  -42.30                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1081 1832   2.237\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    1    3    1   24   33  111  113  168  193  205  244  280  330\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:       96.8407       99    99\n",
      "openf___224_> Open           scfv.BL00090002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000002.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4514\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         250.3216\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      106.57       1.000\n",
      " 2 Bond angle potential               :     194       4     12   5.286   5.286      106.61       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  41.156  41.156      35.420       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.501   1.501      4.3572       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4514       0      0   0.001   0.001     0.42437       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      3   8.409   8.409      14.187       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2 102.014 102.014      17.097       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0 101.449 101.449      6.7393       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  15.176  15.176      1.9202       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 125.402 125.402      1.0249       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      7  54.193 148.069      48.032       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4514       0      9   1.895   1.895     -92.064       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.52    1.35    0.17    6.16    1.35    0.17    6.16\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.38    1.43    0.19    6.38\n",
      "    3     47  98Q  98Q C   CA    742  736    1.76    1.49    0.27    7.88    1.49    0.27    7.88\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.68    1.35    0.19    6.68\n",
      "    5     54  99E  99E N   CA    744  745    1.66    1.43    0.23    7.46    1.43    0.23    7.46\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.73    1.49    0.27    7.73\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.13    1.35    0.20    7.13\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  128.78  107.00   21.78    6.26  107.00   21.78    6.26\n",
      "    2    208  98Q  99E C   N     742  744  141.75  120.00   21.75    4.94  120.00   21.75    4.94\n",
      "    3    211  99E  99E N   CA    744  745  132.21  107.00   25.21    7.24  107.00   25.21    7.24\n",
      "    4    220  99E 100T C   N     751  753  151.63  120.00   31.63    7.19  120.00   31.63    7.19\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  170.09 -136.60   55.83    1.83  -64.10 -158.62   23.66\n",
      "    1         51S  51S N   CA    376  377  167.76  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  146.19 -136.60   86.81    2.67  -64.10 -159.00   25.20\n",
      "    2         52S  52S N   CA    382  383 -169.11  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388  168.99 -124.30  133.10    5.24  -63.50  143.59   27.02\n",
      "    3         53Y  53Y N   CA    388  389 -109.42  135.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -84.81  -72.40   30.26    1.42  -64.10  146.47   11.97\n",
      "    4         54S  54S N   CA    400  401  180.00  152.40                  -35.00\n",
      "    5    553  54S  55S C   N     404  406 -155.65 -136.60   85.40    3.89  -64.10  128.77   15.87\n",
      "    5         55S  55S N   CA    406  407 -125.55  151.20                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -124.06 -124.30   16.40    0.88  -63.50  175.58   30.91\n",
      "    6         97Y  97Y N   CA    723  724  151.80  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -171.74 -121.10   63.75    1.99  -63.80  177.78   31.46\n",
      "    7         98Q  98Q N   CA    735  736  178.44  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -129.15 -117.80   35.97    1.55  -63.60  162.57   25.36\n",
      "    8         99E  99E N   CA    744  745  170.93  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -75.94  -78.10   27.93    1.27  -63.20  140.83   18.90\n",
      "    9        100T 100T N   CA    753  754  177.65  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -119.93 -117.80    7.45    0.33  -63.60 -175.44   27.80\n",
      "   10        240E 240E N   CA   1798 1799  143.94  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815  -62.13  -70.20   18.61    1.17  -62.90  164.43   21.40\n",
      "   11        242K 242K N   CA   1815 1816  123.63  140.40                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824 -100.35 -125.60   29.41    1.29  -63.20  167.98   23.42\n",
      "   12        243H 243H N   CA   1824 1825  153.88  138.80                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834 -152.44 -125.60   82.35    2.57  -63.20  134.81   22.36\n",
      "   13        244H 244H N   CA   1834 1835 -143.35  138.80                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844  -69.26  -67.60   20.01    1.59  -63.20  162.47   19.94\n",
      "   14        245H 245H N   CA   1844 1845  120.06  140.00                  -42.30\n",
      "   15    566 245H 246H C   N    1852 1854 -162.94 -125.60   49.09    1.21  -63.20  177.68   28.56\n",
      "   15        246H 246H N   CA   1854 1855  170.66  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.046\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   24   30   90   96  147  180  181  241  266  325\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:      168.7319       99    99\n",
      "openf___224_> Open           scfv.BL00100002.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.B99990003.pdb\n",
      "readlinef__W> File: scfv.B99990003.pdb, Line: 6\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18101636   17677.379    17.263\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18109380   17684.941    17.270\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18251072   17823.312    17.406\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18253480   17825.664    17.408\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395172   17964.035    17.543\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933      0         0       876\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933   2622      3114       876\n",
      "\n",
      "patch_______> segment topology patched using RTF:   248  ;  HIS  ;  CTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "openf___224_> Open           scfv.B99990003.pdb\n",
      "readlinef__W> File: scfv.B99990003.pdb, Line: 6\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18406145   17974.750    17.553\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18406221   17974.824    17.554\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18406392   17974.992    17.554\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18407242   17975.822    17.555\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18408500   17977.051    17.556\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18410404   17978.910    17.558\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18410444   17978.949    17.558\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18410534   17979.037    17.558\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18413390   17981.826    17.560\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18413450   17981.885    17.560\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18413585   17982.016    17.561\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18417869   17986.199    17.565\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18417957   17986.285    17.565\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18418155   17986.480    17.565\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18424581   17992.754    17.571\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18424713   17992.883    17.571\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18425010   17993.174    17.571\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18434632   18002.570    17.581\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18434832   18002.766    17.581\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18435282   18003.205    17.581\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18449732   18017.316    17.595\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18450032   18017.609    17.595\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18450707   18018.270    17.596\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18472365   18039.418    17.617\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18472813   18039.855    17.617\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     18473821   18040.840    17.618\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18644509   18207.527    17.781\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  scfv\n",
      "  2  scfv\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1884        0\n",
      "openf___224_> Open           scfv.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18397174   17965.990    17.545\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18398624   17967.406    17.546\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18401524   17970.238    17.549\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18407324   17975.902    17.555\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     18524380   18090.215    17.666\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18525284   18091.098    17.667\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     18526272   18092.062    17.668\n",
      "\n",
      "Read the alignment from file       : scfv.ali\n",
      "Total number of alignment positions:   253\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1     2fjhHL     227      1  2fjhHL.pdb\n",
      "  2       scfv     248      1        scfv\n",
      "132 atoms selected for loop refinement\n",
      "make_re_417_> Restraint type to be calculated:  bond\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      141       0       0       0\n",
      "              Total number of restraints before, now         :        0      141\n",
      "make_re_422_> Number of previous, current restraints         :        0      141\n",
      "make_re_423_> Number of previous, current selected restraints:        0      141\n",
      "make_re_417_> Restraint type to be calculated:  angle\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :        0     194       0       0\n",
      "              Total number of restraints before, now         :      141      335\n",
      "make_re_422_> Number of previous, current restraints         :      141      335\n",
      "make_re_423_> Number of previous, current selected restraints:      141      335\n",
      "make_re_417_> Restraint type to be calculated:  improper\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :        0       0       0      81\n",
      "              Total number of restraints before, now         :      335      416\n",
      "make_re_422_> Number of previous, current restraints         :      335      416\n",
      "make_re_423_> Number of previous, current selected restraints:      335      416\n",
      "make_re_417_> Restraint type to be calculated:  dihedral\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :        0       0     235       0\n",
      "              Total number of restraints before, now         :      416      651\n",
      "make_re_422_> Number of previous, current restraints         :      416      651\n",
      "make_re_423_> Number of previous, current selected restraints:      416      651\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395172   17964.035    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      651      670\n",
      "make_re_423_> Number of previous, current selected restraints:      651      670\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      670      687\n",
      "make_re_423_> Number of previous, current selected restraints:      670      670\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      687      702\n",
      "make_re_423_> Number of previous, current selected restraints:      670      685\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      702      713\n",
      "make_re_423_> Number of previous, current selected restraints:      685      696\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      713      716\n",
      "make_re_423_> Number of previous, current selected restraints:      696      699\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     18395188   17964.051    17.543\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL9v21}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  scfv\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      716      717\n",
      "make_re_423_> Number of previous, current selected restraints:      699      700\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;      132      717\n",
      "rmdupl__427_>      103 redundant cosine dihedral restraints were unselected.\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:      717      614\n",
      "openf___224_> Open           scfv.lrsr\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4461\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         342.9465\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       6      7   0.051   0.051      112.07       1.000\n",
      " 2 Bond angle potential               :     194       5     10   4.561   4.561      80.137       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  45.108  45.108      41.799       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   0.746   0.746      1.2502       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4461       0      0   0.001   0.001     0.83647       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      1   7.015   7.015      9.8720       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0  97.353  97.353      8.9300       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0 108.786 108.786      7.8923       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 117.620 117.620      3.9585       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  33.259  33.259      1.0861       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      4  35.295 150.821      23.022       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4461       0     17   1.912   1.912      52.094       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     45  98Q  98Q N   CA    735  736    1.59    1.43    0.16    5.25    1.43    0.16    5.25\n",
      "    2     47  98Q  98Q C   CA    742  736    1.79    1.49    0.30    8.66    1.49    0.30    8.66\n",
      "    3     48  98Q  99E C   N     742  744    1.60    1.35    0.25    8.97    1.35    0.25    8.97\n",
      "    4     54  99E  99E N   CA    744  745    1.67    1.43    0.24    7.86    1.43    0.24    7.86\n",
      "    5     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.85    1.49    0.27    7.85\n",
      "    6     57  99E 100T C   N     751  753    1.54    1.35    0.20    7.06    1.35    0.20    7.06\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  126.64  107.00   19.64    5.64  107.00   19.64    5.64\n",
      "    2    202  98Q  98Q CA  C     736  742  133.42  116.50   16.92    4.86  116.50   16.92    4.86\n",
      "    3    208  98Q  99E C   N     742  744  145.40  120.00   25.40    5.77  120.00   25.40    5.77\n",
      "    4    211  99E  99E N   CA    744  745  129.69  107.00   22.69    6.52  107.00   22.69    6.52\n",
      "    5    220  99E 100T C   N     751  753  147.93  120.00   27.93    6.34  120.00   27.93    6.34\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -157.13  180.00   22.87    4.57  180.00   22.87    4.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   56.96    1.77  -64.10 -165.57   23.03\n",
      "    1         51S  51S N   CA    376  377  175.32  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -137.23 -136.60    7.90    0.39  -64.10 -178.67   18.35\n",
      "    2         52S  52S N   CA    382  383  159.07  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -109.95  -98.40   42.23    4.29  -63.50  139.16   19.97\n",
      "    3         53Y  53Y N   CA    388  389   87.78  128.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -161.59 -136.60   37.20    2.09  -64.10 -173.79   12.48\n",
      "    4         55S  55S N   CA    406  407  123.65  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -123.18 -124.30   13.71    0.77  -63.50  177.85   31.23\n",
      "    5         97Y  97Y N   CA    723  724  149.06  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -153.63 -121.10   59.09    2.12  -63.80  158.57   27.82\n",
      "    6         98Q  98Q N   CA    735  736 -170.98  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -141.14 -117.80   44.15    1.64  -63.60  164.80   26.29\n",
      "    7         99E  99E N   CA    744  745  174.27  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753  -94.68  -78.10   36.23    1.18  -63.20  139.48   19.98\n",
      "    8        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -115.46 -117.80    7.63    0.41  -63.60 -176.86   27.35\n",
      "    9        240E 240E N   CA   1798 1799  144.06  136.80                  -40.30\n",
      "   10    562 241I 242K C   N    1813 1815  -97.23 -118.00   54.58    2.30  -62.90  133.90   15.85\n",
      "   10        242K 242K N   CA   1815 1816   88.63  139.10                  -40.80\n",
      "   11    563 242K 243H C   N    1822 1824 -163.33 -125.60   45.22    2.00  -63.20 -174.49   19.78\n",
      "   11        243H 243H N   CA   1824 1825  113.87  138.80                  -42.30\n",
      "   12    564 243H 244H C   N    1832 1834 -147.23 -125.60   48.56    1.40  -63.20  159.37   25.32\n",
      "   12        244H 244H N   CA   1834 1835 -177.72  138.80                  -42.30\n",
      "   13    565 244H 245H C   N    1842 1844  -65.11  -67.60    2.61    0.22  -63.20  176.91   22.23\n",
      "   13        245H 245H N   CA   1844 1845  140.81  140.00                  -42.30\n",
      "   14    566 245H 246H C   N    1852 1854 -155.17 -125.60   46.55    1.19  -63.20  169.97   27.13\n",
      "   14        246H 246H N   CA   1854 1855  174.76  138.80                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864  -76.52  -67.60   10.56    0.93  -63.20  177.16   21.34\n",
      "   15        247H 247H N   CA   1864 1865  134.36  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   12   27   89   92  126  165  160  241  269  301\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4942\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1375.4840\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      107.00       1.000\n",
      " 2 Bond angle potential               :     194       9     19  12.771  12.771      1135.3       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      5  39.639  39.639      35.266       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.878   1.878      6.3958       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4942       0      0   0.002   0.002      2.3510       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       2      3  11.139  11.139      24.895       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2 106.515 106.515      15.346       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2 108.174 108.174      22.487       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  74.536  74.536      2.7846       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 129.196 129.196      1.0912       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      16      7  39.297 146.565      37.390       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4942       0     20   1.891   1.891     -14.775       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.16    5.73    1.35    0.16    5.73\n",
      "    2     45  98Q  98Q N   CA    735  736    1.61    1.43    0.18    5.98    1.43    0.18    5.98\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.70    1.49    0.26    7.70\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.64    1.35    0.19    6.64\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.31    1.43    0.22    7.31\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.98    1.49    0.27    7.98\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.20    1.35    0.20    7.20\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  127.96  107.00   20.96    6.02  107.00   20.96    6.02\n",
      "    2    211  99E  99E N   CA    744  745  130.12  107.00   23.12    6.64  107.00   23.12    6.64\n",
      "    3    220  99E 100T C   N     751  753  147.17  120.00   27.17    6.17  120.00   27.17    6.17\n",
      "    4    296 246H 246H ND1 CG   1856 1857  145.45  124.00   21.45    4.66  124.00   21.45    4.66\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -156.55  180.00   23.45    4.69  180.00   23.45    4.69\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   56.31    1.76  -64.10 -164.34   23.12\n",
      "    1         51S  51S N   CA    376  377  173.74  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  -85.30  -72.40   70.37    3.71  -64.10  105.58    9.08\n",
      "    2         52S  52S N   CA    382  383 -138.43  152.40                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -119.74 -124.30   28.59    1.71  -63.50  162.99   28.69\n",
      "    3         53Y  53Y N   CA    388  389  163.63  135.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -80.04  -72.40   68.35    4.29  -64.10  120.54    7.92\n",
      "    4         54S  54S N   CA    400  401   84.48  152.40                  -35.00\n",
      "    5    553  54S  55S C   N     404  406  -40.79  -72.40   63.08    2.90  -64.10  134.84   11.32\n",
      "    5         55S  55S N   CA    406  407   97.81  152.40                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -123.18 -124.30   14.72    0.83  -63.50  176.90   31.08\n",
      "    6         97Y  97Y N   CA    723  724  150.08  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -164.71 -121.10   58.16    1.85  -63.80  173.81   30.59\n",
      "    7         98Q  98Q N   CA    735  736  178.19  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -138.31 -117.80   40.99    1.56  -63.60  165.27   26.20\n",
      "    8         99E  99E N   CA    744  745  172.29  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -90.68  -78.10   34.58    1.16  -63.20  138.63   19.61\n",
      "    9        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -115.46 -117.80    8.26    0.45  -63.60 -177.50   27.27\n",
      "   10        240E 240E N   CA   1798 1799  144.73  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815  -95.32 -118.00   22.95    0.93  -62.90  179.54   25.21\n",
      "   11        242K 242K N   CA   1815 1816  142.61  139.10                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824  -66.59  -67.60   10.01    0.74  -63.20  172.38   21.34\n",
      "   12        243H 243H N   CA   1824 1825  130.04  140.00                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834  -80.06  -67.60   34.83    2.30  -63.20  146.15   19.35\n",
      "   13        244H 244H N   CA   1834 1835  172.53  140.00                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844  -83.02  -67.60   31.67    1.98  -63.20  151.33   20.19\n",
      "   14        245H 245H N   CA   1844 1845  167.67  140.00                  -42.30\n",
      "   15    566 245H 246H C   N    1852 1854  -57.85  -67.60   20.86    1.32  -63.20  163.94   20.83\n",
      "   15        246H 246H N   CA   1854 1855  121.55  140.00                  -42.30\n",
      "   16    567 246H 247H C   N    1862 1864  -63.39  -67.60   28.69    2.08  -63.20  153.92   19.23\n",
      "   16        247H 247H N   CA   1864 1865  111.62  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    1   14   26   94   91  129  172  178  246  273  308\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -173.0138       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00010003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4943\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         186.3119\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.049   0.049      104.53       1.000\n",
      " 2 Bond angle potential               :     194       4      8   4.773   4.773      87.205       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      5  45.126  45.126      39.459       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.222   1.222      2.6899       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4943       0      0   0.001   0.001     0.91477       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      2   7.796   7.796      12.193       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      1  88.355  88.355      10.215       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0 118.124 118.124      6.9741       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  68.881  68.881      2.5586       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  93.543  93.543      1.1692       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      13      3  32.583 140.295      18.671       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4943       0      8   1.875   1.875     -100.27       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.52    1.35    0.17    6.04    1.35    0.17    6.04\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.23    1.43    0.19    6.23\n",
      "    3     47  98Q  98Q C   CA    742  736    1.74    1.49    0.25    7.37    1.49    0.25    7.37\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.54    1.35    0.18    6.54\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.24    1.43    0.22    7.24\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.75    1.49    0.27    7.75\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.21    7.28    1.35    0.21    7.28\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  127.87  107.00   20.87    6.00  107.00   20.87    6.00\n",
      "    2    208  98Q  99E C   N     742  744  140.31  120.00   20.31    4.61  120.00   20.31    4.61\n",
      "    3    211  99E  99E N   CA    744  745  131.57  107.00   24.57    7.06  107.00   24.57    7.06\n",
      "    4    220  99E 100T C   N     751  753  146.82  120.00   26.82    6.09  120.00   26.82    6.09\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -157.07  180.00   22.93    4.59  180.00   22.93    4.59\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   56.64    1.76  -64.10 -164.97   23.07\n",
      "    1         51S  51S N   CA    376  377  174.55  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -147.72 -136.60   11.67    0.38  -64.10 -170.33   19.73\n",
      "    2         52S  52S N   CA    382  383  154.75  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -103.07  -98.40   33.96    3.56  -63.50  143.72   20.95\n",
      "    3         53Y  53Y N   CA    388  389   94.77  128.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -149.16 -136.60   38.98    2.17  -64.10  171.82   11.27\n",
      "    4         55S  55S N   CA    406  407  114.29  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -123.18 -124.30   13.75    0.78  -63.50  177.81   31.23\n",
      "    5         97Y  97Y N   CA    723  724  149.11  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -165.01 -121.10   59.60    1.91  -63.80  172.51   30.40\n",
      "    6         98Q  98Q N   CA    735  736 -180.00  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -134.85 -117.80   39.29    1.57  -63.60  163.81   25.83\n",
      "    7         99E  99E N   CA    744  745  172.20  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753  -91.03  -78.10   34.71    1.16  -63.20  138.70   19.64\n",
      "    8        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -115.46 -117.80   10.83    0.58  -63.60  179.96   26.93\n",
      "    9        240E 240E N   CA   1798 1799  147.38  136.80                  -40.30\n",
      "   10    563 242K 243H C   N    1822 1824  -83.19  -67.60   72.22    5.98  -63.20  113.56   13.06\n",
      "   10        243H 243H N   CA   1824 1825   69.49  140.00                  -42.30\n",
      "   11    564 243H 244H C   N    1832 1834 -132.49 -125.60    8.65    0.39  -63.20 -170.97   20.38\n",
      "   11        244H 244H N   CA   1834 1835  133.57  138.80                  -42.30\n",
      "   12    565 244H 245H C   N    1842 1844  -89.10  -67.60   28.31    1.68  -63.20  161.37   21.85\n",
      "   12        245H 245H N   CA   1844 1845  158.42  140.00                  -42.30\n",
      "   13    567 246H 247H C   N    1862 1864 -112.84 -125.60   12.83    0.48  -63.20 -175.64   26.29\n",
      "   13        247H 247H N   CA   1864 1865  140.15  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   16   26   90   91  121  163  176  241  268  294\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:     -106.5762       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      159.2431       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00020003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4878\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         323.5203\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.049   0.049      105.09       1.000\n",
      " 2 Bond angle potential               :     194       5     11   5.183   5.183      101.20       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      3  40.790  40.790      35.080       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.718   1.718      4.8895       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4878       0      0   0.001   0.001     0.91255       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      3   9.177   9.177      16.896       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2 102.096 102.096      20.508       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2 114.323 114.323      17.728       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  53.412  53.412      3.0683       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 107.498 107.498     0.95533       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      5  49.156 145.637      58.035       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4878       0     14   1.878   1.878     -40.843       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.52    1.35    0.17    6.07    1.35    0.17    6.07\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.39    1.43    0.19    6.39\n",
      "    3     47  98Q  98Q C   CA    742  736    1.74    1.49    0.25    7.42    1.49    0.25    7.42\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.50    1.35    0.18    6.50\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.25    1.43    0.22    7.25\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.91    1.49    0.27    7.91\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.15    1.35    0.20    7.15\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    151  52S  53Y C   N     386  388  142.25  120.00   22.25    5.06  120.00   22.25    5.06\n",
      "    2    199  98Q  98Q N   CA    735  736  127.15  107.00   20.15    5.79  107.00   20.15    5.79\n",
      "    3    208  98Q  99E C   N     742  744  141.16  120.00   21.16    4.81  120.00   21.16    4.81\n",
      "    4    211  99E  99E N   CA    744  745  131.18  107.00   24.18    6.95  107.00   24.18    6.95\n",
      "    5    220  99E 100T C   N     751  753  147.61  120.00   27.61    6.27  120.00   27.61    6.27\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -156.88  180.00   23.12    4.62  180.00   23.12    4.62\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   57.44    1.77  -64.10 -166.42   22.96\n",
      "    1         51S  51S N   CA    376  377  176.44  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -144.20 -136.60   14.14    0.80  -64.10 -168.20   12.05\n",
      "    2         52S  52S N   CA    382  383  139.28  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388   17.63  -98.40  124.72    6.21  -63.50  163.96   22.79\n",
      "    3         53Y  53Y N   CA    388  389  174.13  128.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400 -150.75 -136.60   88.12    4.15  -64.10  122.67   15.09\n",
      "    4         54S  54S N   CA    400  401 -121.83  151.20                  -35.00\n",
      "    5    555  56S  57Y C   N     416  418 -112.29  -63.50   50.43    9.22  -63.50   50.43    9.22\n",
      "    5         57Y  57Y N   CA    418  419  -56.15  -43.40                  -43.40\n",
      "    6    556  96V  97Y C   N     721  723 -123.18 -124.30   13.25    0.75  -63.50  178.28   31.30\n",
      "    6         97Y  97Y N   CA    723  724  148.61  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -163.80 -121.10   60.07    1.95  -63.80  170.22   30.00\n",
      "    7         98Q  98Q N   CA    735  736 -178.05  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -135.13 -117.80   38.81    1.54  -63.60  164.53   25.94\n",
      "    8         99E  99E N   CA    744  745  171.53  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -90.34  -78.10   34.46    1.16  -63.20  138.57   19.58\n",
      "    9        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -115.46 -117.80   10.84    0.58  -63.60  179.95   26.93\n",
      "   10        240E 240E N   CA   1798 1799  147.39  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815  -91.33  -70.20   26.48    1.60  -62.90  165.30   23.12\n",
      "   11        242K 242K N   CA   1815 1816  156.37  140.40                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824  -55.63  -67.60   24.66    1.54  -63.20  160.92   20.59\n",
      "   12        243H 243H N   CA   1824 1825  118.44  140.00                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834  -81.61  -67.60   21.49    1.93  -63.20  167.01   19.79\n",
      "   13        244H 244H N   CA   1834 1835  123.69  140.00                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844  -96.85  -67.60   36.04    2.16  -63.20  160.22   22.22\n",
      "   14        245H 245H N   CA   1844 1845  161.05  140.00                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864 -113.35 -125.60   16.46    0.41  -63.20  177.34   19.58\n",
      "   15        247H 247H N   CA   1864 1865  127.80  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   16   26   93   89  124  168  173  260  272  310\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:      115.1200       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00030003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4898\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         385.8716\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.050   0.050      105.96       1.000\n",
      " 2 Bond angle potential               :     194       4     12   5.224   5.224      103.68       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  41.374  41.374      38.861       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      1   2.558   2.558      11.451       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4898       0      0   0.002   0.002      1.8507       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      3   9.191   9.191      16.947       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      3  94.403  94.403      18.232       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2  87.369  87.369      16.825       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  71.524  71.524      3.1788       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  14.995  14.995     0.35463       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      16      9  64.582 147.083      64.214       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4898       0     16   1.884   1.884      4.3229       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.17    6.01    1.35    0.17    6.01\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.30    1.43    0.19    6.30\n",
      "    3     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.60    1.49    0.26    7.60\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.63    1.35    0.19    6.63\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.24    1.43    0.22    7.24\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.90    1.49    0.27    7.90\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.25    1.35    0.20    7.25\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  127.01  107.00   20.01    5.75  107.00   20.01    5.75\n",
      "    2    211  99E  99E N   CA    744  745  130.83  107.00   23.83    6.85  107.00   23.83    6.85\n",
      "    3    220  99E 100T C   N     751  753  147.08  120.00   27.08    6.15  120.00   27.08    6.15\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -157.16  180.00   22.84    4.57  180.00   22.84    4.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   58.66    1.80  -64.10 -168.46   22.81\n",
      "    1         51S  51S N   CA    376  377  179.10  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  -99.96  -72.40   68.74    3.25  -64.10  115.35   10.86\n",
      "    2         52S  52S N   CA    382  383 -144.63  152.40                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -127.54 -124.30   21.94    1.30  -63.50  169.65   24.09\n",
      "    3         53Y  53Y N   CA    388  389  113.70  135.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -83.07  -72.40   14.14    0.69  -64.10  164.41   13.14\n",
      "    4         54S  54S N   CA    400  401  161.68  152.40                  -35.00\n",
      "    5    553  54S  55S C   N     404  406   57.73  -72.40  145.81   11.67  -64.10  162.03   13.86\n",
      "    5         55S  55S N   CA    406  407 -141.82  152.40                  -35.00\n",
      "    6    554  55S  56S C   N     410  412   68.70  -64.10  156.32   15.22  -64.10  156.32   15.22\n",
      "    6         56S  56S N   CA    412  413 -117.46  -35.00                  -35.00\n",
      "    7    556  96V  97Y C   N     721  723 -123.18 -124.30   13.20    0.75  -63.50  178.33   31.31\n",
      "    7         97Y  97Y N   CA    723  724  148.56  135.40                  -43.40\n",
      "    8    557  97Y  98Q C   N     733  735 -163.11 -121.10   60.82    2.00  -63.80  168.42   29.69\n",
      "    8         98Q  98Q N   CA    735  736 -176.32  139.70                  -40.30\n",
      "    9    558  98Q  99E C   N     742  744 -138.57 -117.80   41.63    1.59  -63.60  164.86   26.16\n",
      "    9         99E  99E N   CA    744  745  172.88  136.80                  -40.30\n",
      "   10    559  99E 100T C   N     751  753  -90.29  -78.10   34.45    1.16  -63.20  138.56   19.57\n",
      "   10        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "   11    560 239V 240E C   N    1796 1798 -115.46 -117.80   10.05    0.54  -63.60 -179.27   27.03\n",
      "   11        240E 240E N   CA   1798 1799  146.57  136.80                  -40.30\n",
      "   12    562 241I 242K C   N    1813 1815 -144.73 -118.00   48.63    2.51  -62.90  161.54   18.59\n",
      "   12        242K 242K N   CA   1815 1816   98.48  139.10                  -40.80\n",
      "   13    563 242K 243H C   N    1822 1824 -141.35 -125.60   18.62    0.47  -63.20 -173.82   28.37\n",
      "   13        243H 243H N   CA   1824 1825  148.72  138.80                  -42.30\n",
      "   14    564 243H 244H C   N    1832 1834 -109.68 -125.60   47.84    2.07  -63.20  141.64   20.75\n",
      "   14        244H 244H N   CA   1834 1835 -176.09  138.80                  -42.30\n",
      "   15    565 244H 245H C   N    1842 1844 -133.38 -125.60   97.84    3.57  -63.20  107.45   17.78\n",
      "   15        245H 245H N   CA   1844 1845 -123.67  138.80                  -42.30\n",
      "   16    567 246H 247H C   N    1862 1864  -99.97 -125.60   44.66    2.02  -63.20  147.00   20.78\n",
      "   16        247H 247H N   CA   1864 1865  175.38  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   14   27   95   88  133  171  174  244  270  307\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:       95.4391       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00040003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5276\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1937.1132\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141      16     23   0.108   0.108      424.58       1.000\n",
      " 2 Bond angle potential               :     194       8     23   7.028   7.028      190.90       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      5  42.214  42.214      40.571       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       1      3   3.800   3.800      29.809       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5276      26     31   0.038   0.038      910.59       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       2      3  13.674  13.674      37.513       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      3  92.566  92.566      14.557       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      1  70.487  70.487      10.152       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  66.978  66.978      2.5518       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 105.716 105.716     0.96909       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      17     11  66.181 158.900      70.613       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5276       0     63   1.897   1.897      204.31       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.51    1.35    0.17    5.99    1.35    0.17    5.99\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.17    1.43    0.19    6.17\n",
      "    3     47  98Q  98Q C   CA    742  736    1.74    1.49    0.25    7.32    1.49    0.25    7.32\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.59    1.35    0.19    6.59\n",
      "    5     54  99E  99E N   CA    744  745    1.64    1.43    0.21    7.08    1.43    0.21    7.08\n",
      "    6     56  99E  99E C   CA    751  745    1.75    1.49    0.26    7.71    1.49    0.26    7.71\n",
      "    7     57  99E 100T C   N     751  753    1.54    1.35    0.20    6.92    1.35    0.20    6.92\n",
      "    8     65 241I 241I C   CA   1813 1808    1.67    1.49    0.18    5.12    1.49    0.18    5.12\n",
      "    9     72 242K 242K N   CA   1815 1816    1.57    1.43    0.14    4.67    1.43    0.14    4.67\n",
      "   10     74 242K 242K C   CA   1822 1816    1.66    1.49    0.17    4.82    1.49    0.17    4.82\n",
      "   11     82 243H 243H N   CA   1824 1825    1.57    1.43    0.14    4.75    1.43    0.14    4.75\n",
      "   12     84 243H 243H C   CA   1832 1825    1.65    1.49    0.16    4.66    1.49    0.16    4.66\n",
      "   13     87 244H 244H CB  CA   1838 1835    2.00    1.54    0.46   12.56    1.54    0.46   12.56\n",
      "   14     88 244H 244H CG  CB   1837 1838    2.44    1.50    0.94   26.10    1.50    0.94   26.10\n",
      "   15     93 244H 244H N   CA   1834 1835    1.58    1.43    0.15    4.90    1.43    0.15    4.90\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  126.93  107.00   19.93    5.73  107.00   19.93    5.73\n",
      "    2    211  99E  99E N   CA    744  745  132.88  107.00   25.88    7.44  107.00   25.88    7.44\n",
      "    3    220  99E 100T C   N     751  753  144.84  120.00   24.84    5.64  120.00   24.84    5.64\n",
      "    4    234 241I 242K C   N    1813 1815  139.81  120.00   19.81    4.50  120.00   19.81    4.50\n",
      "    5    260 243H 244H C   N    1832 1834  144.24  120.00   24.24    5.51  120.00   24.24    5.51\n",
      "    6    268 244H 244H CG  CB   1837 1838  155.29  113.00   42.29   10.38  113.00   42.29   10.38\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -145.17  180.00   34.83    6.97  180.00   34.83    6.97\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   53.81    1.78  -64.10 -158.66   23.54\n",
      "    1         51S  51S N   CA    376  377  166.46  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  -96.56  -72.40   41.76    1.91  -64.10  142.28   12.52\n",
      "    2         52S  52S N   CA    382  383 -173.53  152.40                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388  -26.03  -98.40   73.15    2.89  -63.50 -178.59   26.97\n",
      "    3         53Y  53Y N   CA    388  389  139.10  128.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -97.39  -72.40   75.65    3.68  -64.10  106.53   10.05\n",
      "    4         54S  54S N   CA    400  401 -136.20  152.40                  -35.00\n",
      "    5    553  54S  55S C   N     404  406  -82.27  -72.40   17.61    0.80  -64.10  159.06   12.70\n",
      "    5         55S  55S N   CA    406  407  166.98  152.40                  -35.00\n",
      "    6    554  55S  56S C   N     410  412  153.51 -136.60  137.68    5.18  -64.10  152.71   20.41\n",
      "    6         56S  56S N   CA    412  413  -90.18  151.20                  -35.00\n",
      "    7    556  96V  97Y C   N     721  723 -123.18 -124.30   13.66    0.77  -63.50  177.89   31.24\n",
      "    7         97Y  97Y N   CA    723  724  149.02  135.40                  -43.40\n",
      "    8    557  97Y  98Q C   N     733  735 -167.81 -121.10   61.22    1.93  -63.80  174.76   30.85\n",
      "    8         98Q  98Q N   CA    735  736  179.27  139.70                  -40.30\n",
      "    9    558  98Q  99E C   N     742  744 -146.32 -117.80   56.84    2.16  -63.60  157.25   25.51\n",
      "    9         99E  99E N   CA    744  745 -174.04  136.80                  -40.30\n",
      "   10    559  99E 100T C   N     751  753  -93.08  -78.10   35.53    1.17  -63.20  139.13   19.83\n",
      "   10        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "   11    560 239V 240E C   N    1796 1798 -115.46 -117.80   45.77    2.34  -63.60  146.66   22.48\n",
      "   11        240E 240E N   CA   1798 1799 -177.49  136.80                  -40.30\n",
      "   12    562 241I 242K C   N    1813 1815  152.46 -118.00   90.01    3.56  -62.90 -136.23   26.53\n",
      "   12        242K 242K N   CA   1815 1816  129.93  139.10                  -40.80\n",
      "   13    563 242K 243H C   N    1822 1824  123.57   56.30  143.37   10.65  -63.20  178.61   30.04\n",
      "   13        243H 243H N   CA   1824 1825  -85.81   40.80                  -42.30\n",
      "   14    564 243H 244H C   N    1832 1834    1.40  -67.60   69.00    5.17  -63.20 -170.47   20.56\n",
      "   14        244H 244H N   CA   1834 1835  139.51  140.00                  -42.30\n",
      "   15    565 244H 245H C   N    1842 1844 -105.50 -125.60   46.74    2.07  -63.20  143.10   20.66\n",
      "   15        245H 245H N   CA   1844 1845 -179.00  138.80                  -42.30\n",
      "   16    566 245H 246H C   N    1852 1854  -55.46  -67.60   14.54    0.88  -63.20  174.48   22.30\n",
      "   16        246H 246H N   CA   1854 1855  132.00  140.00                  -42.30\n",
      "   17    567 246H 247H C   N    1862 1864  -66.31  -67.60   15.59    1.16  -63.20  166.79   20.66\n",
      "   17        247H 247H N   CA   1864 1865  124.46  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1077 1837   1.510\n",
      "  serious non-bonded atom clash:  1077 1838   2.080\n",
      "  serious non-bonded atom clash:  1077 1840   2.047\n",
      "  serious non-bonded atom clash:  1091 1837   2.258\n",
      "  serious non-bonded atom clash:  1093 1835   2.206\n",
      "  serious non-bonded atom clash:  1096 1836   2.213\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "  serious non-bonded atom clash:  1558 1836   2.282\n",
      "  serious non-bonded atom clash:  1558 1841   2.287\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    0    5    6    4   20   33   98   95  138  176  182  262  259  307\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      124.0812       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00050003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5049\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         957.9610\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       9     13   0.071   0.071      192.02       1.000\n",
      " 2 Bond angle potential               :     194      12     29   8.365   8.365      233.78       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  43.697  43.697      40.763       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       2      2   3.503   3.503      25.198       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5049      12     16   0.021   0.021      266.52       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       2      3  13.896  13.896      38.741       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2  95.663  95.663      18.613       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2 102.106 102.106      13.323       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  17.541  17.541      1.9409       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  13.985  13.985     0.33803       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      14      4  34.676 146.191      31.305       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5049       0     36   1.887   1.887      95.425       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1      8  53Y  53Y CB  CA    390  389    1.84    1.54    0.30    8.22    1.54    0.30    8.22\n",
      "    2      9  53Y  53Y CG  CB    391  390    1.95    1.49    0.46   12.81    1.49    0.46   12.81\n",
      "    3     39  97Y  98Q C   N     733  735    1.50    1.35    0.16    5.63    1.35    0.16    5.63\n",
      "    4     45  98Q  98Q N   CA    735  736    1.61    1.43    0.18    5.82    1.43    0.18    5.82\n",
      "    5     47  98Q  98Q C   CA    742  736    1.75    1.49    0.26    7.57    1.49    0.26    7.57\n",
      "    6     48  98Q  99E C   N     742  744    1.53    1.35    0.19    6.58    1.35    0.19    6.58\n",
      "    7     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.35    1.43    0.22    7.35\n",
      "    8     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.87    1.49    0.27    7.87\n",
      "    9     57  99E 100T C   N     751  753    1.55    1.35    0.21    7.32    1.35    0.21    7.32\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    155  53Y  53Y CA  CB    389  390  149.18  107.50   41.68    9.64  107.50   41.68    9.64\n",
      "    2    160  53Y  53Y CG  CD1   391  392  143.18  120.00   23.18    4.71  120.00   23.18    4.71\n",
      "    3    161  53Y  53Y CG  CD2   391  393  147.05  120.00   27.05    5.50  120.00   27.05    5.50\n",
      "    4    168  53Y  53Y C   CA    398  389  144.66  108.00   36.66    8.50  108.00   36.66    8.50\n",
      "    5    199  98Q  98Q N   CA    735  736  127.25  107.00   20.25    5.82  107.00   20.25    5.82\n",
      "    6    208  98Q  99E C   N     742  744  141.05  120.00   21.05    4.78  120.00   21.05    4.78\n",
      "    7    211  99E  99E N   CA    744  745  131.88  107.00   24.88    7.15  107.00   24.88    7.15\n",
      "    8    220  99E 100T C   N     751  753  146.31  120.00   26.31    5.98  120.00   26.31    5.98\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    336  51S  51S C   CA    380  377   14.03    0.00   14.03    4.94    0.00   14.03    4.94\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    569  52S  52S CA  C     383  386 -136.78  180.00   43.22    8.64  180.00   43.22    8.64\n",
      "    2    576  99E  99E CA  C     745  751 -156.33  180.00   23.67    4.73  180.00   23.67    4.73\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   51.69    2.13  -64.10 -144.03   24.61\n",
      "    1         51S  51S N   CA    376  377  148.24  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  -79.15  -72.40   71.15    4.42  -64.10  117.54    7.75\n",
      "    2         52S  52S N   CA    382  383   81.57  152.40                  -35.00\n",
      "    3    552  53Y  54S C   N     398  400 -174.09 -136.60   56.04    1.86  -64.10  171.93   20.38\n",
      "    3         54S  54S N   CA    400  401 -167.15  151.20                  -35.00\n",
      "    4    553  54S  55S C   N     404  406 -159.02 -136.60   27.19    0.83  -64.10 -175.32   20.22\n",
      "    4         55S  55S N   CA    406  407  166.58  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -123.18 -124.30   14.47    0.81  -63.50  177.13   31.12\n",
      "    5         97Y  97Y N   CA    723  724  149.83  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -165.14 -121.10   57.65    1.82  -63.80  175.11   30.80\n",
      "    6         98Q  98Q N   CA    735  736  176.89  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -137.31 -117.80   39.60    1.51  -63.60  165.74   26.22\n",
      "    7         99E  99E N   CA    744  745  171.25  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753  -91.04  -78.10   34.72    1.16  -63.20  138.71   19.64\n",
      "    8        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -115.46 -117.80   11.52    0.61  -63.60  179.28   26.84\n",
      "    9        240E 240E N   CA   1798 1799  148.08  136.80                  -40.30\n",
      "   10    562 241I 242K C   N    1813 1815  -62.82  -70.20   23.11    1.50  -62.90  159.30   20.69\n",
      "   10        242K 242K N   CA   1815 1816  118.50  140.40                  -40.80\n",
      "   11    563 242K 243H C   N    1822 1824 -107.00 -125.60   24.20    1.09  -63.20  169.18   24.01\n",
      "   11        243H 243H N   CA   1824 1825  154.29  138.80                  -42.30\n",
      "   12    564 243H 244H C   N    1832 1834 -111.32 -125.60   22.60    1.03  -63.20  168.40   24.20\n",
      "   12        244H 244H N   CA   1834 1835  156.32  138.80                  -42.30\n",
      "   13    565 244H 245H C   N    1842 1844  -86.11  -67.60   27.63    1.64  -63.20  158.84   21.33\n",
      "   13        245H 245H N   CA   1844 1845  160.52  140.00                  -42.30\n",
      "   14    567 246H 247H C   N    1862 1864 -113.90 -125.60   15.04    0.68  -63.20  176.88   25.43\n",
      "   14        247H 247H N   CA   1864 1865  148.24  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   274  390   2.285\n",
      "  serious non-bonded atom clash:   379  391   2.205\n",
      "  serious non-bonded atom clash:   379  392   2.249\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    3    1    1   21   32   90  101  128  177  182  240  279  302\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -161.5141       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00060003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4960\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1384.6857\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141      10     13   0.105   0.105      384.15       1.000\n",
      " 2 Bond angle potential               :     194      12     22   8.037   8.037      219.92       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      4  40.975  40.975      38.323       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      1   2.305   2.305      11.469       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4960      14     21   0.030   0.030      535.90       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      3   9.870   9.870      19.546       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0  89.558  89.558      11.189       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2  82.415  82.415      15.352       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  97.331  97.331      3.3353       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 127.770 127.770      1.0643       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      16      4  35.709 150.077      36.311       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4960       0     49   1.901   1.901      108.13       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1      8  53Y  53Y CB  CA    390  389    2.01    1.54    0.47   13.03    1.54    0.47   13.03\n",
      "    2      9  53Y  53Y CG  CB    391  390    2.42    1.49    0.93   25.99    1.49    0.93   25.99\n",
      "    3     19  53Y  53Y C   CA    398  389    1.73    1.49    0.24    6.94    1.49    0.24    6.94\n",
      "    4     39  97Y  98Q C   N     733  735    1.52    1.35    0.18    6.30    1.35    0.18    6.30\n",
      "    5     45  98Q  98Q N   CA    735  736    1.63    1.43    0.20    6.53    1.43    0.20    6.53\n",
      "    6     47  98Q  98Q C   CA    742  736    1.77    1.49    0.28    8.08    1.49    0.28    8.08\n",
      "    7     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.48    1.35    0.18    6.48\n",
      "    8     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.32    1.43    0.22    7.32\n",
      "    9     56  99E  99E C   CA    751  745    1.75    1.49    0.26    7.47    1.49    0.26    7.47\n",
      "   10     57  99E 100T C   N     751  753    1.54    1.35    0.20    7.08    1.35    0.20    7.08\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    151  52S  53Y C   N     386  388  141.58  120.00   21.58    4.90  120.00   21.58    4.90\n",
      "    2    155  53Y  53Y CA  CB    389  390  155.95  107.50   48.45   11.20  107.50   48.45   11.20\n",
      "    3    161  53Y  53Y CG  CD2   391  393  143.23  120.00   23.23    4.72  120.00   23.23    4.72\n",
      "    4    168  53Y  53Y C   CA    398  389  132.67  108.00   24.67    5.72  108.00   24.67    5.72\n",
      "    5    199  98Q  98Q N   CA    735  736  128.67  107.00   21.67    6.23  107.00   21.67    6.23\n",
      "    6    208  98Q  99E C   N     742  744  141.74  120.00   21.74    4.94  120.00   21.74    4.94\n",
      "    7    211  99E  99E N   CA    744  745  131.59  107.00   24.59    7.07  107.00   24.59    7.07\n",
      "    8    220  99E 100T C   N     751  753  145.98  120.00   25.98    5.90  120.00   25.98    5.90\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -156.47  180.00   23.53    4.71  180.00   23.53    4.71\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   51.60    2.04  -64.10 -146.73   24.41\n",
      "    1         51S  51S N   CA    376  377  151.55  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  -61.64  -72.40   36.35    1.80  -64.10  152.70   11.21\n",
      "    2         52S  52S N   CA    382  383  117.68  152.40                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388  -67.46  -98.40   52.94    4.58  -63.50  145.30   23.27\n",
      "    3         53Y  53Y N   CA    388  389  171.35  128.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -96.45  -72.40   25.62    1.54  -64.10  166.94   14.27\n",
      "    4         54S  54S N   CA    400  401  161.23  152.40                  -35.00\n",
      "    5    553  54S  55S C   N     404  406  -88.39  -72.40   63.15    4.27  -64.10  128.62    8.19\n",
      "    5         55S  55S N   CA    406  407   91.30  152.40                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -123.18 -124.30    7.88    0.47  -63.50 -176.61   32.12\n",
      "    6         97Y  97Y N   CA    723  724  143.20  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -164.37 -121.10   49.03    1.51  -63.80 -173.59   32.50\n",
      "    7         98Q  98Q N   CA    735  736  162.75  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -120.80 -117.80   35.68    1.74  -63.60  158.06   24.30\n",
      "    8         99E  99E N   CA    744  745  172.35  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -89.88  -78.10   34.30    1.16  -63.20  138.48   19.54\n",
      "    9        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -115.46 -117.80   11.61    0.62  -63.60  179.20   26.83\n",
      "   10        240E 240E N   CA   1798 1799  148.17  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815  -80.50  -70.20   24.48    1.98  -62.90  159.96   19.88\n",
      "   11        242K 242K N   CA   1815 1816  118.19  140.40                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824 -111.46 -125.60   33.07    1.46  -63.20  156.63   22.74\n",
      "   12        243H 243H N   CA   1824 1825  168.69  138.80                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834 -101.62 -125.60   24.15    0.90  -63.20 -179.82   25.03\n",
      "   13        244H 244H N   CA   1834 1835  141.66  138.80                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844  -68.21  -67.60    4.56    0.37  -63.20  177.86   21.92\n",
      "   14        245H 245H N   CA   1844 1845  135.49  140.00                  -42.30\n",
      "   15    566 245H 246H C   N    1852 1854 -106.42 -125.60   23.63    0.59  -63.20  172.78   19.29\n",
      "   15        246H 246H N   CA   1854 1855  124.99  138.80                  -42.30\n",
      "   16    567 246H 247H C   N    1862 1864 -127.21  -63.20   64.24    9.92  -63.20   64.24    9.92\n",
      "   16        247H 247H N   CA   1864 1865  -36.89  -42.30                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   259  390   2.055\n",
      "  serious non-bonded atom clash:   259  391   2.204\n",
      "  serious non-bonded atom clash:   268  391   2.062\n",
      "  serious non-bonded atom clash:   268  392   2.287\n",
      "  serious non-bonded atom clash:   274  390   2.181\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    1    2    1    1   18   29   99   89  129  179  175  243  275  301\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      139.5757       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00070003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5194\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         363.0529\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.049   0.049      104.66       1.000\n",
      " 2 Bond angle potential               :     194       4     11   5.180   5.180      102.61       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      6  45.199  45.199      40.681       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   2.325   2.325      9.4988       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5194       0      0   0.002   0.002      3.0884       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      4  10.003  10.003      20.076       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0  94.032  94.032      11.606       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      2  71.190  71.190      15.320       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  43.078  43.078      3.1254       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   1.442   1.442     0.22977       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      19      6  63.379 159.028      70.888       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5194       0     20   1.894   1.894     -18.725       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.52    1.35    0.17    6.14    1.35    0.17    6.14\n",
      "    2     45  98Q  98Q N   CA    735  736    1.62    1.43    0.19    6.40    1.43    0.19    6.40\n",
      "    3     47  98Q  98Q C   CA    742  736    1.74    1.49    0.25    7.25    1.49    0.25    7.25\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.50    1.35    0.18    6.50\n",
      "    5     54  99E  99E N   CA    744  745    1.65    1.43    0.22    7.12    1.43    0.22    7.12\n",
      "    6     56  99E  99E C   CA    751  745    1.76    1.49    0.27    7.83    1.49    0.27    7.83\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.20    7.18    1.35    0.20    7.18\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  128.16  107.00   21.16    6.08  107.00   21.16    6.08\n",
      "    2    208  98Q  99E C   N     742  744  139.94  120.00   19.94    4.53  120.00   19.94    4.53\n",
      "    3    211  99E  99E N   CA    744  745  131.11  107.00   24.11    6.93  107.00   24.11    6.93\n",
      "    4    220  99E 100T C   N     751  753  147.26  120.00   27.26    6.19  120.00   27.26    6.19\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    576  99E  99E CA  C     745  751 -154.88  180.00   25.12    5.02  180.00   25.12    5.02\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   59.18    1.81  -64.10 -169.27   22.75\n",
      "    1         51S  51S N   CA    376  377 -179.83  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -100.14  -72.40   67.86    3.19  -64.10  116.39   10.95\n",
      "    2         52S  52S N   CA    382  383 -145.67  152.40                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -119.40  -98.40   33.93    2.85  -63.50  155.54   22.18\n",
      "    3         53Y  53Y N   CA    388  389  101.75  128.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400  -78.38  -72.40   24.53    1.65  -64.10  164.23   11.13\n",
      "    4         54S  54S N   CA    400  401  128.61  152.40                  -35.00\n",
      "    5    553  54S  55S C   N     404  406  104.07 -136.60  145.93    4.46  -64.10 -169.35   19.43\n",
      "    5         55S  55S N   CA    406  407 -124.80  151.20                  -35.00\n",
      "    6    554  55S  56S C   N     410  412   74.08  -64.10  150.53   16.18  -64.10  150.53   16.18\n",
      "    6         56S  56S N   CA    412  413  -94.70  -35.00                  -35.00\n",
      "    7    555  56S  57Y C   N     416  418 -116.01  -63.50   54.04    9.85  -63.50   54.04    9.85\n",
      "    7         57Y  57Y N   CA    418  419  -56.15  -43.40                  -43.40\n",
      "    8    556  96V  97Y C   N     721  723 -123.18 -124.30   14.79    0.83  -63.50  176.83   31.07\n",
      "    8         97Y  97Y N   CA    723  724  150.14  135.40                  -43.40\n",
      "    9    557  97Y  98Q C   N     733  735 -166.61 -121.10   61.37    1.96  -63.80  172.74   30.49\n",
      "    9         98Q  98Q N   CA    735  736 -179.12  139.70                  -40.30\n",
      "   10    558  98Q  99E C   N     742  744 -135.30 -117.80   42.21    1.71  -63.60  161.30   25.51\n",
      "   10         99E  99E N   CA    744  745  175.21  136.80                  -40.30\n",
      "   11    559  99E 100T C   N     751  753  -92.36  -78.10   35.23    1.16  -63.20  138.98   19.76\n",
      "   11        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "   12    560 239V 240E C   N    1796 1798 -115.46 -117.80    2.37    0.10  -63.60 -175.08   22.23\n",
      "   12        240E 240E N   CA   1798 1799  137.20  136.80                  -40.30\n",
      "   13    561 240E 241I C   N    1805 1807  -78.37  -63.40   15.67    2.92 -120.60 -176.55    8.90\n",
      "   13        241I 241I N   CA   1807 1808  -48.22  -43.60                  130.30\n",
      "   14    562 241I 242K C   N    1813 1815 -144.86 -118.00   29.13    0.97  -62.90 -172.35   29.28\n",
      "   14        242K 242K N   CA   1815 1816  150.39  139.10                  -40.80\n",
      "   15    563 242K 243H C   N    1822 1824  -62.70  -67.60   12.73    0.83  -63.20  170.56   21.35\n",
      "   15        243H 243H N   CA   1824 1825  128.26  140.00                  -42.30\n",
      "   16    564 243H 244H C   N    1832 1834 -104.00 -125.60   77.55    3.30  -63.20  112.10   16.68\n",
      "   16        244H 244H N   CA   1834 1835 -146.71  138.80                  -42.30\n",
      "   17    565 244H 245H C   N    1842 1844  -59.53  -67.60   18.54    1.18  -63.20  165.65   20.94\n",
      "   17        245H 245H N   CA   1844 1845  123.31  140.00                  -42.30\n",
      "   18    566 245H 246H C   N    1852 1854  -84.42  -67.60   50.13    4.29  -63.20  136.73   15.87\n",
      "   18        246H 246H N   CA   1854 1855   92.78  140.00                  -42.30\n",
      "   19    567 246H 247H C   N    1862 1864 -110.92 -125.60   49.46    1.58  -63.20  142.12   15.44\n",
      "   19        247H 247H N   CA   1864 1865   91.57  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   18   26   94   91  127  176  178  251  281  308\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -114.6192       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00080003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     5074\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         266.4602\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.052   0.052      115.23       1.000\n",
      " 2 Bond angle potential               :     194       4     12   5.347   5.347      105.69       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      4  39.045  39.045      34.294       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   2.271   2.271      9.5616       1.000\n",
      " 5 Soft-sphere overlap restraints     :    5074       0      1   0.003   0.003      5.1030       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      2   8.726   8.726      15.276       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      2  98.869  98.869      14.390       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  99.999  99.999      9.8984       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 108.596 108.596      4.0400       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 122.797 122.797     0.98950       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      14      6  37.240 144.988      29.216       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    5074       0     16   1.867   1.867     -77.224       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.53    1.35    0.18    6.48    1.35    0.18    6.48\n",
      "    2     45  98Q  98Q N   CA    735  736    1.64    1.43    0.21    7.04    1.43    0.21    7.04\n",
      "    3     47  98Q  98Q C   CA    742  736    1.74    1.49    0.25    7.27    1.49    0.25    7.27\n",
      "    4     48  98Q  99E C   N     742  744    1.55    1.35    0.20    7.18    1.35    0.20    7.18\n",
      "    5     54  99E  99E N   CA    744  745    1.64    1.43    0.21    7.07    1.43    0.21    7.07\n",
      "    6     56  99E  99E C   CA    751  745    1.77    1.49    0.28    8.18    1.49    0.28    8.18\n",
      "    7     57  99E 100T C   N     751  753    1.55    1.35    0.21    7.42    1.35    0.21    7.42\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  132.55  107.00   25.55    7.34  107.00   25.55    7.34\n",
      "    2    211  99E  99E N   CA    744  745  128.01  107.00   21.01    6.04  107.00   21.01    6.04\n",
      "    3    214  99E  99E CA  C     745  751  132.27  116.50   15.77    4.53  116.50   15.77    4.53\n",
      "    4    220  99E 100T C   N     751  753  151.74  120.00   31.74    7.21  120.00   31.74    7.21\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   57.23    1.77  -64.10 -166.04   22.99\n",
      "    1         51S  51S N   CA    376  377  175.95  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382 -165.11 -136.60   42.69    1.42  -64.10  174.29   19.92\n",
      "    2         52S  52S N   CA    382  383 -177.03  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388 -110.29 -124.30   33.21    2.36  -63.50  158.17   27.48\n",
      "    3         53Y  53Y N   CA    388  389  165.51  135.40                  -43.40\n",
      "    4    553  54S  55S C   N     404  406 -154.41 -136.60   24.00    1.33  -64.10 -167.39   12.42\n",
      "    4         55S  55S N   CA    406  407  135.12  151.20                  -35.00\n",
      "    5    556  96V  97Y C   N     721  723 -123.18 -124.30   23.39    1.29  -63.50  168.74   29.78\n",
      "    5         97Y  97Y N   CA    723  724  158.77  135.40                  -43.40\n",
      "    6    557  97Y  98Q C   N     733  735 -145.99 -121.10   72.57    2.96  -63.80  138.78   24.48\n",
      "    6         98Q  98Q N   CA    735  736 -152.13  139.70                  -40.30\n",
      "    7    558  98Q  99E C   N     742  744 -165.21 -117.80   57.40    1.70  -63.60 -178.37   29.75\n",
      "    7         99E  99E N   CA    744  745  169.16  136.80                  -40.30\n",
      "    8    559  99E 100T C   N     751  753  -91.65  -78.10   34.95    1.16  -63.20  138.83   19.70\n",
      "    8        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "    9    560 239V 240E C   N    1796 1798 -115.46 -117.80   14.98    0.79  -63.60  175.92   26.39\n",
      "    9        240E 240E N   CA   1798 1799  151.60  136.80                  -40.30\n",
      "   10    562 241I 242K C   N    1813 1815 -124.35 -118.00   11.30    0.58  -62.90 -178.72   21.06\n",
      "   10        242K 242K N   CA   1815 1816  129.75  139.10                  -40.80\n",
      "   11    563 242K 243H C   N    1822 1824 -121.39 -125.60   14.08    0.60  -63.20  175.40   25.74\n",
      "   11        243H 243H N   CA   1824 1825  152.23  138.80                  -42.30\n",
      "   12    565 244H 245H C   N    1842 1844 -117.69 -125.60   79.36    3.16  -63.20  113.82   17.76\n",
      "   12        245H 245H N   CA   1844 1845 -142.24  138.80                  -42.30\n",
      "   13    566 245H 246H C   N    1852 1854  -63.87  -67.60    6.96    0.62  -63.20  171.83   21.52\n",
      "   13        246H 246H N   CA   1854 1855  145.87  140.00                  -42.30\n",
      "   14    567 246H 247H C   N    1862 1864  -97.34 -125.60   29.54    1.20  -63.20  173.69   23.93\n",
      "   14        247H 247H N   CA   1864 1865  147.39  138.80                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   16   26   87   93  122  181  180  248  274  308\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:      113.2789       53    53\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_486_> OE1/2 will be swapped:      133.1296       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00090003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "openf___224_> Open           scfv.IL00000003.pdb\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     18395212   17964.074    17.543\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:     1884      132        1        5.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "preppdf_457W> Both soft sphere and statistical potential terms selected.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      614     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     4688\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    7.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         264.0857\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     141       7      7   0.049   0.049      103.16       1.000\n",
      " 2 Bond angle potential               :     194       5     11   5.312   5.312      105.21       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      7  46.392  46.392      44.630       1.000\n",
      " 4 Stereochemical improper torsion pot:      81       0      0   1.574   1.574      4.4591       1.000\n",
      " 5 Soft-sphere overlap restraints     :    4688       0      0   0.001   0.001     0.63730       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      3  10.899  10.899      23.832       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0  89.394  89.394      10.474       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      1  59.398  59.398      7.0791       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  44.340  44.340      3.2341       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  28.970  28.970     0.86579       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      15      9  56.727 160.345      47.725       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :    4688       0      6   1.866   1.866     -87.218       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39  97Y  98Q C   N     733  735    1.52    1.35    0.18    6.20    1.35    0.18    6.20\n",
      "    2     45  98Q  98Q N   CA    735  736    1.63    1.43    0.20    6.43    1.43    0.20    6.43\n",
      "    3     47  98Q  98Q C   CA    742  736    1.76    1.49    0.27    7.85    1.49    0.27    7.85\n",
      "    4     48  98Q  99E C   N     742  744    1.53    1.35    0.18    6.53    1.35    0.18    6.53\n",
      "    5     54  99E  99E N   CA    744  745    1.66    1.43    0.23    7.47    1.43    0.23    7.47\n",
      "    6     56  99E  99E C   CA    751  745    1.73    1.49    0.24    6.96    1.49    0.24    6.96\n",
      "    7     57  99E 100T C   N     751  753    1.54    1.35    0.20    7.03    1.35    0.20    7.03\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    199  98Q  98Q N   CA    735  736  129.23  107.00   22.23    6.39  107.00   22.23    6.39\n",
      "    2    208  98Q  99E C   N     742  744  142.23  120.00   22.23    5.05  120.00   22.23    5.05\n",
      "    3    211  99E  99E N   CA    744  745  133.86  107.00   26.86    7.72  107.00   26.86    7.72\n",
      "    4    220  99E 100T C   N     751  753  145.68  120.00   25.68    5.84  120.00   25.68    5.84\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    549  50S  51S C   N     374  376  171.80 -136.60   54.11    1.77  -64.10 -159.47   23.48\n",
      "    1         51S  51S N   CA    376  377  167.48  151.20                  -35.00\n",
      "    2    550  51S  52S C   N     380  382  147.47 -136.60   76.12    3.16  -64.10 -127.36   27.55\n",
      "    2         52S  52S N   CA    382  383  145.86  151.20                  -35.00\n",
      "    3    551  52S  53Y C   N     386  388  159.64 -124.30   81.07    6.68  -63.50 -156.40   28.18\n",
      "    3         53Y  53Y N   CA    388  389  107.34  135.40                  -43.40\n",
      "    4    552  53Y  54S C   N     398  400   93.18 -136.60  136.52    4.47  -64.10 -154.15   17.89\n",
      "    4         54S  54S N   CA    400  401 -167.80  151.20                  -35.00\n",
      "    5    553  54S  55S C   N     404  406 -175.53 -136.60   70.73    2.57  -64.10  159.95   19.57\n",
      "    5         55S  55S N   CA    406  407 -149.75  151.20                  -35.00\n",
      "    6    556  96V  97Y C   N     721  723 -123.18 -124.30   13.12    0.74  -63.50  178.41   31.32\n",
      "    6         97Y  97Y N   CA    723  724  148.47  135.40                  -43.40\n",
      "    7    557  97Y  98Q C   N     733  735 -174.07 -121.10   61.15    1.88  -63.80 -174.28   32.77\n",
      "    7         98Q  98Q N   CA    735  736  170.25  139.70                  -40.30\n",
      "    8    558  98Q  99E C   N     742  744 -117.49 -117.80   37.37    1.88  -63.60  155.19   23.74\n",
      "    8         99E  99E N   CA    744  745  174.17  136.80                  -40.30\n",
      "    9    559  99E 100T C   N     751  753  -89.72  -78.10   34.25    1.16  -63.20  138.45   19.52\n",
      "    9        100T 100T N   CA    753  754 -177.98  149.80                  -42.10\n",
      "   10    560 239V 240E C   N    1796 1798 -115.46 -117.80    9.11    0.49  -63.60 -178.34   27.16\n",
      "   10        240E 240E N   CA   1798 1799  145.60  136.80                  -40.30\n",
      "   11    562 241I 242K C   N    1813 1815 -124.64 -118.00   25.17    1.12  -62.90  167.61   25.46\n",
      "   11        242K 242K N   CA   1815 1816  163.38  139.10                  -40.80\n",
      "   12    563 242K 243H C   N    1822 1824 -125.39 -125.60   11.71    0.44  -63.20 -179.56   19.56\n",
      "   12        243H 243H N   CA   1824 1825  127.09  138.80                  -42.30\n",
      "   13    564 243H 244H C   N    1832 1834 -102.17 -125.60   62.77    2.75  -63.20  126.80   18.40\n",
      "   13        244H 244H N   CA   1834 1835 -162.97  138.80                  -42.30\n",
      "   14    565 244H 245H C   N    1842 1844  -81.70  -67.60   28.55    1.78  -63.20  153.99   20.44\n",
      "   14        245H 245H N   CA   1844 1845  164.83  140.00                  -42.30\n",
      "   15    567 246H 247H C   N    1862 1864  -93.61  -67.60   26.67    2.18  -63.20  178.98   20.64\n",
      "   15        247H 247H N   CA   1864 1865  134.08  140.00                  -42.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1140 1647   2.043\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0   17   26   88   89  127  175  169  242  266  298\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "iupac_m_486_> OE1/2 will be swapped:     -114.1313       99    99\n",
      "iupac_m_483_> CD1/2 CE1/2 swapped:       90.0400      227   227\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.2706\n",
      "openf___224_> Open           scfv.BL00100003.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:      248     1884     1884\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "scfv.B99990001.pdb            3035.92285\n",
      "scfv.B99990002.pdb            3101.36060\n",
      "scfv.B99990003.pdb            3076.25513\n",
      "\n",
      "\n",
      ">> Summary of successfully produced loop models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "scfv.BL00010001.pdb            325.11578\n",
      "scfv.BL00020001.pdb            213.92548\n",
      "scfv.BL00030001.pdb            245.99390\n",
      "scfv.BL00040001.pdb           1172.73059\n",
      "scfv.BL00050001.pdb           1338.15576\n",
      "scfv.BL00060001.pdb            353.32950\n",
      "scfv.BL00070001.pdb           1256.36072\n",
      "scfv.BL00080001.pdb            329.94745\n",
      "scfv.BL00090001.pdb            258.37354\n",
      "scfv.BL00100001.pdb            241.16164\n",
      "scfv.BL00010002.pdb            355.89447\n",
      "scfv.BL00020002.pdb           1337.02356\n",
      "scfv.BL00030002.pdb            241.02191\n",
      "scfv.BL00040002.pdb            316.39154\n",
      "scfv.BL00050002.pdb            268.30432\n",
      "scfv.BL00060002.pdb            409.23645\n",
      "scfv.BL00070002.pdb            279.02911\n",
      "scfv.BL00080002.pdb            255.42993\n",
      "scfv.BL00090002.pdb           1004.46570\n",
      "scfv.BL00100002.pdb            250.32158\n",
      "scfv.BL00010003.pdb           1375.48401\n",
      "scfv.BL00020003.pdb            186.31194\n",
      "scfv.BL00030003.pdb            323.52032\n",
      "scfv.BL00040003.pdb            385.87158\n",
      "scfv.BL00050003.pdb           1937.11316\n",
      "scfv.BL00060003.pdb            957.96100\n",
      "scfv.BL00070003.pdb           1384.68567\n",
      "scfv.BL00080003.pdb            363.05292\n",
      "scfv.BL00090003.pdb            266.46017\n",
      "scfv.BL00100003.pdb            264.08569\n",
      "\n"
     ]
    }
   ],
   "source": [
    " # Loop refinement of an existing model\n",
    "from modeller import *\n",
    "from modeller.automodel import *\n",
    "\n",
    "log.verbose()\n",
    "env = environ()\n",
    "\n",
    "# directories for input atom files\n",
    "env.io.atom_files_directory = './:../atom_files'\n",
    "\n",
    "# Create a new class based on 'loopmodel' so that we can redefine\n",
    "# select_loop_atoms (necessary)\n",
    "class MyLoop(loopmodel):\n",
    "    # This routine picks the residues to be refined by loop modeling\n",
    "    def select_loop_atoms(self):\n",
    "        # 10 residue insertion \n",
    "        #return selection(self.residue_range('58', '74'))\n",
    "        return\n",
    "m = loopmodel(env,\n",
    "              alnfile='scfv.ali',\n",
    "              knowns=[\"2fjhHL\"], \n",
    "              sequence=\"scfv\")          # code of the target\n",
    "\n",
    "m.starting_model= 1                 # index of the first model \n",
    "m.ending_model  = 1                 # index of the last model\n",
    "\n",
    "m.loop.starting_model= 1           # index of the first loop model \n",
    "m.loop.ending_model  = 10          # index of the last loop model\n",
    "m.loop.md_level = refine.fast # loop refinement method; this yields\n",
    "                                   # models quickly but of low quality;\n",
    "                                   # use refine.slow for better models\n",
    "\n",
    "m.make()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     11239320   10975.898    10.719\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     11239848   10976.414    10.719\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     11288448   11023.875    10.766\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     11301748   11036.863    10.778\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11441184   11173.031    10.911\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11442784   11174.594    10.913\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11446764   11178.480    10.916\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11448764   11180.434    10.918\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11451764   11183.363    10.921\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11456264   11187.758    10.926\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11461836   11193.199    10.931\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11464636   11195.934    10.934\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11468836   11200.035    10.938\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11475136   11206.188    10.944\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11477908   11208.895    10.946\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:     11479308   11210.262    10.948\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           scfv.BL00040003.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11490281   11220.978    10.958\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11490357   11221.052    10.958\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11490528   11221.219    10.958\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11491378   11222.049    10.959\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11492636   11223.277    10.960\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11494540   11225.137    10.962\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11494580   11225.176    10.962\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11494670   11225.264    10.962\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11497526   11228.053    10.965\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11497586   11228.111    10.965\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11497721   11228.243    10.965\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11502005   11232.427    10.969\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11502093   11232.513    10.969\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11502291   11232.706    10.969\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11508717   11238.981    10.976\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11508849   11239.110    10.976\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11509146   11239.400    10.976\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11518768   11248.797    10.985\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11518968   11248.992    10.985\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11519418   11249.432    10.986\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11533868   11263.543    11.000\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11534168   11263.836    11.000\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11534843   11264.495    11.000\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11556501   11285.646    11.021\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11556949   11286.083    11.022\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:     11557957   11287.067    11.023\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     11728645   11453.755    11.185\n",
      "read_mo_297_> Segments, residues, atoms:        1      248     1884\n",
      "read_mo_298_> Segment:        1     1   248     1884\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     11736389   11461.317    11.193\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     11878081   11599.688    11.328\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     11880489   11602.040    11.330\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:     12022181   11740.411    11.465\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     12051661   11769.200    11.493\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933      0         0       876\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     12132921   11848.556    11.571\n",
      "\n",
      "patch_______> segment topology patched using RTF:     1  ;  MET  ;  NTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1883  1933   2622      3114       876\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:     12209697   11923.532    11.644\n",
      "patch_______> segment topology patched using RTF:   248  ;  HIS  ;  CTER\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues atoms bonds angles dihedrals impropers:\n",
      "                     1      248  1884  1934   2624      3116       877\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:     1884        0\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  HIS:OXT  HIS\n",
      "preppdf_453W> No fixed restraints selected; there may be some dynamic ones.\n",
      "preppdf_454W> Restraints file was probably not read; use restraints.append().\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:     12458877   12166.872    11.882\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     12796697   12496.774    12.204\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     12841913   12540.931    12.247\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     12932345   12629.243    12.333\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     13113209   12805.868    12.506\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     13474937   13159.118    12.851\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     14198393   13865.618    13.541\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:     15645305   15278.618    14.921\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      248\n",
      "Number of all, selected real atoms                :     1884    1884\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :        0       0\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   357371\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -22423.1699\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 2 Bond angle potential               :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 3 Stereochemical cosine torsion poten:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 4 Stereochemical improper torsion pot:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "13 Mainchain Omega dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "31 Non-bonded restraints              :  357312       0    547   6.778   6.778     -22423.       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       0.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       0.000\n",
      "do_prof_492W> Are you sure you want to calculate a normalized energy profile!\n",
      "\n",
      "\n",
      "\n",
      "openf___224_> Open           scfv.BL00040003.profile\n",
      "# Energy of each residue is written to: scfv.BL00040003.profile\n",
      "# The profile IS normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:   13\n",
      "# The sum of all numbers in the file:      -6.7015\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -22423.169922\n"
     ]
    },
    {
     "ename": "FileNotFoundError",
     "evalue": "[Errno 2] No such file or directory: 'scfv.BL00040003.dssp'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mFileNotFoundError\u001b[0m                         Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-27-7a221c5f7f81>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0mpdb_loopref_name\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m\"%s.BL%04d0003\"\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mloopsel\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0mdo_profile\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpdb_loopref_name\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0mseq_Loop\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstructure2D_Loop\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mdo_dssp\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpdb_loopref_name\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m<ipython-input-26-c5e73b339ca2>\u001b[0m in \u001b[0;36mdo_dssp\u001b[0;34m(pdbIN)\u001b[0m\n\u001b[1;32m      5\u001b[0m     \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msystem\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcomando\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 7\u001b[0;31m     \u001b[0;32mwith\u001b[0m \u001b[0mopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"%s.dssp\"\u001b[0m\u001b[0;34m%\u001b[0m\u001b[0mpdbIN\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"r\"\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mf\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      8\u001b[0m         \u001b[0;31m#head = \"#  RESIDUE AA STRUCTURE BP1 BP2  ACC     N-H-->O    O-->H-N    N-H-->O    O-->H-N    TCO  KAPPA ALPHA  PHI   PSI    X-CA   Y-CA   Z-CA\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      9\u001b[0m         \u001b[0;31m#print [x for x in head.split(\" \") if len(x) > 0]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mFileNotFoundError\u001b[0m: [Errno 2] No such file or directory: 'scfv.BL00040003.dssp'"
     ]
    }
   ],
   "source": [
    "loopsel = 4\n",
    "pdb_loopref_name = \"%s.BL%04d0003\" % (name, loopsel)\n",
    "do_profile(pdb_loopref_name)\n",
    "seq_Loop, structure2D_Loop = do_dssp(pdb_loopref_name)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     11239320   10975.898    10.719\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     11239848   10976.414    10.719\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     11288448   11023.875    10.766\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:     11301748   11036.863    10.778\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/radii14.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL9v19}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           scfv.ali\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     11303750   11038.818    10.780\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     11305200   11040.234    10.781\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     11308100   11043.066    10.784\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     11313900   11048.730    10.790\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:     11430956   11163.043    10.901\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11431860   11163.926    10.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:     11432848   11164.891    10.903\n",
      "\n",
      "Read the alignment from file       : scfv.ali\n",
      "Total number of alignment positions:   253\n",
      "\n",
      "  #  Code        #_Res #_Segm PDB_code    Name\n",
      "-------------------------------------------------------------------------------\n",
      "  1     2fjhHL     227      1  2fjhHL.pdb\n",
      "  2       scfv     248      1        scfv\n"
     ]
    },
    {
     "ename": "NameError",
     "evalue": "name 'structure2D_Loop' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-28-aa32b432f3a6>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     26\u001b[0m \u001b[0mhelix\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mbeta\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mturn\u001b[0m    \u001b[0;34m=\u001b[0m \u001b[0mget_2dprofile\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstructure2D\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     27\u001b[0m \u001b[0;31m#secondary elements model\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 28\u001b[0;31m \u001b[0mhelixL\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mbetaL\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mturnL\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mget_2dprofile\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstructure2D_Loop\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m-\u001b[0m\u001b[0;36m0.045\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     29\u001b[0m \u001b[0;31m#Cut-off line\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     30\u001b[0m \u001b[0mline_cutoff\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mget_profileline\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstructure2D\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mNameError\u001b[0m: name 'structure2D_Loop' is not defined"
     ]
    }
   ],
   "source": [
    "def get_profile(profile_file, seq):\n",
    "    \"\"\"Read `profile_file` into a Python array, and add gaps corresponding to\n",
    "       the alignment sequence `seq`.\"\"\"\n",
    "    # Read all non-comment and non-blank lines from the file:\n",
    "    f = open(profile_file)\n",
    "    vals = []\n",
    "    for line in f:\n",
    "        if not line.startswith('#') and len(line) > 10:\n",
    "            spl = line.split()\n",
    "            vals.append(float(spl[-1]))\n",
    "    # Insert gaps into the profile corresponding to those in seq:\n",
    "    for n, res in enumerate(seq.residues):\n",
    "        for gap in range(res.get_leading_gaps()):\n",
    "            vals.insert(n, None)\n",
    "    # Add a gap at position '0', so that we effectively count from 1:\n",
    "    vals.insert(0, None)\n",
    "    return vals\n",
    "\n",
    "e = modeller.environ()\n",
    "a = modeller.alignment(e, file='%s.ali' % (name))\n",
    "template = get_profile('%s.profile' % pdb_template_name, a[pdb_template_name])\n",
    "model    = get_profile('%s.profile' % pdb_model_name, a[name])\n",
    "loop     = get_profile('%s.profile' % pdb_loopref_name, a[name])  \n",
    "\n",
    "#secondary elements template\n",
    "helix, beta, turn    = get_2dprofile(structure2D)\n",
    "#secondary elements model\n",
    "helixL, betaL, turnL = get_2dprofile(structure2D_Loop, -0.045)\n",
    "#Cut-off line\n",
    "line_cutoff = get_profileline(structure2D)\n",
    "\n",
    "\n",
    "#Plot CDR ranges\n",
    "# los CDR son obtenidos desde el anlisis \n",
    "import numpy as np\n",
    "cdrH = np.array([[27,35],[50,66],[99,109]])\n",
    "cdrL = np.array([[24,34],[49,56],[89,97]])+len(SEQH + linker)\n",
    "lista = np.append(cdrH, cdrL, 0)\n",
    "cdr   = get_rangeprofile(structure2D_gaps, lista, value=-0.050)\n",
    "\n",
    "# Plot the template and model profiles in the same plot for comparison:\n",
    "pylab.figure(1, figsize=(20,6))\n",
    "pylab.xlabel('Alignment position')\n",
    "pylab.ylabel('DOPE per-residue score')\n",
    "pylab.plot(template, color='green', linewidth=2, label='Template')\n",
    "pylab.plot(model, color='red', linewidth=2, label='Model')\n",
    "pylab.plot(loop, color='blue', linewidth=2, label='Loop')\n",
    "\n",
    "#pylab.plot(helix, color='orange', linewidth=8, label='2D')\n",
    "#pylab.plot(beta, color='magenta', linewidth=8)\n",
    "#pylab.plot(turn, color='green', linewidth=8)\n",
    "\n",
    "pylab.plot(helixL, color='orange', linewidth=8,label='2D')\n",
    "pylab.plot(betaL, color='magenta', linewidth=8)\n",
    "pylab.plot(turnL, color='green', linewidth=8)\n",
    "pylab.plot(cdr, color='blue', linewidth=8, label='CDRs')\n",
    "\n",
    "\n",
    "\n",
    "pylab.plot(line_cutoff, color='gray', linestyle='--', linewidth=2, label='Cut-off')\n",
    "\n",
    "pylab.legend()\n",
    "pylab.savefig('dope_profileFull.png', dpi=65)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Cleaning the folder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import os,datetime, shutil\n",
    "archivos = [x for x in os.listdir(\".\") if (x.count(\".ipynb\")==0 and os.path.isdir(x)==False)]\n",
    "dt = str(datetime.datetime.now())\n",
    "folder_file = \"01_modelling_antibody_like_%s\" % dt\n",
    "os.mkdir(folder_file)\n",
    "for f in archivos:\n",
    "    shutil.move(f, folder_file)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
